<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Syst Biol</span> <span class="iso-abbrev" title="iso-abbrev">BMC Syst Biol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Systems Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1752-0509</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6329140">6329140</a> </span> <span class="publisher-id" title="publisher-id">677</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12918-018-0677-4">10.1186/s12918-018-0677-4</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Modelling overflow metabolism in <i>Escherichia coli</i> with flux balance analysis incorporating differential proteomic efficiencies of energy pathways </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">Hong</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">hong.zeng@eng.ox.ac.uk</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Aidong</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">aidong.yang@eng.ox.ac.uk</span> </span> <a href="#Aff1" /> </span> <span class="citation_author_institution" id="Aff1">[], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1936 8948</span> <span class="institution-id" title="institution-id">grid.4991.5</span> <span class="institution" title="institution">Department of Engineering Science, </span> <span class="institution" title="institution">University of Oxford, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-1-1</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-1-1</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">13</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">3</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-9-18</span> <span class="accepted" title="accepted">accepted: 2018-12-28</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>The formation of acetate by fast-growing <i>Escherichia coli</i> ( <i>E. coli</i>) is a commonly observed phenomenon, often referred to as overflow metabolism. Among various studies that have been carried over decades, a recent work (Basan, M. et al. <i>Nature</i> <b>528,</b> 99–104, 2015) suggested and validated that it is the differential proteomic efficiencies in energy biogenesis between fermentation and respiration that lead to the production of acetate at rapid growth conditions, as the consequence of optimally allocating the limited proteomic resource. In the current work, we attempt to incorporate this newly developed proteome allocation theory into flux balance analysis (FBA) to capture quantitatively the extent of overflow metabolism in different <i>E. coli</i> strains. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>A concise constraint was introduced into a FBA-based model with three proteomic cost parameters to represent constrained allocation of proteome over two energy (respiration and fermentation) pathways and biomass synthesis. Linear relationships were shown to exist between the three proteomic cost parameters. Tests with three different strains revealed that the proteomic cost of fermentation was consistently lower than that of respiration. A slow-growing strain appeared to have a higher proteomic cost for biomass synthesis than fast-growing strains. Different assumed levels of carbon flowing into pentose phosphate pathway affected the absolute value of model parameters, but had no qualitative impact on the comparative proteomic costs. For the prediction of biomass yield, significant errors that occurred for one of the tested strains (ML308) were rectified by adjusting the cellular energy demand according to literature data. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>With the aid of a concise proteome allocation constraint, our FBA-based model is able to quantitatively predict the onset and extent of the overflow metabolism in various <i>E. coli</i> strains. Such prediction is enabled by three linearly-correlated (as opposed to uniquely determinable) proteomic cost parameters. The linear relationships between these parameters, when determined using data from cell culturing experiments, render biologically meaningful comparative proteomic costs between fermentation and respiration pathways and between the biomass synthesis sectors of slow- and fast-growing species. Simultaneous prediction of acetate production and biomass yield in the overflow region requires the use of reliable cellular energy demand data. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (10.1186/s12918-018-0677-4) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Flux balance analysis</div> <div class="kwd" title="kwd">
Proteomic efficiency</div> <div class="kwd" title="kwd">
Overflow metabolism</div> <div class="kwd" title="kwd">
Acetate production</div> <div class="kwd" title="kwd">
Biomass yield</div> <div class="kwd" title="kwd"> <i>Escherichia coli</i> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004543</span> <span class="institution" title="institution">China Scholarship Council</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">201708060077</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">Hong</span> </span> </span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>The formation of acidic by-products, predominantly acetate, when <i>Escherichia coli</i> ( <i>E. coli</i>) grows under aerobic-glucose conditions is a commonly observed phenomenon, which has been extensively studied over decades [ <a href="#CR1">1</a>– <a href="#CR5">5</a>]. Lee reviewed 19 studies of recombinant <i>E. coli</i> where acetate was accumulated in fed-batch systems [ <a href="#CR6">6</a>]. It has been reported that the portion of glucose converted into acetate can be as high as 15% [ <a href="#CR7">7</a>], representing a seemingly huge waste of feedstock. The accumulation of acetate in the culture medium appears to be a major limiting factor for achieving high cell density [ <a href="#CR8">8</a>], which is particularly severe in the growth of recombinant strains [ <a href="#CR9">9</a>]. Acetate also impairs the microbial production of recombinant proteins [ <a href="#CR1">1</a>] and drug precursors [ <a href="#CR9">9</a>]. These complications of acetate in bioreactors thus call for elucidation of acetate-pertinent metabolic processes. A similar phenomenon has been observed in tumour cells (Warburg effect) [ <a href="#CR10">10</a>– <a href="#CR12">12</a>]. The associated mathematical models for explaining the Warburg effect have recently been reviewed [ <a href="#CR13">13</a>]. </p> <p>Traditionally, the aerobic formation of acetate has been referred to as overflow metabolism: the excess glucose saturates or inhibits the tricarboxylic acid (TCA) cycle, which subsequently forces the cell to modulate the redundant carbon to the acetate pathway [ <a href="#CR3">3</a>, <a href="#CR14">14</a>]. However, the study by Molenaar et al. suggested that the overflow metabolism as shown in the growth phenotype is probably a result of the global allocation of cellular resources, where the enzyme efficiency and the pathway yield were both taken into account to obtain the optimal growth strategies subject to different growth conditions [ <a href="#CR15">15</a>]. Later in 2015, Basan et al. proposed and validated that the overflow metabolism in <i>E. coli</i> originates from the global physiological proteome allocation for rapid growth [ <a href="#CR16">16</a>]. In particular, the proteomic efficiency of energy biogenesis through aerobic fermentation was found to be higher than that of respiration; this difference in proteomic efficiency between fermentation and respiration appears to play a central role in dictating the degree of overflow metabolism in <i>E. coli</i>. </p> <p>Given the importance of the overflow metabolism, several phenomenological models were developed to depict this effect [ <a href="#CR5">5</a>, <a href="#CR8">8</a>, <a href="#CR17">17</a>, <a href="#CR18">18</a>], where the prediction of acetate excretion was dictated by a combination of (i) the constraints on oxygen and carbon supply and (ii) cellular mass and energy balance. Later, models that adopt conventional regulatory mechanisms of acetate metabolism [ <a href="#CR14">14</a>], such as oxygen limitation, carbon source availability and tight regulation of cofactor pools were evaluated [ <a href="#CR19">19</a>], with an attempt to explain the metabolic shifts from a fully aerobic mode to the aerobic acetate fermentation (overflow). More recently, constraint-based metabolic models [ <a href="#CR20">20</a>] were established to analyse the optimal cellular growth strategy, incorporating principles of (i) limitation in the cellular resource on the maximal attainable growth rate, such as the maximum cytoplasmic density adopted by FBAwMC [ <a href="#CR21">21</a>, <a href="#CR22">22</a>] and the finite amount of resource to be allocated between metabolic network and ribosomes, as applied in RBA [ <a href="#CR23">23</a>– <a href="#CR25">25</a>], (ii) metabolic regulation based on enzyme kinetic information, such as mechanistically detailed descriptions of gene expression and the synthesis of functional macromolecules used in ME-Model [ <a href="#CR26">26</a>] and (iii) membrane occupancy-derived competition between glucose transporters and respiration chain (an extension of ME-Model) [ <a href="#CR27">27</a>]. The major target of these models is to predict the maximum cellular growth rate. Predictions were validated quantitatively by the experimental data, while the overflow metabolism in fast-growing phase was mostly captured in a qualitative way. In addition, it was pointed out [ <a href="#CR16">16</a>, <a href="#CR28">28</a>] that cell volumes were empirically found to vary widely with virtually constant densities across different growth conditions [ <a href="#CR29">29</a>], which suggests that the cytoplasmic density-based constraint might not be fully justified. </p> <p>Inspired by a recent experimental work studying the proteomic cost of the core metabolic pathways of <i>E. coli</i> [ <a href="#CR16">16</a>], a model named constrained allocation flux balance analysis (CAFBA) [ <a href="#CR28">28</a>] managed to predict the rates of acetate production in the overflow metabolism for different <i>E. coli</i> strains, with good quantitative agreement with experimental data. However, the proteomic costs adopted in CAFBA were applied to individual metabolic reactions, without focusing on the exploration of the critical role played by specific metabolic modules such as energy biogenesis pathways. </p> <p>In this work, we attempt to depict the overflow metabolism in various <i>E. coli</i> strains with quantitative accuracy, i.e. predicting aerobic steady-state rates of acetate production at different growth rates and validating the model with experimental data in literature. In particular, we adopt a concise proteome allocation constraint as identified by Basan et al. [ <a href="#CR16">16</a>], referred to as the Proteome Allocation Theory (PAT) in this work. The PAT suggests that the choice of energy biogenesis pathways under different growth conditions results from the discrepancy of proteomic efficiencies between fermentation and respiration. <i>E.coli</i> cells tend to use the more protein-efficient fermentation pathway to generate energy in order to accommodate the high proteomic demand in biosynthesis under rapid growth. The key concepts of PAT are fully embedded and realised in our model. With a parsimonious, PAT-based metabolic model capable of accurately capturing the overflow metabolism, we further analyse the interdependency between pathway-level proteomic cost parameters, the disparity in these parameters between different <i>E. coli</i> strains, and the impact of cellular energy demand on the accuracy of the co-prediction of the overflow metabolism and the biomass yield on substrate. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="formulationofthepatconstraint" title="sec"> <div class="title" tagx="title" title="title">
Formulation of the PAT constraint</div> <p>Following Basan et al. [ <a href="#CR16">16</a>], the fractions of three proteome sectors in the entire proteome (i.e. the total protein content) of the cell sum to unity: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">1</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\phi}_f+{\phi}_r+{\phi}_{BM}=1 $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
BM</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>ϕ</i> <sub> <i>f</i> </sub> and <i>ϕ</i> <sub> <i>r</i> </sub> are the fractions of the fermentation- and respiration-affiliated enzymes, respectively, which enable the fluxes for energy generation; <i>ϕ</i> <sub> <i>BM</i> </sub> represents the fraction of the remaining part of the proteome enabling other cellular activities, broadly referred to as the sector of biomass synthesis [ <a href="#CR16">16</a>, <a href="#CR28">28</a>]. <p /> <p>More specifically, <i>ϕ</i> <sub> <i>f</i> </sub> represents the mass abundance of the enzymes that carry fermentation fluxes involved in glycolysis (glucose to acetyl-CoA), oxidative phosphorylation and acetate synthesis pathways (phosphotransacetylase and acetate kinase). <i>ϕ</i> <sub> <i>r</i> </sub> comprises all the enzymes that catalyse the respiration-associated reactions in glycolysis, tricarboxylic acid (TCA) cycle and oxidative phosphorylation system. Same as in Basan et al. [ <a href="#CR16">16</a>], in this work we adopt the linear dependences assumed in Hui et al. [ <a href="#CR30">30</a>] to relate <i>ϕ</i> <sub> <i>f</i> </sub> and <i>ϕ</i> <sub> <i>r</i> </sub> with the fermentation and respiration fluxes respectively, </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">2a</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\phi}_f={w}_f{v}_f $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">2b</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\phi}_r={w}_r{v}_r $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>v</i> <sub> <i>f</i> </sub> ( <i>v</i> <sub> <i>r</i> </sub>) is the fermentation (respiration) pathway flux, which in this work is represented by the enzymatic reaction "acetate kinase ACKr" ("2-oxogluterate dehydrogenase AKGDH"); <i>w</i> <sub> <i>f</i> </sub> ( <i>w</i> <sub> <i>r</i> </sub>) is the pathway-level proteomic cost, denoting the proteome fraction required per unit fermentation (respiration) flux. <p /> <p>On the biomass synthesis sector, <i>ϕ</i> <sub> <i>BM</i> </sub> corresponds to the remaining part of proteome that is not covered by the fermentation and respiration sectors, including ribosomal proteins and anabolic enzymes (the major part, referred to as biomass synthesis), catabolic enzymes and cellular maintenance proteins. Motivated by the observed linear dependency between growth rate and proteome fraction for biomass synthesis [ <a href="#CR30">30</a>– <a href="#CR32">32</a>], the following linear relationship is assumed: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">3</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\phi}_{BM}={\phi}_0+b\uplambda $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
BM</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
λ</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>b</i>λ is the growth rate-associated component with λ being the specific growth rate and the constant <i>b</i> quantifying the proteome fraction required per unit growth rate. In Basan et al. [ <a href="#CR16">16</a>], <i>ϕ</i> <sub>0</sub> was considered as a growth rate independent constant. <p /> <p>Combining Eqs. ( <a href="#Equ1">1</a>)–( <a href="#Equ4">3</a>), we have </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">4</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f{v}_f+{w}_r{v}_r+b\uplambda =1-{\phi}_0 $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>Equation ( <a href="#Equ5">4</a>) implies that the sum of the three proteomic cost terms on the left-hand side remains constant. However, when the growth rate (and hence the fermentation and respiration fluxes) becomes very low, it is difficult to envisage numerically how this sum could still remain at a constant level. In fact, in Basan et al. [ <a href="#CR16">16</a>] (see its Supplementary Information), it was acknowledged that at growth rates lower than that corresponding to the onset of the overflow phenomenon, the proteome sectors would no longer be constrained by the equality indicated by Eq. ( <a href="#Equ5">4</a>). This suggests that across the entire range of possible growth rates, <i>ϕ</i> <sub>0</sub> is unlikely a growth rate independent constant: it may remain at a constant (and minimum) level in the overflow region where the proteomic resource is stretched, but become growth-rate dependent (and larger) at lower growth rates outside the overflow region, i.e. <i>ϕ</i> <sub>0, min</sub> ≤  <i>ϕ</i> <sub>0</sub> ≤ 1, where <i>ϕ</i> <sub>0, min</sub> is a true constant. Defining <i>ϕ</i> <sub>max</sub> ≡ 1 −  <i>ϕ</i> <sub>0, min</sub>, Eq. ( <a href="#Equ5">4</a>) then becomes </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">5</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f{v}_f+{w}_r{v}_r+b\uplambda =1-{\phi}_0\le 1-{\phi}_{0,\min}\equiv {\phi}_{\mathrm{max}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
min</div> </div> </div> <div class="mo_UNKNOWN">
≡</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>In Vazquez and Oltvai (2016) [ <a href="#CR33">33</a>], <i>ϕ</i> <sub>0</sub> was also interpreted as a variable instead of a constant, with a (non-zero) minimum value. For simplicity, both sides of Eq. ( <a href="#Equ6">5</a>) is divided by <i>ϕ</i> <sub> <i>max</i> </sub>, leading to the final form of the proteome constraint adopted in this work, referred to as PAT constraint from this point on: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">6</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }{v}_f+{w}_r^{\ast }{v}_r+{b}^{\ast}\lambda \le 1 $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast}\equiv {w}_f/{\phi}_{max} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
≡</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
max</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast}\equiv {w}_r/{\phi}_{max} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
≡</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mi_UNKNOWN">
max</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup> ≡  <i>b</i>/ <i>ϕ</i> <sub> <i>max</i> </sub>. <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M19">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M21">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup> are referred to as the proteomic cost parameters. <p /> </div> <div class="predictingtheacetateflux" title="sec"> <div class="title" tagx="title" title="title">
Predicting the acetate flux</div> <p>Flux balance analysis (FBA) [ <a href="#CR20">20</a>] is used to determine the optimal flux distribution under different growth condition, with a set of constraints: </p> <p>max <i>f</i> <sub> <i>obj</i> </sub>, subject to </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">7</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M23">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\displaystyle \begin{array}{l}\left(\mathrm{i}\right)\kern0.5em \mathrm{Sv}=0\\ {}\left(\mathrm{i}\mathrm{i}\right)\kern0.5em {\mathrm{v}}^{\mathrm{L}}\le \mathrm{v}\le {\mathrm{v}}^{\mathrm{U}}\\ {}\left(\mathrm{i}\mathrm{i}\mathrm{i}\right)\kern0.5em {w}_f^{\ast }{v}_f+{w}_r^{\ast }{v}_r+{b}^{\ast}\lambda \le 1\end{array}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
Sv</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
ii</div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
L</div> </div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
≤</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
U</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
iii</div> </div> <div class="mspace_UNKNOWN" /> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>f</i> <sub> <i>obj</i> </sub> is the assumed cellular objective. We specified minimizing substrate uptake as the objective function because in this study the commonly used objective ‘growth rate’ was used as the model input (with acetate production as the model output). S is the stoichiometric matrix defined by the metabolic model; v is a column vector comprising the reactions/fluxes described in the metabolic network; v <sup>L</sup> and v <sup>U</sup> represent the lower and upper limits of the reactions, respectively. The inequality constraint (iii) is same as Eq. ( <a href="#Equ7">6</a>) introduced earlier. <p /> <p>The prediction of the extent of overflow metabolism (rate of acetate production) requires of the parameter values of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M25">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M27">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup> in the third constraint (PAT constraint) of Eq. ( <a href="#Equ8">7</a>). We show in the next section that these three parameters cannot be uniquely determined by the experimentally measured growth rate-acetate production profile alone. A set of values for these parameters was randomly chosen from mathematically equivalent sets (see Additional file <a href="#MOESM1">1</a>: Table S1). The PAT-based FBA was run at different growth rates under aerobic-glucose conditions. FBA was carried out using the core <i>E. coli</i> metabolic model [ <a href="#CR34">34</a>], referred to as the core model in the rest of the paper. The optimal flux distribution was solved via COBRA toolbox [ <a href="#CR35">35</a>] in MATLAB (R2016a). LP solution was determined by Gurobi 6.0. Detailed model descriptions such as uptake bounds and flux regulations are given in Additional file <a href="#MOESM2">2</a>, sections 1 and 2. <p /> </div> <div class="interdependencyofproteomiccostparameters" title="sec"> <div class="title" tagx="title" title="title">
Interdependency of proteomic cost parameters</div> <p>A linear relationship was previously shown to hold between the fermentation (or respiration) flux and steady state growth rates in the overflow region [ <a href="#CR16">16</a>]: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">8</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M29">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {v}_f={k}_f\lambda +{v}_{f,0} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">9</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M31">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {v}_r={k}_r\lambda +{v}_{r,0} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where, as introduced earlier, <i>v</i> <sub> <i>f</i> </sub> is the fermentation flux (referred to as "acetate line" in [ <a href="#CR16">16</a>]); <i>v</i> <sub> <i>r</i> </sub> is the respiration flux. <i>k</i> <sub> <i>f</i> </sub> ( <i>k</i> <sub> <i>r</i> </sub>) and <i>v</i> <sub> <i>f</i>, 0 </sub> ( <i>v</i> <sub> <i>r</i>, 0 </sub>) are constants representing the slope and intercept of the fermentation (respiration) line. Substituting Eq. ( <a href="#Equ9">8</a>) and Eq. ( <a href="#Equ10">9</a>) into Eq. ( <a href="#Equ7">6</a>), with the equal sign held for the overflow condition: <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">10</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M33">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \left({w}_f^{\ast }{k}_f+{w}_r^{\ast }{k}_r+{b}^{\ast}\right)\lambda =1-{w}_f^{\ast }{v}_{f,0}-{w}_r^{\ast }{v}_{r,0} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>Equation ( <a href="#Equ11">10</a>) holds for any growth rate ( <i>λ</i>) in the overflow region, which requires </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">11</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M35">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }{k}_f+{w}_r^{\ast }{k}_r+{b}^{\ast }=0 $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">12</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M37">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ 1-{w}_f^{\ast }{v}_{f,0}-{w}_r^{\ast }{v}_{r,0}=0 $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>Equations ( <a href="#Equ12">11</a>) and ( <a href="#Equ13">12</a>) indicate that (i) there is a linear relationship between the fermentation and respiration proteomic cost parameters <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M39">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M41">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, and (ii) the third growth-rate dependent proteomic cost parameter <i>b</i> <sup>∗</sup> is a linear combination of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M43">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M45">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, thus <i>b</i> <sup>∗</sup> also possesses a linear relationship with <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M47">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> (or <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M49">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>). <p /> <p>If experimental data exist that allow for both the fermentation line and the respiration line to be plotted (such as the steady state growth rate – acetate excretion and growth rate – CO <sub>2</sub> revolution data given in [ <a href="#CR16">16</a>]), their slopes and intercepts, appearing in Eqs. ( <a href="#Equ12">11</a>) and ( <a href="#Equ13">12</a>), can be obtained. However, the three proteomic cost parameters cannot be uniquely determined by the two equations, although specific values of similar parameters have previously been derived from measured cellular protein compositions [ <a href="#CR16">16</a>]. </p> <p>In this work, <i>k</i> <sub> <i>f</i> </sub> and <i>v</i> <sub> <i>f</i>, 0 </sub> were directly determined from the experimentally measured growth rate-acetate excretion profile (data sources are shown in Fig. <a href="#Fig1">1</a>). To our knowledge, no directly experimental data were available for the rate of intracellular respiration. Alternatively we took the growth rate-acetate profile as the input of FBA (setting the objective function to the minimisation of glucose uptake) to estimate the respiration flux at each data point, which was subsequently used to determine <i>k</i> <sub> <i>r</i> </sub> and <i>v</i> <sub> <i>r</i>, 0 </sub>. Flux variability analysis (FVA) was conducted which confirmed that all the relevant fluxes used in the model were uniquely determined. After obtaining <i>k</i> <sub> <i>f</i> </sub> ( <i>k</i> <sub> <i>r</i> </sub>) and <i>v</i> <sub> <i>f</i>, 0 </sub> ( <i>v</i> <sub> <i>r</i>, 0 </sub>), a set of values of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M51">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M53">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup> can be determined by arbitrarily specifying the value for one of the parameters. In this work, we took <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M55">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> to be specified, in a range of [0, 0.11] for MG1655 and [0, 0.07] for ML308 and NCM3722. This thus allows us to present the parameter estimation results in the form of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M57">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>- <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M59">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M61">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>- <i>b</i> <sup>∗</sup> plots. Simulation results presented in this paper were obtained with a randomly chosen value of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M63">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> within the ranges mentioned above and the correspondingly determined values of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M65">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup>. Note that different values chosen for <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M67">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> yielded identical simulation results (Additional file <a href="#MOESM2">2</a>: Figures S9-S17). <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Model predictions of overflow metabolism for MG, NCM and ML at nominal energy demand. The extent of overflow metabolism is represented by the acetate flux. Simulation results of the respiration flux are drawn to show the switch between fully-respiration and respiration-fermentation mode. Comparison is made between model predictions and experimental data for the rates of acetate production. <i>uPPP</i>% was set to 35% according to the flux measurement [ <a href="#CR41">41</a>]. Other <i>uPPP</i>% values render similar results (see Additional file <a href="#MOESM1">1</a>: Table S1 and Additional file <a href="#MOESM2">2</a>, section 3). Experimental data were obtained from different sources [ <a href="#CR3">3</a>, <a href="#CR16">16</a>, <a href="#CR41">41</a>]. Data points from [ <a href="#CR16">16</a>] were converted using 1 mM A <sub>600nm</sub> <sup>− 1</sup> h <sup>− 1</sup> = 2 mmol gDW <sup>− 1</sup> h <sup>− 1</sup> according to [ <a href="#CR28">28</a>]. "-nom" refers to nominal, the default energy demand specified in the core model. ac – acetate flux, vr – respiration flux, simu – simulation results, exp. – experimental data </p>   </div> <p /> </div> <div class="adjustmentofcellularenergydemand" title="sec"> <div class="title" tagx="title" title="title">
Adjustment of cellular energy demand</div> <p>We observed discrepancy in biomass yield between model predictions and the experimental data, especially for ML308 (Fig. <a href="#Fig5">5</a>), and hypothesised the reason being the inaccuracy of cellular energy demand assumed by the core model when applied to this particular species growing under non-overflow and overflow conditions. To test this hypothesis, we collected the growth data in the overflow region for ML308 (Table 7 in [ <a href="#CR3">3</a>]) and found that the reported steady state ATP production rate was lower than what the core model suggested. Therefore, we decided to remodel the cellular energy demand by subtracting the surplus portion, reducing it to the strain-specific values reported in the literature (Additional file <a href="#MOESM1">1</a>: Table S3). </p> <p>The original cellular energy demand embedded in the core model is quantified by </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">13</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M69">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {r}_{ATP, nominal}=\mathrm{ATPM}+\sigma \lambda +{v}_{GLNS}+{v}_{PFK} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ATP</div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
nominal</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mtext_UNKNOWN">
ATPM</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
σλ</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mtext_UNKNOWN">
GLNS</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
PFK</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>r</i> <sub> <i>ATP</i>, <i>nominal</i> </sub> is the overall ATP consumption rate (equivalent to the ATP production rate at steady state); subscript "nominal" indicates the default specification of the core model; <i>v</i> <sub> <i>GLNS</i> </sub> and <i>v</i> <sub> <i>PFK</i> </sub> are the fluxes of the enzymatic reactions glutamine synthetase and phosphofructokinase (one mole ATP is required per mole flux of each reaction); (ATPM +  <i>σλ</i>) denotes the maintenance energy required for non-metabolic processes, where ATPM corresponds to the non-growth-associated maintenance (NGAM) and <i>σ</i> to the growth-associated maintenance (GAM) [ <a href="#CR36">36</a>]. <p /> <p>The adjusted cellular energy demand is formulated as </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">14</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M71">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {r}_{ATP, new}={r}_{ATP, nominal}-S\left(\lambda \right) $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ATP</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
new</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ATP</div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
nominal</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
S</div> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>S</i>( <i>λ</i>) is the offset energy, i.e. the amount of energy over-predicted by the core model. The mathematical analysis of the growth data of ML308 suggested that in the overflow region, the offset energy is linearly related with the growth rate (R <sup>2</sup> = 0.9998): <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">15</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M73">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ S\left(\lambda \right)= k\lambda +c $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
kλ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>k</i> and <i>c</i> are constants. Substituting Eqs. ( <a href="#Equ14">13</a>) and ( <a href="#Equ16">15</a>) into Eq. ( <a href="#Equ15">14</a>), we have <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">16</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M75">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {r}_{ATP, new}=\left(\mathrm{ATPM}-c\right)+\left(\sigma -k\right)\lambda +{v}_{GLNS}+{v}_{PFK} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ATP</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
new</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
ATPM</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
c</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mtext_UNKNOWN">
GLNS</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
PFK</div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>For simplicity, we define M ≡ ATPM −  <i>c</i> and N ≡  <i>σ</i> −  <i>k</i>; M and N are referred to as the (adjusted) maintenance parameters. The adjustment of the cellular energy demand in our FBA is achieved by manipulating the maintenance energy, more specifically, through the maintenance parameters. Note that in the case where growth energetic data is not available (i.e. for MG1655 and NCM3722), such adjustment was not possible, therefore the default maintenance parameter values were used (see Table <a href="#Tab1">1</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Values of the maintenance parameters</p>  <table> <thead> <tr> <th>Strains</th> <th>M</th> <th>N</th> <th>Scope of applicability</th> </tr> </thead> <tbody> <tr> <td>MG-/NCM−/ML-nom</td> <td>8.39 <sup>a</sup> </td> <td>59.81 <sup>a</sup> </td> <td>Complete growth range</td> </tr> <tr> <td>ML-new</td> <td>− 50.90 <sup>b</sup> </td> <td>93.34 <sup>b</sup> </td> <td>Overflow region</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>a</sup>default value specified in the <i>E. coli</i> core model </p> <p> <sup>b</sup>estimated from the growth data (Table 7 in Holms 1996 [ <a href="#CR3">3</a>]) </p> </div>  <p /> </div> <div class="alternativepathwaysincentralmetabolism" title="sec"> <div class="title" tagx="title" title="title">
Alternative pathways in central metabolism</div> <p>The model constructed in this work considers only the central metabolism of <i>E. coli</i> as detailed in the <i>E. coli</i> core model. The energy biogenesis pathways in the model consist of glycolysis (the EMP pathway), the TCA cycle, the acetate pathway (PTA-ACKA) and the terminal oxidative phosphorylation system. However we noted the existence of alternative pathways in the central carbon metabolism, which include the Entner-Doudoraff (ED) pathway, the pentose phosphate (PP) pathway and the more recently explored PEP-glyoxylate cycle [ <a href="#CR37">37</a>]. </p> </div> <div class="edpathway" title="sec"> <div class="title" tagx="title" title="title">
ED pathway</div> <p>The ED pathway was found to be three to five-fold more protein-efficient than the EMP pathway to achieve the same glycolytic flux [ <a href="#CR38">38</a>], which provides a clear rationale for the utilisation of the ED pathway in a number of bacteria, e.g. <i>Sinorhizobium meliloti, Rhodobacter sphaeroides, Zymomonas mobilis, and Paracoccus versutus </i> [ <a href="#CR39">39</a>]. However, Flamholz et al. acknowledged that <i>E. coli</i>, which is capable of using both the ED and the EMP pathways, tends to use the latter. Flux measurements also suggest that the usage of the ED pathway by <i>E.coli</i> K-12 is minimal: only about 2% of glucose catabolism proceeds by means of the ED pathway in batch cultures [ <a href="#CR40">40</a>] and about 6% in mini-scale chemostats [ <a href="#CR41">41</a>]. Furthermore, the activity of the ED pathway was detected only under slow- to mild-growing conditions [ <a href="#CR40">40</a>, <a href="#CR41">41</a>]. To our knowledge, no activity of ED pathway in <i>E. coli</i> has been reported under fast-growing scenarios. </p> </div> <div class="pep-glyoxylatecycle" title="sec"> <div class="title" tagx="title" title="title">
PEP-glyoxylate cycle</div> <p>As for the PEP-glyoxylate cycle, similar to the ED pathway, its usage was identified to be significant only under slow-growing conditions. Not even a trace activity of the PEP-glyoxylate cycle was found in wild-type batch cultures or more rapidly growing chemostats [ <a href="#CR37">37</a>]. Furthermore, the flux comparison between the aceA-pckA knockout strain and the sucC knockout strain [ <a href="#CR16">16</a>] verifies that compared to the TCA cycle, the alternative PEP-glyoxylate cycle plays a less significant role in glucose-limited fast-growing cultures of <i>E. coli</i>. </p> <p>Based on above literature evidences, this work, focusing on the overflow metabolism that occurs at fast-growing cultures of <i>E.coli</i> with relatively sufficient substrate availability, has taken the assumption that the use of alternative ED pathway and PEP-glyoxylate cycle is negligible compared to the glycolysis (i.e. EMP pathway) and the TCA cycle. </p> </div> <div class="pppathway" title="sec"> <div class="title" tagx="title" title="title">
PP pathway</div> <p>The PP pathway, on the other hand, can function as a significant alternative to the upper part of glycolysis for carbon catabolism in <i>E. coli</i>. Previous studies showed that the carbon flow through the PP pathway could reach 20–35% of the total carbon intake and can vary with different growth rates [ <a href="#CR40">40</a>, <a href="#CR41">41</a>] hence neither a constant portion of carbon is diverted into PP pathway nor this portion of carbon flux negligible. The uncertainty embedded in the PP pathway flux motivates us to study the impact of different portion of substrate carbon allocated between the upper part of the EMP pathway and the PP pathway on the proteomic cost parameters and the model predictions. More details can be found in the Additional file <a href="#MOESM2">2</a>, section 3. </p> <p>We define the PP pathway ratio ( <i>PPP</i>%) as the portion of substrate carbon directing to PP pathway to the total carbon intake: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">17</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M77">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ PPP\%=\frac{PGL}{EX\_ glc\left(\mathrm{e}\right)}\times 100\% $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
PPP</div> <div class="mo_UNKNOWN">
%</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
PGL</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
EX</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
glc</div> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> </div> </div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
%</div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where 6-phosphogluconolactonase (PGL) is chosen to represent the PP pathway flux as it is a major and also the beginning enzymatic reaction in the pentose phosphate shunt; EX_glc(e) is the exchange reaction denoting glucose uptake rate. In our simulation, <i>PPP</i>% was controlled by setting the upper bound of the portion of carbon that is directed into the PP pathway, denoted as <i>uPPP</i>%, with the aid of an auxiliary term DM_PPP_RATIO: <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">18</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M79">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ uPPP\%\times EX\_ glc(e)- PGL= DM\_ PPP\_ RATIO\ge 0 $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mtext_UNKNOWN">
uPPP</div> <div class="mo_UNKNOWN">
%</div> <div class="mo_UNKNOWN">
×</div> <div class="mi_UNKNOWN">
EX</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
glc</div> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
PGL</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
DM</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
PPP</div> <div class="mo_UNKNOWN">
_</div> <div class="mtext_UNKNOWN">
RATIO</div> <div class="mo_UNKNOWN">
≥</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p> <i>E.coli</i> MG1655, NCM3722 and ML308 have been selected as the model strains in this work and are referred to as MG, NCM and ML respectively from this point on. In this section, the simulated acetate excretion pattern is presented against experimental data to demonstrate the accuracy of the model prediction. Subsequently we elucidate the linear interdependency of the proteomic cost parameters. In particular, we reveal the similarities and differences between the three <i>E. coli</i> strains. With respect to the PP pathway ratio ( <i>PPP</i>%), previous studies on the slow-growth strain MG show that the portion of carbon that goes into the PP pathway can be approximately 20% of the total carbon intake [ <a href="#CR40">40</a>, <a href="#CR41">41</a>]. In this work, we set the upper bound of the carbon flowing into PP pathway ( <i>uPPP</i>%) to 25, 35 and 40% to investigate the potential effect of the change in <i>PPP</i>% on proteomic cost parameters and model prediction (more justification is provided in Additional file <a href="#MOESM2">2</a>, section 3). On cellular energy demand, we refer to the original energy demand specified in the core model [ <a href="#CR34">34</a>] as nominal energy demand, and present first the set of results which were generated on this basis. Subsequently, we show how an adjusted energy demand (particularly applied to ML, referred to as ML-new) affects the patterns of the estimated proteomic cost parameters and the accuracy of biomass yield prediction. </p> <div class="modelpredictionofoverflowmetabolismwithnominalenergydemand" title="sec"> <div class="title" tagx="title" title="title">
Model prediction of overflow metabolism with nominal energy demand</div> <p>Here, the accuracy of the predicted acetate excretion rate is compared with experimental data. Variation in the proteomic cost parameters with the changed carbon level diverted into the PP pathway is also presented. </p> </div> <div class="modelpredictionofacetateproduction" title="sec"> <div class="title" tagx="title" title="title">
Model prediction of acetate production</div> <p>Figure <a href="#Fig1">1</a> shows that model prediction of the pattern of acetate excretion is in good agreement with the experimental observations for three different <i>E. coli</i> strains. The onset of the production of acetate is concomitant with the drop in the respiratory flux, indicating a switch between fully-respiration to respiration-fermentation mode. As the growth rate further increases, the acetate flux becomes dominant while the extent of respiration is gradually diminishing. It is worth noting that zero acetate production was commonly observed at low growth rates of different strains [ <a href="#CR3">3</a>, <a href="#CR16">16</a>, <a href="#CR41">41</a>, <a href="#CR42">42</a>]; To emphasise the (strain-specific) acetate production pattern, we only collect the data with non-zero acetate production. For all the strains, data involving growth rates lower than those presented in Fig. <a href="#Fig1">1</a> are associated with non-detectable acetate excretion, hence are not shown here. </p> </div> <div class="linearrelationshipsbetweenproteomiccostparameters" title="sec"> <div class="title" tagx="title" title="title">
Linear relationships between proteomic cost parameters</div> <p>When the nominal energy demand is adopted (indicated by "-nom"), the change of <i>uPPP</i>% leads to insignificant changes to the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M81">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  line for each strain (Fig. <a href="#Fig2">2</a>). Between different strains, MG and NCM share nearly identical lines. The lines of ML-nom deviate from those of the former two, but not significantly (although this closeness will be altered with the adjusted energy demand, see Fig. <a href="#Fig2">2</a> ML-new and the section below). In any case, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M83">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> is clearly higher than the corresponding <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M85">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, implying that respiration has a higher (lower) proteomic cost (efficiency) than fermentation for energy production, which is consistent with what was derived from protein abundances data for comparable parameters in [ <a href="#CR16">16</a>]. <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p> <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M87">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  relationship for MG, NCM and ML with nominal energy demand and for ML with new energy demand. "-nom" refers to nominal, the default energy demand specified in the core model. "-new" refers to the adjusted energy demand. <i>uPPP</i>% was set to 25, 35 and 40% for each strain <p />   </div> <p /> <p>To inspect the insignificant disparity in the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M89">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  lines when all strains use the nominal energy demand, Eq. ( <a href="#Equ13">12</a>) is re-arranged to <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">19</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M91">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }=\frac{v_{f,0}}{v_{r,0}}{w}_f^{\ast }+\frac{1}{v_{r,0}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>The slope and intercept of the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M93">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  line are dictated by <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M95">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{v_{f,0}}{v_{r,0}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M97">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{1}{v_{r,0}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, respectively. <i>v</i> <sub> <i>f</i>, 0 </sub> can be determined directly by the experimental measurement of acetate production. <i>v</i> <sub> <i>r</i>, 0 </sub>, on the other hand, is a result of the combination of (measured) rates of acetate production and the mass and energy balance structure of the metabolic model. <p /> <p>For a specific strain, <i>v</i> <sub> <i>f</i>, 0 </sub> only depends on the pattern of acetate excretion, not affected by assumed level of <i>uPPP</i>%. Therefore, the impact of <i>uPPP</i>% on the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M99">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  line is through affecting the value of <i>v</i> <sub> <i>r</i>, 0 </sub>, which turns out to be rather moderate. Between different strains, the ratio of <i>v</i> <sub> <i>f</i>, 0 </sub> and <i>v</i> <sub> <i>r</i>, 0 </sub> and the value of <i>v</i> <sub> <i>r</i>, 0 </sub> are nearly identical between MG and NCM, regardless of the level of <i>uPPP</i>% adopted, resulting in the very much overlapped pattern of the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M101">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> relationship between MG and NCM. For ML, the value of the slope is slightly smaller than MG and NCM, while the intercept is about 25% larger (as shown in Additional file <a href="#MOESM1">1</a>: Table S2). Figure <a href="#Fig2">2</a> also suggests that the proteomic cost (efficiency) of respiration pathways for ML is higher (lower) than that for MG and NCM, regardless the modification in the energy demand. <p /> <p>Compared to the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M103">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  relationship, that of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M105">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> appears to be affected by the level of <i>uPPP</i>% more visibly (Fig. <a href="#Fig3">3</a>). Between different species, the difference is also more pronounced, and closeness is present between the two rapid-growth strains NCM and ML (as presented in Additional file <a href="#MOESM2">2</a>: Figure S1). <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p> <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M107">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  relationship for MG, NCM and ML strains with nominal energy demand. "-nom" refers to nominal, the default energy demand specified in the core model. <i>uPPP</i>% was set to 25 35 and 40% for each strain. The blue arrow shows the increase in <i>b</i> <sup>∗</sup> with the increase of <i>uPPP%</i> at fixed <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M109">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>; the yellow arrow shows the right-shifting trend of the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M111">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> line with the increase of <i>uPPP%</i> <p />   </div> <p /> <p>For a specific strain, the increase of <i>uPPP</i>% gradually moves the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M113">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  line to the right (yellow arrow, Fig. <a href="#Fig3">3</a>), corresponding to an increase in <i>b</i> <sup>∗</sup> (blue arrow, Fig. <a href="#Fig3">3</a>). This trend can be explained by inspecting a re-arrangement of Eq. ( <a href="#Equ12">11</a>): <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">20</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M115">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }=\left({k}_r\frac{v_{f,0}}{v_{r,0}}-{k}_f\right){w}_f^{\ast }+\frac{k_r}{v_{r,0}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
f</div> </div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>Equation ( <a href="#Equ21">20</a>) suggests that the shift of the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M117">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  line results from the change in the respiratory flux (note the intercept, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M119">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{k_r}{v_{r,0}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>). In <i>E. coli</i>, the PP pathway and the TCA cycle are two major sources for the production of NADPH [ <a href="#CR40">40</a>]. At a given growth rate, the amount of NADPH needed for cell growth is fixed based on the mass and energy balance. As <i>uPPP</i>% increases, more carbon is predicted to enter the PP pathway. In the model simulation, an increase in the amount of NADPH produced via PP pathway would force a drop of the flux into the TCA cycle in order to maintain the constant total production rate of NADPH. The reduction in the TCA flux in turn manifests in a lower <i>v</i> <sub> <i>r</i> </sub>. In the overflow region, Eq. ( <a href="#Equ7">6</a>) is bounded by the equal sign. As shown earlier, the level of <i>uPPP</i>% has a negligible impact (when nominal energy demand is adopted) on the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M121">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> line. Also recall that the relation between the rate of acetate excretion and steady state growth rate is fixed by the experimentally measured growth data. With all the other quantities ( <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M123">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, <i>v</i> <sub> <i>f</i> </sub>, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M125">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>λ</i>) fixed in Eq. ( <a href="#Equ7">6</a>), the drop in <i>v</i> <sub> <i>r</i> </sub> due to the increase in <i>uPPP</i>% will necessarily be accompanied by an increase in <i>b</i> <sup>∗</sup>. <p /> <p>Between different strains, <i>b</i> <sup>∗</sup> varies significantly. In particular, <i>b</i> <sup>∗</sup> for MG is remarkably larger than that of NCM and ML (as presented in Additional file <a href="#MOESM2">2</a>: Figure S1). This disparity can again be explained by Eq. ( <a href="#Equ7">6</a>). For a certain value of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M127">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M129">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> is rather similar among different strains (with nominal energy demand) as shown by Fig. <a href="#Fig2">2</a>. In the overflow region, the respiration flux <i>v</i> <sub> <i>r</i> </sub> of MG is much smaller than the others (see Fig. <a href="#Fig1">1</a>), which thus leads to a lower value of the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M131">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }{v}_r $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> term for MG than NCM and ML. As the value of the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M133">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }{v}_f $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> term (for any selected value of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M135">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>) is similar between these strains, due to their similarity in the relationship between <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M137">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M139">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, the value of the remaining term on the left-hand side of Eq. ( <a href="#Equ7">6</a>), <i>b</i> <sup>∗</sup> <i>λ</i>, must be higher for MG than for the other two strains. On the other hand, in the overflow region and at a same acetate excretion rate <i>v</i> <sub> <i>f</i> </sub>, the growth rate of MG has been shown to be much lower than that of NCM and ML. Now, a higher value of <i>b</i> <sup>∗</sup> <i>λ</i> coupled with a lower value of <i>λ</i> will undoubtedly lead to a higher value of <i>b</i> <sup>∗</sup>for MG, compared to the other two strains. <p /> <p>The above mathematical explanation in fact coincides with the known biological fact that the inverse of <i>b</i> <sup>∗</sup> is proportional to the rate of protein synthesis [ <a href="#CR31">31</a>]: the slower the rate of protein synthesis, the higher the value of <i>b</i> <sup>∗</sup>. Thus for the slow-growing strain MG, it is expected to have a higher value of <i>b</i> <sup>∗</sup>compared to the fast-growing strains NCM and ML. </p> </div> <div class="predictedevolutionofpppathwayflux" title="sec"> <div class="title" tagx="title" title="title">
Predicted evolution of PP pathway flux</div> <p>The results presented above show rather moderate impact of the upper limit of PP pathway ratio ( <i>uPPP</i>%) on the linear interdependency of the proteomic cost parameters. With an interest in the FBA solution of the flux distribution in PP pathway (at different growth rates), simulation results were recorded for three strains with <i>uPPP</i>%set to 35%; other <i>uPPP%</i> levels displayed a similar trend (as presented in Additional file <a href="#MOESM2">2</a>: Figures S4 and S5). Flux variability analysis (FVA) [ <a href="#CR43">43</a>] was performed to confirm that the trend of <i>PPP</i>% presented here was unique. </p> <p>In general, <i>PPP</i>% gradually increases with the growth rate. Two turning points can be observed, which divide the whole curve into three distinct phases (Fig. <a href="#Fig4">4</a>a). A close inspection of the model simulation revealed that the variation of the predicted PP pathway ratio was co-related particularly with three fluxes, namely NAD transhydrogenase (NADTRHD), transketolase (TKT2) and NADP transhydrogenase (THD2). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p> <b>a</b> Simulation results of <i>PPP</i>%, NADTRHD, TKT2 and THD2 against growth rates at nominal energy demand. <b>b</b> Comparison between predicted trend of <i>PPP</i>% and experimental data. PP pathway ratio ( <i>PPP</i>%) is divided by ten (0.1*PPP%) to unify the order of magnitude between different data types. Experimental data were obtained from [ <a href="#CR41">41</a>]. <i>uPPP</i>% was set to 35%. Simulation was based on MG1655. NADTRH – NAD transhydrogenase, TKT – transketolase, THD2 – NADP transhydrogenase, simu – simulation results, exp. – experimental data </p>   </div> <p /> <p>In phase I, only NADTRHD is active, with zero fluxes for both TKT2 and THD2. The enzymatic reaction NADTRHD functions to convert NADPH into NADH. Thus in phase I, it is likely that the amount of NADPH produced exceeds the required amount for biosynthesis; NADTRHD is thus activated to consume the surplus NADPH. </p> <p>In phase II, an on/off swap occurs between NADTRHD and TKT2 while THD2 still remains silent. We infer that in this phase, NADPH produced satisfies the demand, but the amount of carbon flowing into the PP pathway surpasses the rate of the carbon withdrawal (for the synthesis of biomass precursors). Therefore, TKT2 is activated to direct the extra amount of four-carbon and five-carbon compounds back to the glycolysis. </p> <p>In phase III, THD2 is finally switched on and becomes significantly active in the high-growth-rates region. TKT2 increases progressively while NADTRHD remains silent. It is presumed that in this phase, as the growth rate becomes higher, more NADPH is required for biomass synthesis. NADP transhydrogenases (THD2) is activated to produce NADPH needed in rapid growth. The surplus carbon flux in the PP pathway, which might result from the high glucose uptake rate at a high growth rate, is directed back to glycolysis via TKT2. </p> <p>It would be desirable to verify the theoretical prediction of the evolution of <i>PPP</i>% with experimental measurements, which unfortunately have not been widely reported in the literature. Nevertheless, Fig. <a href="#Fig4">4</a>b shows a comparison with one set of experimental observations available [ <a href="#CR42">42</a>], which suggests a good degree of qualitative similarity. </p> </div> <div class="adjustingcellularenergydemandimprovesthepredictionofbiomassyield" title="sec"> <div class="title" tagx="title" title="title">
Adjusting cellular energy demand improves the prediction of biomass yield</div> <p>Although combining the PAT constraint with the core model succeeded in predicting the rates of acetate production, the accuracy in biomass yield varied and was especially unsatisfactory for ML strain (Fig. <a href="#Fig5">5</a>). A similar deficiency in yield prediction was also reported in [ <a href="#CR28">28</a>]. Focusing on the yield, two features can be observed: (i) in the overflow region, for a fixed growth rate (associated with an acetate excretion rate) the biomass yield for ML is higher than MG and NCM; and (ii) the rate of the drop in yield (i.e. the slope) of ML is sharper than the other two strains. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Comparison of the biomass yield between model predictions with nominal energy demand and the experimental data. "-nom" refers to nominal, the default energy demand specified in the core model. Biomass yield is calculated as gram biomass produced per gram substrate consumed. <i>uPPP</i>% was set to 35% for all strains. Experimental data were obtained from the same sources [ <a href="#CR3">3</a>, <a href="#CR16">16</a>, <a href="#CR41">41</a>] of the acetate data shown in Fig. <a href="#Fig1">1</a>. simu – simulation results, exp. – experimental data </p>   </div> <p /> <p>Intuitively, feature (i) suggests that in ML, the amount of energy required per unit mass of biomass formation should be less than NCM or MG. Therefore we collected the growth data of ML and remodelled the cellular energy demand (see <a href="#Sec2">Methods</a>). </p> <p>It is worth noting that for ML, the negative value of M (Table <a href="#Tab1">1</a>) clearly indicates a constrained applicable range of the maintenance parameters, i.e. valid only within the overflow region. As growth rate decreases, if M stays unchanged, the overall energy consumption (Eq. ( <a href="#Equ17">16</a>)) will drop to a negative value, which is clearly not biologically feasible. This then implies a certain degree of nonlinearity in the global relationship between (total or maintenance) energy requirement and growth rate. Such proposition was previously referred to as "varied non-growth-associated maintenance" [ <a href="#CR3">3</a>]. Non-linearity in energy consumption manifesting before and after the onset of the overflow metabolism has also been observed and discussed in a recent work [ <a href="#CR44">44</a>]. </p> </div> <div class="modelpredictionofbiomassyieldwithadjustedenergydemand" title="sec"> <div class="title" tagx="title" title="title">
Model prediction of biomass yield with adjusted energy demand</div> <p>We first re-estimated the set of proteomic cost parameters for ML with the adjusted energy demand (see Table <a href="#Tab1">1</a> and <a href="#Sec2">Methods</a>). Applying updated values of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M141">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M143">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup> together with the adjusted maintenance energy, our model is now able to effectively capture the unique trend of biomass yield for ML, without any compromise in the accuracy of predicting acetate excretion (Fig. <a href="#Fig6">6</a>). Simulation results for ML with adjusted energy demand are referred to as "ML-new". <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Model prediction of acetate production and biomass yield for ML with adjusted energy demand compared with experimental data. "-new" refers to the adjusted energy demand. Simulation was done with adjusted energy demand (Table <a href="#Tab1">1</a>, M,N for ML) and updated proteomic cost parameters. Biomass yields are shown as ten times of the original value to unify the order of magnitude between different types of data. <i>uPPP</i>% was set to 35%. Experimental data was obtained from Table 7 in [ <a href="#CR3">3</a>]. ac – acetate flux, Yxs – biomass yield, simu – simulation results, exp. – experimental data </p>   </div> <p /> <p>It is worth noting that, our model also succeeds in matching the elevated reduction in the yield of ML in the overflow region as the growth rate increases. This captured trend appears to originate from the low energy demand of ML. Approximately, the yield reduction rate can be considered as being proportional to the ratio of the increase in the acetate excretion ( <i>ac</i> <sub>2</sub> −  <i>ac</i> <sub>1</sub>) and the increase in glucose uptake rate ( <i>glc</i> <sub>2</sub> −  <i>glc</i> <sub>1</sub>), while the growth rate rises from <i>λ</i> <sub>1</sub> to <i>λ</i> <sub>2</sub>: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">21</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M145">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ yield\ reduction\ rate\propto \frac{ac_2-{ac}_1}{glc_2-{glc}_1}, for\ {\lambda}_1\to {\lambda}_2\ \left({\lambda}_2&amp;gt;{\lambda}_1\right) $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mtext_UNKNOWN">
yield reduction rate</div> <div class="mo_UNKNOWN">
∝</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ac</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ac</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
glc</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
glc</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
for</div> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
→</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mspace_UNKNOWN" /> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>NCM and ML exhibit similar acetate excretion rates, hence a similar value in " <i>ac</i> <sub>2</sub> −  <i>ac</i> <sub>1</sub>". However, the energy demand per unit growth of ML is much lower than NCM, which means that with a similar increase in acetate production, the increase in substrate intake (i.e. <i>glc</i> <sub>2</sub> −  <i>glc</i> <sub>1</sub>) for ML will be lower than NCM to achieve a given increment in the growth rate. According to Eq. ( <a href="#Equ22">21</a>), the yield reduction rate of ML will thus be higher than NCM. </p> </div> <div class="impactoftheadjustedenergydemandon\documentclass[12pt]{minimal}\usepackage{amsmath}\usepackage{wasysym}\usepackage{amsfonts}\usepackage{amssymb}\usepackage{amsbsy}\usepackage{mathrsfs}\usepackage{upgreek}\setlength{\oddsidemargin}{-69pt}\begin{document}$${w}_r^{\ast}-{w}_f^{\ast}$$\end{document}wr∗−wf∗and\documentclass[12pt]{minimal}\usepackage{amsmath}\usepackage{wasysym}\usepackage{amsfonts}\usepackage{amssymb}\usepackage{amsbsy}\usepackage{mathrsfs}\usepackage{upgreek}\setlength{\oddsidemargin}{-69pt}\begin{document}$${b}^{\ast}-{w}_f^{\ast}$$\end{document}b∗−wf∗relationships" title="sec"> <div class="title" tagx="title" title="title">
Impact of the adjusted energy demand on <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M147">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M149">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> relationships </div> <p>To investigate the impact of the change in cellular energy demand on the linear relationships of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M151">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M153">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup> of ML-new, we recalculated constants <i>k</i> <sub> <i>f</i> </sub>, <i>v</i> <sub> <i>f</i>, 0 </sub>, <i>k</i> <sub> <i>r</i> </sub> and <i>v</i> <sub> <i>r</i>, 0 </sub> at different <i>uPPP</i>% values (25, 35 and 40%) to update the linear equations describing <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M155">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> line and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M157">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> (Eqs. ( <a href="#Equ12">11</a>) and ( <a href="#Equ13">12</a>)). The resulting <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M159">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> lines for ML-new are plotted in Fig. <a href="#Fig2">2</a>, together with the results obtained earlier for MG/NCM/ML with nominal energy demand. <p /> <p>The switch to the adjusted energy demand makes the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M161">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  value for ML-new much higher than that of ML-nom, the latter being rather close to those of the MG-nom and NCM-nom. This implies that the adjustment of the energy demand of ML leads to an enlarged gap in the proteomic efficiency between respiration and fermentation. <p /> <p>The similarity among MG-/NCM−/ML-nom has already been discussed in the previous section. Here we mainly focus on the discrepancy with ML-new. We found that both the slope and intercept of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M163">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  line for ML-new are about three times larger than ML-nom (as shown in Additional file <a href="#MOESM1">1</a>: Table S4). The dramatic changes in the slope and intercept of ML-new predominantly result from the reduction in the respiratory flux <i>v</i> <sub> <i>r</i> </sub> when applying the adjusted energy demand (see Fig. <a href="#Fig7">7</a> and Additional file <a href="#MOESM2">2</a>: Figure S3). <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 7</span> </a> </div>   <p>Comparison of the predicted respiration and acetate fluxes between ML-new and ML-nom. "-nom" refers to nominal, the default energy demand specified in the core model. "-new" refers to the adjusted energy demand. The predicted rates of acetate production for ML-nom and ML-new are completely overlapped with each other. <i>uPPP</i>% was set to 35% for both strains. Data source of acetate excretion is shown in Fig. <a href="#Fig1">1</a>. ac – acetate flux, vr – respiration flux, simu – simulation results, exp. – experimental data </p>   </div> <p /> <p>The link between the drop in <i>v</i> <sub> <i>r</i> </sub> and the increase in <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M165">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  has been discussed in the previous section. The results presented herein indicate that it is the energy demand that plays a major role in distinguishing the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M167">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> relationship between different strains, not the <i>uPPP</i>% or the acetate excretion pattern. <p /> <p>Applying the adjusted energy demand also has an impact on the relationship between <i>b</i> <sup>∗</sup> and <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M169">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> . As shown in Fig. <a href="#Fig8">8</a>, the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M171">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> lines are significantly right-shifted when the model is changed from ML-nom to ML-new (i.e. red lines are located in a much right area than yellow lines). Given the identical pattern of acetate excretion between ML-nom and ML-new (as both predicted the same set of experimental data), the amount of energy produced through fermentation remains unchanged. For ML-new, as the energy demand per unit of growth is much lower than that of the nominal strain, the respiratory flux <i>v</i> <sub> <i>r</i> </sub> must decrease significantly to avoid energy overproduction, as confirmed in Fig. <a href="#Fig7">7</a>. Although the value of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M173">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> for ML-new is higher than that for ML-nom (for a given value of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M175">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>, Fig. <a href="#Fig2">2</a>), the value of the product <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M177">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast }{v}_r $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
r</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> for ML-new still becomes lower (as the increase in <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M179">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> is not able to compensate for the sharp drop in <i>v</i> <sub> <i>r</i> </sub>). With no change in <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M181">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }{v}_f $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> between ML-new and ML-nom (for a given <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M183">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>), Eq. ( <a href="#Equ7">6</a>) again dictates <i>b</i> <sup>∗</sup> to become higher for ML-new than ML-nom, hence the right-shifting of the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M185">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> lines. <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 8</span> </a> </div>   <p>Comparison of the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M187">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  relationship between ML-nom and ML-new at different <i>uPPP</i>% levels. "-nom" refers to nominal, the default energy demand specified in the core model. "-new" refers to the adjusted energy demand. <i>uPPP%</i> was set to 25, 35 and 40% for both strains <p />   </div> <p /> <p>In the case of ML-nom, it was shown earlier in Fig. <a href="#Fig3">3</a> that the increase in <i>uPPP</i>% would lead to a reduction of <i>v</i> <sub> <i>r</i> </sub>, which in turn would lead to an increase in <i>b</i> <sup>∗</sup> or right-shifting of the <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M189">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div>  line. Now for ML-new, an enlarged gap between the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M191">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}^{\ast }-{w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> lines at different <i>uPPP</i>% levels is observed compared to the case of ML-nom. This implies that the effect of <i>v</i> <sub> <i>r</i> </sub> reduction due to the increase in <i>uPPP</i>% is more pronounced with the adjusted energy demand. <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="comparisonwithrelevantmodels" title="sec"> <div class="title" tagx="title" title="title">
Comparison with relevant models</div> <p>In the study by Basan et al. [ <a href="#CR16">16</a>], which has been the basis of the PAT constraint formulated in our model, the application of a constraint on proteome fractions (similar to Eq. ( <a href="#Equ5">4</a>)) with parameters derived from measured protein abundances was able to accurately predict the patterns of acetate excretion for <i>E. coli</i> under different growth conditions, when coupled with a simple energy balance equation. In this work, we have embedded the PAT into the core metabolic model of <i>E. coli</i>, taking the advantage of the latter in offering more rigorous modelling of intracellular mass and energy balances. Furthermore, the constraint-based metabolic model allows prediction of detailed metabolic fluxes as opposed to merely acetate production, which could provide more insights about metabolic pathways in connection with the overflow metabolism and pave the way for investigating acetate excretion in junction with possible manipulations of the metabolic network. </p> <p>In addition, the respiratory flux in Basan’s work was associated with the carbon dioxide produced in respiration, termed <i>J</i> <sub> <i>CO</i>2, <i>r</i> </sub>, whose value was deduced by subtracting the fermentation-dependent CO <sub>2</sub> and the growth-dependent CO <sub>2</sub> from the total CO <sub>2</sub> production. As such, <i>J</i> <sub> <i>CO</i>2, <i>r</i> </sub> could not directly correspond to a specific flux in the metabolic network. In our model, the respiratory flux directly refers to a specific flux within the TCA cycle (AKGDH), which appears to be a convenient choice when the PAT is embedded into FBA. Using a constraint-based metabolic model that includes the TCA cycle with a reasonable level of detail, the respiration flux can be directly resolved via FBA, without the need for multi-step calculation along with different levels of assumptions and uncertainties. </p> <p>Another important comparison we would like to make is with the recently developed model – CAFBA. It was mentioned in the <a href="#Sec1">Background</a> section that unlike CAFBA which considers the proteomic cost of every individual reaction in metabolic network, the PAT constraint in our model follows the treatment of Basan et al.’s work and quantifies the proteomic costs at the pathway level. This simplification allows the model to explicitly incorporate the differential proteomic efficiencies of fermentation and respiration that are proven (in both Basan et al. and this work) to govern the flux split between the two pathways. This simplicity comes at the cost of the limited utility of our model: it is intended to be used only for predicting the interplay between the excretion of acetate (or other fermentation products) and the growth rate during overflow metabolism, not other effects of stressed resource allocation. </p> <p>The proteome allocation constraint in CAFBA includes a C-sector (via a term expressed as <i>w</i> <sub> <i>c</i> </sub> <i>v</i> <sub> <i>c</i> </sub>) representing the proteome requirement for uptaking carbon source, which is not explicitly considered in this work. We have ignored this sector as the carbon overflow occurs only at the high growth rate region, where <i>w</i> <sub> <i>c</i> </sub> (proteomic cost of the C <i>-</i>sector) approaches zero at high substrate uptake rates, as shown in CAFBA [ <a href="#CR28">28</a>]. In this region, the low value of <i>w</i> <sub> <i>c</i> </sub> makes the C-sector negligible compared with other proteome sectors. At low growth rate region, the value of <i>w</i> <sub> <i>c</i> </sub> becomes significant, however no acetate is excreted in this region, where the equality relationship in the PAT constraint in our work becomes inactive so that the significance of the C-sector becomes irrelevant. </p> <p>As for the prediction of biomass yield, CAFBA noticed the difficulty in predicting the biomass yield of ML308. In this work, we have found that it is the cellular energy demand that significantly affects the FBA prediction of biomass yield. After replacing the default energy demand with data reported specifically for ML308 strain, our model was able to produce an accurate prediction. Therefore, we consider that it is important to carry out necessary adjustment to the cellular energy demand when applying such a constraint-based modelling approach to specific strains. </p> </div> <div class="parameterisationoftheproteomeallocationconstraint" title="sec"> <div class="title" tagx="title" title="title">
Parameterisation of the proteome allocation constraint</div> <p>The modelling approach proposed in this work can be considered as "halfway" between the coarse-grained proteome allocation model of Basan et al. [ <a href="#CR16">16</a>] and the FBA models that incorporate reaction-level resource allocation constraints such as CAFBA [ <a href="#CR28">28</a>] and FBAwMC [ <a href="#CR21">21</a>]. In CAFBA, the proteome constraint involves ~ 1000 proteomic cost parameters ( <i>w</i> <sub> <i>i</i> </sub>) for a genome-scale model. Similarly in FBAwMC, a large number of crowing coefficients need to be specified. In both cases, the existence of numerous cost parameters originates from associating the resource cost with individual reactions. These parameters conceptually have a clear biological meaning and in principle can be determined experimentally by e.g. proteome measurements or extensive enzyme assays. However, in practice, it has appeared to be difficult to reliably obtain precise values for all the parameters, especially for different strains growing at different growth rates or conditions. In fact, instead of pursuing the exact values for all the individual parameters, CAFBA focused on applying the average value of the proteome fraction invested per unit flux, termed as 〈 <i>w</i>〉, to capture the key flux pattern, along with evaluating the impact of possible heterogeneous values of the proteome parameter <i>w</i> <sub> <i>i</i> </sub> on the model prediction. Similarly, FBAwMC [ <a href="#CR21">21</a>, <a href="#CR22">22</a>] also appears to encounter a certain degree of "randomness" of its crowding coefficients due to the unknown enzyme kinetics and/or turnover numbers. Subsequently, molecular-crowding-based modelling normally treats this "randomness" as noise, where the crowding coefficients are chosen randomly from a distribution of crowding coefficients [ <a href="#CR45">45</a>] or the majority of the crowding coefficients are estimated from a limited number of known enzyme turn-over (k <sub>cat</sub>) values [ <a href="#CR11">11</a>]. </p> <p>In this work, we have intended to formulate a constraint with a greatly reduced number of proteomic cost parameters, while still capturing the essence of constrained cellular resource allocation. This is achieved by formulating the proteome allocation constraint at the pathway (as opposed to reaction) level. The proposal is the concise Eq. ( <a href="#Equ7">6</a>) ( <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M193">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }{v}_f+{w}_r^{\ast }{v}_f+{b}^{\ast}\lambda \le 1 $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
1</div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> ), involving only three proteomic cost parameters (representing proteomic efficiencies of fermentation, respiration and biomass synthesis pathways). In principle, these parameters can be obtained through the direct measurement of protein abundances, following an approach similar to that adopted by Basan et al.’s work [ <a href="#CR16">16</a>]. However, in the current study, we attempted to parameterise this constraint using widely available growth data from cell culturing experiments, in particular growth rate and acetate production rate. It should be noted that cell culturing experiments often yield relatively simple data sets with measurements of a few process variables. It is infeasible to use such data sets to determine a large number of proteomic cost parameters encountered in a proteome constraint expressed at the individual reaction level. Even with the pathway-level constraint adopted in this work, our results show that cell growth and acetate production measurements (alone) cannot uniquely determine the three parameters, but two linear relationships between these parameters can be derived (Eqs. ( <a href="#Equ12">11</a>) and ( <a href="#Equ13">12</a>)). <p /> <p>Furthermore, our model shows that it is the two linear relationships (but not the absolute values) of the proteomic cost parameters that allow an accurate prediction of the overflow metabolism. We thus speculate that for a FBA-based model, the ability of capturing the overflow behaviour is rendered by (i) an extra constraint representing the constrained proteomic resources and (ii) certain relations or relative magnitudes of the proteomic cost parameters embedded in the proteome constraint. In reality, the proteomic efficiencies of the metabolic pathways may vary (within a certain range), at different points in time or between cells in a population which often exhibits heterogeneity [ <a href="#CR28">28</a>]. However, as long as the specific relations or relative magnitudes of these efficiencies are maintained, one can expect that the overflow behaviour will emerge. </p> </div> <div class="applicabilityofthelinearformulationoftheproteomiccost" title="sec"> <div class="title" tagx="title" title="title">
Applicability of the linear formulation of the proteomic cost</div> <p>As indicated in the earlier section, our formulation of the proteomic cost (Eqs. ( <a href="#Equ2">2a</a>) and ( <a href="#Equ3">2b</a>)) reflects the observed linear dependency between proteome fraction and growth rate [ <a href="#CR30">30</a>– <a href="#CR32">32</a>]. Combining this linear dependency with the assumption that the flux processed by a proteome sector <i>i</i> is proportional to the growth rate-dependent component of the associated proteome fraction [ <a href="#CR30">30</a>], we have related <i>ϕ</i> <sub> <i>i</i> </sub> linearly with the flux it carries. A similar model is also adopted in CAFBA [ <a href="#CR28">28</a>], where the linear proteome-flux relation is derived on the assumption that the substrate concentration is proportional to the flux. </p> <p>Note that our model is intended specifically for predicting the steady-state overflow metabolism in <i>E. coli</i> under glucose-limited conditions. In some other circumstances, observations not conforming to this relatively simple model have been reported. For example, Goel and his co-workers found hardly any changes in protein levels in anaerobic slow-growing <i>Lactococcus lactis</i> chemostats <i>,</i> when the cell shifted from a high yield metabolic mode to a low yield metabolic mode with an increased growth rate [ <a href="#CR46">46</a>]. In this case, although the metabolic shift in <i>L. lactis</i> is similar to the overflow metabolism observed in <i>E. coli</i>, proteome allocation did not seem to accompany the changes in metabolic fluxes. In a study on yeast’s transient transcript, enzyme and metabolite responses under metabolic perturbation, it is revealed that the reaction rates are jointly regulated by enzyme capacity and metabolite concentration due to the cell’s tendency in sacrificing the local metabolite homeostasis to maintain fluxes and global metabolite homeostasis upon enzyme perturbation [ <a href="#CR47">47</a>]. Another yeast-based study also suggests that changes in individual flux are predominantly regulated by the levels of metabolites, not enzymes [ <a href="#CR48">48</a>]. </p> <p>The above-mentioned experimental observations suggest that the linear proteome-flux relationship modelled in this work might not be applicable to those circumstances. We hypothesise that this might be at least partially due to the differences between <i>E. coli</i> (the target organism of our model) and the organisms with which those observations were made. Besides, our model has been developed to describe the relationship between (i) the observed steady-state global proteome configuration and (ii) the growth rate or the corresponding flux, drawing on evidences and hypothesises from several previous studies [ <a href="#CR16">16</a>, <a href="#CR28">28</a>, <a href="#CR30">30</a>, <a href="#CR32">32</a>]. Such a model, being global and coarse-grained, is not intended to represent delicate regulatory mechanisms responsible for the transient metabolic changes to maintain cellular homeostasis under perturbations, and might not be suitable for revealing the local regulatory insights on the key factors dictating the individual reaction rates. </p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>With three different <i>E. coli</i> strains, we have evaluated a new model that integrates a previously proposed proteome allocation theory (PAT) into the constraint-based modelling approach – flux balance analysis (FBA), which predicts the distribution of carbon fluxes between fermentation and respiration due to the differential proteomic efficiencies of the two energy biogenesis pathways. Using a simple proteome allocation constraint, our model allows the accurate prediction of acetate production at different steady state growth rates during overflow conditions (with sufficient oxygen and glucose). The model involves three pathway-level proteomic cost parameters linearly interrelated by two equations, which is the consequence of (i) the assumed linear dependency of proteomic costs and the growth rate and (ii) the experimentally observed linear correlation between the fermentation or respiration flux and the growth rate. The non-unique optimal values of the three parameters, or the two linear relationships between them, could be obtained by fitting the model to experimentally measured acetate excretion rates at specific growth rates. </p> <p>The linear relationships between the parameters were shown to be affected, in varying degrees, by (i) the acetate excretion pattern, (ii) the assumed upper limit of the substrate carbon diverting into PP pathway and (iii) the cellular energy demand. The proteomic cost of the fermentation pathway was estimated always to a lower value than that of the respiration pathway, i.e. <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M195">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }&amp;lt;{w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> . The proteomic cost of the biomass synthesis sector was estimated to be higher in a slow-growing strain that excretes acetate at a lower growth rate, in comparison with the other two fast-growing strains, i.e. <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M197">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {b}_{\mathrm{MG}}^{\ast }&amp;gt;{b}_{\mathrm{NCM}/\mathrm{ML}}^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
MG</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
NCM</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
ML</div> </div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span>. The estimated values of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M199">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> and <i>b</i> <sup>∗</sup>both meet qualitatively the expectation from a biological point of view. Furthermore, the relationship between the proteomic efficiencies of fermentation and respiration, i.e. the <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M201">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }-{w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> line, was shown to change between different strains most significantly with the cellular energy demand rather than with the pattern of acetate excretion. This <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M203">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {w}_f^{\ast }-{w}_r^{\ast } $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
∗</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> </span> relationship remained relatively stable when the upper bound of the portion of substrate carbon flowing into PP pathway ( <i>uPPP</i>%) varied in the modelling studies. On the other hand, the increase of <i>uPPP</i>% was shown to lead to a visible increase in the estimated proteomic cost of the biomass synthesis sector <i>b</i> <sup>∗</sup>, which mathematically results from a reduction in the predicted respiration flux. <p /> <p>Finally, and as a general point for constraint-based models, cellular energy demand appeared to have a major impact on the predicted biomass yield; tuning the default energy demand with strain-specific data was shown to be critical in making simultaneously accurate predictions of biomass yield and overflow metabolism. </p> <p>Overall, this work demonstrates the potential of combining a detailed metabolic model with a coarse-grained, pathway-level resource allocation constraint in producing quantitatively accurate predictions of the overflow phenomenon in <i>E coli</i>; similar modelling approaches that feature this type of combination may prove suitable also for other applications. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12918_2018_677_MOESM1_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 1:</span>  <p> Supplementary Tables S1-S6. (XLSX 26 kb)</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12918_2018_677_MOESM2_ESM.docx">LINK</a> <span class="label" tagx="label" title="label">Additional file 2:</span>  <p> Supplementary text and Supplementary Figures S1-S17. (DOCX 229 kb)</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
FBA</div> <div class="def" title="def"> <p>Flux balance analysis</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GAM</div> <div class="def" title="def"> <p>Growth-associated maintenance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NADTRHD</div> <div class="def" title="def"> <p>NAD transhydrogenase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NGAM</div> <div class="def" title="def"> <p>Non-growth-associated maintenance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PAT</div> <div class="def" title="def"> <p>Proteome allocation theory</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PPP%</div> <div class="def" title="def"> <p>Pentose phosphate pathway ratio</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
THD2</div> <div class="def" title="def"> <p>NAD(P) transhydrogenase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TKT2</div> <div class="def" title="def"> <p>Transketolase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
uPPP%</div> <div class="def" title="def"> <p>Upper bound of the pentose phosphate pathway ratio</p> </div> </div> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We would like to thank the two anonymous reviewers for their constructive suggestions and comments on the original manuscript. </p> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>HZ is supported by the China Scholarship Council through a PhD scholarship. The funding body played no role in the design, analysis, data interpretation or manuscript writing of this study. </p> </div> <div class="availabilityofdataandmaterials" title="sec"> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All model equations, parameter values are included in the main text or in the Supporting Information (Additional files <a href="#MOESM1">1</a> and <a href="#MOESM2">2</a>). The MATLAB code for running simulations and the generated datasets in this study are available upon request from the corresponding author. </p> </div> </div> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>HZ performed the modelling work, analysed and interpreted the results, and wrote the manuscript. AY designed and supervised the study, analysed and interpreted the results, and revised the manuscript. Both authors have read and approved the manuscript. </p> <div class="ethicsapprovalandconsenttoparticipate" title="sec"> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>Not applicable.</p> </div> <div class="consentforpublication" title="sec"> <div class="title" tagx="title" title="title">
Consent for publication</div> <p>Not applicable.</p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="publisher’snote" title="sec"> <div class="title" tagx="title" title="title">
Publisher’s Note</div> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eiteman</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altman</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Overcoming acetate in Escherichia coli recombinant protein fermentations</span> <span class="source" tagx="source" title="source">Trends Biotechnol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">530</span> <span class="lpage" tagx="lpage" title="lpage">536</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tibtech.2006.09.001">10.1016/j.tibtech.2006.09.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16971006">16971006</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farmer</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liao</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Reduction of aerobic acetate production by Escherichia coli</span> <span class="source" tagx="source" title="source">Appl Environ Microbiol</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">3205</span> <span class="lpage" tagx="lpage" title="lpage">3210</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9251207">9251207</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holms</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Flux analysis and control of the central metabolic pathways in Escherichia coli</span> <span class="source" tagx="source" title="source">FEMS Microbiol Rev</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">116</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1574-6976.1996.tb00255.x">10.1111/j.1574-6976.1996.tb00255.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8988566">8988566</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rhee</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lebeault</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Physiological constraints in increasing biomass concentration of Escherichiacoli B in fed-batch culture </span> <span class="source" tagx="source" title="source">Biotechnol Lett</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">89</span> <span class="lpage" tagx="lpage" title="lpage">94</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF01032744">10.1007/BF01032744</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jahic</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Enfors</span> <span class="given-names" tagx="given-names" title="given-names">SO</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Modeling of overflow metabolism in batch and fed-batch cultures of Escherichia coli</span> <span class="source" tagx="source" title="source">Biotechnol Prog</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">81</span> <span class="lpage" tagx="lpage" title="lpage">90</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/bp9801087">10.1021/bp9801087</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9933517">9933517</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SY</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High cell-density culture of Escherichia coli</span> <span class="source" tagx="source" title="source">Trends Biotechnol</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">98</span> <span class="lpage" tagx="lpage" title="lpage">105</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0167-7799(96)80930-9">10.1016/0167-7799(96)80930-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8867291">8867291</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holmes</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The central metabolic pathways of Escherichia coli: relationship between flux and control at a branch point, efficiency of conversion to biomass, and excretion of acetate </span> <span class="source" tagx="source" title="source">Curr Top Cell Regul</span> <span class="year" tagx="year" title="year">1986</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">69</span> <span class="lpage" tagx="lpage" title="lpage">105</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/B978-0-12-152828-7.50004-4">10.1016/B978-0-12-152828-7.50004-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3098503">3098503</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luli</span> <span class="given-names" tagx="given-names" title="given-names">GW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strohl</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of growth, acetate production, and acetate inhibition of Escherichia coli strains in batch and fed-batch fermentations </span> <span class="source" tagx="source" title="source">Appl Environ Microbiol</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">1004</span> <span class="lpage" tagx="lpage" title="lpage">1011</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2187400">2187400</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiao</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephanopoulos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Distributing a metabolic pathway among a microbial consortium enhances production of natural products </span> <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">377</span> <span class="lpage" tagx="lpage" title="lpage">383</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.3095">10.1038/nbt.3095</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25558867">25558867</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schulz</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thierbach</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voigt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drewes</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mietzner</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinberg</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Induction of oxidative metabolism by mitochondrial frataxin inhibits cancer growth Otto Warburg revisited </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">281</span> <span class="fpage" tagx="fpage" title="fpage">977</span> <span class="lpage" tagx="lpage" title="lpage">981</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M511064200">10.1074/jbc.M511064200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16263703">16263703</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Shlomi T, Benyamini T, Gottlieb E, Sharan R, Ruppin E. Genome-scale metabolic modeling elucidates the role of proliferative adaptation in causing the Warburg effect. PLoS Comput Biol. 2011;7. 10.1371/journal.pcbi.1002018. </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Vazquez A, Liu J, Zhou Y, Oltvai ZN. Catabolic efficiency of aerobic glycolysis: the Warburg effect revisited. BMC Syst Biol. 2010;4. 10.1186/1752-0509-4-58. </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schuster</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boley</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Möller</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stark</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaleta</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mathematical models for explaining the Warburg effect: a review focussed on ATP and biomass production </span> <span class="source" tagx="source" title="source">Biochem Soc Trans</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">1187</span> <span class="lpage" tagx="lpage" title="lpage">1194</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BST20150153">10.1042/BST20150153</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26614659">26614659</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfe</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The acetate switch</span> <span class="source" tagx="source" title="source">Microbiol Mol Biol Rev</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">12</span> <span class="lpage" tagx="lpage" title="lpage">50</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/MMBR.69.1.12-50.2005">10.1128/MMBR.69.1.12-50.2005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15755952">15755952</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Molenaar D, van Berlo R, de Ridder D, Teusink B. Shifts in growth strategies reflect tradeoffs in cellular economics. Mol Syst Biol. 2009;5. 10.1038/msb.2009.82. </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Basan</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hui</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okano</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williamson</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Overflow metabolism in Escherichia coli results from efficient proteome allocation</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">528</span> <span class="fpage" tagx="fpage" title="fpage">99</span> <span class="lpage" tagx="lpage" title="lpage">104</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature15765">10.1038/nature15765</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26632588">26632588</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Enjalbert</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millard</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dinclaux</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Portais</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Létisse</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Acetate fluxes in Escherichia coli are determined by the thermodynamic control of the Pta-AckA pathway </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">42135</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep42135">10.1038/srep42135</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28186174">28186174</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anane</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">López</span> <span class="given-names" tagx="given-names" title="given-names">CDC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neubauer</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cruz Bournazou</span> <span class="given-names" tagx="given-names" title="given-names">MN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Modelling overflow metabolism in Escherichia coli by acetate cycling</span> <span class="source" tagx="source" title="source">Biochem Eng J</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">125</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">30</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bej.2017.05.013">10.1016/j.bej.2017.05.013</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goel</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wortel</span> <span class="given-names" tagx="given-names" title="given-names">MT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Molenaar</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teusink</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolic shifts: a fitness perspective for microbial cell factories</span> <span class="source" tagx="source" title="source">Biotechnol Lett</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">2147</span> <span class="lpage" tagx="lpage" title="lpage">2160</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10529-012-1038-9">10.1007/s10529-012-1038-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22936303">22936303</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orth</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thiele</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palsson</span> <span class="given-names" tagx="given-names" title="given-names">BØ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">What is flux balance analysis?</span> <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">245</span> <span class="lpage" tagx="lpage" title="lpage">248</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.1614">10.1038/nbt.1614</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20212490">20212490</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Beg QK, Vazquez A, Ernst J, de Menezes MA, Bar-Joseph Z, Barabasi AL, et al. Intracellular crowding defines the mode and sequence of substrate uptake by Escherichia coli and constrains its metabolic activity. P Natl Acad Sci USA. 2007;104. 10.1073/pnas.0609845104. </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Vazquez A, Beg QK, Demenezes MA, Ernst J, Bar-Joseph Z, Barabasi AL, et al. Impact of the solvent capacity constraint on E. coli metabolism. BMC Syst Biol. 2008;2. 10.1186/1752-0509-2-7. </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goelzer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fromion</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scorletti</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cell design in bacteria as a convex optimization problem</span> <span class="source" tagx="source" title="source">Automatica</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">1210</span> <span class="lpage" tagx="lpage" title="lpage">1218</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.automatica.2011.02.038">10.1016/j.automatica.2011.02.038</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goelzer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fromion</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bacterial growth rate reflects a bottleneck in resource allocation</span> <span class="source" tagx="source" title="source">Biochim Biophys Acta (BBA)-General Subj</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">1810</span> <span class="fpage" tagx="fpage" title="fpage">978</span> <span class="lpage" tagx="lpage" title="lpage">988</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbagen.2011.05.014">10.1016/j.bbagen.2011.05.014</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Goelzer A, Fromion V. Resource allocation in living organisms. Biochem Soc Trans. 2017. 10.1042/BST20160436. </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Brien</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lerman</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyduke</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palsson</span> <span class="given-names" tagx="given-names" title="given-names">BO</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-scale models of metabolism and gene expression extend and refine growth phenotype prediction </span> <span class="source" tagx="source" title="source">Mol Syst Biol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">693</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/msb.2013.52">10.1038/msb.2013.52</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhuang K, Vemuri GN, Mahadevan R. Economics of membrane occupancy and respiro-fermentation. Mol Syst Biol. 2011;7. 10.1038/msb.2011.34. </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mori</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hwa</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">OC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Martino</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marinari</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Constrained allocation flux balance analysis</span> <span class="source" tagx="source" title="source">PLoS Comput Biol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">e1004913</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pcbi.1004913">10.1371/journal.pcbi.1004913</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27355325">27355325</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woldringh</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Binnerts</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mans</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Variation in Escherichia coli buoyant density measured in Percoll gradients</span> <span class="source" tagx="source" title="source">J Bacteriol</span> <span class="year" tagx="year" title="year">1981</span> <span class="volume" tagx="volume" title="volume">148</span> <span class="fpage" tagx="fpage" title="fpage">58</span> <span class="lpage" tagx="lpage" title="lpage">63</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6270065">6270065</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hui S, Silverman JM, Chen SS, Erickson DW, Basan M, Wang J, et al. Quantitative proteomic analysis reveals a simple strategy of global resource allocation in bacteria. Mol Syst Biol. 2015;11. 10.15252/msb.20145697. </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scott</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gunderson</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mateescu</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hwa</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Interdependence of cell growth and gene expression: origins and consequences</span> <span class="source" tagx="source" title="source">Science (80- )</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">330</span> <span class="fpage" tagx="fpage" title="fpage">1099</span> <span class="lpage" tagx="lpage" title="lpage">1102</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1192588">10.1126/science.1192588</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">You</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okano</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hui</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gunderson</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Coordination of bacterial proteome with metabolism by cyclic AMP signalling</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">500</span> <span class="fpage" tagx="fpage" title="fpage">301</span> <span class="lpage" tagx="lpage" title="lpage">306</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature12446">10.1038/nature12446</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23925119">23925119</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vazquez</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oltvai</span> <span class="given-names" tagx="given-names" title="given-names">ZN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Macromolecular crowding explains overflow metabolism in cells</span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">31007</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep31007">10.1038/srep31007</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27484619">27484619</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Orth JD, Palsson BØ, Fleming RMT. Reconstruction and use of microbial metabolic networks: the Core Escherichia coli metabolic model as an educational guide. EcoSal Plus. 2010;4. 10.1128/ecosalplus.10.2.1. </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schellenberger</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Que</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fleming</span> <span class="given-names" tagx="given-names" title="given-names">RMT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thiele</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orth</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feist</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantitative prediction of cellular metabolism with constraint-based models: the COBRA toolbox v2. 0 </span> <span class="source" tagx="source" title="source">Nat Protoc</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">1290</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2011.308">10.1038/nprot.2011.308</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21886097">21886097</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, et al. A genome-scale metabolic reconstruction for Escherichia coli K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information. Mol Syst Biol. 2007;3. 10.1038/msb4100155. </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sauer</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A novel metabolic cycle catalyzes glucose oxidation and anaplerosis in hungry Escherichia coli </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">278</span> <span class="fpage" tagx="fpage" title="fpage">46446</span> <span class="lpage" tagx="lpage" title="lpage">46451</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M307968200">10.1074/jbc.M307968200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12963713">12963713</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flamholz</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebermeister</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Glycolytic strategy as a tradeoff between energy yield and protein cost</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">110</span> <span class="fpage" tagx="fpage" title="fpage">10039</span> <span class="lpage" tagx="lpage" title="lpage">10044</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1215283110">10.1073/pnas.1215283110</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23630264">23630264</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Fuhrer T, Fischer E, Sauer U. Experimental identification and quantification of glucose metabolism in seven bacterial species. J Bacteriol. 2005;187. 10.1128/JB.187.5.1581-1590.2005. </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sauer</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canonaco</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heri</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perrenoud</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The soluble and membrane-bound transhydrogenases UdhA and PntAB have divergent functions in NADPH metabolism of Escherichia coli </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">279</span> <span class="fpage" tagx="fpage" title="fpage">6613</span> <span class="lpage" tagx="lpage" title="lpage">6619</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M311657200">10.1074/jbc.M311657200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14660605">14660605</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nanchen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schicker</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sauer</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nonlinear dependency of intracellular fluxes on growth rate in miniaturized continuous cultures of Escherichia coli </span> <span class="source" tagx="source" title="source">Appl Environ Microbiol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">72</span> <span class="fpage" tagx="fpage" title="fpage">1164</span> <span class="lpage" tagx="lpage" title="lpage">1172</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.72.2.1164-1172.2006">10.1128/AEM.72.2.1164-1172.2006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16461663">16461663</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Folsom</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carlson</span> <span class="given-names" tagx="given-names" title="given-names">RP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Physiological, biomass elemental composition and proteomic analyses of Escherichia coli ammonium-limited chemostat growth, and comparison with iron-and glucose-limited chemostat growth </span> <span class="source" tagx="source" title="source">Microbiology</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">161</span> <span class="fpage" tagx="fpage" title="fpage">1659</span> <span class="lpage" tagx="lpage" title="lpage">1670</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/mic.0.000118">10.1099/mic.0.000118</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26018546">26018546</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Becker SA, Feist AM, Mo ML, Hannum G, Palsson BØ, Herrgard MJ. Quantitative prediction of cellular metabolism with constraint-based models: the COBRA toolbox. Nat Protoc. 2007;2. 10.1038/nprot.2007.99. </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kayser</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hecht</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rinas</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolic flux analysis of Escherichia coli in glucose-limited continuous culture. I. Growth-rate-dependent metabolic efficiency at steady state </span> <span class="source" tagx="source" title="source">Microbiology</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">151</span> <span class="fpage" tagx="fpage" title="fpage">693</span> <span class="lpage" tagx="lpage" title="lpage">706</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/mic.0.27481-0">10.1099/mic.0.27481-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15758216">15758216</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Hoek</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Merks</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Redox balance is key to explaining full vs. partial switching to low-yield metabolism</span> <span class="source" tagx="source" title="source">BMC Syst Biol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">22</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1752-0509-6-22">10.1186/1752-0509-6-22</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22443685">22443685</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goel</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eckhardt</span> <span class="given-names" tagx="given-names" title="given-names">TH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puri</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jong</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dos Santos</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giera</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Protein costs do not explain evolution of metabolic strategies and regulation of ribosomal content: does protein investment explain an anaerobic bacterial Crabtree effect? </span> <span class="source" tagx="source" title="source">Mol Microbiol</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">97</span> <span class="fpage" tagx="fpage" title="fpage">77</span> <span class="lpage" tagx="lpage" title="lpage">92</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/mmi.13012">10.1111/mmi.13012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25828364">25828364</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fendt</span> <span class="given-names" tagx="given-names" title="given-names">S-M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buescher</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rudroff</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Picotti</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zamboni</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sauer</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Tradeoff between enzyme and metabolite efficiency maintains metabolic homeostasis upon perturbations in enzyme capacity </span> <span class="source" tagx="source" title="source">Mol Syst Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">356</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20393576">20393576</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanotelli</span> <span class="given-names" tagx="given-names" title="given-names">VRT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goya</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">JO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perlman</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Systems-level analysis of mechanisms regulating yeast metabolic flux</span> <span class="source" tagx="source" title="source">Science (80- )</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">354</span> <span class="fpage" tagx="fpage" title="fpage">aaf2786</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.aaf2786">10.1126/science.aaf2786</a> </span> </span> </li> </ul> </div> </div>  </body></html>