<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Bioeng Biotechnol</span> <span class="iso-abbrev" title="iso-abbrev">Front Bioeng Biotechnol</span> <span class="publisher-id" title="publisher-id">Front. Bioeng. Biotechnol.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Bioengineering and Biotechnology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2296-4185</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6399304">6399304</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fbioe.2019.00032">10.3389/fbioe.2019.00032</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioengineering and Biotechnology</span>: <span class="subject" title="subject">Original Research</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Anadromous Arctic Char Microbiomes: Bioprospecting in the High Arctic</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamilton</span> <span class="given-names" tagx="given-names" title="given-names">Erin F.</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/622927/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Element</span> <span class="given-names" tagx="given-names" title="given-names">Geraint</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/667017/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Coeverden de Groot</span> <span class="given-names" tagx="given-names" title="given-names">Peter</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Engel</span> <span class="given-names" tagx="given-names" title="given-names">Katja</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neufeld</span> <span class="given-names" tagx="given-names" title="given-names">Josh D.</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah</span> <span class="given-names" tagx="given-names" title="given-names">Vishal</span> </span> <a href="#aff3"> <sup>3</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/438252/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">Virginia K.</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#c001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/476125/overview" /> </span> </span> </div> <span class="citation_author_institution" id="aff1">[], <sup>1</sup> <span class="institution" title="institution">Department of Biology, Queen's University</span> <span class="addr-line" title="addr-line">Kingston, ON</span> <span class="country" tagx="country" title="country">Canada</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>2</sup> <span class="institution" title="institution">Department of Biology, University of Waterloo</span> <span class="addr-line" title="addr-line">Waterloo, ON</span> <span class="country" tagx="country" title="country">Canada</span> </span> <span class="citation_author_institution" id="aff3">[], <sup>3</sup> <span class="institution" title="institution">College of the Sciences and Mathematics, West Chester University</span> <span class="addr-line" title="addr-line">West Chester, PA</span> <span class="country" tagx="country" title="country">United States</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Steffen P. Graether, University of Guelph, Canada</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Ashok K. Dubey, Netaji Subhas Institute of Technology, India; Nadia Solovieva, University College London, United Kingdom </p> </div> <div class="corresp" title="corresp">
*Correspondence: Virginia K. Walker <span class="email" tagx="email" title="email">walkervk@queensu.ca</span> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Bioprocess Engineering, a section of the journal Frontiers in Bioengineering and Biotechnology </p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-2-2</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">32</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-10-15</span> <span class="accepted" title="accepted">accepted: 2019-2-05</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Hamilton, Element, van Coeverden de Groot, Engel, Neufeld, Shah and Walker</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Northern populations of Arctic char ( <i>Salvelinus alpinus</i>) can be anadromous, migrating annually from the ocean to freshwater lakes and rivers in order to escape sub-zero temperatures. Such seasonal behavior demands that these fish and their associated microbiomes adapt to changes in salinity, temperature, and other environmental challenges. We characterized the microbial community composition of anadromous <i>S. alpinus</i>, netted by Inuit fishermen at freshwater and seawater fishing sites in the high Arctic, both under ice and in open water. Bacterial profiles were generated by DNA extraction and high-throughput sequencing of PCR-amplified 16S ribosomal RNA genes. Results showed that microbial communities on the skin and intestine of Arctic char were statistically different when sampled from freshwater or saline water sites. This association was tested using hierarchical Ward's linkage clustering, showing eight distinct clusters in each of the skin and intestinal microbiomes, with the clusters reflecting sampling location between fresh and saline environments, confirming a salinity-linked turnover. This analysis also provided evidence for a core composition of skin and intestinal bacteria, with the phyla Proteobacteria, Firmicutes, and Cyanobacteria presenting as major phyla within the skin-associated microbiomes. The intestine-associated microbiome was characterized by unidentified genera from families Fusobacteriaceae, Comamonadaceae, Pseudomonadaceae, and Vibrionaceae. The salinity-linked turnover was further tested through ordinations that showed samples grouping based on environment for both skin- and intestine-associated microbiomes. This finding implies that core microbiomes between fresh and saline conditions could be used to assist in regulating optimal fish health in aquaculture practices. Furthermore, identified taxa from known psychrophiles and with nitrogen cycling properties suggest that there is additional potential for biotechnological applications for fish farm and waste management practices. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Arctic char</div> <div class="kwd" title="kwd">
salmonid fish</div> <div class="kwd" title="kwd">
anadromous</div> <div class="kwd" title="kwd">
microbiomes</div> <div class="kwd" title="kwd">
bioprospecting</div> <div class="kwd" title="kwd">
aquaculture</div> <div class="kwd" title="kwd">
Arctic Ocean</div> <div class="kwd" title="kwd">
aquatic biotechnology</div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Fish carry a mucous layer on their epithelial surfaces that consists of mucins, immunoglobulins, antimicrobial peptides, and commensal bacteria, which serve roles in friction reduction, waste removal, osmoregulation, as well as an early line of defense against pathogens (Esteban and Cerezuela, <a href="#B24">2015</a>). In addition, variations in mucous layer microbiome composition occur across different life stages, among different species, and across distinct geographies in amphibians and marine mammals, which are known to share certain microbial species with the surrounding water (Boutin et al., <a href="#B7">2013</a>; Apprill et al., <a href="#B2">2014</a>; Kueneman et al., <a href="#B33">2014</a>; Chiarello et al., <a href="#B12">2015</a>). Indeed, in farmed Atlantic salmon ( <i>Salmo salar</i>), the abundance of certain bacterial phyla in skin- and intestine-associated microbiomes changed depending on the water source (Lokesh and Kiron, <a href="#B37">2016</a>; Dehler et al., <a href="#B16">2017</a>). Structural changes to the microbiota, however, have not been well-described in wild fish populations. This research is noteworthy because it indicates that deliberate shifts in community structure could potentially hinder the development of dysbiosis. </p> <p>Arctic char ( <i>Salvelinus alpinus</i>), a salmonid species, is of particular interest given the increased popularity of the farmed product in temperate regions. Wild <i>S. alpinus</i> stocks from high latitude waters with an anadromous life history, such that they spend the winter in freshwater lakes to avoid freezing and subsequently migrate to the more nutrient-rich Arctic sea in the summer, could provide insight into the purported turnover in char microbiomes and be of interest to aquaculture biotechnologists. </p> <p>Although relatively unexploited commercially, Arctic char stocks in the lower Northwest Passage of the Kitikmeot region of Nunavut, Canada represent an essential subsistence fishery to indigenous Inuit communities. With recent altered sea ice patterns as a consequence of climate change, and the potential for increasing stress of pollutants associated with future industrialization, we considered it important to undertake genomic, demographic, physiological and microbial analyses on these fish populations. At present, the stocks are considered healthy due a general lack of commercial fisheries in this region, as well as the relatively long-life span of individual fish, which has been reported as up to 33 years in our samples. As part of this effort, an assessment of the mucosa-associated microbiomes of the skin and intestines of <i>S. alpinus</i> from the area in, and surrounding, King William Island Nunavut, has been undertaken. We further explored whether this increased understanding of the microbial communities could inform future biotechnological applications in the management of commercially farmed fish, in addition to other biotechnologies. </p> </div> <div class="materials and methods" title="materials and methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="studyarea" title="sec"> <div class="title" tagx="title" title="title">
Study Area</div> <p>Fishing was done within the Kitikmeot region of Nunavut, Canada in the Western Arctic, at seven distinct sites within 200 km of King William Island, located along the lower Northwest Passage ( <a href="#F1">Figure 1</a>; <a href="#T1">Table 1</a>). Fishing sites were chosen based on Traditional Ecological Knowledge shared by local Inuit elders and in association with the Hunters and Trappers Association of Gjoa Haven, NU, as part of a large-scale fisheries project (Towards; <a href="http://www.arcticfishery.ca">www.arcticfishery.ca</a>). At each fishing site, specific conductance of surface water was obtained with a conductivity meter (Traceable Fisherbrand, Fisher Scientific) to record conductivity and determine our site designations as either freshwater or saline sites ( <a href="#T1">Table 1</a>). The southern region of the study area is unique due to large freshwater influence from major river systems, including Murchison River, Legendary River, Back River, and Hayes River. Therefore, the major sea-water bodies within the region, namely Rasmussen Basin and Chantrey Inlet, are characterized by brackish salinities as defined by Watling ( <a href="#B61">2007</a>), with conductivities between 1,500 and 15,000 μS cm <sup>−1</sup>, diverging from sea salinities more characteristic of the Pacific and Atlantic Oceans (Carmack, <a href="#B11">2007</a>). Fishing sites within these locations are referred to as "saline" sites. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>A map of the lower Northwest Passage in Nunavut, Canada and the location of seven distinct fishing sites initially chosen based on Inuit Traditional Ecological Knowledge. The sites fished include five freshwater sites (Port Perry, Swan Lake, Koka Lake, Murchison River, and Back River); and two saltwater sites (Back House Point and Legendary River estuary). Inset A outlines Nunavut, Canada, in red, while inset B showcases the lower Northwest Passage, in red. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Location and GPS coordinates for each fishing site, followed by designated water source categories and specific conductance measurements shown as conductivity that were determined by a conductivity meter on-site. </p>  <table> <thead> <tr> <th> <b>Location and GPS coordinates</b> </th> <th> <b>Water source</b> </th> <th> <b>Conductivity (μS cm <sup> <b>−1</b> </sup>) </b> </th> </tr> </thead> <tbody> <tr> <td>Port Perry (N69°33′28.764″, W97°26′13.884″)</td> <td>Fresh</td> <td>286</td> </tr> <tr> <td>Swan Lake (N68°40′13.62, W95°56′57.408″)</td> <td>Fresh</td> <td>880</td> </tr> <tr> <td>Koka Lake (N68°32′5.1″, W96°12′45.899″)</td> <td>Fresh</td> <td>670</td> </tr> <tr> <td>Back House Point (N67°27′27.2″, W95°21′38.6″)</td> <td>Saline</td> <td>8,240</td> </tr> <tr> <td>Legendary River (N67°31′17.8″, W96°26′21.8″)</td> <td>Saline</td> <td>3,450</td> </tr> <tr> <td>Murchison River (N68°34′1.2″, W 93°22′37.452″)</td> <td>Fresh</td> <td>225</td> </tr> <tr> <td>Back River (N66°57′30.70″, W95°18′5.20″)</td> <td>Fresh</td> <td>19</td> </tr> </tbody> </table> </div> </div> <div class="fishcollection" title="sec"> <div class="title" tagx="title" title="title">
Fish Collection</div> <p>Commercial (140 mm mesh) and multi-mesh subsistence (5 or 8 panels of 38–140 mm) fishing nets were set during December-June (freshwater lakes under thick ice) and August-September (at open water river estuaries during sea-ice formation and char migration) at distinct geographic sites over a period of 3 years. After setting for several hours, nets were retrieved, and fish were pulled from the nets with nitrile gloves. Using either sterile cotton swabs or an ethanol-sterilized scalpel, the surface mucosal layer of each fish was sampled once per fish with the swabs, or skin scrapings, taken from above the lateral line. The fish were then photographed, weighed, and measured, followed by dissection in an on-site mobile lab. Full intestines were removed using sterile technique, and both skin and intestinal samples were placed in sterile sample tubes or bags, respectively, and frozen at −20°C in a freezer on-site. Samples were shipped frozen and subsequently stored at −20°C until further processing. In addition to the skin and intestine samples, water samples were also taken at each fishing site, in which up to 2 L of water was filtered through sterile 0.22 μm PALL filters in triplicate. The filters were then frozen at −20°C and were subsequently transported and stored at −80°C until further processing. </p> <p>Fish were sampled in accordance with issued licenses to fish for scientific purposes in the waters of the Northwest Territories, Yukon north slope, and Nunavut (in accordance with section 52 of the general fishery regulations of the fisheries act, Fisheries and Oceans, Canada) along with an associated animal care permit issued by the Fresh Water Institute Animal Care Committee of the Department of Fisheries and Oceans (current permit numbers S-18/19-1045-NU and FWI-ACC AUP-2018-63). </p> </div> <div class="sampleprocessinganddnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Sample Processing and DNA Extraction</div> <p>After initial optimization experimentation to determine the selection of appropriate DNA extraction kits for each sample type, DNA from the skin samples was extracted using a NucleoSpin Soil Extraction Kit (Machery-Nagel, Bethlehem, PA), with modifications following the procedures of Kueneman et al. ( <a href="#B33">2014</a>) in which samples were incubated at 65°C for 10 min before mechanical lysis. In order to maximize yield, the initial lysis step was conducted twice, and double-distilled sterile water was left on the filter for 5 min prior to elution. Extracted DNA concentrations were assessed using an Invitrogen Qubit 4 Fluorometer and a QuantiFluor ONE dsDNA system (Promega Corporation, Madison, WI, USA), followed by qualitative analysis with agarose gel electrophoresis and a NanoDrop One Microvolume spectrophotometer (Thermo Scientific) set to A <sub>260/280</sub> absorbance. High concentration samples were diluted to 50 ng μL <sup>−1</sup> prior to PCR amplification, in which starting material for PCR templates ranged from 1 to 50 ng μL <sup>−1</sup>. These skin-associated microbiome samples underwent a pre-amplification PCR step using primers 8F and 1406R (Lane, <a href="#B34">1991</a>; Coolen et al., <a href="#B14">2005</a>) to amplify the variable V1–V9 region of the bacterial 16S ribosomal RNA (rRNA) gene. The PCR mix for each reaction (50 μL total volume) contained 1X ThermoFisher DreamTaq Buffer (with 2 mM MgCl <sub>2</sub>), 0.4 μM forward and reverse primers, 200 μM dNTPs, 400 ng BSA, 2.5 U ThermoFisher DreamTaq DNA Polymerase, and 2 μL of template. The PCR was performed as follows: 95°C for 5 min, 25 cycles of 95°C for 1 min, 52°C for 1 min, 72°C for 1 min, with a final extension of 72°C for 7 min. </p> <p>Intestinal samples were partially thawed to excise three slices within the distal intestine (~2 cm from the vent), comprising a total of 5–100 mg epithelial tissue, avoiding feces and connective tissue. The slices were pooled and DNA was extracted using MOBIO UltraClean Tissue and Cells DNA Isolation Kit (QIAGEN Inc., Toronto, ON) following the manufacturer's instructions, except that elution was achieved with double-distilled sterile water rather than EDTA. The DNA concentrations for the intestine-associated microbiome samples were estimated as described for the skin-associated microbiome preparations, but adjusted to 30 ng μL <sup>−1</sup>. Pre-amplified PCR products were also prepared as described above. </p> <p>In addition, the V4 region of the 16S rRNA gene was amplified on an Illumina sequence platform from DNA isolated from the triplicate water filters using primers 515F and 806R (Caporaso et al., <a href="#B10">2011</a>). </p> </div> <div class="16srrnagenesequencingofmucosalmicrobiomes" title="sec"> <div class="title" tagx="title" title="title">
16S rRNA Gene Sequencing of Mucosal Microbiomes</div> <p>The V4-V5 region of the 16S rRNA gene was amplified from each of the skin and intestinal amplification products using primers 515F-Y (Parada et al., <a href="#B45">2016</a>) and 926R (Quince et al., <a href="#B46">2011</a>). Each primer contained a 6-base index sequence for sample multiplexing as well as Illumina flow cell binding and sequencing sites (Bartram et al., <a href="#B5">2011</a>). The PCR mix (25 μL total volume) contained 1X ThermoPol Buffer, 0.2 μM forward and reverse primers, 200 μM dNTPs, 15 μg BSA, 0.625 U Hot Start <i>Taq</i> DNA polymerase, and 1 μL of template. The PCR was performed as follows: 95°C for 3 min, 35 cycles of 95°C for 30 s, 50°C for 30 s, 68°C for 1 min, and a final extension of 68°C for 7 min. Each amplification reaction was done in triplicate. Equal quantities of each amplicon were pooled. Samples that did not yield a PCR product were not included. No-template controls were added to the Illumina sequencing pool (5 μL), even when amplicons were not detected. Pooled 16S rRNA gene amplicons were subsequently excised from an agarose gel and purified using the Wizard SV Gel and PCR Clean-Up System (Promega, WI, USA). A 5 pM library containing 15% PhiX Control v3 (Illumina Canada Inc, NB, Canada) was sequenced on a MiSeq instrument (Illumina Inc, CA, USA) using a 2 × 250 cycle MiSeq Reagent Kit v2 (Illumina Canada Inc., BC, Canada). </p> </div> <div class="sequencedataanalysis,otutables,andstatistics" title="sec"> <div class="title" tagx="title" title="title">
Sequence Data Analysis, OTU Tables, and Statistics</div> <p>Sequence reads were demultiplexed using Illumina MiSeq Reporter software version 2.5.0.5. Reads were assembled using the paired-end assembler for Illumina sequences (PANDAseq version 2.8, Masella et al., <a href="#B39">2012</a>) with a quality threshold of 0.9, an 8 nucleotide minimum overlap, and 32 nucleotide minimum assembled read length. Assembled reads were analyzed using Quantitative Insights Into Microbial Ecology (QIIME version 1.9.0, Caporaso et al., <a href="#B9">2010</a>). Sequences were clustered into operational taxonomic units (OTUs) using UPARSE algorithm USEARCH version 7.0.1090 (Edgar, <a href="#B21">2013</a>) at 97% identity and aligned with the Python Nearest Alignment Space Termination tool (PyNAST version 1.2.2, Caporaso et al., <a href="#B9">2010</a>). All representative sequences were classified using the Ribosomal Database Project (RDP version 2.2, Wang et al., <a href="#B59">2007</a>) with a stringent confidence threshold (0.8) and the Greengenes database (McDonald et al., <a href="#B40">2012</a>) was used to assign taxonomy. Chimeric sequences were filtered with UCHIME (Edgar et al., <a href="#B22">2011</a>). Before performing statistical analysis, OTUs observed in no-template PCR controls for a sample type were filtered from all samples of that type. OTUs with three or fewer reads within negative PCR controls were retained within the samples, since their low representation in the negative controls rendered them viable representatives in the samples. The OTUs were then rarefied to ~2,000 reads for skin and intestinal microbiome samples. Alpha and beta diversity for samples were analyzed based on rarified OTU tables generated using QIIME with principal coordinate analysis (PCoA). Additional visualizations were done using EMPeror (Vázquez-Baeza et al., <a href="#B58">2013</a>). Primer 7 software version 7.0.13 was used for analysis of similarity (ANOSIM) and similarity percentages (SIMPER) analyses (Clarke et al., <a href="#B13">2014</a>). Clustering analysis was performed using Statistica 13.0 Academic software. </p> <p>The hypothesis that skin microbial community composition was linked to salinity was tested by performing hierarchical Ward's linkage clustering (Dillner et al., <a href="#B19">2005</a>). Tree cluster analysis was carried out using Ward's method as the amalgamation rule and the distance measured as Euclidean units (Dillner et al., <a href="#B19">2005</a>). Using K-means clustering, the samples were divided into K clusters by selecting the number of iterations as 10, and the initial cluster centers chosen to maximize initial between-cluster distances. The output of this step is the list of fish samples that are present in each cluster. The sequence data is publicly available in the European Nucleotide Archive within the European Bioinformatics Institute, under the study: Characterization and Analysis of Skin- and Intestine-associated Microbiomes in the Lower Northwest Passage (Nunavut, Canada) with the following accession number, <a href="PRJEB29173">PRJEB29173</a>. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="comparisonsofskinandintestinalmicrobiotawithinindividualfish" title="sec"> <div class="title" tagx="title" title="title">
Comparisons of Skin and Intestinal Microbiota Within Individual Fish</div> <p>Based on fish sampled at seven distinct sites on or within 200 km of King William Island, located along the lower Northwest Passage ( <a href="#F1">Figure 1</a>; <a href="#T1">Table 1</a>), microbiome sequences were successfully obtained for 118 skin mucosal samples and 202 intestinal samples. Grouping the skin microbiome consortia by environment showed that the freshwater samples contained significantly more Shannon diversity overall than those from saline water ( <i>R</i> <sup>2</sup> = 0.11, <i>F</i> = 16.0, <i>p</i> &amp;lt; 0.05), whereas log <sub>10</sub> Chao1 OTU richness was not significantly different between the two environment types ( <i>R</i> <sup>2</sup> = 0.00, <i>F</i> = 0.1, <i>p</i> &amp;gt; 0.05). In contrast, when intestinal microbiomes were grouped by environment, the saline water samples were more diverse overall than freshwater samples when log <sub>10</sub> Chao1 OTU richness was considered, though this difference was small ( <i>R</i> <sup>2</sup> = 0.03, <i>F</i> = 4.7, <i>p</i> &amp;lt; 0.05). A significant difference was not observed in the intestinal microbiomes between the two environment types when Shannon diversity was considered ( <i>R</i> <sup>2</sup> = 0.00, <i>F</i> = 0.1, <i>p</i> &amp;gt; 0.05). Of the 320 skin and intestine samples combined, 60 individual Arctic char had both sets of microbiome data, allowing a comparison of skin and intestinal flora from the same fish ( <a href="#F2">Figure 2</a>). When comparing across these 60 fish, skin-associated microbiomes were significantly more diverse than intestine-associated microbiomes when both log <sub>10</sub> Chao1 OTU richness ( <i>R</i> <sup>2</sup> = 0.60, <i>F</i> = 180.0, <i>p</i> &amp;lt; 0.05) and Shannon diversity ( <i>R</i> <sup>2</sup> = 0.52, <i>F</i> = 127.4, <i>p</i> &amp;lt; 0.05) were considered. In addition, though the skin and intestine communities were distinct, the skin- and intestine-associated microbiomes among individual fish appeared to be more similar at freshwater sites ( <a href="#F2">Figures 2A,B</a>) compared to saline water sites ( <a href="#F2">Figures 2C,D</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Phyla comparison of skin and intestinal complementary microbiomes for individual fish, shown for samples collected at <b>(A)</b> freshwater sites Port Perry, Swan Lake, and Koka Lake for skin-associated microbiomes, <b>(B)</b> freshwater sites Port Perry, Swan Lake, and Koka Lake for intestine-associated microbiomes, <b>(C)</b> saltwater sites, Back House Point and Legendary River for skin-associated microbiomes, and <b>(D)</b> saltwater sites, Back House Point and Legendary River for intestine-associated microbiomes. Phyla with relative abundances (RA) ≥ 0.5% are shown. Phyla present at RA &amp;lt; 0.5% were pooled together. </p>   </div> <p>The relative abundance of rarified OTUs identified to the genus level, if known, for each of the total 320 skin and intestinal microbiome samples is presented in supplemental tables ( <a href="#SM1">Tables S1</a>, <a href="#SM2">S2</a>, respectively). </p> </div> <div class="skinandintestinalmicrobiotacompositionslinkedtosalinity" title="sec"> <div class="title" tagx="title" title="title">
Skin and Intestinal Microbiota Compositions Linked to Salinity</div> <p>At freshwater sites, most taxa from skin- and intestine-associated microbiome sequence data were affiliated with Proteobacteria, but many also classified to Actinobacteria, Cyanobacteria, and Firmicutes ( <a href="#F2">Figures 2A,B</a>; <a href="#SM1">Tables S1</a>, <a href="#SM2">S2</a>). Although fish skin from saline environments was also colonized by Proteobacteria and Actinobacteria ( <a href="#F2">Figure 2C</a>), other phyla, such as Acidobacteria and Bacteroidetes, were also prominent. Intestines derived from fish sampled in saline waters were similarly characterized by Proteobacteria, Actinobacteria, and Bacteroidetes, but also by Firmicutes, Spirochaetes and Tenericutes, with considerable variation across individual fish ( <a href="#F2">Figure 2D</a>). Given these observations, depending on whether fish were sampled from fresh or saline waters, skin or intestinal bacterial communities appeared to change. For example, relative abundance based on rarified data of the psychrophile <i>Photobacterium</i> increased 200-fold in relative abundance in skin-associated microbiomes from saline sites, compared to freshwater sites ( <a href="#T2">Table 2A</a>). Similarly, relative abundance for <i>Deinoccoccus</i>, known for its ability to resist a variety of environmental stresses, increased 150-fold between fresh- and saline-caught skin microbiome fish samples. Taxa in the intestinal bacterial communities were also shown to change between fresh and saline-caught fish, with similar increases, but to a lesser magnitude including those belonging to the order Vibrionales (18-fold), the genus <i>Photobacterium</i> (14-fold), and an unknown genus in the class Mollicutes (14-fold; <a href="#T2">Table 2B</a>). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p> <b>(A)</b> SIMPER analysis (Primer 7) output showing relative abundances and impact ratio of environmental change between freshwater and saline locations across 118 skin-associated microbiome samples <b>(B)</b> SIMPER analysis (Primer 7) output showing relative abundances and impact ratio of environmental change between freshwater and saline locations across 202 intestine-associated samples. </p>  <table> <thead> <tr> <th> <b>Taxonomy</b> </th> <th> <b>Relative abundance</b> </th> <th> <b>Impact ratio (Saline: Fresh)</b> </th> </tr> <tr> <th /> <th> <b>Saline</b> </th> <th> <b>Fresh</b> </th> <th /> </tr> </thead> <tbody> <tr> <td> <b>(A)</b> </td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Vibrionaceae g: <i>Photobacterium</i> </td> <td>4.22</td> <td>0.02</td> <td>211</td> </tr> <tr> <td>p: Deinococcus c: Deinococci o: Deinococcales f: Deinococcaceae g: <i>Deinococcus</i> </td> <td>7.72</td> <td>0.05</td> <td>154</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Rhodospirillales f: Acetobacteraceae g: Unknown </td> <td>8.58</td> <td>1.52</td> <td>5.64</td> </tr> <tr> <td>p: Firmicutes c: Clostridia o: Clostridiales f: Clostridiaceae g: <i>Clostridium</i> </td> <td>3.84</td> <td>2.50</td> <td>1.54</td> </tr> <tr> <td>p: Proteobacteria c: Betaproteobacteria o: Burkholderiales f: Comamonadaceae g: Other</td> <td>2.97</td> <td>8.01</td> <td>0.37</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Sphingomonadales f: Sphingomonadaceae g: <i>Sphingomonas</i> </td> <td>1.69</td> <td>5.00</td> <td>0.34</td> </tr> <tr> <td>p: Bacteroidetes c: Saprospirae o: Saprospirales f: Chitinophagaceae g: Unknown</td> <td>1.23</td> <td>4.90</td> <td>0.25</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Rhodospirillales f: Rhodospirillaceae g: Unknown </td> <td>0.90</td> <td>5.69</td> <td>0.16</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Caulobacterales f: Caulobacteraceae g: Other </td> <td>0.69</td> <td>4.39</td> <td>0.16</td> </tr> <tr> <td>p: Proteobacteria c: Betaproteobacteria o: Other f: Other g: Other</td> <td>0.73</td> <td>5.27</td> <td>0.14</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Sphingomonadales f: Sphingomonadaceae g: <i>Novosphingobium</i> </td> <td>0.75</td> <td>6.42</td> <td>0.12</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Xanthomonadales f: Sinobacteraceae g: Unknown </td> <td>0.40</td> <td>6.76</td> <td>0.06</td> </tr> <tr> <td>p: Proteobacteria c: Betaproteobacteria o: Burkholderiales f: Oxalobacteraceae g: <i>Cupriavidus</i> </td> <td>0.12</td> <td>4.90</td> <td>0.02</td> </tr> <tr> <td>p: Cyanobacteria c: Nostocophycideae o: Stigonematales f: Rivulariaceae g: <i>Rivularia</i> </td> <td>4.80</td> <td>0.00</td> <td>0.00</td> </tr> <tr> <td>p: Cyanobacteria c: Oscillatoriophycideae o: Oscillatoriales f: Phormidiaceae g: <i>Phormidium</i> </td> <td>7.27</td> <td>0.00</td> <td>0.00</td> </tr> <tr> <td> <b>(B)</b> </td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Other g: Other</td> <td>3.10</td> <td>0.17</td> <td>18.23</td> </tr> <tr> <td>p: Tenericutes c: Mollicutes o: Unknown f: Unknown g: Unknown</td> <td>7.32</td> <td>0.52</td> <td>14.08</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Vibrionaceae g: <i>Photobacterium</i> </td> <td>20.8</td> <td>1.48</td> <td>14.03</td> </tr> <tr> <td>p: Spirochaetes c: Brevinematae o: Brevinematales f: Brevinemataceae g: Unknown</td> <td>4.54</td> <td>0.54</td> <td>8.41</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Vibrionaceae g: Other</td> <td>6.06</td> <td>1.09</td> <td>5.56</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Vibrionaceae g: <i>Aliivibrio</i> </td> <td>5.08</td> <td>1.22</td> <td>4.16</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Sphingomonadales f: Sphingomonadaceae g: <i>Sphingomonas</i> </td> <td>5.91</td> <td>4.06</td> <td>1.46</td> </tr> <tr> <td>p: Tenericutes c: Mollicutes o: Mycoplasmatales f: Mycoplasmataceae g: <i>Mycoplasma</i> </td> <td>4.17</td> <td>4.42</td> <td>0.94</td> </tr> <tr> <td>p: Fusobacteria c: Fusobacteriia o: Fusobacteriales f: Fusobacteriaceae g: <i>u114</i> </td> <td>2.77</td> <td>4.12</td> <td>0.67</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Pseudomonadales f: Pseudomonadaceae g: <i>Pseudomonas</i> </td> <td>1.65</td> <td>3.50</td> <td>0.47</td> </tr> <tr> <td>p: Proteobacteria c: Betaproteobacteria o: Burkholderiales f: Comamonadaceae g: Other</td> <td>2.50</td> <td>5.96</td> <td>0.42</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Pseudomonadales f: Pseudomonadaceae g: Other </td> <td>2.23</td> <td>7.91</td> <td>0.28</td> </tr> <tr> <td>Unclassified Taxa</td> <td>1.23</td> <td>7.65</td> <td>0.16</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Caulobacterales f: Caulobacteraceae g: <i>Phenylobacterium</i> </td> <td>0.40</td> <td>5.71</td> <td>0.07</td> </tr> <tr> <td>p: Proteobacteria c: Betaproteobacteria o: Neisseriales f: Neisseriaceae g: <i>Deefgea</i> </td> <td>0.05</td> <td>2.86</td> <td>0.02</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>Taxonomy from phylum (p) to clade (c) to order (o) to family (f) to genus (g) is shown</i>. </p> </div> </div> <p>Overall, PCoA combined with ANOSIM showed a statistically significant separation between samples obtained from freshwater and saline sites for both skin microbiomes ( <i>p</i> &amp;lt; 0.001; <a href="#F3">Figure 3A</a>), and intestinal microbiomes ( <i>p</i> &amp;lt; 0.001; <a href="#F3">Figure 3B</a>). This indicates that salinity is a primary factor in defining these microbial communities. For the 60 skin and intestinal microbiome samples used for within-fish comparisons where possible, ANOSIM showed that the communities obtained from the two sample types were significantly different ( <i>p</i> &amp;lt; 0.001; <a href="#F4">Figure 4</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Principal coordinate analysis (PCoA) showing grouping of <b>(A)</b> individual skin microbiomes at fresh ( <i>n</i> = 55), and saline ( <i>n</i> = 63) sites and <b>(B)</b> individual intestinal microbiomes at fresh ( <i>n</i> = 144) and saline ( <i>n</i> = 58) sites. PCoA ordinations are based on the Bray-Curtis dissimilarity metric. Freshwater sites include lake and river sites sampled in winter and spring while saline sites refer to sea shoreline locations sampled in autumn during the annual char run. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Principal coordinate analysis (PCoA) showing clustering of individual skin and intestinal microbiome communities based on Bray-Curtis dissimilarity. For every skin-associated community represented, a counterpart intestine-associated community from the same <i>S. alpinus</i> individual is shown. Legend refers to the sample type and environment of where an <i>S. alpinus</i> individual was caught and includes skin microbiomes from freshwater sites ( <i>n</i> = 30), and from saline sites ( <i>n</i> = 30), as well as intestinal microbiomes from freshwater sites ( <i>n</i> = 30) and from saline sites ( <i>n</i> = 30). Fresh refers to lake and river sites sampled in winter and spring while saline refers to sea shoreline locations sampled in autumn during the annual char run. </p>   </div> <p>In addition, preliminary results show that the community structure of identified bacteria from water samples at sites corresponding to where fish were caught (excluding Back River) differs from that of the fish, suggesting that the distribution of microbiota identified on the fish are influenced by physiological processes inherent to the fish ( <a href="#SM7">Figure S1</a>). However, the phyla Proteobacteria, Bacteroidetes, and Actinobacteria are represented in both the water microbiomes and in both skin- and intestine-associated microbiota. Across water samples, no significant difference was observed between freshwater and saline sites overall for either Chao1 OTU richness ( <i>R</i> <sup>2</sup> = 0.01, <i>F</i> = 0.2, <i>p</i> &amp;gt; 0.05), or Shannon diversity ( <i>R</i> <sup>2</sup> = 0.07, <i>F</i> = 1.6, <i>p</i> &amp;gt; 0.05). The most similarity was observed between samples from similar geographic regions. For example, Port Perry, Swan Lake, and Koka Lake were from freshwater sites on King William Island whereas the saline sites Back House Point, Legendary River, and the freshwater site Murchison River were from Chantrey Inlet. </p> </div> <div class="characterizationoftheskinmicrobiota" title="sec"> <div class="title" tagx="title" title="title">
Characterization of the Skin Microbiota</div> <p>When the 118 skin-associated microbiome samples were analyzed using hierarchical Ward's linkage clustering (Dillner et al., <a href="#B19">2005</a>), eight distinct clusters were apparent in the amalgamation schedule graph ( <a href="#F5">Figure 5A</a>), with the decreasing linkage distance after the seventh fusion step indicating a minimal difference for any newly formed clusters. Euclidean distances between the centers of clusters confirmed the distinct nature of each cluster, with all clusters largely apart from each other ( <a href="#SM3">Table S3</a>). Subsequent K-means clustering was then carried out to identify members of each of the clusters ( <a href="#T3">Table 3A</a>). The largest cluster (#6) contained 98% of the freshwater (54/55) and 52% of the saline (32/62) samples, supporting the distinct grouping of saline and freshwater samples observed in the PCoA ordinations ( <a href="#F3">Figure 3</a>). Because many of the fish were netted in autumn, during the annual char migration, it is probable that some saline-associated microbiota could remain in the same cluster as the microbiota from freshwater fish. The ANOVA results show that of the 899 taxa used in the clustering analysis, 56 have a <i>p</i>-value below 0.05. These results also indicated that Proteobacteria, Cyanobacteria, and Firmicutes contributed most to the clustering analysis ( <a href="#SM4">Table S4</a>). As an approach to identify a core microbiome for Arctic char skin, bacteria present across the eight clusters were identified ( <a href="#T4">Table 4A</a>). The fact that none of the nine taxa, except for <i>Clostridium</i>, matched to any known genus likely emphasizes the paucity of research for this wild species. For example, an unknown genus from the Acetobacteraceae family represents over 5% of the total microbiota. In contrast to the clustering based on a saline environment, no correlation was obtained between the presence of any of the microbial genus species and the sex, age, or size of the Arctic char samples ( <i>p</i> &amp;gt; 0.05; data not shown). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Amalgamation schedule used to identify the number of major clusters (K) for <b>(A)</b> OTUs from skin-associated microbiomes and <b>(B)</b> OTUs from intestine-associated microbiomes. Linkage distance is shown in addition to the visualization of steps. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p> <b>(A)</b> Sample counts of each cluster, showing number of samples either caught in a saline or freshwater environment, as obtained using K mean clustering analysis using the skin microbiome data at genus level and <b>(B)</b> Sample counts of each cluster, showing number of samples either caught in a saline or fresh environment, as obtained using K mean clustering analysis using the intestinal microbiome data at genus level. </p>  <table> <thead> <tr> <th> <b>Cluster</b> </th> <th> <b>Saline</b> </th> <th> <b>Fresh</b> </th> </tr> </thead> <tbody> <tr> <td> <b>(A)</b> </td> </tr> <tr> <td>1</td> <td>9</td> <td>0</td> </tr> <tr> <td>2</td> <td>12</td> <td>0</td> </tr> <tr> <td>3</td> <td>3</td> <td>0</td> </tr> <tr> <td>4</td> <td>1</td> <td>1</td> </tr> <tr> <td>5</td> <td>1</td> <td>0</td> </tr> <tr> <td>6</td> <td>31</td> <td>54</td> </tr> <tr> <td>7</td> <td>1</td> <td>0</td> </tr> <tr> <td>8</td> <td>3</td> <td>0</td> </tr> <tr> <td> <b>(B)</b> </td> </tr> <tr> <td>1</td> <td>2</td> <td>7</td> </tr> <tr> <td>2</td> <td>0</td> <td>8</td> </tr> <tr> <td>3</td> <td>1</td> <td>11</td> </tr> <tr> <td>4</td> <td>21</td> <td>82</td> </tr> <tr> <td>5</td> <td>0</td> <td>14</td> </tr> <tr> <td>6</td> <td>19</td> <td>2</td> </tr> <tr> <td>7</td> <td>1</td> <td>19</td> </tr> <tr> <td>8</td> <td>8</td> <td>1</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p> <b>(A)</b> Microorganisms, classified to genus where possible, present in eight clusters along with the percentage within the skin microbiome of Arctic char and <b>(B)</b> Microorganisms, classified to genus where possible, present in eight clusters along with the percentage within the intestinal microbiome of Arctic char. </p>  <table> <thead> <tr> <th> <b>Taxonomy</b> </th> <th> <b>Cluster number and percent abundance</b> </th> </tr> <tr> <th /> <th> <b>1</b> </th> <th> <b>2</b> </th> <th> <b>3</b> </th> <th> <b>4</b> </th> <th> <b>5</b> </th> <th> <b>6</b> </th> <th> <b>7</b> </th> <th> <b>8</b> </th> </tr> </thead> <tbody> <tr> <td> <b>(A)</b> </td> </tr> <tr> <td>Other bacteria</td> <td>0.96</td> <td>1.61</td> <td>2.04</td> <td>0.28</td> <td>0.52</td> <td>1.93</td> <td>0.81</td> <td>0.74</td> </tr> <tr> <td>p: Actinobacteria c: Actinobacteria o: Actinomycetales f: Other g: Other</td> <td>0.30</td> <td>0.81</td> <td>0.85</td> <td>0.05</td> <td>0.19</td> <td>0.80</td> <td>0.05</td> <td>0.30</td> </tr> <tr> <td>p: Proteobacteria c: Other o: Other f: Other g: Other</td> <td>0.07</td> <td>0.20</td> <td>0.03</td> <td>0.17</td> <td>0.09</td> <td>0.55</td> <td>0.05</td> <td>0.03</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Rhodospirillales f: Acetobacteraceae g: Unknown </td> <td>0.45</td> <td>4.14</td> <td>4.73</td> <td>0.05</td> <td>1.52</td> <td>5.15</td> <td>0.14</td> <td>0.03</td> </tr> <tr> <td>p: Cyanobacteria c: Other o: Other f: Other g: Other</td> <td>0.41</td> <td>3.99</td> <td>0.19</td> <td>0.00</td> <td>0.19</td> <td>0.32</td> <td>0.05</td> <td>0.06</td> </tr> <tr> <td>p: Firmicutes c: Bacilli o: Lactobacillales f: Other g: Other</td> <td>2.67</td> <td>0.20</td> <td>0.02</td> <td>0.02</td> <td>0.00</td> <td>0.17</td> <td>3.94</td> <td>57.1</td> </tr> <tr> <td>p: Firmicutes c: Clostridia o: Clostridiales f: Clostridiaceae g: <i>Clostridium</i> </td> <td>2.42</td> <td>0.22</td> <td>72.0</td> <td>0.00</td> <td>0.14</td> <td>0.78</td> <td>0.09</td> <td>0.27</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Rhodobacterales f: Rhodobacteraceae g: Other </td> <td>0.11</td> <td>0.30</td> <td>0.14</td> <td>0.02</td> <td>70.2</td> <td>0.94</td> <td>0.00</td> <td>0.09</td> </tr> <tr> <td>p: Proteobacteria c: Betaproteobacteria o: Burkholderiales f: Comamonadaceae g: Other</td> <td>0.13</td> <td>0.21</td> <td>0.08</td> <td>87.42</td> <td>0.47</td> <td>2.73</td> <td>0.00</td> <td>0.03</td> </tr> <tr> <td> <b>(B)</b> </td> </tr> <tr> <td>Unclassified</td> <td>2.16</td> <td>2.17</td> <td>1.48</td> <td>8.20</td> <td>6.26</td> <td>0.21</td> <td>1.28</td> <td>0.15</td> </tr> <tr> <td>Other Bacteria</td> <td>0.39</td> <td>0.38</td> <td>3.62</td> <td>1.37</td> <td>1.03</td> <td>0.38</td> <td>0.16</td> <td>1.09</td> </tr> <tr> <td>p: Actinobacteria c: Actinobacteria o: Actinomycetales f: Other g: Other</td> <td>3.59</td> <td>0.01</td> <td>0.02</td> <td>0.14</td> <td>0.01</td> <td>0.03</td> <td>0.05</td> <td>0.16</td> </tr> <tr> <td>p: Firmicutes c: Bacilli o: Lactobacillales f: Streptococcaceae g: <i>Streptococcus</i> </td> <td>0.03</td> <td>0.28</td> <td>1.02</td> <td>2.35</td> <td>1.81</td> <td>0.08</td> <td>0.34</td> <td>0.44</td> </tr> <tr> <td>p: Fusobacteria c: Fusobacteriia o: Fusobacteriales f: Fusobacteriaceae g: <i>u114</i> </td> <td>0.03</td> <td>0.04</td> <td>84.2</td> <td>0.79</td> <td>0.02</td> <td>1.64</td> <td>0.01</td> <td>2.70</td> </tr> <tr> <td>p: Proteobacteria c: Alphaproteobacteria o: Sphingomonadales f: Sphingomonadaceae g: <i>Sphingomonas</i> </td> <td>9.56</td> <td>0.02</td> <td>0.17</td> <td>10.5</td> <td>0.23</td> <td>0.62</td> <td>0.06</td> <td>0.46</td> </tr> <tr> <td>p: Proteobacteria c: Betaproteobacteria o: Burkholderiales f: Comamonadaceae g: Other</td> <td>0.07</td> <td>0.11</td> <td>0.01</td> <td>2.48</td> <td>52.5</td> <td>0.61</td> <td>1.01</td> <td>0.17</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Alteromonadales f: Shewanellaceae g: <i>Shewanella</i> </td> <td>2.74</td> <td>0.57</td> <td>0.03</td> <td>1.29</td> <td>1.50</td> <td>0.57</td> <td>0.04</td> <td>0.03</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Pseudomonadales f: Pseudomonadaceae g: Other </td> <td>0.08</td> <td>8.23</td> <td>2.46</td> <td>1.27</td> <td>1.32</td> <td>0.45</td> <td>85.0</td> <td>0.20</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Pseudomonadales f: Pseudomonadaceae g: <i>Pseudomonas</i> </td> <td>0.01</td> <td>0.70</td> <td>0.14</td> <td>2.66</td> <td>1.47</td> <td>0.31</td> <td>3.80</td> <td>0.15</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Vibrionaceae g: Other</td> <td>0.11</td> <td>0.01</td> <td>0.17</td> <td>2.43</td> <td>0.47</td> <td>6.11</td> <td>0.01</td> <td>0.09</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Vibrionaceae g: <i>Aliivibrio</i> </td> <td>0.20</td> <td>0.01</td> <td>0.13</td> <td>3.20</td> <td>0.80</td> <td>3.47</td> <td>0.01</td> <td>0.04</td> </tr> <tr> <td>p: Proteobacteria c: Gammaproteobacteria o: Vibrionales f: Vibrionaceae g: <i>Photobacterium</i> </td> <td>0.24</td> <td>0.27</td> <td>0.68</td> <td>1.78</td> <td>0.02</td> <td>71.0</td> <td>0.38</td> <td>4.49</td> </tr> <tr> <td>p: Tenericutes c: Mollicutes o: Mycoplasmatales f: Mycoplasmataceae g: <i>Mycoplasma</i> </td> <td>73.8</td> <td>0.20</td> <td>1.37</td> <td>2.11</td> <td>0.36</td> <td>1.19</td> <td>0.42</td> <td>2.14</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>Taxonomy from phylum (p) to clade (c) to order (o) to family (f) to genus (g) is shown</i>. </p> </div> </div> </div> <div class="characterizationoftheintestinalmicrobiota" title="sec"> <div class="title" tagx="title" title="title">
Characterization of the Intestinal Microbiota</div> <p>The 202 intestinal microbiome samples were tested by hierarchical Ward's linkage clustering (Dillner et al., <a href="#B19">2005</a>) to investigate the link between salinity and intestine-associated microbiota. The amalgamation schedule graph again showed seven major fusion steps, resulting in eight distinct clusters ( <a href="#F5">Figure 5B</a>) with Euclidean distances between clusters showing their distinctive nature ( <a href="#SM5">Table S5</a>). Of the 59 intestinal microbiome samples from freshwater and 143 from saline sites, two clusters (#6 and #8) were dominated by samples from saline water sites whereas another two clusters (#2 and #5) only showed samples from freshwater ( <a href="#T3">Table 3B</a>). Therefore, these clusters provide further support to the PCoA analysis ( <a href="#F3">Figure 3</a>) that salinity is a statistically significant variable defining the Arctic char intestine-associated microbiome. The ANOVA results indicate that of the 507 taxa used in the clustering analysis, 50 have a <i>p</i>-value below 0.05. The phylum Proteobacteria accounted for 25 of these taxa ( <a href="#SM6">Table S6</a>). Members defined as belonging to an "unidentified kingdom" represented over 6% of the microbial community in two clusters (#4 and #5). Overall, the intestinal core microbiome appears more diverse than that of the skin, with 14 taxa present in all eight clusters ( <a href="#T4">Table 4B</a>). Clusters (#1–#7) included taxa that represented ~8% or more of the community, with 4 and 8 clusters with a single genus occupying over 50% of the entire intestinal microbiome. The taxa included unidentified genera from families Fusobacteriaceae, Comamonadaceae, Pseudomonadaceae, and Vibrionaceae. Other members of the intestinal-associated microbiome include <i>Streptococcus, Sphingomonas, Shewanella, Pseudomonas, Aliivibrio, Photobacterium</i>, and <i>Mycoplasma</i> ( <a href="#T4">Table 4B</a>). Again, no correlation was obtained between the presence of any of the microbial genus species and the sex, age, or size of the Arctic char ( <i>p</i> &amp;gt; 0.05; not shown). Additionally, since hierarchical Ward's linkage clustering was implemented on skin and intestine samples without regard to fishing site, we postulate that the following results provide further evidence that the changes in microbiome between fish caught in saline and freshwater environments are strongly linked to salinity. </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="microbialstructureintheskinandintestinalmucous" title="sec"> <div class="title" tagx="title" title="title">
Microbial Structure in the Skin and Intestinal Mucous</div> <p>The commensal microbes contributing to these Arctic char skin- and intestine-associated microbiome communities were found to be distinct from one another ( <a href="#T2">Table 2</a>; <a href="#F2">Figure 2</a>; <a href="#SM1">Tables S1</a>, <a href="#SM2">S2</a>). There have been few published reports on skin-associated microbiomes, especially in wild fish, possibly because of the challenges associated with ensuring aseptic collection practices and optimal preservation techniques (Kueneman et al., <a href="#B33">2014</a>). Microbiome analysis on the skin of anadromous Atlantic salmon, <i>S. salar</i>, showed that Proteobacteria was the dominant phylum (Lokesh and Kiron, <a href="#B37">2016</a>), and it was also dominant in the majority of our skin samples taken from freshwater Arctic char ( <a href="#F2">Figure 2A</a>). Significant differences in the microbiome from fish caught in fresh and saline waters suggests that the skin microbial community changes as the fish swim from one environment to another. Part of the challenge, and likely reflecting the generally unstudied field, is the large proportion of unknown genera in the skin-associated microbiome; 8 of 9 organisms making up the putative core microbiome are currently unknown, and in addition, over 1% of the microbial community belonged to unknown phyla ( <a href="#T4">Table 4A</a>). Some of these currently unknown members likely serve important roles in the skin mucous, suggesting that metagenomic analysis could perhaps be used to assign putative roles in waste removal, osmoregulation, glycoprotein-mediated drag reduction, and in immune function, and would be of interest for future experiments. </p> <p>In contrast to the paucity of information on skin-associated microbiomes, intestine-associated microbiomes have been better studied, although not extensively in wild, ocean-going fish (Egerton et al., <a href="#B23">2018</a>). Previously reported microorganisms present in the Arctic char gut include <i>Aeromonas, Flavobacterium, Pseudomonas, Lactobacillus</i>, and <i>Vibrio</i> (Ringø and Strøm, <a href="#B50">1994</a>; Ringø et al., <a href="#B51">1995</a>; Nyman et al., <a href="#B42">2017</a>). Lactic acid bacteria (phylum Firmicutes), Fusobacteria, Bacteroidetes, Gammaproteobacteria, Planctomycetes, Clostridia, Verrumicrobia and Bacilli have all been reported as a normal part of the intestine-associated microbiomes in several fish species (Ringø and Gatesoupe, <a href="#B48">1998</a>; Befring-Hovda, <a href="#B6">2007</a>; Ingerslev et al., <a href="#B28">2014</a>; Ghanbari et al., <a href="#B26">2015</a>). The results reported here are therefore consistent with the previous literature, demonstrating that the community found in wild adult <i>S. alpinus</i> intestine-associated microbiomes is not fundamentally different from other fish. </p> <p>The core intestine-associated microbiome in Arctic char includes taxa from the Gammaproteobacteria class as well as three genera within the Vibrionaceae family ( <a href="#T4">Table 4B</a>), similar to observations from Antarctic notothenioid fish species (Ward et al., <a href="#B60">2009</a>). We also noted the presence of <i>Streptococcus</i> and <i>Mycoplasma. Mycoplasma</i> have been reportedly abundant in Antarctic fish and Atlantic salmon (Holben et al., <a href="#B27">2002</a>; Song et al., <a href="#B56">2016</a>; Dehler et al., <a href="#B16">2017</a>). Unlike in humans where <i>Mycoplasma</i> and certain <i>Streptococcus</i> are associated with dysbiosis, these genera are proposed to be important for lipid and sugar metabolism and are therefore included in the core intestinal microbiome for Atlantic salmon (Dehler et al., <a href="#B16">2017</a>). In vertebrates, <i>Sphingomonas</i> have been associated with the early-life education of the immune system (Olszak et al., <a href="#B43">2012</a>; Wingender et al., <a href="#B62">2012</a>; Caballero and Pamer, <a href="#B8">2015</a>; Gensollen et al., <a href="#B25">2016</a>), and therefore may play a similar role in the gut of Arctic char. </p> <p>Notwithstanding the distinct profiles of the skin- and intestine-associated microbiomes, they appear to share certain taxa, including psychrophilic genera, such as <i>Psychrobacter, Shewanella, Flavobacterium, Acinetobacter, Photobacterium, Planctomyces</i>, and <i>Psychromonas</i>, and including bacteria belonging to the phyla Firmicutes and Cyanobacteria, which could contribute to geochemical cycles, including the nitrogen cycle. Certainly, both microbial community compositions were significantly influenced by the environment in which the fish were caught ( <a href="#F3">Figure 3</a>; <a href="#T3">Table 3</a>), with <i>Photobacterium</i> and <i>Deinococcus</i> increasing over 1–2 orders of magnitude when intestine- and skin-associated microbiomes, respectively, were recovered from saline water samples. There was no apparent correlation with other measured biotic factors. Our results on wild Arctic char populations are in accord with previous reports including Atlantic salmon, in that they showed salinity-mediated turnover in microbial distributions across the skin and intestines (Schmidt et al., <a href="#B54">2015</a>; Lokesh and Kiron, <a href="#B37">2016</a>; Dehler et al., <a href="#B16">2017</a>). Manipulation of the skin and intestinal microbiomes by deliberate salinity changes in aquaculture has not yet been explored, but our results suggest that a directed turnover in taxa might be achieved by such a protocol, and should be considered to prevent or inhibit dysbiosis. Although there has been some examination of the survival of farmed <i>S. alpinus</i> at higher salinities, consensus has not yet been reached on best practices for inland recirculating systems (Jørgensen et al., <a href="#B31">1993</a>; Larsson and Berglund, <a href="#B35">1998</a>; Summerfelt et al., <a href="#B57">2004</a>; Duston et al., <a href="#B20">2007</a>). To date, existing aquaculture studies have focused on the osmoregulatory consequences of keeping stocks at low salinity or the transfer of young fish from freshwater to a potential saltwater grow-out phase (Arnesen et al., <a href="#B3">1993</a>; Aarset, <a href="#B1">1999</a>; Duston et al., <a href="#B20">2007</a>), but there has been little consideration of the Arctic char skin-associated microbiome, nor of skin- and intestine-associated microbiomes together, and yet the biotechnological applications for manipulation of the microbiota by probiotics is of considerable interest. </p> </div> <div class="bioprospectingintheskinandintestinalmicrobiomes" title="sec"> <div class="title" tagx="title" title="title">
Bioprospecting in the Skin and Intestinal Microbiomes</div> <p>Anadromous Arctic char from this high Arctic region grow more rapidly than other local salmonids and live several decades (McPhedran et al., unpublished data). Therefore, we posit that knowledge of the microbiota could be helpful in the development of sustainable and efficient aquaculture practices and to provide an alternate modality to overcome the adverse effects of antibiotics and drugs (Nayak, <a href="#B41">2010</a>). As suggested above, consideration could be given to the manipulation of salt concentrations in Arctic char aquaculture to deliberately direct bacterial communities and possibly bypass dysbiotic episodes. Alternatively, individual isolates may prove valuable. Previously, lactic acid bacteria including <i>Streptococcus</i> have been used as probiotics and specifically for furunculosis in rainbow trout, <i>Oncorhynchus mykiss</i> (Balcázar et al., <a href="#B4">2007</a>; Pandiyan et al., <a href="#B44">2013</a>). This is a serious infection of farmed fish causing skin lesions, hemorrhaging, intestinal tissue damage, and death (Ringø et al., <a href="#B49">2004</a>). Other studies have reported the beneficial effect of <i>Shewanella</i> and <i>Vibrio</i> probiotics (Kamei et al., <a href="#B32">1988</a>; Irianto and Austin, <a href="#B29">2002</a>; Díaz-Rosales et al., <a href="#B18">2006a</a>, <a href="#B17">b</a>). The bacteria that we detected in Arctic char skin- and intestine-associated microbiomes, especially those identified as part of the core microbiome, therefore have potential in probiotic applications for aquaculture where they could play a role in immune development and promote the health and growth of farmed fish. </p> <p>Many OTUs in the skin and intestinal microbiomes represented unidentified microorganisms, and it suggests that the microbial diversity associated with these wild Arctic char provides fertile ground for bioprospecting for psychrophiles and osmotolerant organisms. For example, some known organisms that are psychrophilic and halophilic are a good source for polyunsaturated fatty acids (PUFA) (Russell and Nichols, <a href="#B53">1999</a>). Russell ( <a href="#B52">1998</a>) proposed that PUFA in marine psychrophiles allows them to balance the requirement for a fluid membrane at low temperatures with the retention of a requisite level of order. Given these functions, PUFA-producing bacteria may represent an alternate source for human use, with the added advantage of containing only one long-chain PUFA rather than the multiple components present in fish or algal oils (de Pascale et al., <a href="#B15">2012</a>). </p> <p>As indicated, taxa involved in nitrogen fixation (Cyanobacteria and Rhizobiales), nitrate oxidation (Nitrospirales), and nitrite oxidization ( <i>Nitrospira</i>) are present in the skin- and intestine-associated microbiomes ( <a href="#SM1">Tables S1</a>, <a href="#SM2">S2</a>). Their presence suggests that they may provide an inorganic source of nitrogen when organic nitrogen is low, such as when prey may not be as readily available during the winter under ice. Similar results have been observed by Lee and Childress ( <a href="#B36">1994</a>) and Shah et al. ( <a href="#B55">in press</a>) in marine invertebrates. Isolates of Nitrospirales, <i>Nitrospira</i>, and <i>Paracoccus</i>, identified in Arctic char that are adapted to low temperature conditions might be employed in municipal and domestic wastewater treatment facilities where wastewater temperatures fall below 10 °C in winter, and when microbial activity is normally severely depressed (Xu et al., <a href="#B63">2018</a>). Similarly, isolates of <i>Pseudomonas, Shewanella, Bacillus, Arthrobacter</i>, and <i>Sphingobacterium</i> in the intestine-associated microbiome ( <a href="#T4">Table 4B</a>) could play an important role in degradation of pollutants during wastewater treatment. Some bacteria (e.g., members of the Rhizobiales) could even find utility as nitrogen fixers if used for agricultural applications under low temperature conditions. </p> <p>Bioprospecting need not be restricted to probiotics, however. Further exploration of identified extremophiles may be useful for other applications. For example, between 6 and 8 million tons of waste crab, shrimp, and lobster shells are produced globally, with the chitin-rich wastes dumped in landfills or the sea (Yan and Chen, <a href="#B64">2015</a>). The presence of OTUs representing <i>Photobacterium</i>, in the intestine-associated microbiome of Arctic char ( <a href="#T2">Table 2B</a>), with members of this taxon known to aid in chitin digestion (MacDonald et al., <a href="#B38">1986</a>; Ramesh and Venugopalan, <a href="#B47">1989</a>; Itoi et al., <a href="#B30">2006</a>) is noteworthy. It suggests that in the future, chitin-containing seafood waste could be used as supplements for fish food in aquaculture facilities, and also used as a probiotic supplement for Arctic char and other fish. </p> <p>In general, our results highlight the need to further explore structural and functional aspects of the microbial communities that naturally inhabit <i>S. alpinus</i>. The turnover of both the skin- and intestine-associated communities together during migration represents a previously underappreciated stressor for the species that could be exploited for biotechnological applications including advanced aquaculture systems. The specific taxa observed suggest the potential for isolating useful probiotics, whilst the number of undefined taxa exposes a lack of understanding of this ecosystem, the resolution of which could benefit our understanding of Arctic marine ecosystems as a whole. </p> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability</div> <p>The datasets generated for this study can be found in European Nucleotide Archive, <a href="ERP111452">ERP111452</a>. </p> </div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics Statement</div> <p>Fish were sampled in accordance with an approved and issued license to fish for scientific purposes in the waters of the Northwest Territories, Yukon north slope, and Nunavut (in accordance with section 52 of the general fishery regulations of the fisheries act, Fisheries and Oceans, Canada; current permit number S-18/19-1045-NU) along with an associated animal care permit approved and issued by the Fresh Water Institute Animal Care Committee of the Department of Fisheries and Oceans (current permit number FWI-ACC AUP-2018-63). </p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>PvCdG organized the fish collection. JDN and KE undertook sequencing and provided bioinformatics analysis. EFH and GE isolated the DNA and conducted statistical analyses. VS did additional statistical analyses. VKW designed the study. EFH, VS, and VKW drafted the manuscript. All authors have read and accepted the final manuscript. </p> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest Statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank the Gjoa Haven, NU community residents, the Hunters and Trappers Association (HTA), and all of the associated supporters of the Towards a Sustainable Fishery for Nunavummiut. We would like to thank all sampling personnel that provided invaluable help in the field, and particularly the Gjoa Haven fishermen who were integral to the fish sampling efforts. We would also like to thank Eric T. Daechsel for producing the map for this publication, Kristy Moniz for her valuable technical support throughout the sampling and processing efforts, and Dr. Charles W. Greer (National Research Council) for facilitating and providing the initial water microbiome analyses with alternative primers. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p> <b>Funding.</b> This work was funded by the Government of Canada through Genome Canada and the Ontario Genomics Institute (OGI-096). We also acknowledge funding from the Natural Sciences and Engineering Research Council (Canada), the Ontario Ministry of Research and Innovation, CanNor, the Government of Nunavut, and the Northern Scientific Training Program (Polar Knowledge Canada). </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fbioe.2019.00032/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fbioe.2019.00032/full#supplementary-material</a> </p> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_1.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_2.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_3.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_4.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_5.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_6.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_1.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aarset</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Aquacultural development, institution building and research and development policy: Norwegian salmon and Arctic char farming as cases </span>. <span class="source" tagx="source" title="source">Aquacult. Econ. Manage.</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">177</span>– <span class="lpage" tagx="lpage" title="lpage">191</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1080/13657309909380244">10.1080/13657309909380244</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Apprill</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robbins</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eren</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pack</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reveillaud</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mattila</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Humpback whale populaions share a core skin bacterial community: towards a health index for marine mammals? </span> <span class="source" tagx="source" title="source">PLoS ONE.</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e90785</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0090785">10.1371/journal.pone.0090785</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24671052">24671052</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnesen</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jørgensen</span> <span class="given-names" tagx="given-names" title="given-names">E. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jobling</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">Feed intake, growth and osmoregulation in Arctic charr, <i>Salvelinus alpinus</i> (L.), transferred from freshwater to saltwater at 8 °C during summer and winter </span>. <span class="source" tagx="source" title="source">FISH.</span> <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">281</span>– <span class="lpage" tagx="lpage" title="lpage">292</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24202870">24202870</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balcázar</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Blas</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruiz-Zarzuela</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vendrell</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gironés</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muzquiz</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Enhancement of the immune response and protection induced by probiotic lactic acid bacteria against furuncolosis in rainbow trout ( <i>Oncorhynchus mykiss</i>) </span>. <span class="source" tagx="source" title="source">FEMS Immunol. Med. Microbiol.</span> <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">185</span>– <span class="lpage" tagx="lpage" title="lpage">193</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1574-695X.2007.00294.x">10.1111/j.1574-695X.2007.00294.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17645738">17645738</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartram</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M. D. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stearns</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moreno-Hagelsieb</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neufeld</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Generation of multimillion-sequence 16S rRNA gene libraries from complex microbial communities by assembling paired-end Illumina reads </span>. <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span> <span class="volume" tagx="volume" title="volume">77</span>, <span class="fpage" tagx="fpage" title="fpage">3846</span>– <span class="lpage" tagx="lpage" title="lpage">3852</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.02772-10">10.1128/AEM.02772-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21460107">21460107</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Befring-Hovda</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="source" tagx="source" title="source">Application of PCR and DGGE to Characterize the Microflora of Farmed Fish.</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Bergen</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Doctoral Disseration, University of Bergen</span>. </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boutin</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernatchez</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Audet</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Derôme</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Network analysis highlights complex interactions between pathogen, host and commensal microbiota </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e84772</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0084772">10.1371/journal.pone.0084772</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24376845">24376845</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caballero</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pamer</span> <span class="given-names" tagx="given-names" title="given-names">E. G.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Microbiota-mediated inflammation and antimicrobial defense in the intestine</span>. <span class="source" tagx="source" title="source">Annu. Rev. Immunol.</span> <span class="volume" tagx="volume" title="volume">33</span>, <span class="fpage" tagx="fpage" title="fpage">227</span>– <span class="lpage" tagx="lpage" title="lpage">256</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev-immunol-032713-120238">10.1146/annurev-immunol-032713-120238</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25581310">25581310</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caporaso</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuczynski</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stombaugh</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bittinger</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bushman</span> <span class="given-names" tagx="given-names" title="given-names">F. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Costello</span> <span class="given-names" tagx="given-names" title="given-names">E. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">QIIME allows analysis of high-throughput community sequencing data</span>. <span class="source" tagx="source" title="source">Nat. Methods.</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">335</span>– <span class="lpage" tagx="lpage" title="lpage">336</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.f.303">10.1038/nmeth.f.303</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20383131">20383131</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caporaso</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lauber</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walters</span> <span class="given-names" tagx="given-names" title="given-names">W. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berg-Lyons</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lozupone</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turnbaugh</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample</span>. <span class="source" tagx="source" title="source">PNAS</span> <span class="volume" tagx="volume" title="volume">108</span>, <span class="fpage" tagx="fpage" title="fpage">4516</span>– <span class="lpage" tagx="lpage" title="lpage">4522</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1000080107">10.1073/pnas.1000080107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20534432">20534432</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carmack</span> <span class="given-names" tagx="given-names" title="given-names">E. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">The alpha/beta ocean distinction: a perspective on freshwater fluxes, convection, nutrients and productivity in high-latitude seas </span>. <span class="source" tagx="source" title="source">Deep Sea Res. Part II: Top Stud. Oceanogr.</span> <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">2578</span>– <span class="lpage" tagx="lpage" title="lpage">2598</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.dsr2.2007.08.018">10.1016/j.dsr2.2007.08.018</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiarello</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Villéger</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouvier</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bettarel</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouvier</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">High diversity of skin-associated bacterial communities of marine fishes is promoted by their high variability among body parts, individuals and species </span>. <span class="source" tagx="source" title="source">FEMS Microbiol. Ecol.</span> <span class="volume" tagx="volume" title="volume">91</span>: <span class="fpage" tagx="fpage" title="fpage">fiv061</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/femsec/fiv061">10.1093/femsec/fiv061</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26048284">26048284</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clarke</span> <span class="given-names" tagx="given-names" title="given-names">K. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gorely</span> <span class="given-names" tagx="given-names" title="given-names">R. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Somerfield</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Warwick</span> <span class="given-names" tagx="given-names" title="given-names">R. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="source" tagx="source" title="source">Change in Marine Communities: An Approach to Statistical Analysis and Interpretation</span>, <span class="edition" tagx="edition" title="edition">3rd edition</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Plymouth</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">PRIMER-E</span>. </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coolen</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Post</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">C. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forney</span> <span class="given-names" tagx="given-names" title="given-names">L. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization of microbial communities found in the human vagina by analysis of terminal restriction fragment length polymorphisms of 16S rRNA genes </span>. <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span> <span class="volume" tagx="volume" title="volume">71</span>, <span class="fpage" tagx="fpage" title="fpage">8729</span>– <span class="lpage" tagx="lpage" title="lpage">8737</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.71.12.8729-8737.2005">10.1128/AEM.71.12.8729-8737.2005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16332868">16332868</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Pascale</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Santi</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landfald</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">The microbial diversity of Polar environments is a fertile ground for bioprospecting</span>. <span class="source" tagx="source" title="source">Mar. Genom.</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">15</span>– <span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.margen.2012.04.004">10.1016/j.margen.2012.04.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23199876">23199876</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dehler</span> <span class="given-names" tagx="given-names" title="given-names">C. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Secombes</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">S. A. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Seawater transfer alters the intestinal microbiota profiles of Atlantic salmon ( <i>Salmo salar</i> L.) </span>. <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">13877</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41598-017-13249-8">10.1038/s41598-017-13249-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29066818">29066818</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Díaz-Rosales</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chabrillón</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arijo</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martinez-Manzanares</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moriñigo</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balebona</span> <span class="given-names" tagx="given-names" title="given-names">M. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006b</span>). <span class="mixed-article-title" title="mixed-article-title">Superoxide dismutase and catalase activities in photobacterium <i>Damselae</i> ssp </span>. <span class="source" tagx="source" title="source">piscicida. J. Fish Dis.</span> <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">355</span>– <span class="lpage" tagx="lpage" title="lpage">364</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2761.2006.00726.x">10.1111/j.1365-2761.2006.00726.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16768716">16768716</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Díaz-Rosales</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salinas</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cuesta</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chabrillon</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balebona</span> <span class="given-names" tagx="given-names" title="given-names">M. C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2006a</span>). <span class="mixed-article-title" title="mixed-article-title">Gilthead seabream ( <i>Sparus aurata</i> L.) innate immune response after dietary administration of heat-inactivated potential probiotics </span>. <span class="source" tagx="source" title="source">Fish Shellfish Immun.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">482</span>– <span class="lpage" tagx="lpage" title="lpage">492</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.fsi.2005.06.007">10.1016/j.fsi.2005.06.007</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16169250">16169250</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dillner</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schauer</span> <span class="given-names" tagx="given-names" title="given-names">J. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christensen</span> <span class="given-names" tagx="given-names" title="given-names">W. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cass</span> <span class="given-names" tagx="given-names" title="given-names">G. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">A quantitiative method for clustering size distributions of elements</span>. <span class="source" tagx="source" title="source">Atmos. Environ.</span> <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">1525</span>– <span class="lpage" tagx="lpage" title="lpage">1537</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.atmosenv.2004.11.035">10.1016/j.atmosenv.2004.11.035</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duston</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Astatkie</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murray</span> <span class="given-names" tagx="given-names" title="given-names">S. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Effect of salinity at constant 10 °C on grow-out of anadromous Arctic charr from Labrador</span>. <span class="source" tagx="source" title="source">Aquaculture.</span> <span class="volume" tagx="volume" title="volume">273</span>, <span class="fpage" tagx="fpage" title="fpage">679</span>– <span class="lpage" tagx="lpage" title="lpage">686</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.aquaculture.2007.10.035">10.1016/j.aquaculture.2007.10.035</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">UPARSE: highly accurate OTU sequences from microbial amplicon reads</span>. <span class="source" tagx="source" title="source">Nat. Methods.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">996</span>– <span class="lpage" tagx="lpage" title="lpage">998</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.2604">10.1038/nmeth.2604</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23955772">23955772</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">B. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clemente</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quince</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">UCHIME improves sensitivity and speed of chimera detection</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">2194</span>– <span class="lpage" tagx="lpage" title="lpage">2200</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btr381">10.1093/bioinformatics/btr381</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674">21700674</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Egerton</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Culloty</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whooley</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stanton</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ross</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">The gut microbiota of marine fish</span>. <span class="source" tagx="source" title="source">Front. Microbiol.</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">873</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2018.00873">10.3389/fmicb.2018.00873</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29780377">29780377</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Esteban</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cerezuela</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">4-Fish mucosal immunity: skin A2</span>, in <span class="source" tagx="source" title="source">Mucosal Health in Aquaculture</span>, eds <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beck</span> <span class="given-names" tagx="given-names" title="given-names">H.B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peatman</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> ( <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier Inc.</span>), <span class="fpage" tagx="fpage" title="fpage">67</span>– <span class="lpage" tagx="lpage" title="lpage">92</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/B978-0-12-417186-2.00004-2">10.1016/B978-0-12-417186-2.00004-2</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gensollen</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iyer</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasper</span> <span class="given-names" tagx="given-names" title="given-names">D. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blumberg</span> <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">How colonization by microbiota in early life shapes the immune system</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">352</span>, <span class="fpage" tagx="fpage" title="fpage">539</span>– <span class="lpage" tagx="lpage" title="lpage">544</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.aad9378">10.1126/science.aad9378</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27126036">27126036</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghanbari</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kneifel</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Domig</span> <span class="given-names" tagx="given-names" title="given-names">K. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">A new view of the fish gut microbiome: advances from next-generation sequencing</span>. <span class="source" tagx="source" title="source">Aquaculture</span> <span class="volume" tagx="volume" title="volume">448</span>, <span class="fpage" tagx="fpage" title="fpage">464</span>– <span class="lpage" tagx="lpage" title="lpage">475</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.aquaculture.2015.06.033">10.1016/j.aquaculture.2015.06.033</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holben</span> <span class="given-names" tagx="given-names" title="given-names">W. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saarinen</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Särkilahti</span> <span class="given-names" tagx="given-names" title="given-names">L. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Apajalahti</span> <span class="given-names" tagx="given-names" title="given-names">J. H. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of intestinal microflora indicates a novel <i>Mycoplasma</i> phylotype in farmed and wild salmon </span>. <span class="source" tagx="source" title="source">Microb. Ecol.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">175</span>– <span class="lpage" tagx="lpage" title="lpage">185</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00248-002-1011-6">10.1007/s00248-002-1011-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12082453">12082453</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ingerslev</span> <span class="given-names" tagx="given-names" title="given-names">H. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Gersdorff Jørgensen</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strube</span> <span class="given-names" tagx="given-names" title="given-names">M. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larsen</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dalsgaard</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boye</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The development of the gut microbiota in rainbow trout ( <i>Oncorhynchus mykiss</i>) is affected by first feeding and diet type </span>. <span class="source" tagx="source" title="source">Aquaculture</span> <span class="volume" tagx="volume" title="volume">424</span>, <span class="fpage" tagx="fpage" title="fpage">24</span>– <span class="lpage" tagx="lpage" title="lpage">34</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.aquaculture.2013.12.032">10.1016/j.aquaculture.2013.12.032</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irianto</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Austin</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Use of probiotics to control furunculosis in rainbow trout, <i>Oncorhynchus mykiss</i> (Walbaum) </span>. <span class="source" tagx="source" title="source">J. Fish Dis.</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">333</span>– <span class="lpage" tagx="lpage" title="lpage">342</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-2761.2002.00375.x">10.1046/j.1365-2761.2002.00375.x</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Itoi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okamura</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koyama</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugita</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Chitinolytic bacteria in the intestinal tract of Japanese coastal fishes</span>. <span class="source" tagx="source" title="source">Can. J. Microbiol.</span> <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">1158</span>– <span class="lpage" tagx="lpage" title="lpage">1163</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1139/w06-082">10.1139/w06-082</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17473885">17473885</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jørgensen</span> <span class="given-names" tagx="given-names" title="given-names">E. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christiansen</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jobling</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">Effects of stocking density on food intake, growth performance and oxygen consumption in Arctic charr ( <i>Salvelinus alpinus</i>) </span>. <span class="source" tagx="source" title="source">Aquaculture</span> <span class="volume" tagx="volume" title="volume">110</span>, <span class="fpage" tagx="fpage" title="fpage">191</span>– <span class="lpage" tagx="lpage" title="lpage">204</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0044-8486(93)90272-Z">10.1016/0044-8486(93)90272-Z</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamei</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshimizu</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ezura</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kimura</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">1988</span>). <span class="mixed-article-title" title="mixed-article-title">Screening of bacteria with antiviral activity from fresh water salmonid hatcheries</span>. <span class="source" tagx="source" title="source">Microbiol. Immunol.</span> <span class="volume" tagx="volume" title="volume">32</span>, <span class="fpage" tagx="fpage" title="fpage">67</span>– <span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1348-0421.1988.tb01366.x">10.1111/j.1348-0421.1988.tb01366.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3374405">3374405</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kueneman</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parfrey</span> <span class="given-names" tagx="given-names" title="given-names">L. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woodhams</span> <span class="given-names" tagx="given-names" title="given-names">D. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archer</span> <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKenzie</span> <span class="given-names" tagx="given-names" title="given-names">V. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The amphibian skin-associated microbiome across species, space and life history stages</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">1238</span>– <span class="lpage" tagx="lpage" title="lpage">1250</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/mec.12510">10.1111/mec.12510</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24171949">24171949</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lane</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">16S/23S rRNA sequencing</span>. <span class="source" tagx="source" title="source">Nucl. Acid Techniq. Bact. Systemat.</span> <span class="fpage" tagx="fpage" title="fpage">115</span>– <span class="lpage" tagx="lpage" title="lpage">175</span>. </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larsson</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berglund</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Growth and food consumption of 0+ Arctic charr fed pelleted or natural food at six different temperatures </span>. <span class="source" tagx="source" title="source">J. Fish Biol.</span> <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">230</span>– <span class="lpage" tagx="lpage" title="lpage">242</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1095-8649.1998.tb00795.x">10.1111/j.1095-8649.1998.tb00795.x</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">R. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Childress</span> <span class="given-names" tagx="given-names" title="given-names">J. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Assimilation of inorganic nitrogen by marine invertebrates and their chemoautotrophic and methanotrophic symbionts </span>. <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span> <span class="volume" tagx="volume" title="volume">60</span>, <span class="fpage" tagx="fpage" title="fpage">1852</span>– <span class="lpage" tagx="lpage" title="lpage">1858</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16349279">16349279</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lokesh</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kiron</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Transition from freshwater to seawater reshapes the skin-associated microbiota of Atlantic salmon </span>. <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">19707</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep19707">10.1038/srep19707</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26806545">26806545</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MacDonald</span> <span class="given-names" tagx="given-names" title="given-names">N. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stark</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Austin</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">1986</span>). <span class="mixed-article-title" title="mixed-article-title">Bacterial microflora in the gastro-intestinal tract of Dover sole ( <i>Solea solea</i> L.), with emphasis on the possible role of bacteria in the nutrition of the host </span>. <span class="source" tagx="source" title="source">FEMS Microbiol. Lett.</span> <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">107</span>– <span class="lpage" tagx="lpage" title="lpage">111</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1574-6968.1986.tb01508.x">10.1111/j.1574-6968.1986.tb01508.x</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masella</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartram</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Truszkowski</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">D. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neufeld</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">PANDAseq: paired-end assembler for Illumina sequences</span>. <span class="source" tagx="source" title="source">BMC Bioinformat.</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">31</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-13-31">10.1186/1471-2105-13-31</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22333067">22333067</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McDonald</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">M. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodrich</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nawrocki</span> <span class="given-names" tagx="given-names" title="given-names">E. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeSantis</span> <span class="given-names" tagx="given-names" title="given-names">T. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Probst</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea </span>. <span class="source" tagx="source" title="source">ISME J.</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">610</span>– <span class="lpage" tagx="lpage" title="lpage">618</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ismej.2011.139">10.1038/ismej.2011.139</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22134646">22134646</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nayak</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Probiotics and immunity: a fish perspective</span>. <span class="source" tagx="source" title="source">Fish Shellfish Immun.</span> <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">2</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.fsi.2010.02.017">10.1016/j.fsi.2010.02.017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20219683">20219683</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nyman</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huyben</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lundh</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dicksved</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Effects of microbe-and mussel-based diets on the gut microbiota in Arctic charr ( <i>Salvelinus alpinus</i>) </span>. <span class="source" tagx="source" title="source">Aquacult. Rep.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">34</span>– <span class="lpage" tagx="lpage" title="lpage">40</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.aqrep.2016.12.003">10.1016/j.aqrep.2016.12.003</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olszak</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeissig</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vera</span> <span class="given-names" tagx="given-names" title="given-names">M. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richter</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franke</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial exposure during early life has persistent effects on natural killer T cell function </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">336</span>, <span class="fpage" tagx="fpage" title="fpage">489</span>– <span class="lpage" tagx="lpage" title="lpage">493</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1219328">10.1126/science.1219328</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22442383">22442383</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandiyan</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balaraman</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thirunavukkarasu</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">George</span> <span class="given-names" tagx="given-names" title="given-names">E. G. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Subaramaniyan</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manikkam</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Probiotics in aquaculture</span>. <span class="source" tagx="source" title="source">Drug Invent. Today</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">55</span>– <span class="lpage" tagx="lpage" title="lpage">59</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.dit.2013.03.003">10.1016/j.dit.2013.03.003</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parada</span> <span class="given-names" tagx="given-names" title="given-names">A. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Needham</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fuhrman</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Every base matters: assessing small subunit rRNA primers for marine microbiomes with mock communities, time series and global field samples </span>. <span class="source" tagx="source" title="source">Environ. Microbiol.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">1403</span>– <span class="lpage" tagx="lpage" title="lpage">1414</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/1462-2920.13023">10.1111/1462-2920.13023</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26271760">26271760</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quince</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lanzen</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davenport</span> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turnbaugh</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Removing noise from pyrosequenced amplicons</span>. <span class="source" tagx="source" title="source">BMC Bioinformat.</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">38</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-12-38">10.1186/1471-2105-12-38</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21276213">21276213</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramesh</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venugopalan</span> <span class="given-names" tagx="given-names" title="given-names">V. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">1989</span>). <span class="mixed-article-title" title="mixed-article-title">Response of enteric luminous bacteria to environmental conditions in the gut of the fish </span>. <span class="source" tagx="source" title="source">J. Appl. Bacteriol.</span> <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">529</span>– <span class="lpage" tagx="lpage" title="lpage">533</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2672.1989.tb02536.x">10.1111/j.1365-2672.1989.tb02536.x</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ringø</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gatesoupe</span> <span class="given-names" tagx="given-names" title="given-names">F. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Lactic acid bacteria in fish: a review</span>. <span class="source" tagx="source" title="source">Aquaculture.</span> <span class="volume" tagx="volume" title="volume">160</span>, <span class="fpage" tagx="fpage" title="fpage">177</span>– <span class="lpage" tagx="lpage" title="lpage">203</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0044-8486(97)00299-8">10.1016/S0044-8486(97)00299-8</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ringø</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jutfelt</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanapathippillai</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bakken</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sundell</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glette</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Damaging effect of the fish pathogen <i>Aeromonas salmonicida</i> ssp. <i>salmonicida</i> on intestinal enterocytes of Atlantic salmon <i>(Salmo salar L.)</i> </span>. <span class="source" tagx="source" title="source">Cell Tissue Res.</span> <span class="volume" tagx="volume" title="volume">318</span>, <span class="fpage" tagx="fpage" title="fpage">305</span>– <span class="lpage" tagx="lpage" title="lpage">311</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00441-004-0934-2">10.1007/s00441-004-0934-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15503156">15503156</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ringø</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strøm</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Microflora of Arctic charr, <i>Salvelinus alpinus</i> (L.): gastrointestinal microflora of free-living fish and effect of diet and salinity on intestinal microflora </span>. <span class="source" tagx="source" title="source">Aquacult. Res.</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">623</span>– <span class="lpage" tagx="lpage" title="lpage">629</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2109.1994.tb00726.x">10.1111/j.1365-2109.1994.tb00726.x</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ringø</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strøm</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tabachek</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Intestinal microflora of salmonids: a review</span>. <span class="source" tagx="source" title="source">Aquacult Res.</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">773</span>– <span class="lpage" tagx="lpage" title="lpage">789</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2109.1995.tb00870.x">10.1111/j.1365-2109.1995.tb00870.x</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Russell</span> <span class="given-names" tagx="given-names" title="given-names">N. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular adaptations in psychrophilic bacteria: potential for biotechnological applications</span>, in <span class="source" tagx="source" title="source">Biotechnology of Extremophiles. Advances in Biochemical Engineering/Biotechnology</span>, <span class="volume" tagx="volume" title="volume">Vol 61</span>, ed <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Antranikian</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> ( <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Heidelberg</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">21</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BFb0102287">10.1007/BFb0102287</a> </span> </span> </li> <li tag="ref"> <a name="B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Russell</span> <span class="given-names" tagx="given-names" title="given-names">N. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nichols</span> <span class="given-names" tagx="given-names" title="given-names">D. S.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Polyunsaturated fatty acids in marine bacteria- a dogma rewritten</span>. <span class="source" tagx="source" title="source">Microbiology</span> <span class="volume" tagx="volume" title="volume">145</span>, <span class="fpage" tagx="fpage" title="fpage">767</span>– <span class="lpage" tagx="lpage" title="lpage">779</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1099/13500872-145-4-767">10.1099/13500872-145-4-767</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10220156">10220156</a> </span> </span> </li> <li tag="ref"> <a name="B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">V. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">K. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melvin</span> <span class="given-names" tagx="given-names" title="given-names">D. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amaral-Zettler</span> <span class="given-names" tagx="given-names" title="given-names">L. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Community assembly of a euryhaline fish microbiome during salinity acclimation</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">2537</span>– <span class="lpage" tagx="lpage" title="lpage">2550</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/mec.13177">10.1111/mec.13177</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25819646">25819646</a> </span> </span> </li> <li tag="ref"> <a name="B55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanacredi</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> </span> (in press). <span class="mixed-article-title" title="mixed-article-title">Microbial flora associated with laboratory breed horseshoe crabs</span>, in <span class="source" tagx="source" title="source">The Conservation Biology of Horseshoe Crabs: Social Ecological Global Perspectives.</span> ed <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanacredi</span> <span class="given-names" tagx="given-names" title="given-names">J. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Botton</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheung</span> <span class="given-names" tagx="given-names" title="given-names">S. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carmichael</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iwasaki</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dordrecht</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Publishers</span>). </span> </li> <li tag="ref"> <a name="B56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The gut microbial community of antarctic fish detected by 16S rRNA gene sequence analysis</span>. <span class="source" tagx="source" title="source">Biomed Res. Int.</span> <span class="volume" tagx="volume" title="volume">2016</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1155/2016/3241529">10.1155/2016/3241529</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27957494">27957494</a> </span> </span> </li> <li tag="ref"> <a name="B57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Summerfelt</span> <span class="given-names" tagx="given-names" title="given-names">S. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilton</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roberts</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rimmer</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fonkalsrud</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Developments in recirculating systems for Arctic char culture in North America</span>. <span class="source" tagx="source" title="source">Aquacult. Eng.</span> <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">31</span>– <span class="lpage" tagx="lpage" title="lpage">71</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.aquaeng.2003.09.001">10.1016/j.aquaeng.2003.09.001</a> </span> </span> </li> <li tag="ref"> <a name="B58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vázquez-Baeza</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pirrung</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">EMPeror: a tool for visualizing high-throughput microbial community data</span>. <span class="source" tagx="source" title="source">Gigascience</span> <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">16</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/2047-217X-2-16">10.1186/2047-217X-2-16</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24280061">24280061</a> </span> </span> </li> <li tag="ref"> <a name="B59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garrity</span> <span class="given-names" tagx="given-names" title="given-names">G. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiedje</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cole</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy </span>. <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span> <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">5261</span>– <span class="lpage" tagx="lpage" title="lpage">5267</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.00062-07">10.1128/AEM.00062-07</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664">17586664</a> </span> </span> </li> <li tag="ref"> <a name="B60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ward</span> <span class="given-names" tagx="given-names" title="given-names">N. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steven</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penn</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meth,é</span> <span class="given-names" tagx="given-names" title="given-names">B. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Detrich</span> <span class="given-names" tagx="given-names" title="given-names">W. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization of the intestinal microbiota of two Antarctic notothenioid fish species</span>. <span class="source" tagx="source" title="source">Extremophiles</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">679</span>– <span class="lpage" tagx="lpage" title="lpage">685</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00792-009-0252-4">10.1007/s00792-009-0252-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19472032">19472032</a> </span> </span> </li> <li tag="ref"> <a name="B61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Watling</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="source" tagx="source" title="source">Queensland Government Natural Resources and Water.</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Measuring salinity. (Report No. L137)</span>. </span> </li> <li tag="ref"> <a name="B62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wingender</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stepniak</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krebs</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McBride</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Intestinal microbes affect phenotypes and functions of invariant natural killer T cells in mice </span>. <span class="source" tagx="source" title="source">Gastroenterology</span> <span class="volume" tagx="volume" title="volume">143</span>, <span class="fpage" tagx="fpage" title="fpage">418</span>– <span class="lpage" tagx="lpage" title="lpage">428</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1053/j.gastro.2012.04.017">10.1053/j.gastro.2012.04.017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22522092">22522092</a> </span> </span> </li> <li tag="ref"> <a name="B63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">J. Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ainiwaer</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of bacterial community structure of activated sludge from wastewater treatment plants in winter </span>. <span class="source" tagx="source" title="source">Biomed Res. Int.</span> <span class="volume" tagx="volume" title="volume">2018</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1155/2018/8278970">10.1155/2018/8278970</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29707578">29707578</a> </span> </span> </li> <li tag="ref"> <a name="B64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Sustainability: don't waste seafood waste</span>. <span class="source" tagx="source" title="source">Nature News.</span> <span class="volume" tagx="volume" title="volume">524</span>: <span class="fpage" tagx="fpage" title="fpage">7155</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/524155a">10.1038/524155a</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26268177">26268177</a> </span> </span> </li> </ul> </div> </div>  </body></html>