<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Evol Biol</span> <span class="iso-abbrev" title="iso-abbrev">BMC Evol. Biol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Evolutionary Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2148</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6714099">6714099</a> </span> <span class="publisher-id" title="publisher-id">1494</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12862-019-1494-0">10.1186/s12862-019-1494-0</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Elevated mutation rates are unlikely to evolve in sexual species, not even under rapid environmental change </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7824-8706</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romero-Mujalli</span> <span class="given-names" tagx="given-names" title="given-names">Daniel</span> </span> <span class="address" title="address"> <span class="phone" tagx="phone" title="phone">+49331/977-5604</span> <span class="email" tagx="email" title="email">dmujalli@uni-potsdam.de</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeltsch</span> <span class="given-names" tagx="given-names" title="given-names">Florian</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">jeltsch@uni-potsdam.de</span> </span> <a href="#Aff2">2</a> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiedemann</span> <span class="given-names" tagx="given-names" title="given-names">Ralph</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">tiedeman@uni-potsdam.de</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0942 1117</span> <span class="institution-id" title="institution-id">grid.11348.3f</span> <span class="institution" title="institution">University of Potsdam, Evolutionary Biology/Systematic Zoology, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0942 1117</span> <span class="institution-id" title="institution-id">grid.11348.3f</span> <span class="institution" title="institution">Plant Ecology and Nature Conservation, </span> <span class="institution" title="institution">University of Potsdam, </span> </div> </span> <span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.452299.1</span> <span class="institution" title="institution">Berlin-Brandenburg Institute of Advanced Biodiversity Research (BBIB), </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-8-8</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-8-8</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">19</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">175</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-11-21</span> <span class="accepted" title="accepted">accepted: 2019-8-6</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Organisms are expected to respond to changing environmental conditions through local adaptation, range shift or local extinction. The process of local adaptation can occur by genetic changes or phenotypic plasticity, and becomes especially relevant when dispersal abilities or possibilities are somehow constrained. For genetic changes to occur, mutations are the ultimate source of variation and the mutation rate in terms of a mutator locus can be subject to evolutionary change. Recent findings suggest that the evolution of the mutation rate in a sexual species can advance invasion speed and promote adaptation to novel environmental conditions. Following this idea, this work uses an individual-based model approach to investigate if the mutation rate can also evolve in a sexual species experiencing different conditions of directional climate change, under different scenarios of colored stochastic environmental noise, probability of recombination and of beneficial mutations. The color of the noise mimicked investigating the evolutionary dynamics of the mutation rate in different habitats. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>The results suggest that the mutation rate in a sexual species experiencing directional climate change scenarios can evolve and reach relatively high values mainly under conditions of complete linkage of the mutator locus and the adaptation locus. In contrast, when they are unlinked, the mutation rate can slightly increase only under scenarios where at least 50% of arising mutations are beneficial and the rate of environmental change is relatively fast. This result is robust under different scenarios of stochastic environmental noise, which supports the observation of no systematic variation in the mutation rate among organisms experiencing different habitats. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Given that 50% beneficial mutations may be an unrealistic assumption, and that recombination is ubiquitous in sexual species, the evolution of an elevated mutation rate in a sexual species experiencing directional climate change might be rather unlikely. Furthermore, when the percentage of beneficial mutations and the population size are small, sexual species (especially multicellular ones) producing few offspring may be expected to react to changing environments not by adaptive genetic change, but mainly through plasticity. Without the ability for a plastic response, such species may become – at least locally – extinct. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (10.1186/s12862-019-1494-0) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Individual-based models</div> <div class="kwd" title="kwd">
Mutation rate</div> <div class="kwd" title="kwd">
Mutator locus</div> <div class="kwd" title="kwd">
Directional climate change</div> <div class="kwd" title="kwd">
Recombination</div> <div class="kwd" title="kwd">
Beneficial mutations, sexual species</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004238</span> <span class="institution" title="institution">Universität Potsdam</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">not applicable</span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Local adaptation to changing environmental conditions, such as directional climate change, becomes of high importance for organisms with limited dispersal abilities, or when physical barriers preventing dispersal are present. </p> <p>In such scenarios, organisms can adapt by genetic changes and / or phenotypic plasticity. For genetic changes to occur, mutations are the ultimate source of novel variation, and it is generally assumed that a mutation is a rare event [ <a href="#CR1">1</a>]. Consequently, individual-based models of explicit genetics typically assume small and constant (i.e., non-evolving) mutation rates [ <a href="#CR2">2</a>– <a href="#CR4">4</a>]. According to evidence, only few mutations are adaptive; many deleterious; and some are neutral [ <a href="#CR5">5</a>]. There is however genetic variation in DNA repair and replication processes [ <a href="#CR6">6</a>– <a href="#CR9">9</a>], affecting the probability of a mutation to occur. Genetic loci affecting the origin of new variation (i.e., the mutation rate) have been termed "mutator" loci. They may be subject to selection, and selective forces may depend on the environmental context or scenario. </p> <p>Previous work investigating the evolution of the mutation rate has found that the fate of mutator alleles may differ for sexual and asexual organisms [ <a href="#CR10">10</a>, <a href="#CR11">11</a>]. In asexual organisms mutator alleles are associated with the mutations they caused, thus mutator alleles leading to an increase of the mutation rate can increase in frequency by hitchhiking with beneficial mutations at other loci [ <a href="#CR12">12</a>]. In contrast, in sexual organisms given that recombination breaks linkage disequilibrium, the mutator allele will be separated soon from a beneficial mutation it has caused and will not hitchhike to high frequency [ <a href="#CR10">10</a>]. Consequently, the evolution of mutation rate for a sexual organism is rather unlikely, and the mutation rate is then expected to stay close to a minimum achievable, limited either, by the costs of replication fidelity or by the drift limit [ <a href="#CR13">13</a>, <a href="#CR14">14</a>]. </p> <p>Contrary to previous authors, a recent simulation study suggests that the mutation rate can evolve to relatively high values also in sexual species and this can advance invasion speed and promote adaptation to novel environmental conditions along an environmental gradient [ <a href="#CR15">15</a>]. The proposed mechanism is induced linkage disequilibrium between the dispersal locus and the mutator locus (both were evolving traits) that arise from spatial sorting and iterated founder event [ <a href="#CR15">15</a>]. The same authors also found that these results still held under assumptions of 90% of the mutations being lethal. </p> <p>The present work uses an individual-based modeling approach focused on local adaptation to test the evolution of the mutation rate in a population of a sexual species experiencing directional climate change, under different scenarios of linkage disequilibrium (unlinked to complete linkage). The aim was to investigate, following the findings of Cobben et al. [ <a href="#CR15">15</a>], whether directional climate change scenarios can also lead to the evolution of an elevated mutation rate in a sexual species. This work employs an alternative method for the simulation of beneficial mutations, i.e., implementing a distribution of mutation effects inspired by the concept of slightly deleterious mutations [ <a href="#CR16">16</a>, <a href="#CR17">17</a>]. Furthermore, different scenarios of environmental stochasticity or noise color were investigated to test whether the mutation rate could vary among organisms experiencing different habitats. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>In our simulations, the mutation rate followed different evolutionary trajectories, relative to the percentage of beneficial mutation <i>bm</i> and the probability of recombination <i>pR</i>: Overall, the evolved mutation rate reached higher values when increasing the percentage of beneficial mutations <i>bm</i>, and this was independent of scenarios of probability of recombination <i>pR</i> and rate of environmental change (Fig. <a href="#Fig1">1</a>, Additional file <a href="#MOESM1">1</a>: Figure S2). The mutation rate evolved to relatively high values mainly under scenarios of complete linkage ( <i>pR = 1</i>), relatively rapid directional climate change, and 25% or higher percentage of beneficial mutations (Fig. <a href="#Fig1">1</a>). Under the unlinked recombination scenario ( <i>pR = 0</i>), the mutation rate evolved to relatively high values only if the rate of environmental change was fast ( <i>η = 0.04</i>) and the percentage of beneficial mutations was high (50%; Fig. <a href="#Fig1">1</a>). An intermediate recombination rate ( <i>pR = 0.5</i>) yielded intermediate results (Additional file <a href="#MOESM1">1</a>: Figure S1). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>The evolution of the mutation rate under scenarios of directional environmental change, probability of recombination <i>pR</i>, and percentage of beneficial mutations <i>bm</i>. Each data point corresponds to the mean mutation rate present in the population at the end of each simulation run (200 generations). The number of data points per box is shown in parenthesis (selection parameter <i>γ = 2.2</i>). A weaker selection regime ( <i>γ = 3.2</i>) showed a similar pattern (data not shown). Stable env: stable environment (no directional environmental change) </p>   </div> <p /> <p>In general terms, for a population of a sexual species, it seems unlikely for the mutation rate to evolve to higher values under directional climate change scenarios, unless the rate of change is very rapid ( <i>η = 0.04</i>) and the percentage of beneficial mutations is high (50%). This result is robust under different scenarios of environmental stochasticity or noise color (Fig. <a href="#Fig2">2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>The evolution of the mutation rate under scenarios of blue (left) and red (right) noise, directional environmental change, probability of recombination <i>pR</i>, and percentage of beneficial mutations <i>bm</i>. Each data point corresponds to the mean mutation rate present in the population at the end of each simulation run (200 generations). Stable env: stable environment (no directional environmental change) </p>   </div> <p /> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The aim of this study was to investigate whether the mutation rate could evolve to higher values in the course of local adaptation of an isolated population of a sexual species experiencing directional climate change. This work was inspired by a previous study that found adaptive evolution of the mutation rate which advances invasion speed of a sexual population inhabiting an environmental gradient [ <a href="#CR15">15</a>]. According to our results, the mutation rate can evolve to relatively high values only under conditions of complete linkage or, without such linkage, under the assumption of 50% beneficial mutations. Under complete linkage, the mutation rate can evolve under intermediate to high rates of environmental change, and when the percentage of beneficial mutations is at least 25%. The scenario of complete linkage suggests mutation rates can be expected to evolve in asexual organisms, which is in line with current theory [ <a href="#CR10">10</a>, <a href="#CR11">11</a>]. For example, it was demonstrated for <i>Escherichia coli</i> populations that mutator alleles could be fixed by hitchhiking on beneficial mutations at other loci [ <a href="#CR12">12</a>]. In a sexual species, scenarios resembling intermediate to complete linkage may occur if the mutator locus is somehow linked to the trait under selection (e.g., by close proximity on the same chromosome), such that it can hitchhike on beneficial mutations occurring on the evolving trait. Scenarios of intermediate linkage ( <i>pR = 0.5</i>) can also yield slightly elevated mutation rates, relative to the unlinked scenario. However, assuming a definite location of the mutator locus in the genome (as, e.g., a locus encoding for a DNA polymerase), most traits under selection can be expected to be encoded far away from the mutator locus, such that the unlinked scenario seems most appropriate for sexually reproducing species. </p> <p>On the other hand, under unlinked recombination scenarios, the mutation rate evolved to relatively higher values only under scenarios of 50% beneficial mutations, particularly when the rate of environmental change was fast. When the percentage of beneficial mutations was low (10 and 25%), either there were too few data points [ <a href="#CR2">2</a>] to derive any conclusion, or the population went extinct for all runs when the rate of environmental change was increased. Thus, the likelihood of beneficial mutations occurring under scenarios of rapid climate change becomes of high importance for organisms with limited plasticity and standing genetic variation, in order to allow for evolutionary rescue in specialist species. These results are in contrast with those in Cobben et al. [ <a href="#CR15">15</a>] where the mutation rate facilitated range expansion and evolved towards higher values even under the assumption of 90% lethal mutations. In our model, given that we implemented the percentage of beneficial mutations <i>bm</i> directly into the distribution of fitness effects of mutations, the negative effects of mutations (especially when <i>bm</i> was low) became stronger, the higher the rate of directional environmental change. Under these conditions, elevated mutation rates only evolved in scenarios of high percentage of beneficial mutations. This emphasizes potential outcome differences in studies focused on the evolution of the mutation rate depending on the implementation methods for deleterious mutations. Since our method is consistent with the concept of slightly deleterious mutations [ <a href="#CR16">16</a>, <a href="#CR17">17</a>], our simulation of beneficial mutations is presumably more realistic than using a gaussian distribution centered on zero, typically for IBMs of explicit genetics (e.g., [ <a href="#CR2">2</a>, <a href="#CR15">15</a>, <a href="#CR18">18</a>]) under scenarios of directional selection. According to empirical data, however, most mutations are negative, some are neutral, and only few are beneficial [ <a href="#CR5">5</a>]. The common assumption in IBM eco-evolutionary models of 50% beneficial mutations is therefore unrealistic and may overestimate evolutionary rescue (or invasion speed) under directional climate change scenarios. We interpret our results as indication that implementation of beneficial mutation percentages more in line with reality may preclude evolution of elevated mutation rates in sexual species, unless there is strong linkage between mutator and trait locus. </p> <p>According to our model, the evolution of the mutation rate may be possible in asexual unicellular organisms, provided that inherited differences in the function of repair and replication mechanisms exist which could serve as "mutator". In sexual unicellular organisms the mutation rate can evolve if the organisms experience high rate of environmental change and the probability of occurring beneficial mutations is high, or if mutator genes happen to occur close to genome sequences coding for the evolving trait, such that they can hitchhike with beneficial mutations at other loci. In multicellular organisms, the mutator locus is expected to be present and affect germ and soma alike. This was not considered in our model and represents a model limitation. In these organisms an increase in genomic mutation rate may lead to tissue damage and reduced survival. In the model, the mutation rate affected only the process of inheritance of the traits into the next generations. Therefore, the mutation rate may still evolve in multicellular organisms as suggested by the model if the effects of increased mutation rates compromise survival after reproduction takes place [ <a href="#CR15">15</a>], or unless the mutation rate can be somehow increased for segments of the genome (e.g., traits, locus, germ-line). The later scenario may not be consistent with the mutator locus approach as implemented here (e.g., a locus encoding for a DNA polymerase). However, the immune system of vertebrates is an example of high frequency mutations locally restricted to few genes and cell types in multicellular organisms and that has evolved in response to the constant need of novel mutations in the arm race between host and pathogens [ <a href="#CR14">14</a>, <a href="#CR19">19</a>]. </p> <p>Our findings that mutation rates evolve to higher values only with unrealistically high percentages of beneficial mutations or complete linkage seem to be independent from some characteristics of the environmental change, as it was replicated for the different levels of environmental stochasticity or noise color. This supports and expands the observation of no systematic variation in the mutation rate among asexual organisms experiencing different habitats [ <a href="#CR14">14</a>, <a href="#CR20">20</a>], to sexual species. Thus, the mutation rate is expected to be determined by "deep general forces" balancing the deleterious effects of mutations and the physiological cost of further reducing mutation rates [ <a href="#CR20">20</a>], and not by external properties of the habitat. However, there are environmental conditions (e.g., chemicals, radiation) that produce DNA damage or modify the chemistry of enzymes potentially affecting replication fidelity and promoting mutations [ <a href="#CR21">21</a>]. Such conditions lead to non-adaptive elevations of the mutation rate. Another potential source of variation in the mutation rate is the difference in condition between individuals and populations. Increased mutation rates could occur through costs of DNA repair in maladapted individuals/populations experiencing a lag-load to the moving environmental optimum under environmental change. It has been shown that environmental and genetic stress compromise DNA repair mechanisms and therefore, causes individuals to pass on a greater mutation load to their offspring [ <a href="#CR9">9</a>, <a href="#CR22">22</a>, <a href="#CR23">23</a>]. The phenotypic condition can be directly related to the ability of individuals of overcoming the physiological cost of high-fidelity replication [ <a href="#CR8">8</a>]. Thus, individuals able to pay the cost of fidelity are expected to reduce the mutation rate down to the drift limit, as further reductions will be effectively neutral [ <a href="#CR14">14</a>]. Those not able to pay the cost are expected to experience relatively elevated values of the mutation rate [ <a href="#CR8">8</a>, <a href="#CR14">14</a>]. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>In conclusion, considering that 50% beneficial mutations may be an unrealistic assumption, and that recombination is ubiquitous in a sexual species, results in this study suggest that, it is unlikely for the mutation rate to evolve to elevated values in a sexual species experiencing directional climate change scenarios. Instead, mutation rate will be under stabilizing selection at the minimal value allowed by limitations by costs of replication fidelity or limitations imposed by the drift limit, as already proposed in the literature [ <a href="#CR13">13</a>]. Furthermore and in line with Drake’s observation [ <a href="#CR20">20</a>], our results indicate that the mutation rate can be expected to be relatively similar among organisms experiencing different habitats. Though the frequency of beneficial mutations remains an elusive quantity [ <a href="#CR5">5</a>], empirical estimations of beneficial mutations in <i>E. coli</i> seem to be far below the lowest scenario investigated in this model [ <a href="#CR24">24</a>]. If this observation applies to sexual species as well, the evolution of the mutation rate towards elevated values under sexual recombination will become even less likely. It is important to consider that conclusions derived from this study apply under the assumption of a mutator locus affecting replication fidelity. The presence and action of other mechanisms affecting the mutation rate (e.g., epigenetics mechanisms) may affect the results as reported in this study [ <a href="#CR14">14</a>, <a href="#CR19">19</a>]. Therefore, when the percentage of beneficial mutations is small, and populations are not large enough, sexual species (especially multicellular ones) producing few offspring may be expected to buffer their ability for local adaptation mainly through standing genetic variation and plasticity, provided that movement opportunities are constrained. Future work should focus on understanding the potential role of standing genetic variation, polygenic selection, epigenetics and phenotypic plasticity in the ability for local adaptation of sexual species under scenarios of directional climate change and when the probability of beneficial mutations is low. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>In order to investigate the evolution of the mutation rate under directional climate change scenarios, we designed a spatially implicit individual-based model (IBM) of a panmictic diploid population of a sexual species with non-overlapping generations experiencing directional climate change (a trend of the mean climatic variable such as temperature). Our model is based on two previous IBMs: Björklund et al. [ <a href="#CR25">25</a>] for the simulation of environmental scenarios (including noise color) and the density dependence effect on fecundity; and Cobben et al. [ <a href="#CR15">15</a>], for the simulation of explicit genetics on the inheritance of mutator and adaptation loci. The population was assumed to be geographically isolated, thus no migration was possible, such that the focus was on local adaptation. The mutation rate could evolve, and different scenarios of linkage disequilibrium and beneficial mutations were considered. </p> <div class="environment" title="sec"> <div class="title" tagx="title" title="title">
Environment</div> <p>The environment was stochastic and defined an optimum mean phenotype <i>θ</i> <sub> <i>t</i> </sub> that moved at constant speed per generation. This environmental scenario has been considered best suit to investigate the effect of climate change [ <a href="#CR18">18</a>, <a href="#CR26">26</a>]. Thus, <i>θ</i> <sub> <i>t</i> </sub> <i>= θ</i> <sub> <i>0</i> </sub> <i>+ η t</i> impose the directional climate trend, where <i>θ</i> <sub> <i>0</i> </sub> <i>= 0</i> was the initial environmental optimum (when <i>t = 0</i>) and <i>η</i>, the rate of environmental change. The parameter <i>η</i> was changed to simulate different scenarios of environmental change (e.g., no change, slow, medium, rapid change). </p> <p>Environmental stochasticity or noise color was implemented as follows: The parameter <i>θ</i> <sup> <i>*</i> </sup> <sub> <i>t</i> </sub> <i>= θ</i> <sub> <i>t</i> </sub> <i>+ ϕ</i> <sub> <i>t</i> </sub>, was the realized environmental state with noise <i>ϕ</i> <sub> <i>t</i> </sub> <i>= αϕ</i> <sub> <i>t-1</i> </sub> <i>+ βξ</i> <sub> <i>t</i> </sub>. The autocorrelation coefficient <i>α</i> indicated the level of environmental correlation and therefore the noise color: <i>− 1 &amp;lt; α &amp;lt; 0</i>, blue noise; <i>α = 0</i>, white noise, and <i>0 &amp;lt; α &amp;lt; 1</i>, red noise. Three scenarios of <i>α</i> were considered, based on Björklund et al. [ <a href="#CR25">25</a>]: blue noise ( <i>α = − 0.7</i>); white noise ( <i>α = 0</i>), and red noise ( <i>α = 0.7</i>). The parameter <i>β</i> determined the environmental variance, according to <i>β = σ</i> <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \sqrt{1-{\alpha}^2} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , as in [ <a href="#CR27">27</a>], where <i>σ</i> <sup> <i>2</i> </sup> <i>= 1</i> was the environmental variance. The parameter <i>ξ</i> <sub> <i>t</i> </sub> was a random value, normally distributed with zero mean and unity of variance. <p /> </div> <div class="populationdynamics" title="sec"> <div class="title" tagx="title" title="title">
Population dynamics</div> <p>ndividuals in the population were characterized by the following traits: sex, stage (whether adult or juvenile), phenotype <i>zi</i>, determined by the alleles at the adaptation locus, and a mutator locus whose alleles determine the genetic mutation rate. The phenotype and the mutator locus were considered evolving traits for the model. At the beginning of each simulation run, the population composed of 1000 individuals, at carrying capacity <i>K</i>, was assumed to be locally adapted. Therefore the alleles coding for the phenotype were initialized randomly from a normal distribution centered in <i>θ</i> <sub> <i>0</i> </sub> and variance <i>V = VG / 2 L</i>, where <i>VG</i> is the initial genetic variance present in the population, and <i>L</i> the number of loci which was set to 1 for all runs in this study. The phenotype <i>z</i> of individual <i>i</i> was determined by additive effects of alleles at the adaptation locus. As in Cobben et al. [ <a href="#CR15">15</a>] the mutation rate was given by <i>μ = 10</i> <sup> <i>-exp</i> </sup>, where <i>exp = (l</i> <sub> <i>m,1</i> </sub> <i>+ l</i> <sub> <i>m,2</i> </sub> <i>) / 2</i>. The alleles at the mutator locus <i>l</i> <sub> <i>m,1</i> </sub> and <i>l</i> <sub> <i>m,2</i> </sub> were initialized randomly according to a discrete uniform distribution in range <i>[2; 4]</i>. Therefore, the model’s initial conditions already incorporated intraspecific variation in the mutation rate between individuals in the population (initial mean mutation rate of <i>10</i> <sup> <i>− 3</i> </sup>). Assuming initial intraspecific variation in the mutation rate facilitates its evolutionary dynamic and reduces an otherwise relatively high extinction risk during early simulation time. </p> </div> <div class="degreeoflocaladaptation" title="sec"> <div class="title" tagx="title" title="title">
Degree of local adaptation</div> <p>Each individual was tested on its ability for local adaptation given by the match of its phenotype <i>zi</i> to the current environmental optimum <i>θ</i> <sup> <i>*</i> </sup> <sub> <i>t</i> </sub>. Stabilizing selection was assumed according to: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ wi={e}^{\frac{-{\left( zi-{\theta}^{\ast }t\right)}^2}{2{\upgamma}^2}} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
wi</div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
zi</div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>The variable <i>wi</i> indicates the degree of local adaptation of individual <i>i</i>, and <i>γ</i>, the strength of selection, affected the width of the fitness function, and was set to 2.2. This means that individuals having a phenotype departing from the optimum in <i>1SD</i> phenotypic units ( <i>SD</i>: standard deviation) will have a fitness of 90% (relative to the maximum fitness) which would be moderate selection according to [ <a href="#CR25">25</a>]. </p> </div> <div class="fecundity" title="sec"> <div class="title" tagx="title" title="title">
Fecundity</div> <p>The mating system was random mating, where females mated only once and males could repeat mating (lottery polygyny). The fecundity <i>λ</i> of the reproductive couple was the sum of the scaled degree of local adaptation values <i>w’i</i> after considering density dependence effects, as in Björklund et al. [ <a href="#CR25">25</a>]: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ w^{\prime }i={wie}^{\left(1-\frac{N}{K}\right)} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
′</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
wie</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
K</div> </div> </div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>where <i>N</i> is the population size and <i>K</i>, the carrying capacity. Each couple (partners <i>i</i>, <i>j</i>) produced a number of offspring randomly drawn from a Poisson distribution with expectancy <i>λ = w’i + w’j.</i> <p /> </div> <div class="inheritance" title="sec"> <div class="title" tagx="title" title="title">
Inheritance</div> <p>Two scenarios of recombination were considered: unlinked and complete linkage (Table <a href="#Tab1">1</a>). During the process of inheritance, allele values for each locus (adaptation locus and mutator locus) of the inherited haplotype were picked randomly from the corresponding parental locus (unlinked scenario). In complete linkage scenario, alleles at the mutator locus were linked and migrated together with the corresponding alleles at the adaptation locus. For comparison, we also implemented an intermediate linkage (probability of recombination <i>pR = 0.5</i>). Then, mutations took place with probability <i>μ</i> determined by the alleles at the mutator locus as explained above. In case that a mutation occurred, its effect for the adaptation locus was randomly drawn from a normal distribution with zero mean and variance equals to the variance of the distribution of fitness effect size of mutations, which was an input parameter [ <a href="#CR18">18</a>]. The assumption of a Gaussian distribution is consistent with analysis of mutation effects [ <a href="#CR28">28</a>, <a href="#CR29">29</a>]. In the model, the mean <i>x</i> of the distribution of mutation effect size could change according to the scenario of percentage of beneficial mutations (input parameter, Table <a href="#Tab1">1</a>). This applied for the adaptation locus only. This approach granted that beneficial mutations – mutations that pushed the trait in the direction of <i>θ</i> <sub> <i>t</i> </sub> – actually occurred at different probabilities as shown in the distribution of mutations fitness effect size (Fig. <a href="#Fig3">3</a>). This can be important since under scenarios of directional selection the common assumption in individual-based models of explicit genetics of symmetric distribution of beneficial and deleterious mutations can overestimate the amount of arising beneficial mutations (e.g., [ <a href="#CR12">12</a>, <a href="#CR17">17</a>]). In fact, the shifted distributions approximates the mutational effects according to the model of slightly deleterious mutations [ <a href="#CR16">16</a>, <a href="#CR17">17</a>], albeit mutational effects under this model often exhibit skewed distributions. When a mutation occurred at the mutator locus, its effect consisted in adding a value randomly drawn from [− 1, 0, 1] to the mutated allele. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Parameters values and description (values in parentheses were implemented to evaluate the robustness of outcomes) </p>  <table> <thead> <tr> <th>Parameter</th> <th>Value</th> <th>Description</th> </tr> </thead> <tbody> <tr> <td> <i>K</i> </td> <td>1000</td> <td>Carrying capacity</td> </tr> <tr> <td> <i>γ</i> </td> <td>2.2 (3.2)</td> <td>Strength of selection</td> </tr> <tr> <td> <i>σ</i> <sup> <i>2</i> </sup> </td> <td>1</td> <td>Variance of the stochastic environment</td> </tr> <tr> <td> <i>θ</i> <sub> <i>0</i> </sub> </td> <td>0</td> <td>Initial environmental optimum</td> </tr> <tr> <td> <i>η</i> </td> <td>0, 0.01, 0.02, 0.03, 0.04</td> <td>Rate of environmental change</td> </tr> <tr> <td> <i>z</i> </td> <td>Evolving trait</td> <td>Ecological phenotype</td> </tr> <tr> <td> <i>VG</i> </td> <td>0.2</td> <td>Initial genetic variance present in the population</td> </tr> <tr> <td> <i>L</i> </td> <td>Number of loci per evolving trait</td> <td>1</td> </tr> <tr> <td> <i>μ</i> </td> <td>Evolving trait, range [0; 1]</td> <td>Mutation rate</td> </tr> <tr> <td> <i>MV</i> </td> <td>0.2</td> <td>Variance of the distribution of mutations fitness effect size</td> </tr> <tr> <td> <i>bm</i> </td> <td>10, 25, 40, 50</td> <td>Percentage (%) of beneficial mutations</td> </tr> <tr> <td> <i>pR</i> </td> <td>0 (unlinked), (0.5), 1 (complete linkage),</td> <td>Probability of recombination</td> </tr> <tr> <td> <i>t</i> </td> <td>300</td> <td>Time limit per simulation, in generations. The last 200 generations were exposed to the treatment of directional climate change </td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Distribution of fitness effects of mutations according to different scenarios of percentage of beneficial mutations <i>bm</i>. Beneficial mutations are shown in light grey color. Variance of the distribution <i>MV = 0.1</i> </p>   </div> <p /> <p>After the process of inheritance, all adults died, and newborns took over the population (non-overlapping generations). </p> </div> <div class="methodforbeneficialmutations" title="sec"> <div class="title" tagx="title" title="title">
Method for beneficial mutations</div> <p>The mean <i>x</i> of the distribution of mutation effect size was given by <i>x = ε – y</i> <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \sqrt{MV} $$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN">
MV</div> </div> </div> <span class="inline-graphic" title="inline-graphic" /> </div> , where <i>ε = 0</i> was the Z-score cutting value between deleterious and beneficial mutations. The parameter <i>MV</i> was the variance of the distribution, and <i>y</i> was given by the quantile function of the normal distribution with probability <i>p = 1 – q</i>. The parameter <i>q</i> was the desired proportion of beneficial mutation (e.g., 0.25 for 25% scenario of beneficial mutations). <p /> </div> <div class="simulationexperiments" title="sec"> <div class="title" tagx="title" title="title">
Simulation experiments</div> <p>To study the evolution of the mutation rate, different scenarios of rate of environmental change, stochastic noise color, probability of recombination, and percentage of beneficial mutations were implemented (Table <a href="#Tab1">1</a>). The scenarios of beneficial mutations were implemented only for the environmental conditions <i>η &amp;gt; 0</i> (directional climate change). For the stable environment ( <i>η = 0</i>), the effect of mutations was drawn from a gaussian distribution centered in zero as it is commonly done in IBMs of explicit genetics [ <a href="#CR2">2</a>, <a href="#CR4">4</a>, <a href="#CR15">15</a>] and acted as a control. Note that under directional environmental change, this is exactly our scenario of 50% beneficial mutations. Simulation experiments consisted of 200 replicates, lasting for 300 generations each: 100 generations of stable environment followed by 200 generations of treatment of directional climate change. Simulations were run under white stochastic environmental noise unless otherwise specified. The sequence of operations in the model were: update of the environmental optimum, degree of local adaptation, reproduction, mortality of adults, and update of phenotype of the new generation before repeating the loop. If extinctions occurred, the data was not used for the analysis. </p> <p>The visualization of the data was done in r project v3.4.2 and the model was programmed in Netlogo v6.0.2. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional file</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12862_2019_1494_MOESM1_ESM.docx">LINK</a> <span class="label" tagx="label" title="label">Additional file 1:</span>  <p>Additional supporting data. (DOCX 756 kb)</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>bm</i> </div> <div class="def" title="def"> <p>Percentage of beneficial mutations</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>exp</i> </div> <div class="def" title="def"> <p>Exponent value</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IBM</div> <div class="def" title="def"> <p>Individual-based model</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>K</i> </div> <div class="def" title="def"> <p>Carrying capacity</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>L</i> </div> <div class="def" title="def"> <p>Number of loci</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>l</i> <sub> <i>m,1</i> </sub> and <i>l</i> <sub> <i>m,2</i> </sub> </div> <div class="def" title="def"> <p>Alleles at the mutator locus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>MV</i> </div> <div class="def" title="def"> <p>Variance of the distribution of mutations effects</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>N</i> </div> <div class="def" title="def"> <p>Population size</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>p</i> </div> <div class="def" title="def"> <p>Probability of non-beneficial mutations</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>pR</i> </div> <div class="def" title="def"> <p>Probability of recombination;</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>q</i> </div> <div class="def" title="def"> <p>Probability of beneficial mutations</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>SD</i> </div> <div class="def" title="def"> <p>Standard deviation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
stable env</div> <div class="def" title="def"> <p>Stable environment (no directional environmental change)</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>t</i> </div> <div class="def" title="def"> <p>Time in generations</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>V</i> </div> <div class="def" title="def"> <p>Phenotypic variance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>VG</i> </div> <div class="def" title="def"> <p>Initial genetic variance present in the population;</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>w’i</i> </div> <div class="def" title="def"> <p>Scaled degree of local adaptation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>wi</i> </div> <div class="def" title="def"> <p>Degree of local adaptation (fitness proxy)</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>x</i> </div> <div class="def" title="def"> <p>Mean of the distribution of mutations effects</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>y</i> </div> <div class="def" title="def"> <p>Quantile function of the normal distribution</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>zi</i> </div> <div class="def" title="def"> <p>Phenotype of the <i>i-</i>th individual </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>α</i> </div> <div class="def" title="def"> <p>Autocorrelation coefficient</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>β</i> </div> <div class="def" title="def"> <p>Adjusted environmental variance;</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>γ</i> </div> <div class="def" title="def"> <p>Strength of selection</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>ε</i> </div> <div class="def" title="def"> <p>Z-score cutting value between deleterious and beneficial mutations</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>η</i> </div> <div class="def" title="def"> <p>Rate of directional environmental change</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>θ</i> <sup> <i>*</i> </sup> <sub> <i>t</i> </sub> </div> <div class="def" title="def"> <p>Realized environmental state after considering stochasticity</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>θ</i> <sub> <i>0</i> </sub> </div> <div class="def" title="def"> <p>Initial environmental optimum</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>θt</i> </div> <div class="def" title="def"> <p>Optimum mean phenotype</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>λ</i> </div> <div class="def" title="def"> <p>Fecundity</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>μ</i> </div> <div class="def" title="def"> <p>Mutation rate</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>ξ</i> <sub> <i>t</i> </sub> </div> <div class="def" title="def"> <p>Random normally distributed variable</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>σ</i> <sup> <i>2</i> </sup> </div> <div class="def" title="def"> <p>Environmental variance</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Publisher’s Note</b> </p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Ulrike Schlägel for her support in the development of the method for beneficial mutations based on the distribution of mutations effects, and the reviewers for their valuable comments and feedback. We acknowledge the support of the Deutsche Forschungsgemeinschaft and Open Access Publishing Fund of University of Potsdam. </p> </div> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>The idea of this study was conceived by RT and DRM. The model was designed and implemented by DRM. The same author ran the experiments and analyzed the data. RT, FJ and DRM wrote the manuscript. All authors read and approved the final manuscript. </p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was funded by the University of Potsdam. The funding body had no role in the design of the study, the collection, analysis, interpretation of data nor in the writing of the manuscript. </p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data generated or analyzed during this study are available at <a href="https://github.com/danielrm84?tab=repositories">https://github.com/danielrm84?tab=repositories</a>. </p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>Not applicable</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p>Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Futuyma DJ, Edwards SV, True JR. Evolution. Sinauer Associates; 2005. (Evolution). Available from: <a href="https://books.google.de/books?id=rRGNQgAACAAJ">https://books.google.de/books?id=rRGNQgAACAAJ</a> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution and extinction in a changing environment: a quantitative-genetic analysis</span> <span class="source" tagx="source" title="source">Evolution.</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">151</span> <span class="lpage" tagx="lpage" title="lpage">163</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1558-5646.1995.tb05967.x">10.1111/j.1558-5646.1995.tb05967.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28593664">28593664</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferrière</span> <span class="given-names" tagx="given-names" title="given-names">Régis</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dieckmann</span> <span class="given-names" tagx="given-names" title="given-names">Ulf</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Couvet</span> <span class="given-names" tagx="given-names" title="given-names">Denis</span> </span> </span> <span class="source" tagx="source" title="source">Evolutionary Conservation Biology</span> <span class="year" tagx="year" title="year">2004</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romero-Mujalli</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeltsch</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiedemann</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Individual-based modeling of eco-evolutionary dynamics: state of the art and future directions </span> <span class="source" tagx="source" title="source">Reg Environ Chang</span> <span class="year" tagx="year" title="year">2019</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">12</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10113-018-1406-7">10.1007/s10113-018-1406-7</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eyre-Walker</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keightley</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The distribution of fitness effects of new mutations</span> <span class="source" tagx="source" title="source">Nat Rev Genet.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="fpage" tagx="fpage" title="fpage">610</span> <span class="lpage" tagx="lpage" title="lpage">618</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg2146">10.1038/nrg2146</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17637733">17637733</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drake</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charlesworth</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charlesworth</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crow</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rates of spontaneous mutation</span> <span class="source" tagx="source" title="source">Genetics.</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">148</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">1667</span> <span class="lpage" tagx="lpage" title="lpage">1686</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9560386">9560386</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baer</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyamoto</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Denver</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mutation rate variation in multicellular eukaryotes: causes and consequences</span> <span class="source" tagx="source" title="source">Nat Rev Genet</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">619</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg2158">10.1038/nrg2158</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17637734">17637734</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sniegowski</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerrish</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaver</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The evolution of mutation rates: separating causes from consequences</span> <span class="source" tagx="source" title="source">BioEssays.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">1057</span> <span class="lpage" tagx="lpage" title="lpage">1066</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/1521-1878(200012)22:12&lt;1057::AID-BIES3&gt;3.0.CO;2-W">10.1002/1521-1878(200012)22:12&amp;lt;1057::AID-BIES3&amp;gt;3.0.CO;2-W</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11084621">11084621</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharp</span> <span class="given-names" tagx="given-names" title="given-names">NP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agrawal</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evidence for elevated mutation rates in low-quality genotypes</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">109</span> <span class="issue" tagx="issue" title="issue">16</span> <span class="fpage" tagx="fpage" title="fpage">6142</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1118918109">10.1073/pnas.1118918109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22451943">22451943</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leigh</span> <span class="given-names" tagx="given-names" title="given-names">EG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Natural selection and mutability</span> <span class="source" tagx="source" title="source">Am Nat</span> <span class="year" tagx="year" title="year">1970</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="issue" tagx="issue" title="issue">937</span> <span class="fpage" tagx="fpage" title="fpage">301</span> <span class="lpage" tagx="lpage" title="lpage">305</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1086/282663">10.1086/282663</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Beneficial mutations, hitchhiking and the evolution of mutation rates in sexual populations</span> <span class="source" tagx="source" title="source">Genetics.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">151</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">1621</span> <span class="lpage" tagx="lpage" title="lpage">1631</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10101182">10101182</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaver</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dombrowski</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sweeney</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Treis</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zappala</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sniegowski</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fitness evolution and the rise of mutator alleles in experimental Escherichia coli populations </span> <span class="source" tagx="source" title="source">Genetics.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">162</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">557</span> <span class="lpage" tagx="lpage" title="lpage">566</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12399371">12399371</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The lower bound to the evolution of mutation rates</span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">1107</span> <span class="lpage" tagx="lpage" title="lpage">1118</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evr066">10.1093/gbe/evr066</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21821597">21821597</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martincorena</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luscombe</span> <span class="given-names" tagx="given-names" title="given-names">NM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Non-random mutation: the evolution of targeted hypermutation and hypomutation</span> <span class="source" tagx="source" title="source">BioEssays.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">123</span> <span class="lpage" tagx="lpage" title="lpage">130</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bies.201200150">10.1002/bies.201200150</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23281172">23281172</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cobben</span> <span class="given-names" tagx="given-names" title="given-names">MMP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitesser</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kubisch</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolving mutation rate advances the invasion speed of a sexual species</span> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">150</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12862-017-0998-8">10.1186/s12862-017-0998-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28651517">28651517</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohta</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Slightly deleterious mutant substitutions in evolution</span> <span class="source" tagx="source" title="source">Nature.</span> <span class="year" tagx="year" title="year">1973</span> <span class="volume" tagx="volume" title="volume">246</span> <span class="fpage" tagx="fpage" title="fpage">96</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/246096a0">10.1038/246096a0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4585855">4585855</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eyre-Walker</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keightley</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">NGC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaffney</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantifying the slightly deleterious mutation model of molecular evolution</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">2142</span> <span class="lpage" tagx="lpage" title="lpage">2149</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a004039">10.1093/oxfordjournals.molbev.a004039</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12446806">12446806</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vincenzi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Extinction risk and eco-evolutionary dynamics in a variable environment with increasing frequency of extreme events </span> <span class="source" tagx="source" title="source">J R Soc Interface</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">97</span> <span class="fpage" tagx="fpage" title="fpage">20140441</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1098/rsif.2014.0441">10.1098/rsif.2014.0441</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24920116">24920116</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Odegard</span> <span class="given-names" tagx="given-names" title="given-names">VH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schatz</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Targeting of somatic hypermutation</span> <span class="source" tagx="source" title="source">Nat Rev Immunol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">573</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nri1896">10.1038/nri1896</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16868548">16868548</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drake</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A constant rate of spontaneous mutation in DNA-based microbes</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="year" tagx="year" title="year">1991</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="issue" tagx="issue" title="issue">16</span> <span class="fpage" tagx="fpage" title="fpage">7160</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.88.16.7160">10.1073/pnas.88.16.7160</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1831267">1831267</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guengerich</span> <span class="given-names" tagx="given-names" title="given-names">FP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mechanisms of mutagenesis: DNA replication in the presence of DNA damage</span> <span class="source" tagx="source" title="source">Mutat Res Rev Mutat Res</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">768</span> <span class="fpage" tagx="fpage" title="fpage">53</span> <span class="lpage" tagx="lpage" title="lpage">67</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.mrrev.2016.03.006">10.1016/j.mrrev.2016.03.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27234563">27234563</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agrawal</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic loads under fitness-dependent mutation rates</span> <span class="source" tagx="source" title="source">J Evol Biol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">1004</span> <span class="lpage" tagx="lpage" title="lpage">1010</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1420-9101.2002.00464.x">10.1046/j.1420-9101.2002.00464.x</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berger</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stångberg</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grieshop</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martinossi-Allibert</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnqvist</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Temperature effects on life-history trade-offs, germline maintenance and mutation rate under simulated climate warming </span> <span class="source" tagx="source" title="source">Proc Biol Sci</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">284</span> <span class="issue" tagx="issue" title="issue">1866</span> <span class="fpage" tagx="fpage" title="fpage">20171721</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1098/rspb.2017.1721">10.1098/rspb.2017.1721</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29118134">29118134</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Imhof</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schlötterer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fitness effects of advantageous mutations in evolving Escherichia coli populations</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">98</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">1113</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.98.3.1113">10.1073/pnas.98.3.1113</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11158603">11158603</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Björklund</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ranta</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaitala</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bach</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lundberg</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">NChr</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantitative trait evolution and environmental change</span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">e4521</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0004521">10.1371/journal.pone.0004521</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19229330">19229330</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kopp</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matuszewski</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid evolution of quantitative traits: theoretical perspectives</span> <span class="source" tagx="source" title="source">Evol Appl</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">169</span> <span class="lpage" tagx="lpage" title="lpage">191</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/eva.12127">10.1111/eva.12127</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24454555">24454555</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwager</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johst</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeltsch</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Does red noise increase or decrease extinction risk? Single extreme events versus series of unfavorable conditions </span> <span class="source" tagx="source" title="source">Am Nat</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">167</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">879</span> <span class="lpage" tagx="lpage" title="lpage">888</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1086/503609">10.1086/503609</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16615033">16615033</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lynch M, Walsh B. Genetics and analysis of quantitative traits: Sinauer; 1998. Available from: <a href="https://books.google.de/books?id=UhCCQgAACAAJ">https://books.google.de/books?id=UhCCQgAACAAJ</a> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lenormand</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodnight</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The fitness effect of mutations across environments: a survey in light of fitness landscape models </span> <span class="source" tagx="source" title="source">Evolution.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">60</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">2413</span> <span class="lpage" tagx="lpage" title="lpage">2427</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.0014-3820.2006.tb01878.x">10.1111/j.0014-3820.2006.tb01878.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17263105">17263105</a> </span> </span> </li> </ul> </div> </div>  </body></html>