<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4853719">4853719</a> </span> <span class="pii" title="pii">srep24041</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/srep24041">10.1038/srep24041</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Contributions of historical and contemporary geographic and environmental factors to phylogeographic structure in a Tertiary relict species, <i>Emmenopterys henryi</i> (Rubiaceae) </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Yong-Hua</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Ian J.</span> </span> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Comes</span> <span class="given-names" tagx="given-names" title="given-names">Hans Peter</span> </span> <a href="#a3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">Hua</span> </span> <a href="#a4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">Ying-Xiong</span> </span> <a href="#c1">a</a> <a href="#a1">1</a> </span> <span class="citation_author_institution" id="a1">[1], <span class="institution" title="institution">Key Laboratory of Conservation Biology for Endangered Wildlife of the Ministry of Education, and College of Life Sciences, Zhejiang University </span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="a2">[2], <span class="institution" title="institution">Department of Environmental Science, Policy, and Management, University of California</span> <span class="country" tagx="country" title="country">USA</span> </span> <span class="citation_author_institution" id="a3">[3], <span class="institution" title="institution">Department of Ecology &amp;amp; Evolution, Salzburg University</span> <span class="country" tagx="country" title="country">Austria</span> </span> <span class="citation_author_institution" id="a4">[4], <span class="institution" title="institution">Laboratory of Biodiversity and Biogeography, Kunming Institute of Botany, Chinese Academy of Sciences </span> <span class="country" tagx="country" title="country">China</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">a</span> <span class="email" tagx="email" title="email">qyxhero@zju.edu.cn</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-05-05</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">24041</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-10-20</span> <span class="accepted" title="accepted">accepted: 2016-03-21</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Macmillan Publishers Limited</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Examining how historical and contemporary geographic and environmental factors contribute to genetic divergence at different evolutionary scales is a central yet largely unexplored question in ecology and evolution. Here, we examine this key question by investigating how environmental and geographic factors across different epochs have driven genetic divergence at deeper (phylogeographic) and shallower (landscape genetic) evolutionary scales in the Chinese Tertiary relict tree <i>Emmenopterys henryi</i>. We found that geography played a predominant role at all levels – phylogeographic clades are broadly geographically structured, the deepest levels of divergence are associated with major geological or pre-Quaternary climatic events, and isolation by distance (IBD) primarily explained population genetic structure. However, environmental factors are clearly also important – climatic fluctuations since the Last Interglacial (LIG) have likely contributed to phylogeographic structure, and the population genetic structure (in our AFLP dataset) was partly explained by isolation by environment (IBE), which may have resulted from natural selection in environments with divergent climates. Thus, historical and contemporary geography and historical and contemporary environments have all shaped patterns of genetic structure in <i>E. henryi</i>, and, in fact, changes in the landscape through time have also been critical factors. </p> </div> </div> </div> <div class="body" title="body"> <p>Understanding the contemporary and historical ecological (climatic, geographical) factors shaping population genetic diversity, structure, and divergence is of great interest to molecular ecology, evolutionary biology and conservation biology <a href="#b1">1</a> <a href="#b2">2</a> <a href="#b3">3</a>. Populations separated by physical geographical barriers (including geographic distance) may diverge under any combination of natural selection and random genetic drift resulting from reduced gene flow and population connectivity <a href="#b4">4</a> <a href="#b5">5</a>. In the absence of extrinsic (e.g. physical) barriers to gene flow, population divergence may still occur when reproductive isolation evolves between populations as a result of ecologically-based divergent selection in different environments <a href="#b5">5</a> <a href="#b6">6</a> <a href="#b7">7</a> <a href="#b8">8</a> <a href="#b9">9</a> <a href="#b10">10</a>. Thus, population genetic divergence can result from both geographical and environmental factors (including climate and soil, among others). Disentangling the roles of geographic and environmental forces in driving genetic structure during certain periods (usually contemporary) has seen a large body of research in both plants and animals over recent years <a href="#b4">4</a> <a href="#b11">11</a> <a href="#b12">12</a> <a href="#b13">13</a> <a href="#b14">14</a> <a href="#b15">15</a>. However, few studies have examined how these factors contribute to genetic structure through time, including both historical and contemporary geographic and environmental factors, to better understand how changing climates and geographic landscape features can influence patterns of genetic structure observed presently <a href="#b16">16</a> <a href="#b17">17</a>. </p> <p>For example, climatic fluctuations during the Quaternary which resulted in population isolation in multiple refugia are considered major drivers of population divergence and broad phylogeographic patterns <a href="#b18">18</a> <a href="#b19">19</a> <a href="#b20">20</a>. Of course, major geographic barriers, like oceans, rivers, and mountains, are also recognized as key drivers of biogeo-graphic structure <a href="#b18">18</a>, and thus, both environmental and geographic factors can contribute to genetic divergence at deeper evolutionary scales. Likewise, geography and the environment have also been recognized as critical factors underlying genetic differentiation over shorter evolutionary scales, like the evolution of population genetic structure among contemporary populations on a landscape <a href="#b5">5</a> <a href="#b21">21</a> <a href="#b22">22</a>. </p> <p>So, how do historical and contemporary geographic and environmental factors contribute to genetic divergence at different evolutionary scales? This is a central yet largely unexplored question in ecology and evolution <a href="#b16">16</a> <a href="#b17">17</a>. In general, the flora of subtropical (Central-Southeast) China presents some excellent systems for such studies, including several genera of Tertiary relict trees that have inhabited topographically and ecologically heterogeneous environments (in terms of climate, soil, etc.) for millions of years <a href="#b23">23</a> <a href="#b24">24</a> <a href="#b25">25</a> <a href="#b26">26</a>. These genera (e.g. <i>Cathaya</i>, <i>Ginkgo</i>, <i>Metasequoia</i>, <i>Davidia</i>, <i>Emmenopterys</i>) are thought to represent remnants of the so-called ‘boreotropical flora’ that likely formed a belt of vegetation around the Northern Hemisphere during the Early Tertiary/Eocene <a href="#b27">27</a> <a href="#b28">28</a> <a href="#b29">29</a>. Here, we examine this key question by investigating how environmental and geographic factors across different epochs have driven genetic divergence at deeper (phylogeographic) and shallower (landscape genetic) evolutionary scales in the Chinese flowering tree <i>Emmenopterys henryi</i> Oliv. (Rubiaceae). </p> <p> <i>Emmenopterys henryi</i> is a particular suitable species for addressing these issues. This deciduous tree, which is the only extant species of its genus, is native to subtropical China, where it occurs in disjunct montane valleys of mainly warm-temperate deciduous (WTD) forests, at elevations ranging from <i>c</i>. 400–1600 (2000) m above sea level <a href="#b30">30</a> ( <a href="#f1">Fig. 1A</a>). Landscape characteristics, climatic conditions, and soil types vary between regions within the distribution range of <i>E. henryi</i> <a href="#b31">31</a>. Well-preserved infructescences of now-extinct <i>Emmenopterys</i> species are known from the Eocene of North America and Germany <a href="#b32">32</a>, but there are no reliable fossils of <i>E. henryi</i>. However, there is dated molecular evidence to suggest that the origin of this species dates back to the Early Miocene <a href="#b33">33</a>. Previous results based on inter-simple sequence repeat (ISSR) markers indicate that <i>E. henryi</i> exhibits significant population genetic structure and divergence <a href="#b34">34</a>; however it remains unclear whether this is the result of long-term geographical barriers to gene flow, ecologically-based divergent selection, or recent habitat fragmentation. </p> <p>In this study, we integrate genetic markers that capture signatures from historical (chloroplast DNA) and contemporary (amplified fragment length polymorphisms; AFLPs) divergence <a href="#b35">35</a>, ecological niche modelling (ENM), and spatial genetic modelling approaches to disentangle the relative roles of geography, climate, and ecology in shaping the population genetic structure of <i>E. henryi</i> across subtropical China. Our main objectives were to: (i) estimate the timing and pattern of divergence among populations of <i>E. henryi</i>; (ii) investigate how climatic and geographical variation over space and time explain patterns of phylogeographic and population genetic structure; and (iii) explore the specific environmental variables that may underlie local adaptation through natural selection in divergent environments. </p> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="cpdnaanditsphylogeographyanddiversity" title="sec"> <div class="title" tagx="title" title="title">
cpDNA and ITS phylogeography and diversity</div> <p>The three cpDNA-IGS regions surveyed across the 443 individuals of <i>E. henryi</i> were aligned along a total length of 2163 bp with 26 single-site mutations (including two 1-bp indels), 18 length polymorphisms (2–78 bp) and one inversion (27 bp) observed ( <a href="#S1">Table S5</a>). A total of 40 haplotypes (‘chlorotypes’; H1–40) were detected in the 38 <i>E. henryi</i> populations across subtropical China ( <a href="#S1">Table S1</a>; <a href="#f1">Fig. 1A</a>). Of the 40 haplotypes, 20 were shared by at least two populations while the other 20 haplotypes were only found in a single population ( <a href="#S1">Table S1</a>; <a href="#f1">Fig. 1A</a>). The most common haplotypes were H12 (found in 10 populations with a frequency of 0.25), H2 (17.5% of all populations), H28 (15%), H14 (12.5%), and H6 (10%). Total haplotype diversity ( <i>h</i> <sub>T</sub>) was estimated to be 0.928 and within-population diversity ( <i>h</i> <sub>S</sub>) was 0.332 ( <a href="#S1">Table S1</a>). Regression analyses showed that <i>h</i> <sub>S</sub> was not dependent on longitude or latitude ( <i>P</i> = 0.23, 0.66, respectively), but <i>π</i> <sub>s</sub> was weakly related to latitude ( <i>R</i> <sup> <i>2</i> </sup> = 0.116, <i>P</i> = 0.036). </p> <p>The Bayesian haplotype tree from <div class="sc_UNKNOWN">
beast</div> supported the monophyly of <i>E. henryi</i> [posterior probability ( <i>PP</i>) = 1], and two main (‘Northern’ vs. ‘Southern’) lineages were recognized with weak support ( <i>PP</i> = 0.57, 0.70, respectively; <a href="#f1">Fig. 1B</a>). The haplotypes of the Northern lineage were located in the northern part of the species’ range, except for H12 from populations S17 and S18, while those of the Southern lineage were present only in southern populations ( <a href="#f1">Fig. 1A</a>). In the haplotype network, the two major lineages are also recognized ( <a href="#f1">Fig. 1B,C</a>). Most haplotypes close to each other in the phylogeny and haplotype network tended to occur in nearby populations ( <a href="#f1">Fig. 1</a>). In the <div class="sc_UNKNOWN">
samova</div> analysis, <i>F</i> <sub>CT</sub> values increased progressively as the value of <i>K</i> increased from 2 to 26. However, with <i>K</i> ranging between 5 and 26, <i>F</i> <sub>CT</sub> values did not increase significantly, and in most cases the newly defined groups comprised single populations ( <a href="#S1">Fig. S1</a>). Thus, we retained the configuration of <i>K</i> = 4 ( <i>F</i> <sub>CT</sub> = 0.523). The four cpDNA groups identified are populations S1–S6 in the east (Southeast group), S7–S23 in the middle (Central-Southwest group), N1–N6 in the northeast (Northeast group), and N7–N15 in the northwest (Northwest group). This grouping is mostly consistent with the phylogenetic analyses ( <a href="#f1">Fig. 1B,C</a>). </p> <p>Non-hierarchical AMOVA ( <a href="#t1">Table 1</a>) revealed a strong population genetic structure for cpDNA sequence variation at the species level ( <i>Φ</i> <sub>ST</sub> = 0.779; <i>P </i>&amp;lt; 0.001). Hierarchical AMOVA, however, revealed that of the total genetic variation, 36.48% was distributed between the Northern lineage and the Southern lineage ( <i>Φ</i> <sub>CT</sub> = 0.365), 45.43% was explained by variation among populations within regions ( <i>Φ</i> <sub>SC</sub> = 0.715), and only 18.10% was found within populations ( <i>Φ</i> <sub>ST</sub> = 0.819; <a href="#t1">Table 1</a>). Nevertheless, there was greater population subdivision in the Southern lineage ( <i>Φ</i> <sub>ST</sub> = 0.823) when compared to the Northern lineage ( <i>Φ</i> <sub>ST</sub> = 0.551; <a href="#t1">Table 1</a>). Moreover, significant phylogeographic structure for cpDNA was observed at the range-wide scale ( <i>G</i> <sub>ST</sub>/ <i>N</i> <sub>ST</sub> = 0.704/0.802, <i>P </i>&amp;lt; 0.01) and in the Southern lineage ( <i>G</i> <sub>ST</sub>/ <i>N</i> <sub>ST</sub> = 0.757/0.868, <i>P </i>&amp;lt; 0.05), but no such structure was found in the Northern lineage ( <i>G</i> <sub>ST</sub>/ <i>N</i> <sub>ST</sub> = 0.521/0.379, <i>P</i> &amp;gt; 0.05; <a href="#t1">Table 1</a>). Finally, Mantel tests uncovered a strong pattern of IBD for cpDNA both at the range-wide scale ( <i>r</i> = 0.237, <i>P</i> = 0.001) and the region scale (Northern lineage: <i>r</i> = 0.506, <i>P</i> = 0.001; Southern lineage: <i>r</i> = 0.293, <i>P</i> = 0.001). </p> <p>The ITS sequences of 212 individuals (38 populations) of <i>E. henryi</i> were aligned with a total of length of 767 bp, exhibiting 10 nucleotide substitutions (ITS-1: 4; ITS-2: 6; <a href="#S1">Table S6</a>). Together, these 10 polymorphic sites identified nine ITS haplotypes (‘ribotypes’, R1–9; <a href="#S1">Table S6</a>). Of those, five ribotypes were specific to the Southern cpDNA lineage (R2–6) and three to the Northern cpDNA lineage (R7–9; <a href="#f2">Fig. 2A</a>). These lineage-specific ribotypes formed separate <div class="sc_UNKNOWN">
tcs</div> clades, except for the shared ribotype R1 ( <a href="#f2">Fig. 2B</a>). </p> </div> <div class="moleculardatingandhistoricaldemographybasedoncpdnasequencevariation" title="sec"> <div class="title" tagx="title" title="title">
Molecular dating and historical demography based on cpDNA sequence variation</div> <p>Based on the assumption that <i>E. henryi</i> and <i>P. bracteata</i> diverged at <i>c</i>. 22 Ma (Manns <i>et al</i>. <a href="#b33">33</a>; <a href="#f1">Fig. 1B</a>; node 1), the time since divergence between the Northern and Southern lineages was estimated by <div class="sc_UNKNOWN">
beast</div> analysis to have occurred 5.06 Ma (95% HPD: 1.68–8.91 Ma; node 2). The onset of lineage diversification was estimated as 3.42 Ma (Northern lineage, 95% HPD: 0.99–6.40 Ma; node 4) and 3.64 Ma (Southern lineage, 95% HPD: 1.18–6.42 Ma; node 3). For this cpDNA-IGS chronogram, we recovered an average substitution rate of 6.225 × 10 <sup>−10</sup> s/s/y by the <div class="sc_UNKNOWN">
beast</div> analysis, which is much lower than the average values generally reported for noncoding regions of the chloroplast genome (e.g. 1.2–1.7 × 10 <sup>−9</sup> s/s/y) <a href="#b36">36</a>; but in accordance with that in woody taxa and⁄or phylogenetic relicts (viz. ‘living fossils’) <a href="#b37">37</a> <a href="#b38">38</a> <a href="#b39">39</a> <a href="#b40">40</a>. </p> <p>For the four cpDNA population groups of <i>E. henryi</i>, at least one of the estimates of Tajima’s <i>D</i> and Fu’s <i>F</i> <sub>S</sub> was generally significant, except for the Central-Southwest group ( <a href="#t2">Table 2</a>). In contrast, almost all four groups failed to reject the spatial expansion model ( <i>SSD</i>, <i>H</i> <sub>Rag</sub> values <i>P</i> &amp;gt; 0.05), with the exception of the Northeast group ( <i>P </i>&amp;lt; 0.05; <a href="#t2">Table 2</a>). In addition, the mismatch distributions of the Southeast, Northeast and Northwest groups were unimodal, except for the Central-Southwest group ( <a href="#S1">Fig. S3</a>). Consequently, the Southeast, Northeast and Northwest groups were found to fit to the distributions expected under a spatial expansion model. Based on the corresponding <i>τ</i> values, and assuming a substitution rate of 6.225 × 10 <sup>−10</sup> s/s/y (see above), we dated the three spatial expansions to the last glacial cycle(s) (Southeast: <i>c</i>. 0.23 Ma, 95% CI: 0.000–0.811 Ma; Northwest: <i>c</i>. 0.19 Ma, 95% CI: 0.111–0.314 Ma; Northeast: <i>c</i>. 0.26 Ma, 95% CI: 0.133–0.376 Ma; <a href="#t2">Table 2</a>). </p> </div> <div class="aflpanalysis" title="sec"> <div class="title" tagx="title" title="title">
AFLP analysis</div> <p>The nine primer combinations employed with samples from 37 populations (394 individuals) of <i>E. henryi</i> generated a total of 457 fragments, of which 431 (94.31%) were polymorphic. Different primer pairs amplified variable numbers of fragments, from 38 to 69, with an average of 50.8 ± 10.1 fragments per primer combination ( <a href="#S1">Table S3</a>). The percentage of polymorphic fragments varied among primer pairs from 88.89 to 98.55% ( <a href="#S1">Table S3</a>). Because of the high degree of polymorphism, these primer combinations distinguished all 394 individuals as separate phenotypes. AFLP variation within populations (in terms of <i>PPF</i>, <i>I</i>, <i>H</i> <sub>E</sub>, <i>DW</i>) varied widely ( <a href="#S1">Table S1</a>). The highest genetic diversity was observed in a northern population (N12), with <i>PPF</i> = 74.18%, <i>I</i> = 0.243, and <i>H</i> <sub>E</sub> = 0.162. By contrast, genetic diversity was lowest in a southern population (S22), with <i>PPF</i> = 57.77%, <i>I</i> = 0.035, and <i>H</i> <sub>E</sub> = 0.023. Regression analyses showed that <i>H</i> <sub>E</sub> was weakly related to latitude ( <i>R</i> <sup>2</sup> = 0.110, <i>P</i> = 0.045). </p> <p>The partition with the highest log marginal likelihood (spatial: -51313.01, non-spatial: -51225.49) produced by <div class="sc_UNKNOWN">
baps</div> identified nine different clusters ( <a href="#S1">Fig. S2A</a>). All populations from the Northern and two (S17, S18) from the Southern (cpDNA) lineage formed one cluster, while the remaining populations from the Southern lineage were further divided into eight clusters ( <a href="#f3">Fig. 3A,B</a>). In the PCoA-plot, the first axis (42.20% of the variation) and the third axis (12.84%) indicated that the main split in the dataset was between the central-southeastern and the northern samples, while the second axis (16.10%) separated the southwestern samples from the northern group ( <a href="#f3">Fig. 3C</a>). The NJ analysis based on genetic distances among populations also identified three regional groups, again comprising populations from the central-southeastern region (67% bootstrap support), southwestern region (92%), and northern region (&amp;lt;50%; <a href="#S1">Fig. S2B</a>). With few exceptions, S16 and S17 clustered with the northern group. Populations from central-southeastern and southwestern China showed higher among-population divergence when compared with those from within the northern group ( <a href="#f3">Figs 3C</a> and <a href="#S1">S2B</a>). Non-hierarchical AMOVA indicated high overall levels of population differentiation for AFLPs in <i>E. henryi</i> ( <i>Φ</i> <sub>ST</sub> = 0.344; <a href="#t1">Table 1</a>). When accounting for the species’ significant hierarchical (regional) substructure ( <i>Φ</i> <sub>CT</sub> = 0.061), levels of population subdivision still remained high ( <i>Φ</i> <sub>SC</sub> = 0.364), but were markedly higher in the Southern lineage ( <i>Φ</i> <sub>ST</sub> = 0.416) than in the Northern one ( <i>Φ</i> <sub>ST</sub> = 0.175; <a href="#t1">Table 1</a>). </p> </div> <div class="ecologicalnichemodellingacrosstemporalscales" title="sec"> <div class="title" tagx="title" title="title">
Ecological niche modelling across temporal scales</div> <p>The <div class="sc_UNKNOWN">
maxent</div> model for <i>E. henryi</i> had high predictive power and did not overfit the presence data (AUC = 0.804 ± 0.003). The current distributional predictions ( <a href="#f4">Fig. 4A</a>) were accurate representations of the species’ extant distribution, except for some predicted areas where the species does not occur at present (e.g. southeastern Qinghai–Tibetan Plateau and Taiwan). Palaeodistribution modeling suggested more restricted ranges of the species during the Last Interglacial (LIG) compared with its current distribution, particularly in north-central China (e.g. northern Sichuan Basin and Daba/Qinling Mts.; <a href="#f4">Fig. 4B</a>), with subsequent expansion at the Last Glacial Maximum (LGM) to cover a slightly greater area than that predicted under current climatic conditions ( <a href="#f4">Fig. 4A,C</a>). However, ENMs for the future (2080) predict that climate change will result in a reduction of the species’ potential range in China. Most evident is a loss of suitable habitat in areas south of the Yangtze Delta, where only small and disjunct mountain areas are predicted as suitable ( <a href="#f4">Fig. 4D</a>). </p> </div> <div class="ibdandibe" title="sec"> <div class="title" tagx="title" title="title">
IBD and IBE</div> <p>The structural equation modelling (SEM) and multiple matrix regression with randomization (MMRR) analyses both revealed significant effects of IBD and IBE on neutral (non-outlier) AFLP divergence ( <i>F</i> <sub>ST</sub>) in this species. In both analyses, IBD explained more of the variation in genetic differentiation than IBE (SEM: ≈2 times as much; MMRR: ≈1.25 times as much; <a href="#t3">Table 3</a>). SEM identified significant pairwise relationships among environmental distance, geographic distance, and genetic distance ( <i>P </i>&amp;lt; 0.01) and estimated that IBD (β <sub>D</sub> = 0.360 ± 0.037) contributed about twice as much as IBE (β <sub>E</sub> = 0.181 ± 0.151) to genetic distance. Similarly, MMRR also estimated the contribution of IBD (β <sub>D</sub> = 0.307, <i>P </i>&amp;lt; 0.001) to be larger than that of IBE (β <sub>E</sub> = 0.247, <i>P</i> = 0.002). Each analysis detected low levels of covariation between geographical and environmental distance (0.133 ± 0.028 in SEM and 0.488 in MMRR), although the covariation based on the MMRR analysis was approaching the reliability limit of 0.5 ( <a href="#t3">Table 3</a>). The difference between these estimates from each analysis is due to the different environmental distance matrices used, estimated as Euclidian distances among all observed environmental variables in MMRR and as a latent variable constructed from the observed environmental variables in SEM. </p> <p>For the SEM analysis, we also quantified the contributions of the individual environmental variables to the composition of the environmental dissimilarity latent variable ( <a href="#S1">Table S7</a>). The results revealed that <div class="sc_UNKNOWN">
bio</div>4 (temperature seasonality) and <div class="sc_UNKNOWN">
bio</div>7 (temperature annual range) were the primary contributors, followed by <div class="sc_UNKNOWN">
bio</div>1 (annual mean temperature), <div class="sc_UNKNOWN">
bio</div>5 (max temperature of warmest month), <div class="sc_UNKNOWN">
bio</div>9 (mean temperature of driest quarter), and <div class="sc_UNKNOWN">
bio</div>15 (precipitation seasonality). By contrast, slope, soil, and <div class="sc_UNKNOWN">
bio</div>12 (annual precipitation) had very minor contributions. </p> </div> <div class="detectingpotentiallociunderselection:outlierlocitestandmlranalysis" title="sec"> <div class="title" tagx="title" title="title">
Detecting potential loci under selection: outlier loci test and MLR analysis</div> <p>Using <div class="sc_UNKNOWN">
fdist</div>, 67 of 457 loci (14.66%) showed high probability (99.5%) for divergent selection among the nine <div class="sc_UNKNOWN">
baps</div> groups, including 21 loci under directional selection and 46 loci under balancing selection ( <a href="#f5">Fig. 5A</a>). Additionally, we identified 16 outlier loci in <div class="sc_UNKNOWN">
bayescan</div> (3.50% of all 457 loci; <a href="#f5">Fig. 5B</a>). However, only six loci were identified by both <div class="sc_UNKNOWN">
fdist</div> and <div class="sc_UNKNOWN">
bayescan</div> ( <a href="#f5">Fig. 5</a>). Eventually, four potential loci under selection (L128, L144, L294, and L305) were confirmed by the MLR analysis with <span class="inline-formula" title="inline-formula"> <span class="inline-graphic" title="inline-graphic" /> </span> &amp;gt; 0.5 ( <a href="#S1">Table S8</a>). When we ran linear regressions using each environmental variable individually, all six loci were significantly ( <i>P </i>&amp;lt; 0.05) associated with at least two of the eight selected environmental variables ( <a href="#S1">Table S7</a>). Among the environmental variables, <div class="sc_UNKNOWN">
bio</div>2 (mean diurnal temperature range), <div class="sc_UNKNOWN">
bio</div>4 (temperature seasonality), <div class="sc_UNKNOWN">
bio</div>5 (maximum temperature of the warmest month), <div class="sc_UNKNOWN">
bio</div>12 (annual precipitation) and <div class="sc_UNKNOWN">
bio15</div> (precipitation seasonality) were most highly associated with the potential loci under selection. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The results of our phylogeographic and landscape genetic analyses reveal how historical and contemporary environmental and geographic factors have all contributed to presently observed patterns of genetic divergence in <i>E. henryi</i>. At broad scales, the results of our cpDNA and ITS phylogeography show that genetic variation is, largely, geographically structured. In the cpDNA phylogeography, we found two major lineages corresponding to the Northern and Southern populations (with minor exceptions). The boundary between these lineages is located in the region of the Yangtze River and next to the Three Gorges Mountain Region (TGMR; <a href="#f1">Figs 1</a>, <a href="#f2">2</a>, <a href="#f3">3</a>). We found evidence of admixture of Northern lineage cpDNA haplotypes into the Southern lineage (S16, S17, S18), which is most likely explained by both introgression following secondary contact during glacial periods and incomplete lineage sorting due to recent, postglacial divergence. Nuclear ITS ribotype data did not show such a distinct division between the Northern and Southern populations but did, nevertheless, indicate geographically structured ribotype frequency differences between these sets of populations and the presence of ribotypes found only in the Northern and Southern lineages. In fact, of the nine ribotypes we detected, only one was shared between lineages (R1), which was found in many populations, including those around the Yungui Plateau ( <a href="#f2">Fig. 2</a>). Several ribotypes and chlorotypes are also unique to Southeastern populations, where the climate is warmest and wettest, or to the Northernmost populations, where the climate is coolest and driest. Whether these haplotypes and populations sharing similar haplotype frequencies are clustered in these areas because they are environmentally divergent from the intermediate climate of central China, where the most haplotype diversity is found, or because of geographical restrictions to the mountain chains in these regions is still unclear. Nevertheless, it appears that major geographic regions harbor phylogeographic structure, but that geographic distances and major geographic barriers (like the Yangtze River) are not always major barriers to genetic introgression. For example, there is cpDNA evidence to suggest that two populations from the Northern lineage (N6 and N15 in the Qinling Mts.) were shaped by dispersal events from a southern population (S12 in central China), as a chlorotype (H1) unique to this latter population differs from its derived haplotype (H26) in the Qinling Mts. by only one step ( <a href="#f1">Fig. 1A,C</a>; <a href="#S1">Table S1</a>). These populations are separated by several hundred kilometers, suggesting that very long distance dispersal may be possible in this species, that these populations have some shared ancestry, or that they may have been more widespread and come in contact in the past, possibly during the LGM when there was much more suitable habitat available to <i>E. henryi</i> ( <a href="#f4">Fig. 4</a>). Hence, the observed phylogeographic patterns may also reflect historical factors, particularly relating to the expansion of suitable habitat during the LGM from heavily fragmented habitat during the LIG, followed by the restriction, again, of suitable habitat leading to the present disjunct montane distribution of <i>E. henryi</i> populations ( <a href="#f4">Fig. 4</a>). </p> <p>Our ENM analysis through time suggested that <i>E. henryi</i> populations likely experienced cycles of expansion and retraction into and out of local refugia, and Quaternary climatic fluctuations may have played a role in generating phylogeographic structure as well <a href="#b26">26</a> <a href="#b41">41</a> <a href="#b42">42</a>. In fact, the earliest diversification events in <i>E. henryi</i> are associated with major geological and climatic events. The estimated divergence time between the two major lineages (Northern and Southern) of <i>E. henryi</i>, at approximately the Mio-/Pliocene boundary [ <i>c</i>. 5.06 Ma (1.68–8.91) Ma; see node 2 in <a href="#f1">Fig. 1B</a>], coincides with global climate cooling during the Late Miocene/Early Pliocene <a href="#b43">43</a> <a href="#b44">44</a>. This cooling is hypothesized to have been a key trigger of aridification in East Asia along with stronger winter and/or weaker summer monsoon circulations <a href="#b45">45</a> <a href="#b46">46</a>. Concurrently, the mid-Pliocene abrupt uplift of the eastern Tibetan Plateau ( <i>c</i>. 3.4 Ma) <a href="#b46">46</a> <a href="#b47">47</a> induced dramatic geomorphological changes in Southwest China. These climatic and geological changes may have triggered early lineage diversification in <i>E. henryi</i> through habitat fragmentation and the formation of physical barriers to gene flow <a href="#b46">46</a>, which we see reflected in early branching events within each of the major Northern and Southern lineages beginning around 3.64 to 3.42 Ma (see nodes 3 and 4 in <a href="#f1">Fig. 1B</a>). Such tectonic/climate-induced vicariance has also been invoked to explain similar patterns of north-south differentiation in other forest tree species from subtropical China (e.g. <i>Taxus wallichiana</i> <a href="#b48">48</a>, <i>Cercidiphyllum japonicum</i> <a href="#b39">39</a>, <i>Kalopanax septemlobus</i> <a href="#b49">49</a>). Expansion of potential suitable habitat of <i>E. henryi</i> during the last glaciations, indeed, is supported by our ENM analysis, which indicates larger distribution ranges of this lineage at the LGM compared to both the LIG and present ( <a href="#f4">Fig. 4A–C</a>). </p> <p>In addition to the interplay between geographic and climatic factors, population demography and colonization history also likely contributed to the observed pattern of phylogeographic structure. For instance, our phylogeographic and ENM analyses are consistent with the relatively recent expansion of <i>E. henryi</i> into the northernmost regions (Northwest and Northeast groups) of its range. These groups (particular the Northeast group; N1–6) harbor lower genetic diversity ( <a href="#S1">Fig. S3</a>) and the most derived haplotypes found in our <i>E. henryi</i> phylogeny ( <a href="#f1">Fig. 1B</a>) and <div class="sc_UNKNOWN">
tcs</div> networks ( <a href="#f1">Fig. 1C</a>), which both suggest populations founded following range expansion. Additionally, the MDA indicated a spatial (and demographic) expansion within the Northwest and Northeast groups at <i>c</i>. 0.19 Ma (95% CI: 0.111–0.314 Ma) and <i>c</i>. 0.26 Ma (95% CI: 0.133–0.376 Ma; <a href="#t2">Table 2</a>), respectively, possibly coinciding with the penultimate (Riss) glacial ( <i>c</i>. 0.12–0.35 Ma), although, due to the broad confidence intervals, this interpretation requires some caution. In fact, the species is not predicted to have occurred in the northern Sichuan Basin and the Daba/Qinling Mts. during the LIG ( <a href="#f4">Fig. 4B</a>), i.e. when temperatures were at least 5 °C higher than at present <a href="#b47">47</a>. Accordingly, we hypothesize that the climate might have warmed enough during the LIG to extirpate <i>E. henryi</i> from the above regions, followed by recolonization via northward expansion during the LGM ( <a href="#f4">Fig. 4C</a>). Notably, such a significant impact of the LIG has also been detected for other plants and animals in subtropical China (e.g. <i>Pinus kwangtungensis</i> <a href="#b50">50</a>, <i>Aegithalos concinnus</i> <a href="#b51">51</a>, <i>Parus monticolus</i> <a href="#b52">52</a>). </p> <p>The Southeast group may also carry signatures of expansion – these populations also have reduced haplotype diversity ( <a href="#S1">Fig. S3A</a>) and significantly negative Tajima’s <i>D</i> and Fu’s <i>F</i> <sub>S</sub> statistics ( <a href="#t2">Table 2</a>). This provides strong evidence for a relatively recent spatial (and demographic) expansion in the Tianmu and Wuyi Mts. located southwest of the Yangtze Delta Region (YDR). Although the range of our time estimates for this expansion, again, is broad (approx. 0–0.81 Ma; <a href="#t2">Table 2</a>), the respective point estimate ( <i>c</i>. 0.23 Ma) once more coincides with the penultimate (Riss) glacial (see above). In addition to the ENM and MDA analyses ( <a href="#f4">Figs 4B,C</a> and <a href="#S1">S3A</a>), this expansion scenario is further supported by vegetation reconstructions based on fossil pollen data, which indicate that the YDR still sustained patches of northern peripheral WTD forest in the lowland areas of East China during the LGM <a href="#b53">53</a>. Thus, we again find evidence that Quaternary climatic oscillations across geographic regions of China likely contributed to phylogeographic patterns in this species. </p> <p>Clearly, at broad phylogeographic scales, our results demonstrate distinct geographic genetic structure across the range of <i>E. henryi</i>. At finer spatial and evolutionary scales, our landscape genetic analysis of AFLP genotype data also revealed that geographical isolation played the predominant role in driving genetic divergence of <i>E. henryi</i> populations. However, we also found that environmental/climatic variation also contributed significantly to explaining population genetic structure (SEM: IBD ≈ 36.0 vs. IBE ≈ 18.1%; MMRR: IBD ≈ 30.72 vs. IBE ≈ 24.67%). Thus, we found evidence of both isolation by distance (IBD) <a href="#b21">21</a> and isolation by environment (IBE) <a href="#b5">5</a> in our population genetic dataset. IBD could be explained by geographical distance, extensive restriction and fragmentation of WTD forests <a href="#b54">54</a>, physical barriers to gene flow (like the Yangtze River), or increasing fragmentation from other land uses in subtropical China (such as forest, agriculture and residential), all of which are commonly barriers to gene flow in plants <a href="#b3">3</a>. Patterns of IBE often result from divergent selection between different environments; however, environmental factors can also shape gene flow through other processes (e.g. environmental differences affecting phenological differences among populations) <a href="#b55">55</a>. Thus, IBE may not be due solely to selection but could also be explained by other diverse mechanisms associated with environmental factors <a href="#b5">5</a>. Hence, multiple processes, associated with both geographic and environmental variables, all could have effectively disrupted gene flow between <i>E. henryi</i> populations, and broad-scale geographic and climatic factors appear to have shaped the spatial distribution of genetic variation in this species at both deeper evolutionary and more recent ecological time scales. </p> <p>To examine the potential mechanism driving the pattern of IBE, we tested for loci under selection, in our AFLP dataset, and their associations with environmental/climatic variables. In identifying outlier AFLP loci, we sought to determine how selection might play a role in shaping genetic differentiation of the nine <div class="sc_UNKNOWN">
baps</div> clusters of <i>E. henryi</i> along environmental clines. All four loci identified by both <div class="sc_UNKNOWN">
fdist</div> and <div class="sc_UNKNOWN">
bayescan</div> as undergoing putative diversifying selection ( <a href="#f5">Fig. 5</a>) were associated with environmental predictors across environmental gradients ( <a href="#S1">Table S7</a>), suggesting these regions of the genome are diverging and that climate may play a role. As expected, temperature and precipitation were estimated as the major driving factors influencing allele frequencies at outlier loci, consistent with other studies examining drivers of adaptive genetic divergence in plants <a href="#b56">56</a>. These variables were also identified as important drivers of neutral genetic divergence by our SEM analysis (specifically <div class="sc_UNKNOWN">
bio</div>5, maximum temperature of the warmest month, and <div class="sc_UNKNOWN">
bio15</div>, precipitation seasonality). Altogether, these results are consistent with population genetic divergence under natural selection in divergent environments, suggesting this may have been the process generating IBE in this system. </p> <p>In conclusion, by using phylogeographic, landscape genetic, and ecological niche modeling analysis together, we were able to identify the many factors, both historical and contemporary, that have shaped spatial genetic structure in this Tertiary relict species. At various spatial and evolutionary scales, we found that both geographical and environmental/climatic factors contribute to patterns of genetic structure. Geography played a predominant role at all levels – phylogeographic clades are broadly geographically structured, the deepest levels of divergence are associated with major geological or pre-Quaternary climatic events, and IBD primarily explained population genetic structure. However, environmental factors are clearly also important – climatic fluctuations since the LIG have likely contributed to phylogeographic structure, and the population genetic structure (in our AFLP dataset) was partly explained by IBE, which may have resulted from natural selection in environments with divergent climates. Thus, historical and contemporary geography and historical and contemporary environments have all shaped patterns of genetic structure in <i>E. henryi</i>, and, in fact, changes in the landscape through time have also been critical factors. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialandsamplingdesign" title="sec"> <div class="title" tagx="title" title="title">
Plant material and sampling design</div> <p>We obtained silica-dried leaf material from 38 populations of <i>Emmenopterys henryi</i> throughout its range ( <a href="#S1">Table S1</a>, <a href="#f1">Fig. 1A</a>; see <a href="#S1">Supplementary Method S1</a> for more details of the study species). In each population, representative samples of 10–20 plants were taken, resulting in a total of 433 individuals. Total genomic DNA was extracted from the dried leaf tissue using a DNeasy plant tissue kit (Qiagen). All samples were sequenced at three intergenic spacer (IGS) regions of chloroplast DNA (cpDNA), while a subset of individuals ( <i>n</i> = 212) was also sequenced at the entire internal transcribed spacer (ITS) region of nuclear ribosomal DNA (nrDNA). Of these 433 individuals, 394 (representing 37 populations; population S23 with 1 individual was excluded) were surveyed for AFLPs ( <a href="#S1">Table S1</a>; <a href="#f1">Fig. 1A</a>). <i>Pinckneya bracteata</i> (Bartram) Raf., collected from the JC Raulston Arboretum (Raleigh, NC, USA), was selected as an outgroup for the phylogenetic analyses based on a previous molecular phylogenetic study of Rubiaceae <a href="#b33">33</a>. Voucher specimens of this species and all sampled populations of <i>E. henryi</i> are stored at the Herbarium of Zhejiang University (HZU; Hangzhou, Zhejiang, China). </p> </div> <div class="dnaextraction,dnasequencingandaflpfingerprinting" title="sec"> <div class="title" tagx="title" title="title">
DNA extraction, DNA sequencing and AFLP fingerprinting</div> <p>For the phylogeographic DNA analyses, we sequenced three cpDNA-IGS regions ( <i>psb</i>A– <i>trn</i>H, <i>trn</i>L– <i>trn</i>F, <i>trn</i>T– <i>trn</i>L) and the ITS region (i.e. ITS1 + 5.8S + ITS2). The primers and methodology for amplification of these four DNA regions were described in Shaw <i>et al</i>. <a href="#b57">57</a> and Gielly <i>et al</i>. <a href="#b58">58</a>. Sequences were generated on an ABI 377XL DNA sequencer and were edited, assembled, and aligned in <div class="sc_UNKNOWN">
geneious</div> v.4.8.5 <a href="#b59">59</a>. All sequences were deposited in GenBank (see <a href="#S1">Table S2</a> for accession numbers). </p> <p>The AFLP protocol followed the procedure described by Vos <i>et al</i>. <a href="#b60">60</a>, with minor modifications that included the use of fluorescent-dye-labeled primers (Applied Biosystems, Foster City, California, USA) for selective amplification in multiplex analysis. Selective primer pairs were initially screened on 50 individuals from 25 populations. Of the 64 primer pair combinations tested, nine pairs that gave the best results with respect to polymorphism and clarity of AFLP profiles ( <a href="#S1">Table S3</a>) were chosen for the full survey (further details in <a href="#S1">Supplementary Method S2</a>). We also performed four separate test runs with 15 individuals and each chosen primer combination. These test runs confirmed that AFLP fragments were reproducible, resulting in an average error rate of 0.21% (±0.07) per fragment (see also Knowles &amp;amp; Richards <a href="#b61">61</a>). The AFLP data matrix is available in <a href="#S1">Supplementary Dataset</a>. </p> </div> <div class="phylogeographicalandpopulationgeneticdataanalyses" title="sec"> <div class="title" tagx="title" title="title">
Phylogeographical and population genetic data analyses</div> <p>For both cpDNA and ITS, haplotype ( <i>h</i>) and nucleotide ( <i>π</i>) diversities were calculated for each population and the species overall using <div class="sc_UNKNOWN">
dnasp</div> v.5.10 <a href="#b62">62</a>. Genealogical relationships of the haplotypes identified were inferred from a 95% statistical parsimony network constructed in <div class="sc_UNKNOWN">
tcs</div> v.1.21 <a href="#b63">63</a>, with gaps (indels) coded as substitutions (A or T). For cpDNA, the permutation test implemented in <div class="sc_UNKNOWN">
permut</div> was employed to compare parameters of population differentiation with unordered and ordered alleles ( <i>G</i> <sub>ST</sub> and <i>N</i> <sub>ST</sub>, respectively) based on 1000 random permutations <a href="#b64">64</a>. Spatial analysis of molecular variance (SAMOVA), as implemented in <div class="sc_UNKNOWN">
samova</div> v.1.0 <a href="#b65">65</a>, was used to identify regional groups of populations that are geographically homogeneous and maximally differentiated from each other. For cpDNA data, isolation by distance (IBD) effects <a href="#b21">21</a> were assessed by regressions of <i>F</i> <sub>ST</sub>/(1 −  <i>F</i> <sub>ST</sub>) against the logarithm (log <sub>10</sub>) of geographic distance for all pairs of populations, or subsets thereof, following Rousset <a href="#b66">66</a>. </p> <p>For the AFLP dataset, we calculated the total number of AFLP fragments per population ( <i>F</i> <sub>T</sub>), the percentage of fragments that are polymorphic within each population ( <i>PPF</i>), Nei’s (1973) gene diversity ( <i>H</i> <sub>E</sub>), and Shannon’s information index <i>I</i> following the method of Lynch &amp;amp; Milligan <a href="#b67">67</a> using <div class="sc_UNKNOWN">
aflpsurv</div> v.1.0 <a href="#b68">68</a>. In addition, ‘frequency-down-weighted-marker values’ ( <i>DW</i>; according to Schönswetter &amp;amp; Tribsch <a href="#b69">69</a>), were calculated for each population. To infer the most likely number of population genetic clusters ( <i>K</i>) in the AFLP dataset, we used three approaches. First, we used <div class="sc_UNKNOWN">
baps</div> v.6.0 <a href="#b70">70</a> to detect clusters of genetically similar populations and to estimate individual coefficients of ancestry ( <i>q</i>) with regard to the detected clusters. Second, we utilized <div class="sc_UNKNOWN">
genalex</div> v.6.4 <a href="#b71">71</a> to run a principal coordinates analysis (PCoA), which non-hierarchically grouped the samples without prior knowledge of their source location. Third, we constructed an unrooted neighbour-joining (NJ) tree using the <div class="sc_UNKNOWN">
phylip</div> package 3.6 with 1000 bootstraps (see <a href="#S1">Supplementary Method S3</a> for more details). To quantify variation in cpDNA sequences and AFLPs among populations and genetic clusters (as identified by SAMOVA), we performed analyses of molecular variance (AMOVAs) in <div class="sc_UNKNOWN">
arlequin</div> v.3.5 <a href="#b72">72</a> using <i>Φ</i>- statistics, respectively. The significance of fixation indices was tested using 10,000 permutations <a href="#b73">73</a>. </p> </div> <div class="divergencetimeestimationanddemographicanalysesbasedoncpdnasequences" title="sec"> <div class="title" tagx="title" title="title">
Divergence time estimation and demographic analyses based on cpDNA sequences</div> <p>To relate differentiation among cpDNA haplotypes of <i>E. henryi</i> to pre-Quaternary and Quaternary events, we estimated divergence time under a Bayesian approach as implemented in <div class="sc_UNKNOWN">
beast</div> v.1.8.0 <a href="#b74">74</a>. We estimated the divergence time using an unlinked substitution model ( <i>psb</i>A– <i>trn</i>H: HKY; <i>trn</i>L– <i>trn</i>F: HKY; <i>trn</i>T– <i>trn</i>L: GTR+I) selected by <div class="sc_UNKNOWN">
jmodeltest</div> v. <div class="sc_UNKNOWN">
2.0</div> <a href="#b75">75</a>. Given the intraspecific nature of our data, an uncorrelated lognormal distributed relaxed clock ( <div class="sc_UNKNOWN">
ucld</div>) model, and a coalescent model assuming constant population size were used to model the tree prior. Since no reliable fossils of <i>E. henryi</i> are currently known (see Introduction), we used the divergence time between <i>E. henryi</i> and <i>P. bracteata</i>, inferred from a fossil-calibrated cpDNA phylogeny of Rubiaceae, as a secondary calibration point to calibrate the stem node of <i>E. henryi</i> <a href="#b33">33</a> (node 1 in <a href="#f1">Fig. 1B</a>: mean = 21.77 Ma; SD = 5.0 Ma; 95% CI = 7.92–33.64 Ma). For divergence time estimations, the Markov chain Monte Carlo (MCMC) was run for three independent 50 million generation chains, sampling from the chain every 5000 generations. The output was visualized using <div class="sc_UNKNOWN">
tracer</div> v.1.5 ( <a href="http://tree.bio.ed.ac.uk/software/tracer/">http://tree.bio.ed.ac.uk/software/tracer/</a>) to ensure that parameter values were fluctuating at stable levels. Based on these results, the first 5000 trees were discarded as burn-in, and the remaining samples were summarized as a maximum clade credibility (MCC) tree with mean divergence times and 95% highest posterior density (HPD) intervals of age estimates in <div class="sc_UNKNOWN">
treeannotator</div> v.1.8.0 <a href="#b76">76</a>. Finally, these results were summarized in a single tree visualized in <div class="sc_UNKNOWN">
figtree</div> v.1.3.1 ( <a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>). </p> <p>Tajima’s <i>D</i> <a href="#b77">77</a> and Fu’s <i>F</i> <sub>S</sub> <a href="#b78">78</a> neutrality tests were used to assess population demographic history. We also used <div class="sc_UNKNOWN">
arlequin</div> to infer the demographic history of two main lineages identified (‘Northern’ vs. ’Southern’; see the Results Section) from their combined cpDNA-IGS sequences by mismatch distribution analysis (MDA) (further details in <a href="#S1">Supplementary Method S4</a>). </p> </div> <div class="present,pastandfuturedistributionmodelling" title="sec"> <div class="title" tagx="title" title="title">
Present, past and future distribution modelling</div> <p>Assuming <i>E. henryi</i> has not changed (and will not change) its climatic preference over at least the last glacial/interglacial cycle and into the future <a href="#b79">79</a>, we reconstructed ENMs to determine the potential distribution of <i>E. henryi</i> across its range under present (1950 <i>–</i>2000), past [the Last Glacial Maximum (LGM, <i>c</i>. 21 kya) <a href="#b80">80</a> <a href="#b81">81</a> and the Last Interglacial (LIG, <i>c</i>. 130–114 kya) <a href="#b82">82</a>] and future (2080) climatic conditions using the maximum entropy method implemented in <div class="sc_UNKNOWN">
maxent</div> v.3.2.1. In addition to the 38 distribution records included in this study, 76 herbarium-recorded presences were sourced from the Chinese Virtual Herbarium ( <a href="http://www.cvh.org.cn">www.cvh.org.cn</a>) and the National Specimen Information Infrastructure of China ( <a href="http://www.nsii.org.cn">www.nsii.org.cn</a>). Based on a total of 114 records, a current distribution model was developed using six bioclimatic data layers (annual mean temperature, annual precipitation, precipitation of wettest, driest, warmest and coldest quarter) available from the WorldClim database ( <a href="http://www.worldclim.org">www.worldclim.org</a>) <a href="#b83">83</a> at 2.5 arc-min resolution for the present (1950–2000), assumed to be important for temperate species in East Asia <a href="#b39">39</a> <a href="#b49">49</a>. This restricted dataset was used to avoid including highly correlated variables (data not shown), and thus to prevent potential over-fitting <a href="#b84">84</a>. The established model was then projected onto the set of climatic variables simulated by the Model for Interdisciplinary Research on Climate (MIROC) <a href="#b85">85</a> to infer the extent of suitable habitat during the LGM and the LIG (see above). Future projections for 2080 were performed based on the Canadian Centre for Climate Modelling and Analysis model (CCCMA-CGCM31) under the A2 scenario that were provided by the CIAT downscaled GCM Data Portal ( <a href="http://gisweb.ciat.cgiar.org/GCMPage/">http://gisweb.ciat.cgiar.org/GCMPage/</a>) with 30 arc-second resolution. </p> <p>Model performance was evaluated using receiver operating characteristic (ROC) analyses in <div class="sc_UNKNOWN">
maxent</div>. Values of the area under the ROC curve (AUC) between 0.7 and 0.9 indicate good fit <a href="#b86">86</a>. We modelled the modern distribution 10 times, using different subsets of 70% of the localities to train the model and 30% to test the model, and visually compared AUC scores and jackknife tests of variable importance to assess consistency between runs. </p> </div> <div class="ibdandibeanalyses" title="sec"> <div class="title" tagx="title" title="title">
IBD and IBE analyses</div> <p>To quantify the roles of geography and ecology in spatial genetic divergence, we quantified isolation by distance (IBD) <a href="#b21">21</a> and isolation by environment (IBE) <a href="#b87">87</a> using our AFLP dataset, because multi-locus datasets with highly variable markers are ideal for this purpose <a href="#b5">5</a> <a href="#b88">88</a> <a href="#b89">89</a>. We computed population pairwise genetic distances ( <i>F</i> <sub>ST</sub>) with <div class="sc_UNKNOWN">
arlequin</div> using all putatively neutral (non-outlier) AFLP loci (i.e. 451 loci) described above. We obtained a total of 21 environmental and geographical variables for our study area, including 19 bioclimatic variables with 30 arc-second resolution ( <a href="http://www.bioclim.org">http://www.bioclim.org</a>), a slope layer based on a digital elevation model with 1-km resolution from the USGS EROS database ( <a href="http://eros.usgs.gov">http://eros.usgs.gov</a>), and soil type from the Chinese soil taxonomy record <a href="#b90">90</a>. We used resistance-based distances, to reflect biological connectivity between populations, instead of direct geographical distances <a href="#b89">89</a> (see <a href="#S1">Supplementary Method S5</a> for more details). </p> <p>We then employed two related methods to quantify IBD and IBE: multiple matrix regression with randomization (MMRR) <a href="#b88">88</a> and structural equation modelling (SEM) <a href="#b89">89</a>. Both methods use a series of regression-based analyses to quantify the effects of multiple explanatory variables on a single response variable, in this case genetic distance ( <i>F</i> <sub>ST</sub>). MMRR provides a straightforward method for estimating linear regressions among distance matrices, while SEM allows for flexibility in defining the relationships among variables and utilizes latent variable modelling. For the MMRR analysis, we constructed an environmental distance matrix by taking the population pairwise Euclidian distances across all 21 environmental variables. We then analyzed the effects of environmental distance and geographic distance, as explanatory variables, on genetic distance, as the response variable, using the ‘MMRR’ function in R <a href="#b88">88</a> with 10,000 permutations. For the SEM analysis, we used all 21 environmental variables to construct an environmental distance latent variable, and then quantified the relationships between environmental distance, geographic distance, and genetic distance using the ‘lavaan’ package in R ( <a href="http://cran.r-project.org/web/packages/lavaan/">http://cran.r-project.org/web/packages/lavaan/</a>). Prior to analysis, all environmental, geographic, and genetic distance matrices were standardized. In each case, the analysis was used to test three alternative hypotheses: (1) only geographical distance contributes significantly to genetic distance, (2) only environmental distance contributes significantly to genetic distance, and (3) geographical distance and environmental distance each contribute significantly to genetic distance. We performed both analyses on AFLP pairwise genetic distances ( <i>F</i> <sub>ST</sub>) for all populations. </p> </div> <div class="potentialaflplociunderselection:detectionofoutlierlociandassociationsbetweenallelefrequenciesandenvironmentalvariables" title="sec"> <div class="title" tagx="title" title="title">
Potential AFLP loci under selection: detection of outlier loci and associations between allele frequencies and environmental variables </div> <p>Two different basic approaches allow screening for AFLP loci that are putatively under selection: (i) a genome scan test procedure based on <i>F</i> <sub>ST</sub> comparisons to identify the loci that are significantly different than expected under neutrality and a given demographic model; and (ii) correlative methods that look for associations between genetic variation and environmental variables. The key to using these approaches together is that they identify links between putative loci under selection and individual environmental variables. Hence, our primary purpose for using these tests was to identify the specific environmental variables that are potential drivers of genetic differentiation via divergent natural selection (e.g. IBE resulting from selection in divergent environments). </p> <p>For the genome scan test, both the <div class="sc_UNKNOWN">
fdist</div> <a href="#b91">91</a> and <div class="sc_UNKNOWN">
bayescan</div> <a href="#b92">92</a> approaches were implemented on 457 polymorphic AFLP loci using the nine genetic clusters defined in the <div class="sc_UNKNOWN">
baps</div> analysis ( <i>K</i> = 9). Then, to detect associations between allele frequencies and environmental variables, we used Multiple Linear Regression (MLR) <a href="#b93">93</a> in R v.3.1.1 (R Development Core Team 2011) to identify potential adaptive loci that are under selection from current environmental factors. For outlier loci confirmed by both <div class="sc_UNKNOWN">
fdist</div> and <div class="sc_UNKNOWN">
bayescan</div>, we computed their population pairwise frequencies of AFLP alleles at the 37 sampling sites. We extracted values for the geographical coordinates of each population from 19 bioclimatic GIS data layers with 30 arc-second resolution (1950–2000; <a href="http://www.bioclim.org">http://www.bioclim.org</a>). We then regressed the allele frequencies of the retained outlier loci (dependent variables) on the selected environmental variables (explanatory variables; standardized) using the MLR model (see <a href="#S1">Table S4</a> for the retained allele frequencies and the selected environmental variables). Potential adaptive loci were identified as <span class="inline-formula" title="inline-formula"> <span class="inline-graphic" title="inline-graphic" /> </span> &amp;gt; 0.5 and significantly correlated to at least one explanatory variable <a href="#b13">13</a>. Univariate regressions were then conducted for each variable individually to estimate its significance (see <a href="#S1">Supplementary Method S6</a> for more details). </p> </div> </div> <div class="additionalinformation" title="sec"> <div class="title" tagx="title" title="title">
Additional Information</div> <p> <b>How to cite this article</b>: Zhang, Y.-H. <i>et al</i>. Contributions of historical and contemporary geographic and environmental factors to phylogeographic structure in a Tertiary relict species, <i>Emmenopterys henryi</i> (Rubiaceae). <i>Sci. Rep</i>. <b>6</b>, 24041; doi: 10.1038/srep24041 (2016). </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Information</div>  <div class="media" title="media"> <a href="srep24041-s1.pdf">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Dataset</div>  <div class="media" title="media"> <a href="srep24041-s2.xlsx">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Shi-Liang Zhou for help for help with the AFLP analysis. We are grateful to Sheng-Nan Zhai, Yun-Rui Mao, Na Yuan, Yi Ren, Zhang-Fu Wang, and Deng-Long Ha for help for help with the field survey and leaf collection, and to Dr. Shota Sakaguchi from the University of Tokyo for help with the ENM analysis. This research was supported by the National Science Foundation of China (Grant Nos 31370241 and 31570214) and the International Cooperation and Exchange of the National Natural Science Foundation of China (Grant Nos 31511140095, 31561143015). </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="b1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loveless</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamrick</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Ecological determinants of genetic structure in plant populations</span>. <span class="source" tagx="source" title="source">Annu. Rev. Ecol. Syst.</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">65</span>– <span class="lpage" tagx="lpage" title="lpage">95</span> ( <span class="year" tagx="year" title="year">1984</span>). </span> </li> <li tag="ref"> <a name="b2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lester</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruttenberg</span> <span class="given-names" tagx="given-names" title="given-names">B. I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaines</span> <span class="given-names" tagx="given-names" title="given-names">S. D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kinlan</span> <span class="given-names" tagx="given-names" title="given-names">B. P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The relationship between dispersal ability and geographic range size</span>. <span class="source" tagx="source" title="source">Ecol. Lett.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">745</span>– <span class="lpage" tagx="lpage" title="lpage">758</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17594430">17594430</a> </span> </span> </li> <li tag="ref"> <a name="b3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holderegger</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buehler</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gugerli</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manel</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Landscape genetics of plants</span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">675</span>– <span class="lpage" tagx="lpage" title="lpage">683</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20940103">20940103</a> </span> </span> </li> <li tag="ref"> <a name="b4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nosil</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Ecological speciation and its alternatives</span> in <span class="source" tagx="source" title="source">Ecological speciation.</span> (ed. <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nosil</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>) <span class="fpage" tagx="fpage" title="fpage">3</span>– <span class="lpage" tagx="lpage" title="lpage">21</span> (Oxford University Press Inc., New York, <span class="year" tagx="year" title="year">2012</span>). </span> </li> <li tag="ref"> <a name="b5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">I. J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bradburd</span> <span class="given-names" tagx="given-names" title="given-names">G. S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation by environment</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">5649</span>– <span class="lpage" tagx="lpage" title="lpage">5662</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25256562">25256562</a> </span> </span> </li> <li tag="ref"> <a name="b6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garant</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kruuk</span> <span class="given-names" tagx="given-names" title="given-names">L. E. B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilkin</span> <span class="given-names" tagx="given-names" title="given-names">T. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCleery</span> <span class="given-names" tagx="given-names" title="given-names">R. H.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheldon</span> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution driven by differential dispersal within a wild bird population</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">433</span>, <span class="fpage" tagx="fpage" title="fpage">60</span>– <span class="lpage" tagx="lpage" title="lpage">65</span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15635409">15635409</a> </span> </span> </li> <li tag="ref"> <a name="b7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Postma</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Noordwijk</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene flow maintains a large genetic difference in clutch size at a small spatial scale</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">433</span>, <span class="fpage" tagx="fpage" title="fpage">65</span>– <span class="lpage" tagx="lpage" title="lpage">68</span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15635410">15635410</a> </span> </span> </li> <li tag="ref"> <a name="b8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rundle</span> <span class="given-names" tagx="given-names" title="given-names">H. D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nosil</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Ecological speciation</span>. <span class="source" tagx="source" title="source">Ecol. Lett.</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">336</span>– <span class="lpage" tagx="lpage" title="lpage">352</span> ( <span class="year" tagx="year" title="year">2005</span>). </span> </li> <li tag="ref"> <a name="b9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dyer</span> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nason</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garrick</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Landscape modelling of gene flow: improved power using conditional genetic distance derived from the topology of population networks </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">3746</span>– <span class="lpage" tagx="lpage" title="lpage">3759</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20723052">20723052</a> </span> </span> </li> <li tag="ref"> <a name="b10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freeland</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biss</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conrad</span> <span class="given-names" tagx="given-names" title="given-names">K. F.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silvertown</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Selection pressures have caused genome–wide population differentiation of <i>Anthoxanthum odoratum</i> despite the potential for high gene flow </span>. <span class="source" tagx="source" title="source">J. Evolution Biol.</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">776</span>– <span class="lpage" tagx="lpage" title="lpage">782</span> ( <span class="year" tagx="year" title="year">2010</span>). </span> </li> <li tag="ref"> <a name="b11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pease</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Landscape genetics of California mule deer ( <i>Odocoileus hemionus</i>): the roles of ecological and historical factors in generating differentiation </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">1848</span>– <span class="lpage" tagx="lpage" title="lpage">1862</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19302356">19302356</a> </span> </span> </li> <li tag="ref"> <a name="b12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freedman</span> <span class="given-names" tagx="given-names" title="given-names">A. H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomassen</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buermann</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">T. B.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genomic signals of diversification along ecological gradients in a tropical lizard</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">3773</span>– <span class="lpage" tagx="lpage" title="lpage">3788</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20618893">20618893</a> </span> </span> </li> <li tag="ref"> <a name="b13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manel</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poncet</span> <span class="given-names" tagx="given-names" title="given-names">B. N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Legendre</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gugerli</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holderegger</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Common factors drive adaptive genetic variation at different spatial scales in <i>Arabis alpina</i> </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">3824</span>– <span class="lpage" tagx="lpage" title="lpage">3835</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20723057">20723057</a> </span> </span> </li> <li tag="ref"> <a name="b14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sork</span> <span class="given-names" tagx="given-names" title="given-names">V. L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Gene movement and genetic association with regional climate gradients in California valley oak ( <i>Quercus lobata</i> Nee) in the face of climate change </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">19</span> , <span class="fpage" tagx="fpage" title="fpage">3806</span>– <span class="lpage" tagx="lpage" title="lpage">3823</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20723054">20723054</a> </span> </span> </li> <li tag="ref"> <a name="b15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Geological events play a larger role than Pleistocene climatic fluctuations in driving the genetic structure of <i>Quasipaa boulengeri</i> (Anura: Dicroglossidae) </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">1120</span>– <span class="lpage" tagx="lpage" title="lpage">1133</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23216961">23216961</a> </span> </span> </li> <li tag="ref"> <a name="b16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Epps</span> <span class="given-names" tagx="given-names" title="given-names">C. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wasser</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keim</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mutayoba</span> <span class="given-names" tagx="given-names" title="given-names">B. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brashares</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantifying past and present connectivity illuminates a rapidly changing landscape for the African elephant </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">1574</span>– <span class="lpage" tagx="lpage" title="lpage">158</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23398457">23398457</a> </span> </span> </li> <li tag="ref"> <a name="b17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">Q. X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edwards</span> <span class="given-names" tagx="given-names" title="given-names">D. L.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knowles</span> <span class="given-names" tagx="given-names" title="given-names">L. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrative testing of how environments from the past to the present shape genetic structure across landscapes </span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">3386</span>– <span class="lpage" tagx="lpage" title="lpage">3402</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24299395">24299395</a> </span> </span> </li> <li tag="ref"> <a name="b18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lomolino</span> <span class="given-names" tagx="given-names" title="given-names">M. V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riddle</span> <span class="given-names" tagx="given-names" title="given-names">B. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whittaker</span> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> Glaciation and biogeoraphic dynamics of the Pleistocene in <span class="source" tagx="source" title="source">Biogeography</span> 4th edn (eds <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lomolino</span> <span class="given-names" tagx="given-names" title="given-names">M. V.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>.) Ch. 9, <span class="fpage" tagx="fpage" title="fpage">313</span>– <span class="lpage" tagx="lpage" title="lpage">357</span> (Sinauer Associates, Inc., Sunderland, Massachussetts, <span class="year" tagx="year" title="year">2010</span>). </span> </li> <li tag="ref"> <a name="b19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hewitt</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The genetic legacy of the Quaternary ice ages</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">405</span>, <span class="fpage" tagx="fpage" title="fpage">907</span>– <span class="lpage" tagx="lpage" title="lpage">913</span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10879524">10879524</a> </span> </span> </li> <li tag="ref"> <a name="b20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kropf</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Comes</span> <span class="given-names" tagx="given-names" title="given-names">H. P.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kadereit</span> <span class="given-names" tagx="given-names" title="given-names">J. W.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Past, present and future of mountain species of the French Massif Central – the case of <i>Soldanella alpina</i> L. subsp. <i>alpina</i> (Primulaceae) and a review of other plant and animal studies </span>. <span class="source" tagx="source" title="source">J. Biogeogr.</span> <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">799</span>– <span class="lpage" tagx="lpage" title="lpage">812</span> ( <span class="year" tagx="year" title="year">2012</span>). </span> </li> <li tag="ref"> <a name="b21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wright</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation by distance</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">114</span>– <span class="lpage" tagx="lpage" title="lpage">138</span> ( <span class="year" tagx="year" title="year">1943</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17247074">17247074</a> </span> </span> </li> <li tag="ref"> <a name="b22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Slatkin</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene flow and the geographic structure of natural populations</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">236</span>, <span class="fpage" tagx="fpage" title="fpage">787</span>– <span class="lpage" tagx="lpage" title="lpage">792</span> ( <span class="year" tagx="year" title="year">1987</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3576198">3576198</a> </span> </span> </li> <li tag="ref"> <a name="b23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ying</span> <span class="given-names" tagx="given-names" title="given-names">T. S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y. L.</span> </span> <span class="source" tagx="source" title="source">The endemic genera of seed plants of China.</span> <span class="fpage" tagx="fpage" title="fpage">516</span>– <span class="lpage" tagx="lpage" title="lpage">518</span> (Science Press, Beijing, <span class="year" tagx="year" title="year">1994</span>) (In Chinese). </span> </li> <li tag="ref"> <a name="b24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qian</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparison of the taxonomic richness of temperate plants in East Asia and North America </span>. <span class="source" tagx="source" title="source">Am. J. Bot.</span> <span class="volume" tagx="volume" title="volume">89</span>, <span class="fpage" tagx="fpage" title="fpage">1818</span>– <span class="lpage" tagx="lpage" title="lpage">1825</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21665610">21665610</a> </span> </span> </li> <li tag="ref"> <a name="b25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z. Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Z. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D. Z.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Origin and differentiation of endemism in the flora of China</span>. <span class="source" tagx="source" title="source">Front. Biol. China</span> <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">125</span>– <span class="lpage" tagx="lpage" title="lpage">143</span> ( <span class="year" tagx="year" title="year">2007</span>). </span> </li> <li tag="ref"> <a name="b26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">Y. X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">C. X.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Comes</span> <span class="given-names" tagx="given-names" title="given-names">H. P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant molecular phylogeography in China and adjacent regions: tracing the genetic imprints of Quaternary climate and environmental change in the world’s most diverse temperate flora </span>. <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">225</span>– <span class="lpage" tagx="lpage" title="lpage">244</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21292014">21292014</a> </span> </span> </li> <li tag="ref"> <a name="b27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiffney</span> <span class="given-names" tagx="given-names" title="given-names">B. H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The Eocene North Atlantic land bridge: its importance in Tertiary and modern phytogeography of the Northern Hemisphere </span>. <span class="source" tagx="source" title="source">J. Arnold Arboretum</span> <span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">243</span>– <span class="lpage" tagx="lpage" title="lpage">273</span> ( <span class="year" tagx="year" title="year">1985</span>). </span> </li> <li tag="ref"> <a name="b28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Latham</span> <span class="given-names" tagx="given-names" title="given-names">R. E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ricklefs</span> <span class="given-names" tagx="given-names" title="given-names">R. E.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Continental comparisons of temperate-zone tree species diversity</span> in <span class="source" tagx="source" title="source">Species diversity in ecological communities</span> (eds <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ricklefs</span> <span class="given-names" tagx="given-names" title="given-names">R. E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schluter</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>) <span class="fpage" tagx="fpage" title="fpage">294</span>– <span class="lpage" tagx="lpage" title="lpage">314</span> (University of Chicago Press, Chicago, IL, USA, <span class="year" tagx="year" title="year">1993</span>). </span> </li> <li tag="ref"> <a name="b29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kubitzki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krutzsch</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Origins of east and south east Asian plant diversity</span> in <span class="source" tagx="source" title="source">Floristic characteristics and diversity of East Asian plants – Proceedings of the first International Symposium on Floristic Characteristics and Diversity of East Asian Plants </span> (eds <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">A. L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>.) <span class="fpage" tagx="fpage" title="fpage">25</span>– <span class="lpage" tagx="lpage" title="lpage">27</span> (Higher Education Press, China, <span class="year" tagx="year" title="year">1996</span>). </span> </li> <li tag="ref"> <a name="b30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Rubiaceae – <i>Emmenopterys</i> </span> in <span class="source" tagx="source" title="source">Flora of China</span>, <span class="volume" tagx="volume" title="volume">Vol. 19</span> (eds <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z. Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raven</span> <span class="given-names" tagx="given-names" title="given-names">P. H.</span> </span>) <span class="fpage" tagx="fpage" title="fpage">102</span> (Science Press &amp;amp; Missouri Botanical Garden Press, 1989–2013). </span> </li> <li tag="ref"> <a name="b31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">H. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oldfield</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qian</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> Global significance of plant diversity in China in <span class="source" tagx="source" title="source">Plants of China</span> (eds <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">D. Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blackmore</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>) Ch. 2, <span class="fpage" tagx="fpage" title="fpage">7</span>– <span class="lpage" tagx="lpage" title="lpage">31</span> (Science Press, Beijing, China, <span class="year" tagx="year" title="year">2013</span>). </span> </li> <li tag="ref"> <a name="b32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manchester</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Z. D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uemura</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Eastern Asian endemic seed plant genera and their paleogeographic history throughout the Northern Hemisphere </span>. <span class="source" tagx="source" title="source">J. Syst. Evol.</span> <span class="volume" tagx="volume" title="volume">47</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">42</span> ( <span class="year" tagx="year" title="year">2009</span>). </span> </li> <li tag="ref"> <a name="b33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manns</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wikström</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bremer</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Historical biogeography of the predominantly neotropical subfamily Cinchonoideae (Rubiaceae): into or out of America? </span> <span class="source" tagx="source" title="source">Int. J. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">173</span>, <span class="fpage" tagx="fpage" title="fpage">261</span>– <span class="lpage" tagx="lpage" title="lpage">286</span> ( <span class="year" tagx="year" title="year">2012</span>). </span> </li> <li tag="ref"> <a name="b34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">Z. X.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic structure of endangered <i>Emmenopterys henryi</i> Oliv. based on ISSR polymorphism and implications for its conservation </span>. <span class="source" tagx="source" title="source">Genetica</span> <span class="volume" tagx="volume" title="volume">133</span>, <span class="fpage" tagx="fpage" title="fpage">227</span>– <span class="lpage" tagx="lpage" title="lpage">234</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17828435">17828435</a> </span> </span> </li> <li tag="ref"> <a name="b35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">I. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Recognizing the temporal distinctions between landscape genetics and phylogeography</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">2605</span>– <span class="lpage" tagx="lpage" title="lpage">2608</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20561197">20561197</a> </span> </span> </li> <li tag="ref"> <a name="b36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graur</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W. H.</span> </span> Rates and patterns of nucleotide substitution in <span class="source" tagx="source" title="source">Fundamentals of molecular evolution</span>, 2nd edn. (eds <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graur</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W. H.</span> </span>) Ch. 4, <span class="fpage" tagx="fpage" title="fpage">99</span>– <span class="lpage" tagx="lpage" title="lpage">164</span> (Sinauer Associates, Inc., Sunderland, MA, USA., <span class="year" tagx="year" title="year">2000</span>). </span> </li> <li tag="ref"> <a name="b37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kay</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whittall</span> <span class="given-names" tagx="given-names" title="given-names">J. B.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodges</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A survey of nuclear ribosomal internal transcribed spacer substitution rates across angiosperms: an approximate molecular clock with life history effects </span>. <span class="source" tagx="source" title="source">BMC Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">36</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16638138">16638138</a> </span> </span> </li> <li tag="ref"> <a name="b38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avise</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nelson</span> <span class="given-names" tagx="given-names" title="given-names">W. S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugita</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A speciational history of living fossils molecular evolutionary patterns in horseshoe crabs </span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">1986</span>– <span class="lpage" tagx="lpage" title="lpage">2001</span> ( <span class="year" tagx="year" title="year">1994</span>). </span> </li> <li tag="ref"> <a name="b39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">X. S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Molecular data and ecological niche modelling reveal a highly dynamic evolutionary history of the East Asian Tertiary relict <i>Cercidiphyllum</i> (Cercidiphyllaceae) </span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">196</span>, <span class="fpage" tagx="fpage" title="fpage">617</span>– <span class="lpage" tagx="lpage" title="lpage">630</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22845876">22845876</a> </span> </span> </li> <li tag="ref"> <a name="b40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Geological and ecological factors drive cryptic speciation of yews in a biodiversity hotspot </span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">199</span>, <span class="fpage" tagx="fpage" title="fpage">1093</span>– <span class="lpage" tagx="lpage" title="lpage">1108</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23718262">23718262</a> </span> </span> </li> <li tag="ref"> <a name="b41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">H. Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xing</span> <span class="given-names" tagx="given-names" title="given-names">Q. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ji</span> <span class="given-names" tagx="given-names" title="given-names">Z. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">L. H.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">Y. H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">An outline of Quaternary development of <i>Fagus</i> forest in China: palynological and ecological perspectives </span>. <span class="source" tagx="source" title="source">Flora</span> <span class="volume" tagx="volume" title="volume">198</span>, <span class="fpage" tagx="fpage" title="fpage">249</span>– <span class="lpage" tagx="lpage" title="lpage">259</span> ( <span class="year" tagx="year" title="year">2003</span>). </span> </li> <li tag="ref"> <a name="b42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">Y. X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guan</span> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">C. X.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Comes</span> <span class="given-names" tagx="given-names" title="given-names">H. P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Did glacials and/or interglacials promote allopatric incipient speciation in East Asian temperate plants? Phylogeographic and coalescent analyses on refugial isolation and divergence in <i>Dysosma versipellis</i> </span>. <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">281</span>– <span class="lpage" tagx="lpage" title="lpage">293</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19405195">19405195</a> </span> </span> </li> <li tag="ref"> <a name="b43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shackleton</span> <span class="given-names" tagx="given-names" title="given-names">N. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">New data on the evolution of Pliocene climatic variability</span> in <span class="source" tagx="source" title="source">Paleoclimate and evolution, with emphasis on human origins</span> (ed. <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vrba</span> <span class="given-names" tagx="given-names" title="given-names">E. S.</span> </span>) <span class="fpage" tagx="fpage" title="fpage">242</span>– <span class="lpage" tagx="lpage" title="lpage">248</span> (Yale University Press, New Haven, USA, <span class="year" tagx="year" title="year">1995</span>). </span> </li> <li tag="ref"> <a name="b44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zachos</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pagani</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sloan</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Billups</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Trends, rhythms, and aberrations in global climate 65 Ma to present</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">292</span>, <span class="fpage" tagx="fpage" title="fpage">686</span>– <span class="lpage" tagx="lpage" title="lpage">693</span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11326091">11326091</a> </span> </span> </li> <li tag="ref"> <a name="b45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heermance</span> <span class="given-names" tagx="given-names" title="given-names">R. V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burbank</span> <span class="given-names" tagx="given-names" title="given-names">D. W.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Chronology and tectonic controls of Late Tertiary deposition in the southwestern Tian Shan foreland, NW China </span>. <span class="source" tagx="source" title="source">Basin Res.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">599</span>– <span class="lpage" tagx="lpage" title="lpage">632</span> ( <span class="year" tagx="year" title="year">2007</span>). </span> </li> <li tag="ref"> <a name="b46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">J. Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Major changes in East Asian climate in the mid-Pliocene: triggered by the uplift of the Tibetan Plateau or global cooling? </span> <span class="source" tagx="source" title="source">J. Asian Earth Sci.</span> <span class="volume" tagx="volume" title="volume">69</span>, <span class="fpage" tagx="fpage" title="fpage">48</span>– <span class="lpage" tagx="lpage" title="lpage">59</span> ( <span class="year" tagx="year" title="year">2013</span>). </span> </li> <li tag="ref"> <a name="b47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">Y. F.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Uplift of the Qinghai-Xizang (Tibetan) Plateau and east Asia environmental changes in the Late Cenozoic </span>. <span class="source" tagx="source" title="source">Acta. Geogr. Sin.</span> <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">10</span>– <span class="lpage" tagx="lpage" title="lpage">20</span> ( <span class="year" tagx="year" title="year">1999</span>). </span> </li> <li tag="ref"> <a name="b48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">L. M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">High variation and strong phylogeographic pattern among cpDNA haplotypes in <i>Taxus wallichiana</i> (Taxaceae) in China and North Vietnam </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">4684</span>– <span class="lpage" tagx="lpage" title="lpage">4698</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17908214">17908214</a> </span> </span> </li> <li tag="ref"> <a name="b49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakaguchi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Climate oscillation during the Quaternary associated with landscape heterogeneity promoted allopatric lineage divergence of a temperate tree <i>Kalopanax septemlobus</i> (Araliaceae) in East Asia </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">3823</span>– <span class="lpage" tagx="lpage" title="lpage">3838</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22646502">22646502</a> </span> </span> </li> <li tag="ref"> <a name="b50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lopez-Pujol</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z. Y.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular evidence for glacial expansion and interglacial retreat during Quaternary climatic changes in a montane temperate pine ( <i>Pinus kwangtungensis</i> Chun ex Tsiang) in southern China </span>. <span class="source" tagx="source" title="source">Plant Syst. Evol.</span> <span class="volume" tagx="volume" title="volume">284</span>, <span class="fpage" tagx="fpage" title="fpage">219</span>– <span class="lpage" tagx="lpage" title="lpage">229</span> ( <span class="year" tagx="year" title="year">2010</span>). </span> </li> <li tag="ref"> <a name="b51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Profound climatic effects on two East Asian black-throated tits (Ave: Aegithalidae), revealed by ecological niche models and phylogeographic analysis </span>. <span class="source" tagx="source" title="source">Plos One</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">e29329</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22195047">22195047</a> </span> </span> </li> <li tag="ref"> <a name="b52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Glacial expansion and diversification of an East Asian montane bird, the green-backed tit ( <i>Parus monticolus</i>) </span>. <span class="source" tagx="source" title="source">J. Biogeogr.</span> <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">1156</span>– <span class="lpage" tagx="lpage" title="lpage">1169</span> ( <span class="year" tagx="year" title="year">2013</span>). </span> </li> <li tag="ref"> <a name="b53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harrison</span> <span class="given-names" tagx="given-names" title="given-names">S. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahara</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prentice</span> <span class="given-names" tagx="given-names" title="given-names">I. C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Palaeovegetation. Diversity of temperate plants in east Asia</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">413</span>, <span class="fpage" tagx="fpage" title="fpage">129</span>– <span class="lpage" tagx="lpage" title="lpage">130</span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11557970">11557970</a> </span> </span> </li> <li tag="ref"> <a name="b54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">M. S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Assessment of forest geospatial patterns over the three giant forest areas of China</span>. <span class="source" tagx="source" title="source">J. Forestry Res.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">25</span>– <span class="lpage" tagx="lpage" title="lpage">31</span> ( <span class="year" tagx="year" title="year">2008</span>). </span> </li> <li tag="ref"> <a name="b55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orsini</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vanoverbeke</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swillen</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mergeay</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Meester</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Drivers of population genetic differentiation in the wild: isolation by dispersal limitation, isolation by adaptation and isolation by colonization </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">5983</span>– <span class="lpage" tagx="lpage" title="lpage">5999</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24128305">24128305</a> </span> </span> </li> <li tag="ref"> <a name="b56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoder</span> <span class="given-names" tagx="given-names" title="given-names">J. B.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genomic signature of adaptation to climate in <i>Medicago truncatula</i> </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">196</span>, <span class="fpage" tagx="fpage" title="fpage">1263</span>– <span class="lpage" tagx="lpage" title="lpage">1275</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24443444">24443444</a> </span> </span> </li> <li tag="ref"> <a name="b57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaw</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lickey</span> <span class="given-names" tagx="given-names" title="given-names">E. B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schilling</span> <span class="given-names" tagx="given-names" title="given-names">E. E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Small</span> <span class="given-names" tagx="given-names" title="given-names">R. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of whole chloroplast genome sequences to choose noncoding regions for phylogenetic studies in angiosperms: the tortoise and the hare III </span>. <span class="source" tagx="source" title="source">Am. J. Bot.</span> <span class="volume" tagx="volume" title="volume">94</span>, <span class="fpage" tagx="fpage" title="fpage">275</span>– <span class="lpage" tagx="lpage" title="lpage">288</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21636401">21636401</a> </span> </span> </li> <li tag="ref"> <a name="b58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gielly</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan,</span> <span class="given-names" tagx="given-names" title="given-names">Y. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Küpfer,</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taberlet,</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic use of noncoding regions in the genus <i>Gentiana</i> L.: chloroplast <i>trn</i>L (UAA) intron versus nuclear ribosomal internal transcribed spacer sequences </span>. <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">460</span>– <span class="lpage" tagx="lpage" title="lpage">466</span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8744760">8744760</a> </span> </span> </li> <li tag="ref"> <a name="b59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drummond</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. Geneious 4.8.5 (Biomatters Ltd., Auckland, New Zealand, 2009). Available at <a href="http://www.geneious.com/previous-versions">http://www.geneious.com/previous-versions</a>. </span> </li> <li tag="ref"> <a name="b60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vos</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">AFLP: a new technique for DNA fingerprinting</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">4407</span>– <span class="lpage" tagx="lpage" title="lpage">4414</span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7501463">7501463</a> </span> </span> </li> <li tag="ref"> <a name="b61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knowles</span> <span class="given-names" tagx="given-names" title="given-names">L. L.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richards</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Importance of genetic drift during Pleistocene divergence as revealed by analyses of genomic variation </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">4023</span>– <span class="lpage" tagx="lpage" title="lpage">4032</span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16262856">16262856</a> </span> </span> </li> <li tag="ref"> <a name="b62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rozas</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez–DelBarrio</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Messeguer</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rozas</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">DnaSP, DNA polymorphism analyses by the coalescent and other methods</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">2496</span>– <span class="lpage" tagx="lpage" title="lpage">2497</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14668244">14668244</a> </span> </span> </li> <li tag="ref"> <a name="b63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clement</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Posada</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crandall</span> <span class="given-names" tagx="given-names" title="given-names">K. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">TCS: a computer program to estimate gene genealogies</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">1657</span>– <span class="lpage" tagx="lpage" title="lpage">1659</span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11050560">11050560</a> </span> </span> </li> <li tag="ref"> <a name="b64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pons</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petit</span> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Measuring and testing genetic differentiation with ordered versus unordered alleles</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">144</span>, <span class="fpage" tagx="fpage" title="fpage">1237</span>– <span class="lpage" tagx="lpage" title="lpage">1245</span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8913764">8913764</a> </span> </span> </li> <li tag="ref"> <a name="b65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dupanloup</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneider</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A simulated annealing approach to define the genetic structure of populations</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">2571</span>– <span class="lpage" tagx="lpage" title="lpage">2581</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12453240">12453240</a> </span> </span> </li> <li tag="ref"> <a name="b66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rousset</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic differentiation and estimation of gene flow from <i>F</i>–statistics under isolation by distance </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">145</span>, <span class="fpage" tagx="fpage" title="fpage">1219</span>– <span class="lpage" tagx="lpage" title="lpage">1228</span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9093870">9093870</a> </span> </span> </li> <li tag="ref"> <a name="b67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milligan</span> <span class="given-names" tagx="given-names" title="given-names">B. G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of population genetic structure with RAPD markers</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">91</span>– <span class="lpage" tagx="lpage" title="lpage">99</span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8019690">8019690</a> </span> </span> </li> <li tag="ref"> <a name="b68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vekemans</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beauwens</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemaire</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roldan-Ruiz</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Data from amplified fragment length polymorphism (AFLP) markers show indication of size homoplasy and of a relationship between degree of homoplasy and fragment size </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">139</span>– <span class="lpage" tagx="lpage" title="lpage">151</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11903911">11903911</a> </span> </span> </li> <li tag="ref"> <a name="b69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schönswetter</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tribsch</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Vicariance and dispersal in the alpine perennial <i>Bupleurum stellatum</i> L. (Apiaceae) </span>. <span class="source" tagx="source" title="source">Taxon</span> <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">725</span>– <span class="lpage" tagx="lpage" title="lpage">732</span> ( <span class="year" tagx="year" title="year">2005</span>). </span> </li> <li tag="ref"> <a name="b70" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corander</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marttinen</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siren</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Enhanced Bayesian modelling in BAPS software for learning genetic structures of populations</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">539</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19087322">19087322</a> </span> </span> </li> <li tag="ref"> <a name="b71" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peakall</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smouse</span> <span class="given-names" tagx="given-names" title="given-names">P. E.</span> </span> <span class="mixed-article-title" title="mixed-article-title">GENALEX 6: genetic analysis in Excel. Population genetic software for teaching and research </span>. <span class="source" tagx="source" title="source">Mol. Ecol. Notes</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">288</span>– <span class="lpage" tagx="lpage" title="lpage">295</span> ( <span class="year" tagx="year" title="year">2006</span>). </span> </li> <li tag="ref"> <a name="b72" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lischer</span> <span class="given-names" tagx="given-names" title="given-names">H. E. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows </span>. <span class="source" tagx="source" title="source">Mol. Ecol. Resour.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">564</span>– <span class="lpage" tagx="lpage" title="lpage">567</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21565059">21565059</a> </span> </span> </li> <li tag="ref"> <a name="b73" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smouse</span> <span class="given-names" tagx="given-names" title="given-names">P. E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quattro</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of molecular variance inferred from metric distances among DNA haplotypes application to human mitochondrial DNA restriction data </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">131</span>, <span class="fpage" tagx="fpage" title="fpage">479</span>– <span class="lpage" tagx="lpage" title="lpage">491</span> ( <span class="year" tagx="year" title="year">1992</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1644282">1644282</a> </span> </span> </li> <li tag="ref"> <a name="b74" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drummond</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suchard</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rambaut</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">1969</span>– <span class="lpage" tagx="lpage" title="lpage">1973</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">22367748</a> </span> </span> </li> <li tag="ref"> <a name="b75" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Darriba</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taboada</span> <span class="given-names" tagx="given-names" title="given-names">G. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doallo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Posada</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">jModelTest 2: more models, new heuristics and parallel computing</span>. <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">772</span>– <span class="lpage" tagx="lpage" title="lpage">772</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22847109">22847109</a> </span> </span> </li> <li tag="ref"> <a name="b76" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drummond</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rambaut</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <span class="source" tagx="source" title="source">BMC Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">214</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a> </span> </span> </li> <li tag="ref"> <a name="b77" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tajima</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">123</span>, <span class="fpage" tagx="fpage" title="fpage">585</span>– <span class="lpage" tagx="lpage" title="lpage">595</span> ( <span class="year" tagx="year" title="year">1989</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255">2513255</a> </span> </span> </li> <li tag="ref"> <a name="b78" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">Y. X.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">147</span>, <span class="fpage" tagx="fpage" title="fpage">915</span>– <span class="lpage" tagx="lpage" title="lpage">925</span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9335623">9335623</a> </span> </span> </li> <li tag="ref"> <a name="b79" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nogues-Bravo</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Predicting the past distribution of species climatic niches</span>. <span class="source" tagx="source" title="source">Global. Ecol. Biogeogr.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">521</span>– <span class="lpage" tagx="lpage" title="lpage">531</span> ( <span class="year" tagx="year" title="year">2009</span>). </span> </li> <li tag="ref"> <a name="b80" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braconnot</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Results of PMIP2 coupled simulations of the Mid-Holocene and Last Glacial Maximum – Part 1: experiments and large-scale features </span>. <span class="source" tagx="source" title="source">Clim. Past.</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">261</span>– <span class="lpage" tagx="lpage" title="lpage">277</span> ( <span class="year" tagx="year" title="year">2007</span>). </span> </li> <li tag="ref"> <a name="b81" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braconnot</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Results of PMIP2 coupled simulations of the Mid-Holocene and Last Glacial Maximum – Part 2: feedbacks with emphasis on the location of the ITCZ and mid- and high latitudes heat budget </span>. <span class="source" tagx="source" title="source">Clim. Past.</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">279</span>– <span class="lpage" tagx="lpage" title="lpage">296</span> ( <span class="year" tagx="year" title="year">2007</span>). </span> </li> <li tag="ref"> <a name="b82" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otto-Bliesner</span> <span class="given-names" tagx="given-names" title="given-names">B. L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Simulating arctic climate warmth and icefield retreat in the last interglaciation</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">311</span>, <span class="fpage" tagx="fpage" title="fpage">1751</span>– <span class="lpage" tagx="lpage" title="lpage">1753</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16556838">16556838</a> </span> </span> </li> <li tag="ref"> <a name="b83" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hijmans</span> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cameron</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parra</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">P. G.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jarvis</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <span class="source" tagx="source" title="source">Int. J. Climatol.</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1965</span>– <span class="lpage" tagx="lpage" title="lpage">1978</span> ( <span class="year" tagx="year" title="year">2005</span>). </span> </li> <li tag="ref"> <a name="b84" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peterson</span> <span class="given-names" tagx="given-names" title="given-names">A. T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakazawa</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Environmental data sets matter in ecological niche modelling: an example with <i>Solenopsis invicta</i> and <i>Solenopsis richteri</i> </span>. <span class="source" tagx="source" title="source">Global. Ecol. Biogeogr.</span> <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">135</span>– <span class="lpage" tagx="lpage" title="lpage">144</span> ( <span class="year" tagx="year" title="year">2008</span>). </span> </li> <li tag="ref"> <a name="b85" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">K-1 Model Developers. K-1 coupled GCM (MIROC) description in <i>K-1 Technical Report 1 of of the Kyousei project</i> (eds Hasumi, H. &amp;amp; Emori, S.) 1–28 (Center for Climate System Research, University of Tokyo, Tokyo, 2004). URL <a href="http://ccsr.aori.u-tokyo.ac.jp/~hasumi/miroc_description.pdf">http://ccsr.aori.u-tokyo.ac.jp/~hasumi/miroc_description.pdf</a>. </span> </li> <li tag="ref"> <a name="b86" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swets</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Measuring the accuracy of diagnostic systems</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">240</span>, <span class="fpage" tagx="fpage" title="fpage">1285</span>– <span class="lpage" tagx="lpage" title="lpage">1293</span> ( <span class="year" tagx="year" title="year">1988</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3287615">3287615</a> </span> </span> </li> <li tag="ref"> <a name="b87" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">I. J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Summers</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic structure is correlated with phenotypic divergence rather than geographic isolation in the highly polymorphic strawberry poison-dart frog </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">447</span>– <span class="lpage" tagx="lpage" title="lpage">458</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20025652">20025652</a> </span> </span> </li> <li tag="ref"> <a name="b88" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">I. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Examining the full effects of landscape heterogeneity on spatial genetic variation: a multiple matrix regression approach for quantifying geographic and ecological isolation </span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">3403</span>– <span class="lpage" tagx="lpage" title="lpage">3411</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24299396">24299396</a> </span> </span> </li> <li tag="ref"> <a name="b89" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">I. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glor</span> <span class="given-names" tagx="given-names" title="given-names">R. E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Losos</span> <span class="given-names" tagx="given-names" title="given-names">J. B.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantifying the roles of ecology and geography in spatial genetic divergence</span>. <span class="source" tagx="source" title="source">Ecol. Lett.</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">175</span>– <span class="lpage" tagx="lpage" title="lpage">182</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23137142">23137142</a> </span> </span> </li> <li tag="ref"> <a name="b90" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H. Z.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gong</span> <span class="given-names" tagx="given-names" title="given-names">Z. T.</span> </span> Regularity of soil distribution and soil division in Chinese soil taxonomic classification (ed. <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gong</span> <span class="given-names" tagx="given-names" title="given-names">Z. T.</span> </span>) Ch. <span class="volume" tagx="volume" title="volume">22,</span> <span class="fpage" tagx="fpage" title="fpage">860</span>– <span class="lpage" tagx="lpage" title="lpage">863</span> (China Science Press, Beijing, China, <span class="year" tagx="year" title="year">1999</span>). </span> </li> <li tag="ref"> <a name="b91" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beaumont</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nichols</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Evaluating loci for use in the genetic analysis of population structure</span>. <span class="source" tagx="source" title="source">P. Roy. Soc. B.: Biol. Sci.</span> <span class="volume" tagx="volume" title="volume">263</span>, <span class="fpage" tagx="fpage" title="fpage">1619</span>– <span class="lpage" tagx="lpage" title="lpage">1626</span> ( <span class="year" tagx="year" title="year">1996</span>). </span> </li> <li tag="ref"> <a name="b92" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foll</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaggiotti</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A genome scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">180</span>, <span class="fpage" tagx="fpage" title="fpage">977</span>– <span class="lpage" tagx="lpage" title="lpage">993</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18780740">18780740</a> </span> </span> </li> <li tag="ref"> <a name="b93" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zulliger</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schnyder</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gugerli</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Are adaptive loci transferable across genomes of related species? Outlier and environmental association analyses in Alpine Brassicaceae species </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">1626</span>– <span class="lpage" tagx="lpage" title="lpage">1639</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23398479">23398479</a> </span> </span> </li> <li tag="ref"> <a name="b94" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Environmental Systems Research Institute. ArcGIS Desktop 9, ArcView 9.3, Single Use. (Environmental Systems Research Institute, Redlands, CA, USA, 2008). Available at <a href="http://www.esri.com/software/arcgis/arcgis-for-desktop">http://www.esri.com/software/arcgis/arcgis-for-desktop</a>. </span> </li> </ul> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Author Contributions</b> Y.X.Q. conceived and designed the study, Y.H.Z. and H.P. collected samples, Y.H.Z. performed experiments, Y.H.Z. and I.J.W. analyzed the data, Y.X.Q., Y.H.Z., H.P.C. and I.J.W. wrote the manuscript. All authors revised the manuscript. </p> </div> </div> </div>  </body></html>