<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6346095">6346095</a> </span> <span class="publisher-id" title="publisher-id">37070</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-018-37070-z">10.1038/s41598-018-37070-z</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Immunoinformatic and systems biology approaches to predict and validate peptide vaccines against Epstein–Barr virus (EBV) </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ali</span> <span class="given-names" tagx="given-names" title="given-names">Arif</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">arif_sjtu@sjtu.edu.cn</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8349-5181</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">Abbas</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaushik</span> <span class="given-names" tagx="given-names" title="given-names">Aman Chandra</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Yanjie</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ali</span> <span class="given-names" tagx="given-names" title="given-names">Syed Shujait</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Junaid</span> <span class="given-names" tagx="given-names" title="given-names">Muhammad</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saleem</span> <span class="given-names" tagx="given-names" title="given-names">Shoaib</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">William C. S.</span> </span> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mao</span> <span class="given-names" tagx="given-names" title="given-names">Xueying</span> </span> <a href="#Aff4">4</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-4200-7502</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">Dong-Qing</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">dqwei@sjtu.edu.cn</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 0368 8293</span> <span class="institution-id" title="institution-id">grid.16821.3c</span> <span class="institution" title="institution">State Key Laboratory of Microbial Metabolism, and College of Life Sciences and Biotechnology, </span> <span class="institution" title="institution">Shanghai Jiao Tong University, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.449683.4</span> <span class="institution" title="institution">Center for Biotechnology and Microbiology, </span> <span class="institution" title="institution">University of Swat, </span> </div> </span> <span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1771 451X</span> <span class="institution-id" title="institution-id">grid.415499.4</span> <span class="institution" title="institution">Department of Clinical Oncology, </span> <span class="institution" title="institution">Queen Elizabeth Hospital, </span> </div> </span> <span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 2323 5732</span> <span class="institution-id" title="institution-id">grid.39436.3b</span> <span class="institution" title="institution">Qianweichang College, </span> <span class="institution" title="institution">Shanghai University, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-1-1</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-1-1</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">720</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-6-29</span> <span class="accepted" title="accepted">accepted: 2018-11-30</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Epstein–Barr virus (EBV), also known as human herpesvirus 4 (HHV-4), is a member of the Herpesviridae family and causes infectious mononucleosis, Burkitt’s lymphoma, and nasopharyngeal carcinoma. Even in the United States of America, the situation is alarming, as EBV affects 95% of the young population between 35 and 40 years of age. In this study, both linear and conformational B-cell epitopes as well as cytotoxic T-lymphocyte (CTL) epitopes were predicted by using the ElliPro and NetCTL.1.2 webservers for EBV proteins (GH, GL, GB, GN, GM, GP42 and GP350). Molecular modelling tools were used to predict the 3D coordinates of peptides, and these peptides were then docked against the MHC molecules to obtain peptide-MHC complexes. Studies of their post-docking interactions helped to select potential candidates for the development of peptide vaccines. Our results predicted a total of 58 T-cell epitopes of EBV; where the most potential were selected based on their TAP, MHC binding and C-terminal Cleavage score. The top most peptides were subjected to MD simulation and stability analysis. Validation of our predicted epitopes using a 0.45 µM concentration was carried out by using a systems biology approach. Our results suggest a panel of epitopes that could be used to immunize populations to protect against multiple diseases caused by EBV. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">This work is supported by the Key Research Area Grant 2016YFA0501703 from the Ministry of Science and Technology of China, and Ph.D. Programs Foundation of Ministry of Education of China (Contract No., 20120073110057). </span> </div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Epstein–Barr virus (EBV), also known as human herpesvirus 4 (HHV-4), is a member of the Herpesviridae family and is one of the eight known types of human herpesvirus. EBV is the most common human virus in the world <sup> <a href="#CR1">1</a> </sup> and was isolated in 1964 from tumor cells (Burkitt’s lymphoma) by Epstein’s group <sup> <a href="#CR2">2</a> </sup>. EBV is related to distinct forms of cancer, such as Burkitt’s lymphoma, stomach cancer, Hodgkin’s lymphoma and nasopharyngeal carcinoma <sup> <a href="#CR3">3</a>, <a href="#CR4">4</a> </sup>. A High number of cases are usually reported. In the United States and other developing countries, most people are infected with EBV <sup> <a href="#CR5">5</a> </sup> as 90% of the adults in the United States have been formally diagnosed with EBV infection. EBV infection can be asymptomatic or symptomatic, and the latter case includes mild fatigue, fever, enlarged spleen, swollen liver, swollen lymph nodes, inflamed throat, or rashes <sup> <a href="#CR6">6</a> </sup>. From 2006 to 2015, several clinical trials were conducted to develop vaccines; however, an EBV vaccine, phase 2 trial, from gp350 protein has been testified. This vaccine reduced the rate of Infectious Mononucleosis (IM) but not virus infection <sup> <a href="#CR7">7</a> </sup>. </p> <p>Precautionary measures, such as avoiding direct contact with patients (including refraining from using a patient’s toothbrush, sharing food, or exchanging bodily fluids), can help reduce the risk of infection. EBV can infect host B cells and booms via a nonlytic mechanism <sup> <a href="#CR8">8</a> </sup>. EBV viral proteins play important roles in lymphoproliferative disease. For example, viral membranous proteins, such as LMP-1, may induce tumorigenic replication in infected B cells <sup> <a href="#CR9">9</a>, <a href="#CR10">10</a> </sup>. There are two types of EBV; Epstein and Yvonne Barr identified EBV in tumor tissue associated with Burkitt’s Lymphoma <sup> <a href="#CR11">11</a> </sup>. Glycoprotein 42 (gp42), Glycoprotein H, Glycoprotein L, and Glycoprotein B aid in the entrance to the host cell. Glycoprotein 42 binds to the HLA class II molecule because it is required for B cells, which inhibit epithelial cell fusion. For epithelial-cell fusion, the GH receptor protein interacts with GP42 <sup> <a href="#CR12">12</a> </sup> and GL is transported to the cell surface, which is essential for the correct folding of GH <sup> <a href="#CR13">13</a> </sup>. </p> <p>EBV glycoprotein B is important for viral fusion events with B cells <sup> <a href="#CR14">14</a> </sup>. The human immune system precisely targets EBV glycoprotein 350 (gp350), which is an example of a lytically expressed gene <sup> <a href="#CR15">15</a> </sup>. The attachment of GP350 to the MHC-II molecules in the cell is aided by the already attached GP42 protein of EBV virus <sup> <a href="#CR16">16</a> </sup>. The fusion of the B-cell membrane and the outer viral envelope of the EBV virion requires functional spicule glycoproteins such as GH, GL, and gp42 <sup> <a href="#CR17">17</a> </sup>. Previous studies suggest that glycoproteins such as GB complement membrane fusion <sup> <a href="#CR18">18</a> </sup>. </p> <p>Vaccination is a significant approach to improve the standard of public health and provide an effective way to control the growing infections. In nature, plants act as bioreactors, which have been used to express efficient vaccine antigens against viral, bacterial and protozoan infections. Besides, we know that antibody epitope prediction using computational tools, one of the crucial steps of vaccine design <sup> <a href="#CR19">19</a> </sup>. Recent advancement in vaccine design has aimed for the expansion of conventional assays designed to quantify T-cell responses against various vaccine candidates <sup> <a href="#CR20">20</a> </sup>. Immunoinformatic approaches have made great contributions to predicting B-cell and T-cell epitopes in the development of subunit vaccines <sup> <a href="#CR21">21</a> </sup>. For subunit vaccine development, identification of continuous B-cell or nonlinear also known as non-continuous and cytotoxic T-lymphocyte (CTL) epitopes are essential. Among B-cell epitopes, &amp;gt;90% are noncontinuous <sup> <a href="#CR22">22</a>, <a href="#CR23">23</a> </sup>. The use of computational tools contribute greatly in biology designing <i>in silico</i> vaccine, prediction of T-cell epitope is crucial which does not only reduce the cost but also the necessity for experimental results <sup> <a href="#CR24">24</a> </sup>. Epitopic vaccine against HIV, malaria and tuberculosis resulted in promising outputs and maintained the defensive and beneficial potential therapeutic uses of the developed vaccines candidates <sup> <a href="#CR25">25</a> </sup>. Immunoinformatics plays an upright role in antibody and immunodiagnostic agents development, and vaccine design. The early phases of vaccine and other therapeutics agents developments were based on solely immunological experimentations. These early developmental techniques were tedious and costly too. The use of bioinformatics such as computational techniques greatly reduces the time and cost of developing such agents for therapeutic purposes. Khan <i>et al</i>. <sup> <a href="#CR26">26</a> </sup>, used multiple bioinformatics tools to predict vaccine against multiple HPV viruses <sup> <a href="#CR26">26</a> </sup>. Thus the development of modern therapeutic medicines and vaccines greatly rely on such tools <sup> <a href="#CR27">27</a>, <a href="#CR28">28</a> </sup>. </p> <p>Systems medicine emphases significantly on the components of pathway kinetics to probe different conditions. Systems medicine could be also utilized to investigate interaction mechanisms between microbes. Metagenomics data could be utilized for such analysis. Rather than whole cell interaction, an insight onto proteins interaction could be also comprehended through systems biology approaches (Singh, P. K. <i>et al</i>. <sup> <a href="#CR29">29</a> </sup>. These latest techniques widely expand the circle of new drugs development. Overall, it is known that multidisciplinary aspects of the production of therapeutic proteins that has gained much more attention (Dangi, A. K. <i>et al</i>. <sup> <a href="#CR30">30</a> </sup>. Structural modelling using computational resources led to success in the development of computer assisted drugs. Likewise molecular docking algorithms scoring functions of different conformers to design new drug candidates. </p> <p>In the present study, potential B-cell and T-cell epitopes (as effective vaccine candidates) were identified with the help of immunoinformatic approaches. T-cell immunogenicity is associated with epitope binding strength to the MHC molecule <sup> <a href="#CR31">31</a> </sup>. Molecular modelling tools were applied to peptide-MHC complexes to investigate their post-docking interaction in order to select potential candidates for the development of peptide vaccines. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="epstein-barrvirussequence" title="sec"> <div class="title" tagx="title" title="title">
Epstein-Barr Virus Sequence</div> <p>The sequences of EBV proteins, including GH, GL, GB, GN, GM, GP42 and GP350, were retrieved from the Universal Protein knowledgebase (Uniprot) database ( <a href="http://www.uniprot.org/">http://www.uniprot.org/</a>). The sequence retrieval accession numbers along with other information are provided in the (Table <a href="#Tab1">1</a>). The 3D coordinates of all the selected proteins were predicted by using online webserver phyre2 ( <a href="http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index">http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id = index</a>) <sup> <a href="#CR32">32</a> </sup>. The overall workflow of the work is shown in the Fig. <a href="#Fig1">1</a>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Detailed information, including individual protein sequence length, and region and accession number is shown in the table below. </p>  <table> <thead> <tr> <th>S. No</th> <th>Uniprot Accession Number</th> <th>Protein Name</th> <th>No of Amino Acids</th> </tr> </thead> <tbody> <tr> <td>1.</td> <td>P0C763</td> <td>Glycoprotein B</td> <td>857</td> </tr> <tr> <td>2.</td> <td>P03231</td> <td>Glycoprotein H</td> <td>706</td> </tr> <tr> <td>3.</td> <td>P03212</td> <td>Glycoprotein L</td> <td>137</td> </tr> <tr> <td>4.</td> <td>P03215</td> <td>Glycoprotein M</td> <td>405</td> </tr> <tr> <td>5.</td> <td>P03196</td> <td>Glycoprotein N</td> <td>102</td> </tr> <tr> <td>6.</td> <td>P03205</td> <td>Glycoprotein 42</td> <td>223</td> </tr> <tr> <td>7.</td> <td>P68343</td> <td>Glycoprotein 350</td> <td>886</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>The figure above is showing the pipeline of the study. Resources, methods, and each step is discussed. </p>   </div> <p /> </div> <div class="predictionoflinearb-cellepitopes" title="sec"> <div class="title" tagx="title" title="title">
Prediction of Linear B-Cell Epitopes</div> <p>After interacting with antigens (such as B-cell epitopes), B-lymphocyte cells differentiate into memory cells and antibody secreting plasma cells <sup> <a href="#CR33">33</a> </sup>. B-cell epitopes have a hydrophilic nature and are accessible for flexible regions <sup> <a href="#CR34">34</a> </sup>. IEDB ( <a href="http://www.iedb.org/">http://www.iedb.org/</a>) online analysis resources were used to obtain the Parker hydrophilicity prediction values <sup> <a href="#CR35">35</a> </sup>, Emini prediction values of surface accessibility <sup> <a href="#CR36">36</a> </sup>, Kolaskar and Tongaonkar’s antigenicity scale values <sup> <a href="#CR37">37</a> </sup>, and Karplus and Schulz Flexibility Prediction values. B-cell epitopes were predicted using ElliPro ( <a href="http://tools.immuneepitope.org/toolsElliPro/">http://tools.immuneepitope.org/toolsElliPro/</a>) using both protein sequences and structural information <sup> <a href="#CR38">38</a> </sup>. ElliPro utilizes the Protrusion Index (PI) of residues, protein shape approximation, and the final neighbouring residues clustering, which rely on PI. </p> </div> <div class="predictionofpotentialcytotoxict-lymphocyte(ctl)epitopes" title="sec"> <div class="title" tagx="title" title="title">
Prediction of Potential Cytotoxic T-lymphocyte (CTL) Epitopes</div> <p>CTL epitopes were predicted using the NetCTL.1.2 ( <a href="http://www.cbs.dtu.dk/services/NetCTL/">http://www.cbs.dtu.dk/services/NetCTL/</a>) server <sup> <a href="#CR28">28</a> </sup>. NetCTL accepts the FASTA sequence format to perform different analyses, such as prediction of MHC class I binding affinity, TAP transport efficiency and C-terminal cleavage. The artificial neural network and weight matrix were used for the prediction of MHC-I binding and proteasome-dependent C-terminal cleavage. </p> </div> <div class="peptidelibraryconstructionandmoleculardocking" title="sec"> <div class="title" tagx="title" title="title">
Peptide Library Construction and Molecular Docking</div> <p>All of the predicted epitopes were modelled using the online webserver PEP-FOLD3 using 200 simulation runs to sample the conformations <sup> <a href="#CR39">39</a> </sup> and sOPEP energy function <sup> <a href="#CR40">40</a> </sup>. Subsequently, we have docked the best ranked peptide models to the selected class I MHC molecules HLA-A (PDB ID: 2GIT) using the PatchDock docking server <sup> <a href="#CR41">41</a> </sup>. The algorithm of the PatchDock server uses structural geometry to find docking transformations with good molecular shape complementarity <sup> <a href="#CR42">42</a> </sup>. The resulting complexes were refined through the FireDock server <sup> <a href="#CR43">43</a>, <a href="#CR44">44</a> </sup>. High energy complexes were subjected to interaction analysis and molecular dynamics simulations <sup> <a href="#CR45">45</a> </sup>. </p> </div> <div class="moleculardynamicssimulations" title="sec"> <div class="title" tagx="title" title="title">
Molecular Dynamics Simulations</div> <p>The accepted complexes were subjected to Molecular Dynamic simulations using the AMBER 14 molecular dynamics package <sup> <a href="#CR46">46</a> </sup>. The system was neutralized using Na <sup>+</sup> ions using tleap. Each system was solvated in a rectangular box with buffer distance of 8.0 A° using TIP3P water molecules. A two-stage energy minimization of the complexes, using the SANDER module of AMBER 14, was performed to relieve the atomic clashes. An initial minimization of 6,000 steps, followed by another round of minimization (6,000 steps), were used to restrain the positions of all atoms in the systems, except those from the water molecules in the first minimization. The <i>pmemd.cuda</i> <sup> <a href="#CR47">47</a> </sup> software was used to simulate the minimized complexes. The SHAKE algorithm and the Particle-Mesh Ewald (PME) method were used to include the long-range interactions, and a non-bonded interaction cutoff radius of 10 A° was considered. For equilibration, 10,000 ps time was applied, followed by a 50 ns simulation carried out at 310 K using the Langevin temperature coupling scheme at constant pressure (1 atm) with isotropic molecule-based scaling. Sampling of the MD trajectories was carried out every 2.0 ps. RMSD and hydrogen bonding analysis were carried out using the integrated CPPTRAJ and PYTRAJ <sup> <a href="#CR48">48</a> </sup> modules in AMBER 14 and were visualized using the online server PDBePISA <sup> <a href="#CR49">49</a> </sup>, UCSF Chimera <sup> <a href="#CR50">50</a> </sup> and PyMOL <sup> <a href="#CR51">51</a> </sup>. </p> </div> <div class="antigenicandallergenicbehaviourthepredictedepitopes" title="sec"> <div class="title" tagx="title" title="title">
Antigenic and Allergenic behaviour the predicted Epitopes</div> <p>To confirm the allergenic and non-allergenic properties of all the designed epitopes, B-cell and T-cell epitopes, AlgPred <sup> <a href="#CR52">52</a> </sup> ( <a href="http://crdd.osdd.net/raghava/algpred/">http://crdd.osdd.net/raghava/algpred/</a>), which is an online web tool was employed with accuracy of 85%. Primary amino acid sequences were used of all the selected proteins for this purpose. The antigenicity of all the epitopes was predicted by using ANTIGENpro <sup> <a href="#CR53">53</a> </sup> ( <a href="http://scratch.proteomics.ics.uci.edu/">http://scratch.proteomics.ics.uci.edu/</a>) using different machine learning algorithms to process the amino acids sequences. </p> </div> <div class="pkpdmodeling" title="sec"> <div class="title" tagx="title" title="title">
PKPD Modeling</div> <p>The design and execution of the EBV signalling cascade was performed based on a literature survey within a virtual cell. Proposed peptides were chosen for kinetic analysis of EBV, where a concentration of 0.45 µm was assigned based on previous literature. Modelling of chemical reaction networks was applied for the analysis of this pathway. </p> <p>This nonlinear kinetics scheme follows the Michaelis–Menten equation as below: </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">1</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\rm{V}}=\frac{({\rm{Vmax}})\,\cdot \,({\rm{S}})}{{\rm{Km}}+{\rm{S}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Vmax</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
·</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Km</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
S</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>This equation can be transformed to, </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\rm{C}}=\frac{({\rm{Cmax}})\,\cdot \,({\rm{D}})}{{\rm{Km}}+{\rm{D}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Cmax</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
·</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Km</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div>or <div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">2</span> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\rm{V}}=\frac{{\rm{d}}[{\rm{P}}]}{{\rm{dt}}}=\frac{({\rm{Amax}})\,\cdot \,({\rm{D}})}{{\rm{Km}}+{\rm{D}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
]</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
dt</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Amax</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
·</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Km</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> <p>Here,</p> <p>C = steady state concentration;</p> <p>C <sub>max</sub> = theoretical maximum for C; </p> <p>A <sub>max</sub> = theoretical maximum for A; </p> <p>D = dose.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="sequenceretrievalandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence retrieval and analysis</div> <p>Uniprot was used to retrieve the primary amino acid sequences of the selected proteins (glycoprotein B, glycoprotein L, glycoprotein N, glycoprotein H, glycoprotein M, glycoprotein 42 and glycoprotein 350) of EBV as shown in Fig. <a href="#Fig2">2</a>. Information about the protein source, accession number, number of active residues, and other information is given in Table <a href="#Tab1">1</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>The 3D structures of the selected proteins. The Phyre 2 online server was used for B-cell and T-cell epitope prediction. ( <b>A</b>) Glycoprotein B ( <b>B</b>) Glycoprotein H ( <b>C</b>) Glycoprotein L ( <b>D</b>) Glycoprotein M ( <b>E</b>) Glycoprotein N ( <b>F</b>) Glycoprotein 42 ( <b>G</b>) Glycoprotein 350. </p>   </div> <p /> </div> <div class="allergenicityandantigenicityprediction" title="sec"> <div class="title" tagx="title" title="title">
Allergenicity and antigenicity prediction</div> <p>The allergic and nonallergenic behaviours of EBV species were predicted using AlgPred ( <a href="http://www.imtech.res.in/raghava/algpred/">http://www.imtech.res.in/raghava/algpred/</a>). Allergenicity prediction of known protein sequences is based on similarity. We checked if the epitopes are antigenic or not using the online server AntigenPro ( <a href="http://www.scratch.proteomics.ics.uci.edu/">http://www.scratch.proteomics.ics.uci.edu/</a>) <sup> <a href="#CR35">35</a> </sup>. All of the proteins were found to be nonallergenic, while they possess antigenic properties (Table <a href="#Tab2">2</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Antigenic and allergenic results of the selected proteins.</p>  <table> <thead> <tr> <th>S.NO</th> <th>Proteins</th> <th>Allergenicity</th> <th>Antigenicity</th> </tr> </thead> <tbody> <tr> <td>1.</td> <td>Glycoprotein B</td> <td>Non-Allergic</td> <td>Antigenic</td> </tr> <tr> <td>2.</td> <td>Glycoprotein H</td> <td>Non-Allergic</td> <td>Antigenic</td> </tr> <tr> <td>3.</td> <td>Glycoprotein L</td> <td>Non-Allergic</td> <td>Antigenic</td> </tr> <tr> <td>4.</td> <td>Glycoprotein M</td> <td>Non-Allergic</td> <td>Antigenic</td> </tr> <tr> <td>5.</td> <td>Glycoprotein N</td> <td>Non-Allergic</td> <td>Antigenic</td> </tr> <tr> <td>6.</td> <td>Glycoprotein 42</td> <td>Non-Allergic</td> <td>Antigenic</td> </tr> <tr> <td>7.</td> <td>Glycoprotein 350</td> <td>Non-Allergic</td> <td>Antigenic</td> </tr> </tbody> </table>  <p /> </div> <div class="b-cellepitopeprediction" title="sec"> <div class="title" tagx="title" title="title">
B-cell epitope prediction</div> <p>The BCPred server predicted 58 B-cell epitopes: five epitopes were for GP 42, eight for GP H, nineteen for GP B, one for GP L and GP N, five for GP M, and nineteen for GP 350. However, epitopes with scores above 0.99 were selected as the most potentially antigenic epitopes. Therefore, only one epitope each from GP42, GL, GM, GN and GH; four epitopes from GB; and fifteen epitopes from GP350 were found to meet the threshold value. B-cell epitopes, along with their scores, are tabulated in Table <a href="#Tab3">3</a>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>B-Cell epitopes predicted by BCPred.</p>  <table> <thead> <tr> <th>Protein</th> <th>Position</th> <th>Epitope</th> <th>Score</th> </tr> </thead> <tbody> <tr> <td>GP42</td> <td>43</td> <td>TWVPKPNVEVWPVDPPPPVN</td> <td>1</td> </tr> <tr> <td>GH</td> <td>623</td> <td>DEKEGLETTTYITSQEVQNS</td> <td>0.994</td> </tr> <tr> <td>GB</td> <td>21</td> <td>GAQTPEQPAPPATTVQPTAT</td> <td>1</td> </tr> <tr> <td>400</td> <td>TTPTSSPPSSPSPPAPSAAR</td> <td>1</td> </tr> <tr> <td>430</td> <td>RRRDAGNATTPVPPTAPGKS</td> <td>1</td> </tr> <tr> <td>257</td> <td>YKIVDYDNRGTNPQGERRAF</td> <td>1</td> </tr> <tr> <td>GL</td> <td>18</td> <td>LPTWGNWAYPCCHVTQLRAQ</td> <td>0.674</td> </tr> <tr> <td>GM</td> <td>357</td> <td>TPSPGRNRRRSSTSSSSSRS</td> <td>1</td> </tr> <tr> <td>GN</td> <td>25</td> <td>TGVLPAGASSPTNAAAASLT</td> <td>1</td> </tr> <tr> <td>GP350</td> <td>514</td> <td>TTPTPNATSPTPAVTTPTPN</td> <td>1</td> </tr> <tr> <td>535</td> <td>TSPTPAVTTPTPNATSPTLG</td> <td>1</td> </tr> <tr> <td>474</td> <td>TSPTPAGTTSGASPVTPSPS</td> <td>1</td> </tr> <tr> <td>598</td> <td>TSPTSAVTTPTPNATGPTVG</td> <td>1</td> </tr> <tr> <td>720</td> <td>PAPRPGTTSQASGPGNSSTS</td> <td>1</td> </tr> <tr> <td>577</td> <td>TSPTSAVTTPTPNATSPTLG</td> <td>1</td> </tr> <tr> <td>556</td> <td>TSPTSAVTTPTPNATSPTLG</td> <td>1</td> </tr> <tr> <td>423</td> <td>KAPESTTTSPTLNTTGFADP</td> <td>1</td> </tr> <tr> <td>835</td> <td>TSPPVTTAQATVPVPPTSQP</td> <td>1</td> </tr> <tr> <td>647</td> <td>TSAVTTGQHNITSSSTSSMS</td> <td>1</td> </tr> <tr> <td>243</td> <td>GILTSTSPVATPIPGTGYAY</td> <td>1</td> </tr> <tr> <td>452</td> <td>THVPTNLTAPASTGPTVSTA</td> <td>1</td> </tr> <tr> <td>626</td> <td>TNHTLGGTSPTPVVTSQPKN</td> <td>1</td> </tr> <tr> <td>746</td> <td>NVTKGTPPQNATSPQAPSGQ</td> <td>1</td> </tr> <tr> <td>767</td> <td>TAVPTVTSTGGKANSTTGGK</td> <td>1</td> </tr> </tbody> </table>  <p /> </div> <div class="surfaceaccessibilityofebv" title="sec"> <div class="title" tagx="title" title="title">
Surface accessibility of EBV</div> <p>Threshold values &amp;gt;1 were set to predict the surface probability values. Amino acids with higher surface probability values (&amp;gt;1) have greater probability to be present on protein surfaces <sup> <a href="#CR36">36</a> </sup>. The maximum surface probability scores for Glycoprotein B (RRRRRD <sub>428–433</sub>), Glycoprotein H (EREDRD <sub>520–525</sub>) <sub>,</sub> Glycoprotein L (KNGSNQ <sub>68–73</sub>), Glycoprotein M (RNRRRS <sub>362–367</sub>), Glycoprotein N (TEAQDQ <sub>44–49</sub>), Glycoprotein 42 (TKKKHT <sub>199–124</sub>), and Glycoprotein 350 (PRPRYN <sub>810–815</sub>) were 9.415, 9.265, 4.395, 9.054, 4.777, 5.691 and 4.859, respectively. The minimum surface probability scores were 0.032 (VVILVI <sub>745–750</sub>), 0.033 (CVFCLV <sub>5–10</sub>), 0.071 (LAICLV <sub>8–13</sub>), 0.067 (IIPILC <sub>309–314</sub>), 0.07 (LVLVII <sub>76–81</sub>), 0.054 (VIVLLL <sub>18–23</sub>), and 0.058 (AALLVC <sub>3–8</sub>). Figure <a href="#MOESM1">S1</a> (Supplementary Materials) shows the graphical representation of the predicted surface accessibility of EBV. Moreover, for all of the other proteins which have maximum and minimum accessibility scores are shown in Table <a href="#MOESM1">S1</a> (Supplementary Materials). </p> </div> <div class="surfaceflexibilityofebvselectedproteins" title="sec"> <div class="title" tagx="title" title="title">
Surface flexibility of EBV selected proteins</div> <p>The Karplus and Schulz flexibility method was used to calculate the motions of atoms (back and forth, considering temperature or B factor). Low B-factor values indicate a highly systematic structure, and high B factors indicate a distorted structure 32. The graphical representation of the surface flexibility results for EBV is shown in Fig. <a href="#MOESM1">S3</a>. Maximum flexibility scores for Glycoprotein B, Glycoprotein H, Glycoprotein L, Glycoprotein M, Glycoprotein N, Glycoprotein 42, and Glycoprotein 350 were 1.13, 1.094, 1.121, 1.157, 1.071, 1.091, 137 for heptapeptides EQNQEQK <sub>803–809,</sub> VITQGPN <sub>346–442</sub>, PKNGSNQ <sub>67–73</sub>, STSSSSS <sub>368–374</sub>, GASSPTN <sub>31–37</sub>, VRGGGRV <sub>31–37</sub>, PGNSSTS <sub>733–739</sub>, respectively. Minimum flexibility scores were 0.88, 0.872, 0.904, 0.866, 0.862, 0.897, and 0.894 for the peptides QAIMLAL <sub>795–801</sub>, LAAMLMA <sub>351–357</sub>, FLAICLV <sub>7–13</sub>, FLWWVVF <sub>193–199</sub>, IYLMYVC <sub>85–91</sub>, VAAAAIT <sub>37–43</sub>, and LLVMADC <sub>877–883</sub>, respectively. Figure <a href="#MOESM1">S2</a> (Supplementary Materials) shows a graphical representation of the predicted surface flexibility of EBV. Moreover, for all of the other proteins which have maximum and minimum flexibility scores are shown in Table <a href="#MOESM1">S2</a> (Supplementary Materials). </p> </div> <div class="parkerhydrophilicitypredictionforebv" title="sec"> <div class="title" tagx="title" title="title">
Parker Hydrophilicity Prediction for EBV</div> <p>Hydrophilicity of the predicted epitopes was calculated using the Parker hydrophilicity approach <sup> <a href="#CR35">35</a> </sup>. The graphical illustration of the predicted Parker hydrophilicity of EBV is shown in Fig. <a href="#MOESM1">S3</a> (Supplementary Materials). From all of the predicted EBV peptides, the maximum hydrophilicity calculated was 6.843 for Glycoprotein 350 at the amino acid positions DNGTESK <sub>496–502</sub>. These regions were predicted to act as active T-cell epitopes. The minimum hydrophilicity score calculated was −7.857 for Glycoprotein M at the amino acid positions FLWWVVF <sub>193–199</sub>. Moreover, for all of the other proteins which have maximum and minimum hydrophilicity scores are shown in Table <a href="#MOESM1">S3</a> (Supplementary Materials). </p> </div> <div class="t-cellepitopeidentification" title="sec"> <div class="title" tagx="title" title="title">
T-cell epitope identification</div> <p>Epitope predictions for all seven proteins were conducted on NetCTL, an online epitope prediction server. MHC-I binding prediction using the SMM method resulted in many potential epitopes against one allele, HLA-A*24:02. The weight matrix and artificial neural network was used for the prediction of MHC-I binding and proteasome dependent C-terminal cleavage. The MHC binding affinity, the TAP score, and the C-terminal cleavage score were considered to select the most promising epitopes among those predicted. The top three epitopes (as shown in Table <a href="#Tab4">4</a>) for glycoprotein B (ETDQMDTIY, QMDTIYQCY, PTTVMSSIY), glycoprotein L (MTAASYARY, LTSAQSGDY, ATSVLLSAY), glycoprotein H (ALENISDIY, LLTTLETLY, SSSALTGHL), glycoprotein N (IADCVAFIY, FLALGNSFY, TTDSEEEIF), glycoprotein M (LTEAQDQFY, IASAIYLMY, ASAIYLMYV), glycoprotein 42 (CAELYPCTY, HTFQVPQNY, NTREYTFSY), and glycoprotein 350 (PTNTTDITY, FLGNNSILY, HAEMQNPVY) were selected for docking. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  </div></p><p>List of the total peptides T-cell vaccines predicted by NetCTL.</p>  <table> <thead> <tr> <th /> <th>S.NO</th> <th>Peptide Sequence</th> <th>MHC Binding Affinity</th> <th>Rescale Binding Affinity</th> <th>C-terminal Cleavage Affinity</th> <th>Transport Affinity</th> <th>Prediction Score</th> <th>MHC-I Binding</th> </tr> </thead> <tbody> <tr> <td>GB</td> <td>131</td> <td>ETDQMDTIY</td> <td>0.7945</td> <td>3.3733</td> <td>0.6371</td> <td>2.4710</td> <td>3.5924</td> <td>Yes</td> </tr> <tr> <td>134</td> <td>QMDTIYQCY</td> <td>0.7028</td> <td>2.9841</td> <td>0.9533</td> <td>2.7440</td> <td>3.2643</td> <td>Yes</td> </tr> <tr> <td>502</td> <td>PTTVMSSIY</td> <td>0.5331</td> <td>2.2633</td> <td>0.6136</td> <td>2.5280</td> <td>2.4817</td> <td>Yes</td> </tr> <tr> <td>GL</td> <td>256</td> <td>MTAASYARY</td> <td>0.7300</td> <td>3.0993</td> <td>0.6470</td> <td>2.9770</td> <td>3.3452</td> <td>Yes</td> </tr> <tr> <td>216</td> <td>LTSAQSGDY</td> <td>0.7246</td> <td>3.0764</td> <td>0.6687</td> <td>2.9570</td> <td>3.3245</td> <td>Yes</td> </tr> <tr> <td>396</td> <td>ATSVLLSAY</td> <td>0.6201</td> <td>2.6327</td> <td>0.9581</td> <td>3.0130</td> <td>2.9271</td> <td>Yes</td> </tr> <tr> <td>GH</td> <td>41</td> <td>ALENISDIY</td> <td>0.4625</td> <td>1.9636</td> <td>0.9238</td> <td>2.9920</td> <td>2.2518</td> <td>Yes</td> </tr> <tr> <td>107</td> <td>LLTTLETLY</td> <td>0.3076</td> <td>1.3059</td> <td>0.9638</td> <td>2.7450</td> <td>1.5877</td> <td>Yes</td> </tr> <tr> <td>96</td> <td>SSSALTGHL</td> <td>0.1437</td> <td>0.6103</td> <td>0.9131</td> <td>1.1760</td> <td>0.8060</td> <td>Yes</td> </tr> <tr> <td>GN</td> <td>86</td> <td>IADCVAFIY</td> <td>0.6262</td> <td>2.6588</td> <td>0.9587</td> <td>2.7190</td> <td>2.9385</td> <td>Yes</td> </tr> <tr> <td>225</td> <td>FLALGNSFY</td> <td>0.5351</td> <td>2.2720</td> <td>0.8655</td> <td>2.9130</td> <td>2.9385</td> <td>Yes</td> </tr> <tr> <td>396</td> <td>TTDSEEEIF</td> <td>0.4510</td> <td>1.9148</td> <td>0.2635</td> <td>2.3520</td> <td>2.0719</td> <td>Yes</td> </tr> <tr> <td>GM</td> <td>43</td> <td>LTEAQDQFY</td> <td>0.7612</td> <td>3.2319</td> <td>0.6749</td> <td>2.8170</td> <td>3.4740</td> <td>Yes</td> </tr> <tr> <td>81</td> <td>IASAIYLMY</td> <td>0.5205</td> <td>2.2100</td> <td>0.7658</td> <td>3.0050</td> <td>2.4752</td> <td>Yes</td> </tr> <tr> <td>82</td> <td>ASAIYLMYV</td> <td>0.1686</td> <td>0.7159</td> <td>0.5794</td> <td>0.5250</td> <td>0.8291</td> <td>Yes</td> </tr> <tr> <td>GP42</td> <td>132</td> <td>CAELYPCTY</td> <td>0.4738</td> <td>2.0117</td> <td>0.9710</td> <td>2.9460</td> <td>2.3047</td> <td>Yes</td> </tr> <tr> <td>86</td> <td>HTFQVPQNY</td> <td>0.4422</td> <td>1.8777</td> <td>0.9494</td> <td>2.9720</td> <td>2.1687</td> <td>Yes</td> </tr> <tr> <td>103</td> <td>NTREYTFSY</td> <td>0.3707</td> <td>1.5740</td> <td>0.9745</td> <td>2.9820</td> <td>1.8693</td> <td>Yes</td> </tr> <tr> <td>GP350</td> <td>316</td> <td>PTNTTDITY</td> <td>0.5572</td> <td>2.3659</td> <td>0.9498</td> <td>2.3650</td> <td>2.6266</td> <td>Yes</td> </tr> <tr> <td>274</td> <td>FLGNNSILY</td> <td>0.4340</td> <td>1.8426</td> <td>0.9710</td> <td>2.8180</td> <td>2.1291</td> <td>Yes</td> </tr> <tr> <td>143</td> <td>HAEMQNPVY</td> <td>0.4028</td> <td>1.7103</td> <td>0.7631</td> <td>2.7190</td> <td>1.9607</td> <td>Yes</td> </tr> </tbody> </table>  <p /> </div> <div class="peptidemodellinganddockingstudiesforhla-a*24:02andepitopeinteractionanalysis" title="sec"> <div class="title" tagx="title" title="title">
Peptide modelling and docking studies for HLA-A*24:02 and epitope interaction analysis</div> <p>The selected top three epitopes from all proteins were docked against HLA-A*24:02 using Fire dock. The epitope QMDTIYQCY (glycoprotein B) was docked to understand the interaction pattern between peptide-MHC complexes. The global docking energy and van der Waals (vdW) energy were reported as −35.20 (kcal/mol) and −25.12 (kcal/mol), respectively. Residues Gln7, Cys8 and Asp3 from the docked peptides and Thr143, Tyr116, Thr80 and Lys146 from the MHC molecules were involved in binding. Peptide MTAASYARY (glycoprotein H) was reported to share a global energy of −34.27 (kcal/mol), with −29.12 (kcal/mol) vdW energy. On the other hand, peptides from Glycoprotein M (TTDSEEEIF), Glycoprotein N (LTEAQDQFY), Glycoprotein 42 (CAELYPCTY), and Glycoprotein 350 (PTNTTDITY) contributed global energies of −36.20 (kcal/mol), −34.25 (kcal/mol), −40.20 (kcal/mol), and −30.48 (kcal/mol), respectively. The vdW interaction energies for these complexes ranged from −29.71(kcal/mol) to −18.80 (kcal/mol). Residues Lys66, Arg97, Tyr99, Gln155, His114, and Thr163 from these complexes were uniformly involved in hydrogen bonding interactions. The Chimera interaction analysis tool predicted the interactions between peptide and MHC-I molecules within 3Ǻ. Overall, the stability was supported by the variable amounts of hydrogen bonding. The molecular interaction patterns are depicted in Fig. <a href="#Fig3">3</a>, while the interacting atoms are shown in Table <a href="#Tab5">5</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Interaction pattern of the docked peptides against MHC I molecules.</p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  <p>Molecular Docking analysis of the final peptides.</p>  <table> <thead> <tr> <th>Peptide</th> <th>Global Energy (kcal/mol)</th> <th>vdW Energy (kcal/mol)</th> <th>H-Bond Energy (kcal/mol)</th> <th>H-Bond Interaction</th> </tr> <tr> <th>Peptide-MHC atom pair</th> <th>d <sub>init</sub> (Å) </th> </tr> </thead> <tbody> <tr> <td>QMDTIYQCY</td> <td>−35.20</td> <td>−25.12</td> <td>−1.09</td> <td>GLN7 NE2-THR143 OG1</td> <td>3.39</td> </tr> <tr> <td>Glycoprotein B</td> <td>CYS8 N-TYR116 OH</td> <td>3.77</td> </tr> <tr> <td>ASP3 OD1-THR80 OG1</td> <td>2.62</td> </tr> <tr> <td>ASP3OD2-LYS146 NZ</td> <td>3.95</td> </tr> <tr> <td>MTAASYARY</td> <td>−34.27</td> <td>−29.12</td> <td>−3.82</td> <td>GLN5 NE2-GLU63 O</td> <td>3.35</td> </tr> <tr> <td>Glycoprotein H</td> <td>GLN5 NE2-GLU63 OE2</td> <td>3.60</td> </tr> <tr> <td>SER6 N-TYR99 OH</td> <td>3.47</td> </tr> <tr> <td>TYR9 N-GLN155 OE1</td> <td>2.42</td> </tr> <tr> <td>THR2 O-THR73 OG1</td> <td>3.56</td> </tr> <tr> <td>THR2 O-ARG97 NH1</td> <td>3.63</td> </tr> <tr> <td>SER3 OG-HIS70 NE2</td> <td>3.01</td> </tr> <tr> <td>GLN5 OE1-TYR99 OH</td> <td>3.77</td> </tr> <tr> <td>ASP8 OD2-HIS114 NE2</td> <td>3.79</td> </tr> <tr> <td>ASP8 OD2-ARG97 NH2</td> <td>3.76</td> </tr> <tr> <td>MTAASYARY</td> <td>−34.26</td> <td>−18.80</td> <td>−1.36</td> <td>GLN5 NE2-THR163 OG1</td> <td>2.74</td> </tr> <tr> <td>Glycoprotein H</td> <td>SER6 O-TYR99 OH</td> <td>2.56</td> </tr> <tr> <td>GLY7 O-ARG7 NH1</td> <td>2.22</td> </tr> <tr> <td>TYR9 OH-TRP147 NE1</td> <td>3.52</td> </tr> <tr> <td>TTDSEEEIF</td> <td>−36.20</td> <td>−29.71</td> <td>−4.82</td> <td>THR1 OG1-GLU63 OE2</td> <td>3.03</td> </tr> <tr> <td>Glycoprotein M</td> <td>THR1 O-LYS66 NZ</td> <td>2.89</td> </tr> <tr> <td>GLU7 OE1-ARG97 NH2</td> <td>3.72</td> </tr> <tr> <td>GLU7 OE2-ARG97 NE</td> <td>3.69</td> </tr> <tr> <td>GLU7 OE2-ARG97 NH2</td> <td>3.02</td> </tr> <tr> <td>GLU7 OE2-HIS114 NE2</td> <td>3.24</td> </tr> <tr> <td>LTEAQDQFY</td> <td>−34.25</td> <td>−24.49</td> <td>−3.30</td> <td>LEU1 N-ARG65 O</td> <td>3.65</td> </tr> <tr> <td>Glycoprotein N</td> <td>GLU3 OE2-TYR99 OH</td> <td>2.17</td> </tr> <tr> <td>ASP6 O-LEU156 N</td> <td>3.88</td> </tr> <tr> <td>GLN7 OE1-HIS70 NE2</td> <td>3.04</td> </tr> <tr> <td>GLU3 OE1-LYS66 NZ</td> <td>3.60</td> </tr> <tr> <td>GLU3 OE2-HIS70 NE2</td> <td>3.32</td> </tr> <tr> <td>CAELYPCTY</td> <td>−40.20</td> <td>−23.44</td> <td>−1.23</td> <td>CYS7 N-THR163 OG1</td> <td>3.50</td> </tr> <tr> <td>Glycoprotein 42</td> <td>THR8 OG1-THR163 O</td> <td>2.95</td> </tr> <tr> <td>TYR9 N-TYR159 OH</td> <td>3.74</td> </tr> <tr> <td>GLU3 O-GLN155 NE2</td> <td>3.13</td> </tr> <tr> <td>PRO6 O-THR163 OG1</td> <td>2.03</td> </tr> <tr> <td>THR8 O-LYS66 NZ</td> <td>2.02</td> </tr> <tr> <td>PTNTTDITY</td> <td>−30.48</td> <td>−29.45</td> <td>−3.12</td> <td>THR5 OG1-THR143 OG1</td> <td>3.48</td> </tr> <tr> <td>Glycoprotein 350</td> <td>ASN3 OD1-TRP147 N</td> <td>3.54</td> </tr> <tr> <td>THR4 O-THR80 OG1</td> <td>3.58</td> </tr> <tr> <td>THR4 OG1-LYS146 NZ</td> <td>2.53</td> </tr> <tr> <td>ASP6 O-TRP147 NE1</td> <td>3.57</td> </tr> <tr> <td>ASP6 OD2-ARG97 NH2</td> <td>3.50</td> </tr> <tr> <td>ASP6 OD1-ARG97 NH2</td> <td>3.89</td> </tr> <tr> <td>ASP6 OD2-ARG97 NH2</td> <td>3.50</td> </tr> <tr> <td>ASP6 OD2-HIS114 NE2</td> <td>3.27</td> </tr> </tbody> </table> </div> <p /> </div> <div class="pkpdmodelling-basedvalidation" title="sec"> <div class="title" tagx="title" title="title">
PKPD modelling-based validation</div> <p>In a time-course simulation with the shortlisted peptides (obtained from the screening), the initial concentration of the top-listed peptides was set at 0.45 µm <sup> <a href="#CR54">54</a>– <a href="#CR56">56</a> </sup>, and after 20 seconds, all of the interactions of the EBV mechanism in the cell signalling cascade were stabilized given in Fig. <a href="#Fig4">4</a> and <a href="#Fig5">5</a>. EBV is a Baltimore Class I virus of the Herpesviridae family and plays a crucial role in lymphoproliferative disease. During tropic EBV infection, EBV binds to the HLA class II molecule, and B cells inhibit epithelial cell fusion, while the GH receptor protein interacts with GP42, and GL is transported to the cell surface where it is essential for the correct folding of GH. EBV GB is important for viral fusion events with B cells. Glycoprotein 350 binding is supported by the binding of EBV gp42 to B-cell MHC-II, while fusion of the B-cell membrane and the outer viral envelope of EBV virion must have functional spicule glycoproteins, such as GH, GL and gp42. Vaccines development requires the identification of extremely competent B-cell linear or nonlinear and CTL epitopes, where T cells act as mediators. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Model of the biochemical pathway of MDS in the presence of peptides.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>The time-course simulation with screened peptides against relation of interacting EBV. </p>   </div> <p /> <p>A systems biology approach is useful to investigate T-cell epitopes in peptide sequences, and earlier reports have accelerated research leading to the development of immune biology. </p> </div> <div class="rootmeansquaredeviation(rmsd)" title="sec"> <div class="title" tagx="title" title="title">
Root Mean Square Deviation (RMSD)</div> <p>Molecular dynamics simulation was conducted to confirm the post-docking stability of the complexes. Trajectories were obtained after 50 ns and subjected to backbone stability using RMSD. RMSD of the selected complexes after 50 ns revealed that all of the complexes were stable and that the peptides had occupied the binding grooves of MHC-I molecules. RMSDs of all the complexes were calculated and plotted on the graph shown in Fig. <a href="#Fig6">6</a>. The RMSDs of all of the complexes ranged from 0.08 to 0.2 nm. These results show that small fluctuations were observed during the simulation time, but these fluctuations are likely because some of the peptides are modelled as loop structures. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Root mean square deviation (RMSD) of the peptide-MHC complexes showing the stability of the peptides in the binding cavity of the MHC molecule. </p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Vaccination is one of the best options for providing immunity against different pathogenic organisms, thus delivering protection against different diseases. Different types of vaccines, such as peptide, conjugated, subunit, and DNA vaccines, can be used to provoke the immune response. However, in this postgenomic and proteomic era, researchers prefer peptide or subunit vaccines over the whole pathogenic agent due to the availability and accessibility of huge data sets for different pathogens. These data can be systematically analyzed through the use of computational tools. Presentation of MHC-Antigen on the surface of T-cells provide a way to kill the infected cell. This presentation of MHC-antigen complex direct apoptosis or self-eating process of the infected cell. The antigenic element of the pathogen provoke the immune response by activating a signalling process. The peptide fragment bound to MHC molecule is primarily presented T-cell which significantly rely on different factors including proteasome cleavage and transport with the aid of ER. TAP which transporting channels or proteins help in the transport to the surface of the cell. Therefore, considering the c-terminal cleavage activity and TAP efficiency greatly help in the selection of effective vaccine candidates <sup> <a href="#CR31">31</a>, <a href="#CR57">57</a>– <a href="#CR59">59</a> </sup>. Immunoinformatic approaches have contributed greatly to the development of vaccines. Therefore, we employed these tools to design peptide vaccines against EBV to provide a means to protect humanity from the multiple diseases caused by EBV, such as infectious mononucleosis, Burkitt’s lymphoma <sup> <a href="#CR60">60</a> </sup>, Hodgkin’s lymphoma <sup> <a href="#CR61">61</a> </sup>, stomach cancer, laryngeal carcinoma <sup> <a href="#CR62">62</a> </sup>, multiple sclerosis <sup> <a href="#CR63">63</a>, <a href="#CR64">64</a> </sup> and lymphomatoid granulomatosis <sup> <a href="#CR65">65</a> </sup>. Additional diseases that have been linked to EBV include Giannotti–Crosstie syndrome, erythema multiform, acute genital ulcers, and oral hairy leukoplakia <sup> <a href="#CR66">66</a> </sup>; furthermore, hypersensitivity to mosquito bites has been associated with EBV infection <sup> <a href="#CR67">67</a> </sup>. EBV has been implicated in disorders related to alpha-synuclein aggregation (e.g., Parkinson’s disease, dementia with Lewy bodies, and multiple system atrophy) <sup> <a href="#CR68">68</a> </sup>. The proteome of EBV has many important functional proteins involved not only in its pathogenesis but also in the maintenance of the pathogenic condition. EBV has been reported to use its glycoproteins, such as glycoprotein B, glycoprotein L, glycoprotein H, glycoprotein N, glycoprotein M, glycoprotein 42, and glycoprotein 350, to attach to and infect its host cell. Therefore, predicting and validating both B-cell and T-cell epitopes from these proteins are important steps to provide immunity against these various diseases. Based on MHC binding affinity, TAP score, C-terminal cleavage score, molecular docking and MD simulation, we propose Glycoprotein B (QMDTIYQCY <sub>134–142</sub>), Glycoprotein L (MTAASYARY <sub>256–264</sub>), Glycoprotein N (TTDSEEEIF <sub>396–404</sub>), Glycoprotein M (LTEAQDQFY <sub>43–52</sub>), Glycoprotein 42 (CAELYPCTY <sub>132–140</sub>), and Glycoprotein 350 (PTNTTDITY <sub>316–324</sub>) as the final T-cell epitopes that could provoke the immune response in the host cell. The systems biology approach with pharmacokinetic/dynamics modelling validated that these epitopes significantly activates the immune response pathway and, thus, provide a strong basis for the testing of these epitopes under <i>in vivo</i> conditions. Structural stability analysis of the peptide-MHC complexes also revealed the stability of these immunogenic complexes. Antigenic and allergenic profiles also confirmed that these epitopes are strong candidates. Furthermore, B-cell epitopes were reported as the primary choice for the development of a B-cell immune response. </p> <p>This study provides a means for the development of peptide-based vaccines against EBV infection that could prevent many important diseases. To date, no such computational meta-analysis integrated with dynamics has been reported for the purpose of developing a peptide vaccine for EBV. This multiple step process has noticeably increased the scope and precision of this study. Our results will facilitate efficient subsequent experimental efforts, as the specified regions from Glycoprotein B (QMDTIYQCY <sub>134–142</sub>), Glycoprotein L (MTAASYARY <sub>256–264</sub>), Glycoprotein N (TTDSEEEIF <sub>396–404</sub>), Glycoprotein M (LTEAQDQFY <sub>43–52</sub>), Glycoprotein 42 (CAELYPCTY <sub>132–140</sub>), and Glycoprotein 350 (PTNTTDITY <sub>316–324</sub>) could be used for the development of candidate CTL epitopes. This study will aid in the progress of peptide vaccines against EBV. </p> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>It is well known that EBV causes many human diseases, including cancer. This study integrated multiple approaches to elucidate possible effective peptide vaccines that could provide protection against multiple infections. This study provides insight into the disease-causing factors of EBV virus and, thus, finalized potential B-cell and T-cell epitopes that will aid in the development of effective vaccines. Despite the prediction and validation of such peptides computationally, testing of our predicted epitopes should be carried out in animal models. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_37070_MOESM1_ESM.pdf">LINK</a>  <p>Supplementary Material</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p> <b>Supplementary information</b> accompanies this paper at 10.1038/s41598-018-37070-z. </p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Part of our computations was carried out at the High Performance Computing Center of Shanghai Jiao Tong University. This work is supported by the Key Research Area Grant 2016YFA0501703 from the Ministry of Science and Technology of China, and Ph.D Programs Foundation of Ministry of Education of China (Contract No., 20120073110057). </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>A.A. has made contribution in the acquisition, analysis and interpretation of the data and drafted the manuscript. A.K., A.C.K. and M.J. involved in the design of the study. D.Q.W. is the academic supervisor and involved in the supervision of the study. W.C.C. and S.S.A. for the critical evaluationof the manuscript. S.S., X.M. and Y.W. is involved in the revision proofreading of the manuscript. All authors read and approved the final manuscript. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chesnokova</span> <span class="given-names" tagx="given-names" title="given-names">LS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishimura</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hutt-Fletcher</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fusion of epithelial cells by Epstein–Barr virus proteins is triggered by binding of viral glycoproteins gHgL to integrins αvβ6 or αvβ8 </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">106</span> <span class="fpage" tagx="fpage" title="fpage">20464</span> <span class="lpage" tagx="lpage" title="lpage">20469</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0907508106">10.1073/pnas.0907508106</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">LS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woodman</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murray</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epstein-Barr virus (EBV) and its associated human cancers–genetics, epigenetics, pathobiology and novel therapeutics </span> <span class="source" tagx="source" title="source">Front Biosci</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">2672</span> <span class="lpage" tagx="lpage" title="lpage">2713</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2741/2000">10.2741/2000</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16720343">16720343</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maeda</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Spectrum of Epstein-Barr virus-related diseases: a pictorial review</span> <span class="source" tagx="source" title="source">Japanese journal of radiology</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">4</span> <span class="lpage" tagx="lpage" title="lpage">19</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11604-008-0291-2">10.1007/s11604-008-0291-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19373526">19373526</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">WC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nasopharyngeal carcinoma: molecular biomarker discovery and progress</span> <span class="source" tagx="source" title="source">Mol Cancer</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1476-4598-6-1">10.1186/1476-4598-6-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17199893">17199893</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abbott</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Asymptomatic primary infection with Epstein-Barr virus: observations on young adult cases </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="fpage" tagx="fpage" title="fpage">e00382</span> <span class="lpage" tagx="lpage" title="lpage">00317</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00382-17">10.1128/JVI.00382-17</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28835490">28835490</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Odumade</span> <span class="given-names" tagx="given-names" title="given-names">OA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hogquist</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balfour</span> <span class="given-names" tagx="given-names" title="given-names">HH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Progress and Problems in Understanding and Managing Primary Epstein-Barr Virus Infections</span> <span class="source" tagx="source" title="source">Clinical Microbiology Reviews</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">193</span> <span class="lpage" tagx="lpage" title="lpage">209</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/cmr.00044-10">10.1128/cmr.00044-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21233512">21233512</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgan</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epstein—Barr virus vaccines</span> <span class="source" tagx="source" title="source">Vaccine</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">563</span> <span class="lpage" tagx="lpage" title="lpage">571</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0264-410X(92)90434-L">10.1016/0264-410X(92)90434-L</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1323899">1323899</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">LS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rickinson</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epstein–Barr virus: 40 years on</span> <span class="source" tagx="source" title="source">Nature Reviews Cancer</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">757</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrc1452">10.1038/nrc1452</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15510157">15510157</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebowitz</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kieff</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An EBV membrane protein expressed in immortalized lymphocytes transforms established rodent cells </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">831</span> <span class="lpage" tagx="lpage" title="lpage">840</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0092-8674(85)90256-9">10.1016/0092-8674(85)90256-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3000618">3000618</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Epstein</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Achong</span> <span class="given-names" tagx="given-names" title="given-names">BG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barr</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Virus particles in cultured lymphoblasts from Burkitt’s lymphoma</span> <span class="source" tagx="source" title="source">The Lancet</span> <span class="year" tagx="year" title="year">1964</span> <span class="volume" tagx="volume" title="volume">283</span> <span class="fpage" tagx="fpage" title="fpage">702</span> <span class="lpage" tagx="lpage" title="lpage">703</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0140-6736(64)91524-7">10.1016/S0140-6736(64)91524-7</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jardetzky</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Longnecker</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Epstein-Barr virus (EBV) glycoprotein B cytoplasmic C-terminal tail domain regulates the energy requirement for EBV-induced membrane fusion </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">11686</span> <span class="lpage" tagx="lpage" title="lpage">11695</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.01349-14">10.1128/JVI.01349-14</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25100836">25100836</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turk</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hutt-Fletcher</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Epstein-Barr virus (EBV) BZLF2 gene product associates with the gH and gL homologs of EBV and carries an epitope critical to infection of B cells but not of epithelial cells </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">3987</span> <span class="lpage" tagx="lpage" title="lpage">3994</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7539502">7539502</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buranathai</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grose</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hutt-Fletcher</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chaperone functions common to nonhomologous Epstein-Barr virus gL and Varicella-Zoster virus gL proteins </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">71</span> <span class="fpage" tagx="fpage" title="fpage">1667</span> <span class="lpage" tagx="lpage" title="lpage">1670</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8995697">8995697</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reimer</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Backovic</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deshpande</span> <span class="given-names" tagx="given-names" title="given-names">CG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jardetzky</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Longnecker</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of Epstein-Barr virus glycoprotein B functional domains via linker insertion mutagenesis </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">83</span> <span class="fpage" tagx="fpage" title="fpage">734</span> <span class="lpage" tagx="lpage" title="lpage">747</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.01817-08">10.1128/JVI.01817-08</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18987135">18987135</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khyatti</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">PC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stefanescu</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Menezes</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epstein-Barr virus (EBV) glycoprotein gp350 expressed on transfected cells resistant to natural killer cell activity serves as a target antigen for EBV-specific antibody-dependent cellular cytotoxicity </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">1991</span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">996</span> <span class="lpage" tagx="lpage" title="lpage">1001</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1846213">1846213</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borza</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hutt-Fletcher</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Alternate replication in B cells and epithelial cells switches tropism of Epstein–Barr virus </span> <span class="source" tagx="source" title="source">Nat Med</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">594</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nm0602-594">10.1038/nm0602-594</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12042810">12042810</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kirschner</span> <span class="given-names" tagx="given-names" title="given-names">AN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Omerović</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Popov</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Longnecker</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jardetzky</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Soluble Epstein-Barr virus glycoproteins gH, gL, and gp42 form a 1: 1: 1 stable complex that acts like soluble gp42 in B-cell fusion but not in epithelial cell fusion </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">80</span> <span class="fpage" tagx="fpage" title="fpage">9444</span> <span class="lpage" tagx="lpage" title="lpage">9454</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00572-06">10.1128/JVI.00572-06</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16973550">16973550</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spear</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Longnecker</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Herpesvirus entry: an update</span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">10179</span> <span class="lpage" tagx="lpage" title="lpage">10185</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.77.19.10179-10185.2003">10.1128/JVI.77.19.10179-10185.2003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12970403">12970403</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dubey, K. K. <i>et al</i>. Vaccine and antibody production in plants: developments and computational tools. <i>Brief Funct Genomics</i>, 10.1093/bfgp/ely020 (2018). </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ip</span> <span class="given-names" tagx="given-names" title="given-names">PP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nijman</span> <span class="given-names" tagx="given-names" title="given-names">HW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daemen</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epitope prediction assays combined with validation assays strongly narrows down putative cytotoxic T lymphocyte epitopes </span> <span class="source" tagx="source" title="source">Vaccines</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">203</span> <span class="lpage" tagx="lpage" title="lpage">220</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/vaccines3020203">10.3390/vaccines3020203</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26343185">26343185</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gillespie</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional heterogeneity and high frequencies of cytomegalovirus-specific CD8+ T lymphocytes in healthy seropositive donors </span> <span class="source" tagx="source" title="source">J Virol</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">8140</span> <span class="lpage" tagx="lpage" title="lpage">8150</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.74.17.8140-8150.2000">10.1128/JVI.74.17.8140-8150.2000</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10933725">10933725</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doering</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsudaira</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cysteine scanning mutagenesis at 40 of 76 positions in villin headpiece maps the F-actin binding site and structural features of the domain </span> <span class="source" tagx="source" title="source">Biochemistry</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">12677</span> <span class="lpage" tagx="lpage" title="lpage">12685</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/bi9615699">10.1021/bi9615699</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8841111">8841111</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Regenmortel</span> <span class="given-names" tagx="given-names" title="given-names">MH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mapping epitope structure and activity: from one-dimensional prediction to four-dimensional description of antigenic specificity </span> <span class="source" tagx="source" title="source">Methods (San Diego, Calif.)</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">465</span> <span class="lpage" tagx="lpage" title="lpage">472</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/meth.1996.0054">10.1006/meth.1996.0054</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Florea</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epitope prediction algorithms for peptide-based vaccine design</span> <span class="source" tagx="source" title="source">Proc IEEE Comput Soc Bioinform Conf</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">17</span> <span class="lpage" tagx="lpage" title="lpage">26</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16826643">16826643</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A systematic bioinformatics approach for selection of epitope-based vaccine targets</span> <span class="source" tagx="source" title="source">Cellular immunology</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">244</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">147</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cellimm.2007.02.005">10.1016/j.cellimm.2007.02.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17434154">17434154</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Computational identification, characterization and validation of potential antigenic peptide vaccines from hrHPVs E6 proteins using immunoinformatics and computational systems biology approaches </span> <span class="source" tagx="source" title="source">PloS one</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">e0196484</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0196484">10.1371/journal.pone.0196484</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29715318">29715318</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brusic</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petrovsky</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Immunoinformatics and its relevance to understanding human immune disease</span> <span class="source" tagx="source" title="source">Expert review of clinical immunology</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">145</span> <span class="lpage" tagx="lpage" title="lpage">157</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1586/1744666X.1.1.145">10.1586/1744666X.1.1.145</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20477662">20477662</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larsen</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Large-scale validation of methods for cytotoxic T-lymphocyte epitope prediction</span> <span class="source" tagx="source" title="source">BMC bioinformatics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">424</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-8-424">10.1186/1471-2105-8-424</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17973982">17973982</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar Singh</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Systems biology as an approach for deciphering microbial interactions</span> <span class="source" tagx="source" title="source">Briefings in Functional Genomics</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">166</span> <span class="lpage" tagx="lpage" title="lpage">168</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bfgp/elu023">10.1093/bfgp/elu023</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24994863">24994863</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dangi</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sinha</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dwivedi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">&amp;amp; Shukla, P. Cell Line Techniques and Gene Editing Tools for Antibody Production: A Review </span> <span class="source" tagx="source" title="source">Frontiers in Pharmacology</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">630</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fphar.2018.00630">10.3389/fphar.2018.00630</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29946262">29946262</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lazarski</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The kinetic stability of MHC class II: peptide complexes is a key parameter that dictates immunodominance </span> <span class="source" tagx="source" title="source">Immunity</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">29</span> <span class="lpage" tagx="lpage" title="lpage">40</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.immuni.2005.05.009">10.1016/j.immuni.2005.05.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16039577">16039577</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kelley, L. A., Mezulis, S., Yates, C. M., Wass, M. N. &amp;amp; Sternberg, M. J. E. The Phyre2 web portal for protein modeling, prediction and analysis. <i>Nature Protocols</i> <b>10</b>, 845, 10.1038/nprot.2015.053 <a href="http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index">(http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id = index)</a> (2015). </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nair</span> <span class="given-names" tagx="given-names" title="given-names">DT</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epitope recognition by diverse antibodies suggests conformational convergence in an antibody response </span> <span class="source" tagx="source" title="source">The Journal of Immunology</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">168</span> <span class="fpage" tagx="fpage" title="fpage">2371</span> <span class="lpage" tagx="lpage" title="lpage">2382</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4049/jimmunol.168.5.2371">10.4049/jimmunol.168.5.2371</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11859128">11859128</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fieser</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tainer</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geysen</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Houghten</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lerner</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Influence of protein flexibility and peptide conformation on reactivity of monoclonal anti-peptide antibodies with a protein alpha-helix </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">8568</span> <span class="lpage" tagx="lpage" title="lpage">8572</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.84.23.8568">10.1073/pnas.84.23.8568</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parker</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodges</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">New hydrophilicity scale derived from high-performance liquid chromatography peptide retention data: correlation of predicted surface residues with antigenicity and X-ray-derived accessible sites </span> <span class="source" tagx="source" title="source">Biochemistry</span> <span class="year" tagx="year" title="year">1986</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">5425</span> <span class="lpage" tagx="lpage" title="lpage">5432</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/bi00367a013">10.1021/bi00367a013</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2430611">2430611</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emini</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hughes</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perlow</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boger</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Induction of hepatitis A virus-neutralizing antibody by a virus-specific synthetic peptide </span> <span class="source" tagx="source" title="source">Journal of virology</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">836</span> <span class="lpage" tagx="lpage" title="lpage">839</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2991600">2991600</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolaskar</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tongaonkar</span> <span class="given-names" tagx="given-names" title="given-names">PC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A semi-empirical method for prediction of antigenic determinants on protein antigens</span> <span class="source" tagx="source" title="source">FEBS letters</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">276</span> <span class="fpage" tagx="fpage" title="fpage">172</span> <span class="lpage" tagx="lpage" title="lpage">174</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0014-5793(90)80535-Q">10.1016/0014-5793(90)80535-Q</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1702393">1702393</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ponomarenko</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">ElliPro: a new structure-based tool for the prediction of antibody epitopes</span> <span class="source" tagx="source" title="source">BMC bioinformatics</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">514</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-9-514">10.1186/1471-2105-9-514</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19055730">19055730</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lamiable, A. <i>et al</i>. PEP-FOLD3: faster de novo structure prediction for linear peptides in solution and in complex. <i>Nucleic acids research</i>, gkw329 (2016). </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maupetit</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuffery</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Derreumaux</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A coarse‐grained protein force field for folding and structure prediction</span> <span class="source" tagx="source" title="source">Proteins: Structure, Function, and Bioinformatics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">394</span> <span class="lpage" tagx="lpage" title="lpage">408</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/prot.21505">10.1002/prot.21505</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Duhovny, D., Nussinov, R. &amp;amp; Wolfson, H. J. In <i>International Workshop on Algorithms in Bioinformatics</i>. 185–200 (Springer). </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneidman‐Duhovny</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Taking geometry to its edge: fast unbound rigid (and hinge‐bent) docking</span> <span class="source" tagx="source" title="source">Proteins: Structure, Function, and Bioinformatics</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">107</span> <span class="lpage" tagx="lpage" title="lpage">112</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/prot.10397">10.1002/prot.10397</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andrusier</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nussinov</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfson</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FireDock: fast interaction refinement in molecular docking</span> <span class="source" tagx="source" title="source">Proteins: Structure, Function, and Bioinformatics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">139</span> <span class="lpage" tagx="lpage" title="lpage">159</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/prot.21495">10.1002/prot.21495</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mashiach</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneidman-Duhovny</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andrusier</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nussinov</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfson</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FireDock: a web server for fast interaction refinement in molecular docking</span> <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">W229</span> <span class="lpage" tagx="lpage" title="lpage">W232</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkn186">10.1093/nar/gkn186</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18424796">18424796</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kingsford</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chazelle</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Solving and analyzing side-chain positioning problems using linear and integer programming</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">1028</span> <span class="lpage" tagx="lpage" title="lpage">1039</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bti144">10.1093/bioinformatics/bti144</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15546935">15546935</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Case, D. A. <i>et al</i>. Amber 14 (2014). </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salomon-Ferrer</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Götz</span> <span class="given-names" tagx="given-names" title="given-names">AW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poole</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le Grand</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 2. Explicit solvent particle mesh Ewald </span> <span class="source" tagx="source" title="source">Journal of chemical theory and computation</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">3878</span> <span class="lpage" tagx="lpage" title="lpage">3888</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ct400314y">10.1021/ct400314y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26592383">26592383</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roe</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheatham</span> <span class="given-names" tagx="given-names" title="given-names">TE</span> <span class="suffix" tagx="suffix" title="suffix">III</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">PTRAJ and CPPTRAJ: software for processing and analysis of molecular dynamics trajectory data </span> <span class="source" tagx="source" title="source">Journal of chemical theory and computation</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">3084</span> <span class="lpage" tagx="lpage" title="lpage">3095</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ct400341p">10.1021/ct400341p</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26583988">26583988</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krissinel</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henrick</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inference of macromolecular assemblies from crystalline state</span> <span class="source" tagx="source" title="source">Journal of molecular biology</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">372</span> <span class="fpage" tagx="fpage" title="fpage">774</span> <span class="lpage" tagx="lpage" title="lpage">797</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jmb.2007.05.022">10.1016/j.jmb.2007.05.022</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17681537">17681537</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">UCSF Chimera. A visualization system for exploratory research and analysis. Resource for Biocomputing, Visualization, and Informatics (RBVI), UCSF. <a href="http://www.rbvi.ucsf.edu/chimera/">http://www.rbvi.ucsf.edu/chimera/</a> (2004). </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">The PyMOL molecular graphics system. Schrödinger, Inc. https://www.pymol.org/ (2002).</span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saha</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghava</span> <span class="given-names" tagx="given-names" title="given-names">GP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">AlgPred: prediction of allergenic proteins and mapping of IgE epitopes</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">W202</span> <span class="lpage" tagx="lpage" title="lpage">209</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkl343">10.1093/nar/gkl343</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16844994">16844994</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Randall</span> <span class="given-names" tagx="given-names" title="given-names">AZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sweredoski</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baldi</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SCRATCH: a protein structure and structural feature prediction server</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">W72</span> <span class="lpage" tagx="lpage" title="lpage">76</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gki396">10.1093/nar/gki396</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15980571">15980571</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Epstein-Barr virus (EBV)-encoded protein kinase, EBV-PK, but not the thymidine kinase (EBV-TK), is required for ganciclovir and acyclovir inhibition of lytic viral production </span> <span class="source" tagx="source" title="source">Journal of virology</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">4534</span> <span class="lpage" tagx="lpage" title="lpage">4542</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.02487-09">10.1128/JVI.02487-09</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20181711">20181711</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pavić</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Flow cytometric analysis of herpes simplex virus type 1 susceptibility to acyclovir, ganciclovir, and foscarnet </span> <span class="source" tagx="source" title="source">Antimicrobial agents and chemotherapy</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">2686</span> <span class="lpage" tagx="lpage" title="lpage">2692</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AAC.41.12.2686">10.1128/AAC.41.12.2686</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9420039">9420039</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saijo</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzutani</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niikura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morikawa</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurane</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Importance of C‐terminus of herpes simplex virus type 1 thymidine kinase for maintaining thymidine kinase and acyclovir‐phosphorylation activities </span> <span class="source" tagx="source" title="source">Journal of medical virology</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">66</span> <span class="fpage" tagx="fpage" title="fpage">388</span> <span class="lpage" tagx="lpage" title="lpage">393</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/jmv.2157">10.1002/jmv.2157</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11793392">11793392</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Alberts, B. <i>et al</i>. <i>Molecular Biology of the Cell, Sixth Edition</i> (Taylor &amp;amp; Francis Group, 2014). </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brusic</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bajic</span> <span class="given-names" tagx="given-names" title="given-names">VB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petrovsky</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Computational methods for prediction of T-cell epitopes—a framework for modelling, testing, and applications </span> <span class="source" tagx="source" title="source">Methods</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">436</span> <span class="lpage" tagx="lpage" title="lpage">443</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymeth.2004.06.006">10.1016/j.ymeth.2004.06.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15542369">15542369</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lundegaard</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lund</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keşmir</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The role of the proteasome in generating cytotoxic T-cell epitopes: insights obtained from improved predictions of proteasomal cleavage </span> <span class="source" tagx="source" title="source">Immunogenetics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">33</span> <span class="lpage" tagx="lpage" title="lpage">41</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00251-005-0781-7">10.1007/s00251-005-0781-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15744535">15744535</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiss</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Malley</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Benign lymphadenopathies</span> <span class="source" tagx="source" title="source">Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="issue" tagx="issue" title="issue">Suppl 1</span> <span class="fpage" tagx="fpage" title="fpage">S88</span> <span class="lpage" tagx="lpage" title="lpage">96</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/modpathol.2012.176">10.1038/modpathol.2012.176</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pannone</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The role of EBV in the pathogenesis of Burkitt’s Lymphoma: an Italian hospital based survey </span> <span class="source" tagx="source" title="source">Infectious agents and cancer</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">34</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1750-9378-9-34">10.1186/1750-9378-9-34</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25364378">25364378</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mechelli</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epstein-Barr virus genetic variants are associated with multiple sclerosis</span> <span class="source" tagx="source" title="source">Neurology</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">1362</span> <span class="lpage" tagx="lpage" title="lpage">1368</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1212/WNL.0000000000001420">10.1212/WNL.0000000000001420</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25740864">25740864</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ascherio</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Munger</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epstein-barr virus infection and multiple sclerosis: a review</span> <span class="source" tagx="source" title="source">Journal of neuroimmune pharmacology: the official journal of the Society on NeuroImmune Pharmacology </span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">271</span> <span class="lpage" tagx="lpage" title="lpage">277</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11481-010-9201-3">10.1007/s11481-010-9201-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20369303">20369303</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tagliavini</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Lymphomatoid granulomatosis: a practical review for pathologists dealing with this rare pulmonary lymphoproliferative process </span> <span class="source" tagx="source" title="source">Pathologica</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">105</span> <span class="fpage" tagx="fpage" title="fpage">111</span> <span class="lpage" tagx="lpage" title="lpage">116</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24466760">24466760</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Di Lernia</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mansouri</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Epstein-Barr virus and skin manifestations in childhood</span> <span class="source" tagx="source" title="source">International journal of dermatology</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">1177</span> <span class="lpage" tagx="lpage" title="lpage">1184</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-4632.2012.05855.x">10.1111/j.1365-4632.2012.05855.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24073903">24073903</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kyriakidis</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Primary EBV infection and hypersensitivity to mosquito bites: a case report</span> <span class="source" tagx="source" title="source">Virologica Sinica</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">517</span> <span class="lpage" tagx="lpage" title="lpage">520</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12250-016-3868-4">10.1007/s12250-016-3868-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27900557">27900557</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woulfe</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoogendoorn</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tarnopolsky</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Munoz</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Monoclonal antibodies against Epstein-Barr virus cross-react with alpha-synuclein in human brain </span> <span class="source" tagx="source" title="source">Neurology</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">1398</span> <span class="lpage" tagx="lpage" title="lpage">1401</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1212/WNL.55.9.1398">10.1212/WNL.55.9.1398</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11087792">11087792</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">McGrath, P. Cancer virus discovery helped by delayed flight. <i>BBC World Service.–2004. Available at</i> <a href="http://www.bbc.com/news/health-26857610">http://www.bbc.com/news/health-26857610</a> (2015). </span> </li> </ul> </div> </div>  </body></html>