<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Nat Commun</span> <span class="iso-abbrev" title="iso-abbrev">Nat Commun</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6376029">6376029</a> </span> <span class="publisher-id" title="publisher-id">8599</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41467-019-08599-y">10.1038/s41467-019-08599-y</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Diversity within the adenovirus fiber knob hypervariable loops influences primary receptor interactions </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baker</span> <span class="given-names" tagx="given-names" title="given-names">Alexander T.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greenshields-Watson</span> <span class="given-names" tagx="given-names" title="given-names">Alexander</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9880-6560</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coughlan</span> <span class="given-names" tagx="given-names" title="given-names">Lynda</span> </span> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davies</span> <span class="given-names" tagx="given-names" title="given-names">James A.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uusi-Kerttula</span> <span class="given-names" tagx="given-names" title="given-names">Hanni</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cole</span> <span class="given-names" tagx="given-names" title="given-names">David K.</span> </span> <a href="#Aff2">2</a> <a href="#Aff4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rizkallah</span> <span class="given-names" tagx="given-names" title="given-names">Pierre J.</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9302-1761</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parker</span> <span class="given-names" tagx="given-names" title="given-names">Alan L.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">ParkerAL@Cardiff.ac.uk</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0807 5670</span> <span class="institution-id" title="institution-id">grid.5600.3</span> <span class="institution" title="institution">Division of Cancer and Genetics, School of Medicine, </span> <span class="institution" title="institution">Cardiff University, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0807 5670</span> <span class="institution-id" title="institution-id">grid.5600.3</span> <span class="institution" title="institution">Division of Infection and Immunity, School of Medicine, </span> <span class="institution" title="institution">Cardiff University, </span> </div> </span> <span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0670 2351</span> <span class="institution-id" title="institution-id">grid.59734.3c</span> <span class="institution" title="institution">Department of Microbiology, </span> <span class="institution" title="institution">Icahn School of Medicine at Mount Sinai, </span> </div> </span> <span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 0485 7917</span> <span class="institution-id" title="institution-id">grid.450850.c</span> <span class="institution" title="institution">Present Address: Immunocore Ltd., </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-2-2</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-2-2</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">10</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">741</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-8-22</span> <span class="accepted" title="accepted">accepted: 2019-1-21</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Adenovirus based vectors are of increasing importance for wide ranging therapeutic applications. As vaccines, vectors derived from human adenovirus species D serotypes 26 and 48 (HAdV-D26/48) are demonstrating promising efficacy as protective platforms against infectious diseases. Significant clinical progress has been made, yet definitive studies underpinning mechanisms of entry, infection, and receptor usage are currently lacking. Here, we perform structural and biological analysis of the receptor binding fiber-knob protein of HAdV-D26/48, reporting crystal structures, and modelling putative interactions with two previously suggested attachment receptors, CD46 and Coxsackie and Adenovirus Receptor (CAR). We provide evidence of a low affinity interaction with CAR, with modelling suggesting affinity is attenuated through extended, semi-flexible loop structures, providing steric hindrance. Conversely, in silico and in vitro experiments are unable to provide evidence of interaction between HAdV-D26/48 fiber-knob with CD46, or with Desmoglein 2. Our findings provide insight into the cell-virus interactions of HAdV-D26/48, with important implications for the design and engineering of optimised Ad-based therapeutics. </p> </div> <div class="abstract" title="abstract"> <p>Adenovirus based (AdV) vectors are promising platforms for therapeutics and vaccines, but receptor usage of serotypes in clinical development remains unclear. Here, based on crystal structures and modeling, Baker et al. show that HAdV-D26/48 fiber knob protein interacts weakly with CAR but not with CD46 or DSG2. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Adenoviruses are increasingly important vectors for wide-ranging therapeutic interventions, from gene delivery and oncolytic agents to platforms for vaccine applications <sup> <a href="#CR1">1</a>– <a href="#CR3">3</a> </sup>. As vaccine vectors, their use clinically has been popularised by their excellent safety profile coupled with their ability to induce robust cellular and humoral immunogenicity in humans <sup> <a href="#CR4">4</a> </sup>. Phylogenetically, the human adenoviruses (HAdV’s) are diverse, subdivided across seven species, A–G <sup> <a href="#CR5">5</a> </sup>, based classically on serological cross-reactivity, receptor usage, haemagglutination properties and, more recently, phylogenetic sequence similarity <sup> <a href="#CR6">6</a>, <a href="#CR7">7</a> </sup>. </p> <p>Most experimental and clinical studies have focussed on the well-studied species C adenovirus, HAdV-C5. Although potently immunogenic, the efficacy of vaccines based on HAdV-C5 appears hampered by high seroprevalence rates in humans, and enthusiasm for their use as clinical vaccine platforms has been dampened by the well-publicised failure of the MERCK sponsored STEP vaccine trial. This trial, to evaluate an HAdV-C5-based HIV vaccine encoding HIV gag/pol/nef antigens, was abandoned due to apparent lack of efficacy upon 1st term analysis. The study also identified a non-significant trend towards increased HIV acquisition in a specific high-risk, uncircumcised subset of patients who also had high levels of baseline pre-existing neutralising antibodies (NAbs) to HAdV-C5 <sup> <a href="#CR8">8</a>, <a href="#CR9">9</a> </sup>. As a result, attention has switched from HAdV-C5-based vectors towards the development of alternative adenoviral serotypes with lower rates of pre-existing immunity. Most notably, vectors under development include those based on species D serotypes including HAdV-D26, which has entered Phase-III clinical trials as an Ebola vaccine and recently reported promising immunogenicity in an HIV trial. Chimeric vectors utilising the hexon hyper variable regions (HVRs) of HAdV-D48 which have undergone Phase-I evaluation as an HIV vaccine <sup> <a href="#CR3">3</a>, <a href="#CR10">10</a>– <a href="#CR12">12</a> </sup>. However, despite extensive clinical advances using these vaccine vectors we possess very limited knowledge of their basic biology, particularly with regards to the determinants underpinning their tropism, mechanisms of cellular entry, and receptor usage. In this study, we address these shortcomings through analysis of adenoviral diversity in the context of their receptor binding, fiber proteins. Whilst adenoviruses are historically divided into seven species, A–G, this may underestimate their diversity <sup> <a href="#CR13">13</a>– <a href="#CR15">15</a> </sup>. Phylogenetic examination of 56 human adenovirus fiber proteins from different species shows deviation from the taxonomy expected based upon whole genome taxonomy, likely due to recombination events as seen in other adenoviral proteins <sup> <a href="#CR14">14</a> </sup>. Here, we have sought to generate high-resolution crystallographic structures of the cellular interacting fiber-knob domains of species D adenoviruses HAdV-D26 and HAdV-D48. The fiber-knob is the receptor interacting domain of the fiber protein, one of three major capsid proteins along with the hexon and penton base, as shown schematically in Fig. <a href="#Fig1">1a</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Phylogenetic analysis of adenoviruses mapped by whole genome and fiber-knob domain. A diagrammatic representation of the adenoviral major capsid proteins shows the icosahedral capsid structure with the Hexon (orange) comprising the facets, pentons (green) at the vertices, from which the fiber proteins (fiber-shaft in light blue, fiber-knob in dark blue) protrude ( <b>a</b>). Condensed maximum likelihood trees (percentage confidence shown by numbers next to nodes) were generated from alignments of fiber-knob domain amino acid sequences of adenoviruses 1–56 ( <b>b</b>) or whole genome NT sequences ( <b>c</b>). Adenoviruses divide into 7 subspecies, as denoted in the key, regardless of alignment used, but the species D adenoviruses divide into additional sub-species when determined by fiber-knob alignment, for readability simple nomenclature is used, all are human adenovirus. Numbers next to nodes denote confidence. Wu–Kabat variability analysis of the Clustal Omega aligned fiber-knob domains amino acid sequences of adenoviruses 1–56 ( <b>d</b>) reveals regions of low amino acid variability corresponding to beta-sheets. The locations of HAdV-C5 β-strands, as described by Xia et al. (1994) <sup> <a href="#CR27">27</a> </sup>, are aligned to the structure and shown by arrows, the corresponding positions are coloured in red </p>   </div> <p /> <p>In this study, we employ an integrative workflow utilising X-ray crystallography, in silico modelling, and in vitro assays to dissect previous findings <sup> <a href="#CR16">16</a>, <a href="#CR17">17</a> </sup> suggesting interactions by HAdV-D26 and HAdV-D48 with Coxsackie and Adenovirus Receptor (CAR) <sup> <a href="#CR5">5</a>, <a href="#CR18">18</a> </sup> and CD46 (Membrane Cofactor Protein, MCP) <sup> <a href="#CR16">16</a>, <a href="#CR17">17</a>, <a href="#CR19">19</a>– <a href="#CR21">21</a> </sup>. Utilising surface plasmon resonance (SPR), we also investigate the potential for HAdV-D26 and HAdV-D48 to interact with Desmoglein 2. Our findings shed new light on the cell–virus interactions of adenovirus and have potential implications for the design and engineering of optimised HAdV-based therapeutics, both for vaccine applications and oncolytic development, allowing us to minimise off-target or undesirable interactions in vivo. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="geneticvariabilityinadenovirusfiber-knobprotein" title="sec"> <div class="title" tagx="title" title="title">
Genetic variability in adenovirus fiber-knob protein</div> <p>We generated phylogenetic trees of human adenovirus serotypes 1–56 which revealed greater diversity in the fiber-knob domain (Fig. <a href="#Fig1">1b</a>) than might be expected based upon the taxonomy of the whole virus (Fig. <a href="#Fig1">1c</a>). These phylogenetic trees have been condensed to 70% bootstrap confidence (500 bootstrap replicates) to exclude poorly supported nodes and display the projected diversity. A full dendrogram showing to-scale branches is provided in Supplementary Figure <a href="#MOESM1">1</a>. In both phylogenetic trees, the adenoviruses divide into seven clades corresponding to the seven adenoviral species, A–G. However, the tree based upon the fiber-knob domain (Fig. <a href="#Fig1">1b</a>) shows the species D adenoviruses forming a greater number of sub-groups than in the whole genome tree suggesting greater diversity in the receptors of this species than might be expected when comparing serotypes. This may be the result of recombination events, as reported previously for other adenoviral proteins <sup> <a href="#CR13">13</a>, <a href="#CR15">15</a>, <a href="#CR22">22</a>– <a href="#CR24">24</a> </sup>. The opposite is observed in species B adenoviruses, where simpler groupings are seen when analysing fiber-knob domain alignment than by whole genome analysis. When divided into sub-species based on whole genome, they divide into species B1 and B2, but when the tree is generated based on fiber-knob alone the species B viruses do not divide into similar groups. The significance of this fiber-knob diversity is unclear, but it has previously been suggested that the species B1/B2 designation may more closely represent the tissue tropism, than receptor usage <sup> <a href="#CR25">25</a>, <a href="#CR26">26</a> </sup>. </p> <p>We next calculated the amino acid variability at each position in the aligned adenoviral fiber-knob sequences, which revealed regions of broad conservation corresponding to β-strands which make up the main fold of the fiber-knob trimer (Fig. <a href="#Fig1">1d</a>). The positions corresponding to the β-strands of HAdV-C5, as originally reported by Xia et al. <sup> <a href="#CR27">27</a> </sup>, are shown by arrows. Comparison shows that the more N-terminal (A, B, and C) and C-terminal (I and J) β-strands have greater homology across the adenoviral species than the other strands. This may relate to the intervening loops between the less tightly conserved β-strands (D, E, F, G, H) being more apical, a region which is often involved in receptor interactions <sup> <a href="#CR26">26</a>, <a href="#CR28">28</a>– <a href="#CR32">32</a> </sup>. </p> </div> <div class="structuralanalysisofhadv-d26andhadv-d48fiber-knob" title="sec"> <div class="title" tagx="title" title="title">
Structural analysis of HAdV-D26 and HAdV-D48 fiber-knob</div> <p>To investigate diversity within the species D adenoviruses fiber-knob protein, recombinant, 6-His-tagged fiber-knob protein from HAdV-D26 and HAdV-D48 (hereafter referred to as HAdV-D26K and HAdV-D48K) were generated, purified, and used to determine X-ray crystallographic structures of HAdV-D26 (PDB: 6FJN) and HAdV-D48 (PDB: 6FJQ) at resolutions of 0.97 and 2.91 Å, respectively (Fig. <a href="#Fig2">2a–d</a>). Table <a href="#Tab1">1</a> shows the data collection and refinement statistics for the crystallographic structures generated in this study. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Overview of the HAdV-D26 and HAdV-D48 fiber-knob protein structures. The surface representation of the trimeric HAdV-D26K (PDB 6FJN) biological assembly is shown side-on with the cartoon representation shown for the nearest monomer ( <b>a</b>) and the top-down view of the same HAdV-D26K trimer, as it would appear looking towards the virion, is seen as a cartoon representation ( <b>b</b>), with each monomer coloured in red, green, or blue with the hypervariable loops extending between the β-strands (dark blue) coloured as follows: AB—green, BC—purple, CD—brown, DE—orange, DG—red, EG—pink, GH—purple, HI—yellow, IJ—light blue. The HAdV-D48K (PDB 6FJQ) trimer is shown similarly ( <b>c</b>, <b>d</b>). The Wu–Kabat variability plot of the fiber-knob domains of species D adenoviruses shows regions of low variability ( <b>e</b>) with the locations of the HAdV-D48K β-strands shown by arrows above the graph, and the positions coloured blue, or red when the position is a β-strand in both HAdV-D26K and HAdV-D48K </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Data collection and refinement statistics (molecular replacement)</p>  <table> <thead> <tr> <th /> <th>HAdV-D26K</th> <th>HAdV-C5K</th> <th>HAdV-D48K</th> </tr> </thead> <tbody> <tr> <td> <b>Data collection</b> </td> </tr> <tr> <td>Space group</td> <td>P 2 <sub>1</sub>3 </td> <td>P 2 <sub>1</sub>2 <sub>1</sub>2 </td> <td>P 4 <sub>3</sub>32 </td> </tr> <tr> <td>Cell dimensions</td> </tr> <tr> <td> <i> a</i>, <i>b</i>, <i>c</i> (Å) </td> <td>86.01, 86.01, 86.01</td> <td>102.16, 102.44, 77.01</td> <td>145.18, 145.18, 145.18</td> </tr> <tr> <td> <i> α</i>, <i>β</i>, <i>γ</i> (°) </td> <td>90.0, 90.0, 90.0</td> <td>90.0, 90.0, 90.0</td> <td>90.0, 90.0, 90.0</td> </tr> <tr> <td>Resolution (Å)</td> <td>0.97–60.82 (0.97–1.00)</td> <td>1.49–61.56 (1.49–1.53)</td> <td>2.91–83.82 (2.91–2.99)</td> </tr> <tr> <td> <i>R</i> <sub>sym</sub> or <i>R</i> <sub>merge</sub> </td> <td>0.043 (0.745)</td> <td>0.134 (1.838)</td> <td>0.125 (302.6)</td> </tr> <tr> <td> <i>I</i>/ <i>σ</i> <i>I</i> </td> <td>27.3 (0.7)</td> <td>7.1 (0.7)</td> <td>22.2 (1.7)</td> </tr> <tr> <td>Completeness (%)</td> <td>94.9 (43.9)</td> <td>99.8 (99.9)</td> <td>100.0 (100)</td> </tr> <tr> <td>Redundancy</td> <td>16.7 (1.6)</td> <td>6.6 (6.3)</td> <td>41.2 (41.4)</td> </tr> <tr> <td> <b>Refinement</b> </td> </tr> <tr> <td>Resolution (Å)</td> <td>0.97–60.82</td> <td>1.49–61.56</td> <td>2.91–83.82</td> </tr> <tr> <td>No. reflections</td> <td>112,612</td> <td>125,479</td> <td>11,371</td> </tr> <tr> <td> <i>R</i> <sub>work</sub>/ <i>R</i> <sub>free</sub> </td> <td>18.2/19.5</td> <td>21.1/23.3</td> <td>20.1/29.1</td> </tr> <tr> <td>No. atoms</td> <td>1811</td> <td>4825</td> <td>3117</td> </tr> <tr> <td>  Protein</td> <td>1579</td> <td>4395</td> <td>3091</td> </tr> <tr> <td>  Ligand/ion</td> <td>8</td> <td>21</td> <td>20</td> </tr> <tr> <td>  Water</td> <td>224</td> <td>409</td> <td>6</td> </tr> <tr> <td> <i>B</i>-factors </td> <td>16.0</td> <td>34.0</td> <td>87.0</td> </tr> <tr> <td>  Protein</td> <td>15.8</td> <td>33.6</td> <td>94.2</td> </tr> <tr> <td>  Ligand/ion</td> <td>29.3</td> <td>35.7</td> <td>129.5</td> </tr> <tr> <td>  Water</td> <td>23.9</td> <td>42.9</td> <td>61.0</td> </tr> <tr> <td>R.M.S. deviations</td> </tr> <tr> <td>  Bond lengths (Å)</td> <td>0.025</td> <td>0.011</td> <td>0.019</td> </tr> <tr> <td>  Bond angles (°)</td> <td>2.339</td> <td>1.534</td> <td>2.293</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>One crystal was used for each dataset</p> <p>Values in parentheses are for highest-resolution shell</p> </div> </div> <p /> <p>The monomers (red, green, blue) form an anti-parallel β-barrel, typical of adenoviral fiber-knob protein, as described by Xia et al. (PDB 1KNB) <sup> <a href="#CR27">27</a> </sup>. Each monomer interacts with two neighbouring copies to form a homotrimer with 3-fold symmetry (Fig. <a href="#Fig2">2b, d</a>) and a highly stable interface (Supplementary Figure <a href="#MOESM1">2A</a>). Stability analysis using PISA (Protein Interactions, Surfaces and Assemblies) software calculates the HAdV-D26 and HAdV-D48 trimers to have &amp;gt;20% lower interface energy than that of HAdV-C5, indicative of a more stable interaction (Supplementary Figure <a href="#MOESM1">2A</a>) between the monomers of the HAdV-D26 and HAdV-D48 fiber-knobs. </p> <p>As with the pan species analysis (Fig. <a href="#Fig1">1d</a>), variability of the aligned species D fiber-knobs (Fig. <a href="#Fig2">2e</a>) confirmed that the β-strands comprising the hydrophobic core of the trimers are highly conserved, with β-strands demonstrating a high degree of overlap in both spatial position and sequence variability (Supplementary Figure <a href="#MOESM1">2B</a>). </p> </div> <div class="fiber-knobloopsarestabilisedbyinter-loopinteractions" title="sec"> <div class="title" tagx="title" title="title">
Fiber-knob loops are stabilised by inter-loop interactions</div> <p>Particularly relevant to this study are the DG, GH, HI, and IJ loops, linking the indicated strands corresponding to those in the originally reported HAdV-C5 fiber-knob loops (Supplementary Figure <a href="#MOESM1">2B</a>) <sup> <a href="#CR27">27</a> </sup>. These loops have previously been shown to be critical in engagement of CD46 for Ad11, 35, and 21 <sup> <a href="#CR33">33</a>– <a href="#CR36">36</a> </sup>. Alignment of these loops with the corresponding loops of HAdV-C5 (species C, CAR interacting), HAdV-B35 (species B, CD46 interacting), Ad11 (species B, CD46 interacting), and Ad37 (species D, CAR and Sialic acid/GD1a glycan interacting) reveals different topologies in these critical receptor interacting regions (Fig. <a href="#Fig3">3a</a>) <sup> <a href="#CR35">35</a>– <a href="#CR38">38</a> </sup>. The HI loops of HAdV-D26K and HAdV-D48K are most homologous to those of HAdV-B35K and HAdV-D37K respectively in terms of amino acid sequence identity (Supplementary Figure <a href="#MOESM1">2B</a>) and spatial alignment. The HAdV-D26K DG loop is most homologous to HAdV-B35K but incorporates a three amino acid insertion, while HAdV-D48K DG loop displays a differing and unique topology. The GH and IJ loops of HAdV-C5K, HAdV-D26K, HAdV-D37K, and HAdV-D48K demonstrate similar spatial arrangements (though the similarity does not extend to the sequence identity) but differ from the CD46 utilising HAd-B11K and HAdV-B35K. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Comparison of HI, DG, GH, and IJ loops of adenoviruses used in this study. <b>a</b> The hypervariable loops of HAdV-D26K (green) and HAdV-D48K (cyan) relevant to this study (HI, DG, GH, and IJ) are shown in the context of the control virus fiber-knob domains, HAdV-C5K (orange), Ad11K (yellow), HAdV-B35K (pink), and Ad37K (purple). The electron density achieved in the loops of HAdV-D26K ( <b>b</b>) and HAdV-D48K ( <b>c</b>) are shown as mesh. The fitted residues are seen as stick representations, with oxygen and nitrogen atoms coloured red and blue, respectively and other atoms coloured according to their relative B-factors with warmer colours indicating higher B-factor values </p>   </div> <p /> <p>The high-resolution structures obtained allowed us to robustly characterise the loops, seen in the electron density maps (Fig. <a href="#Fig3">3b, c</a>). To assess loop flexibility and mobility, we assessed the B-factors (also known as temperature factor), a measure of the confidence in the position of an atom which can be used to infer flexibility. By assessing the B-factors, the relative flexibility of the moieties of interest can be inferred. While the apical domains of some loops demonstrated increased B-factors relative to the rest of the molecule, the loops’ B-factors are surprisingly low (Fig. <a href="#Fig3">3b, c</a>), suggesting that they may exhibit limited flexibility. </p> <p>To investigate whether the different loop conformations were the product of flexibility, or restricted mobility we investigated the inter-loop interactions in the HAdV-D26K and HAdV-D48K structures. This analysis shows that the GH loop of HAdV-D26K (like those of HAdV-C5K, HAdV-D37K, and HAdV-D48K) does not extend directly away from the G and H β-sheets, but forms a β-hairpin (Fig. <a href="#Fig3">3a, b</a>) that is maintained by seven polar contacts within the neighbouring IJ loop which restrict the loops’ orientation (Fig. <a href="#Fig4">4a</a>, Supplementary Figure <a href="#MOESM1">3A</a>). Polar contacts were also observed at the apex of several loops, notably the GH and CD loops (Fig. <a href="#Fig4">4a, b</a>). The IJ loops form fewer intramolecular polar contacts but are stabilised by interactions with the adjacent CD and DG loops (Supplementary Figure <a href="#MOESM1">3C, D</a>). These interactions retain the apical residues in a stable conformation, rather than leaving the side chains fully labile. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Hypervariable loop conformations and contacts residues. The residues comprising the indicated loops of HAdV-D26K ( <b>a</b>) and HAdV-D48K ( <b>b</b>) are shown diagrammatically with numbers indicating the start and end residues of each loop depicted. The network of intraloop polar interactions is shown by solid lines (one polar bond), and dashed lines (two polar bonds, colour variations are only for ease of viewing) between interacting residues, similar interloop bonds are also present as visualised in Supplementary Figure <a href="#MOESM1">2</a>. Residues forming part of a helical motif are shaded in blue. The HAdV-D48K DG-loop is seen to form contacts to the opposing monomer across the inter-monomeric cleft ( <b>c</b>). The labelled residues forming polar contacts (shown as sticks) are coloured by relative B-factor, with warmer colours indicating higher relative B-factors, as is the cartoon representation of the loop. The opposing HAdV-D48K monomer is seen as a ribbon representation of the carbon-α chain in cyan and the surface of the HAdV-D48K trimer seen as a semi-transparent grey surface </p>   </div> <p /> <p>The B-factors of the HAdV-D48K DG loop were observed to be polarised about the hairpin, with the outer face of the loop having higher B-factors compared to the inner face (Fig. <a href="#Fig4">4c</a>). This is likely the result of polar contacts formed between Ser-307, Gln-308, Ala-309, and Leu-304 with Asp-359 and Gln-357 of the opposing monomer stabilising the conformation of the DG loop. The proline-rich nature of this loop provides further rigidity (Supplementary Figure <a href="#MOESM1">2D</a>). </p> <p>Crystal contacts did not reveal any specific interactions between these DG-loops and neighbouring non-trimer copies. We calculated the energy of interaction to be below the background threshold (&amp;lt;−3.0 kcal mol <sup>−1</sup>) for all loops except DG. The DG-loop of HAdV-D26K is calculated to have interaction energy of −6.5 kcal mol <sup>−1</sup> in two separate stretches of this exceptionally long loop (Supplementary Figure <a href="#MOESM1">4</a>). Importantly, no strong contacts are found within the inter-monomer cleft. </p> <p>Based on this analysis of the inter and intra-loop bonds, we suggest that these adenoviral loops may not be fully flexible variable regions, but organised receptor engagement motifs with carefully evolved structures. This has direct implications for receptor engagement of these viruses, as the loops govern previously characterised interactions with CAR and CD46, and are directly involved in their pathogenicity <sup> <a href="#CR25">25</a>, <a href="#CR33">33</a>, <a href="#CR39">39</a> </sup>. </p> </div> <div class="insilicoevaluationofhadv-d26/48kinteractionwithcar" title="sec"> <div class="title" tagx="title" title="title">
In silico evaluation of HAdV-D26/48K interaction with CAR</div> <p>Both CD46 and CAR have been proposed as primary attachment receptors for HAdV-D26 and HAdV-D48 <sup> <a href="#CR16">16</a>, <a href="#CR17">17</a> </sup>. Previously generated crystal structures of HAdV-B11K in complex with full-length CD46 (PDB: 3O8E), and HAd-D37K in complex with CAR-D1 domain (PDB: 2J12) reveal the loops to be essential to receptor interactions <sup> <a href="#CR40">40</a> </sup>. To investigate the ability of HAdV-D26K and HAdV-D48K to bind these receptors we generated homology models by alignment of the new HAdV-D26K and HAdV-D48K fiber-knob structures modelled, using the existing fiber-knobs in complex with the receptor of interest, and performed energy minimisation to optimise the conformation to achieve the lowest possible energy interface with which to analyse the interaction. We performed similar experiments with the well-described CAR and CD46 binding fiber-knob proteins of HAdV-C5 (PDB: 6HCN) and HAdV-B35 (PDB: 2QLK), respectively, as controls. </p> <p>Modelling of HAdV-D26K and HAdV-D48K in complex with the CAR-D1 domain revealed a region of high homology with the CAR utilising HAdV-C5 fiber-knob, hereafter termed the α-interface (Fig. <a href="#Fig5">5a</a>). Sequence alignment with HAdV-C5K shows that many of the residues previously shown to be critical for CAR interaction in HAdV-C5K are conserved in HAdV-D26K and HAdV-D48K (Fig. <a href="#Fig5">5b</a>), including Ser-408, Pro-409, and Tyr-376 <sup> <a href="#CR37">37</a> </sup>. The same is true of residues predicted to interact with CAR directly, such as Lys-417 (number is for HAdV-C5K). The residues predicted to form direct CAR binding interactions for HAdV-C5K, HAdV-D26K, and HAdV-D48K are pictured in complex with the maximum spatial occupancy of the energy minimised CAR-D1 (Fig. <a href="#Fig5">5c</a>). The high levels of homology are seen to extend to the proteins’ fold as well as the linear sequence. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Modelling of the HAdV-D26K and HAdV-D48K interaction with CAR at the α-interface. The α-interface region is shown by the box on the structural alignment of HAdV-C5K (orange), 26K (green), and 48K (cyan) fiber-knob domain crystal structures in complex with CAR-D1 domain (grey) as determined by homology alignment to the previously reported Ad37K CAR-D1 structure (PDB: 2J12) ( <b>a</b>). The aligned amino acid sequence of the investigated fiber-knobs ( <b>b</b>) and the predicted α-interface forming CAR-D1 binding residues are highlighted in blue, with the underlined residues representing the HAdV-C5K amino acids shown by Kirby et al. (2000) <sup> <a href="#CR37">37</a> </sup> to be important for CAR interaction. Conservation of key residues can be seen between HAdV-C5K, HAdV-D26K, and HAdV-D48K fiber-knobs. This conservation is visualised, with the contact residues comprising the α-interface with HAdV-C5K, 26K, and 48K shown as sticks in complex with the energy minimised CAR-D1 domain (grey), shown as the surface of the maximum spatial occupancy of the aligned CAR-D1 monomers from each of the energy minimised models in complex with HAdV-C5K, 26K, and 48K fiber-knobs ( <b>c</b>). <b>d</b> Plots the predicted binding energy of the energy minimised fiber-knob proteins to CAR-D1 complex in the α-interface, only. Lower binding energy indicates a more stable interface with the red line depicting 3.0 kcal mol <sup>−1</sup>, which can be considered background. <i>n</i> = 3, where each calculation is an independent fiber-knob: CAR interface, error bars indicate mean ± SD </p>   </div> <p /> <p>Binding energies were calculated between the modelled fiber-knob proteins and CAR, restricting the calculation to only the α-interface to best model the conserved region. For the modelled complexes, a stable α-interface was predicted for all complexes modelled, albeit weaker for the known non-CAR utilising HAdV-B35K (Fig. <a href="#Fig5">5d</a>) which has lower sequence conservation with HAdV-C5K. However, the interaction is complicated by a second CAR interface, termed the β-interface (Fig. <a href="#Fig6">6a</a>). The loops forming the β-interface with CAR-D1 differ between HAdV-C5, HAdV-D26, and HAdV-D48 fiber-knob (Fig. <a href="#Fig6">6b</a>). The shorter HAdV-C5K DG loop does not clash with the CAR-D1 surface, whereas the extended HAdV-D26K DG loop forms a partial steric clash, with surface seen to clash with the aligned CAR-D1, and HAdV-D48K DG loop is seen to form an even larger steric clash. Whilst the longer loop of HAdV-D26K is expected to be more flexible than that of HAdV-C5K, the HAdV-D48K DG loop is surprisingly stable due to the characteristics described (Fig. <a href="#Fig4">4c</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Modelling of the HAdV-D26K and HAdV-D48K interaction with CAR at the β-interface. The β-interface region is shown by the box on the structural alignment of HAdV-C5K (orange), 26K (green), and 48K (cyan) fiber-knob domain crystal structures in complex with CAR-D1 domain (grey) as determined by homology alignment to the previously reported Ad37K CAR-D1 structure (PDB: 2J12) ( <b>a</b>). A dot surface shows the surface of HAdV-C5K, 26K, and 48K DG-loops in the inter-monomer cleft ( <b>b</b>). The boxes denote the maximum B-factor of the corresponding loops, which are shown as putty representations with thicker regions indicating higher relative B-factors, from which we can infer the relative stability of the loops. Antibody competition inhibition assay ( <b>c</b>) shows the relative inhibitory ability of the HAdV-C5, 35, 26, and 48, fiber-knob domains in CAR expressing CHO-CAR cells, with the calculated IC <sub>50</sub> values shown in boxes. <i>n</i> = 3 biological replicates. Surface plasmon resonance (SPR) traces are shown by coloured lines, and the fitted curves by black lines ( <b>d</b>). The calculated binding coefficients on rate ( <i>K</i> <sub>On</sub>), off rate ( <i>K</i> <sub>Off</sub>), and dissociation coefficient ( <i>K</i> <sub>D</sub>) are given in the table ( <b>e</b>). IC <sub>50</sub> curves are fitted by non-linear regression. Error bars represent standard deviation of 3 biological replicates. Error bars indicate mean ± SD </p>   </div> <p /> </div> <div class="biologicalevaluationofhadv-d26/48kinteractionwithcar" title="sec"> <div class="title" tagx="title" title="title">
Biological evaluation of HAdV-D26/48K interaction with CAR</div> <p>Our modelling studies indicate that the longer, inflexible DG loop of HAdV-D48K would be likely to sterically hinder the HAdV-D48K: CAR interaction at the β-interface to a greater extent than the more modest inhibition of the smaller and more labile loop of HAdV-D26K, which in turn would exhibit more inhibition of CAR binding than that of HAdV-C5K, where no steric inhibition is observed. Competition inhibition assays using recombinant fiber-knob protein to inhibit antibody binding to CAR receptor in CHO-CAR cells (which express CAR, while the parental cell line (CHO-K1) is established to be non-permissive to adenovirus infection) support our observations (Fig. <a href="#Fig6">6c</a>). The IC <sub>50</sub> (the concentration of protein required to inhibit 50% of antibody binding) of HAdV-C5K is 7.0 ng/10 <sup>5</sup> cells, while HAdV-D26K and HAdV-D48K demonstrate IC <sub>50</sub> values 15.7 and 480 times higher at 0.110 μg/10 <sup>5</sup> cells and 3.359 μg/10 <sup>5</sup> cells, respectively, reflecting their reduced ability to engage CAR. </p> <p>SPR analysis indicates that HAdV-C5K binds strongly to CAR (Fig. <a href="#Fig6">6d</a>) with a <i>K</i> <sub>D</sub> of 0.76 nM. HAdV-D26K and HAdV-D48K have lower overall affinities for CAR (Fig. <a href="#Fig6">6e</a>). While the <i>K</i> <sub>Off</sub> of the three fiber knob proteins (Fig. <a href="#Fig6">6e</a>) are similar, the <i>K</i> <sub>On</sub> is fastest for HAdV-C5K, with HAdV-D26K <i>K</i> <sub>On</sub> being slower, and HAdV-D48K exhibiting the slowest <i>K</i> <sub>On</sub>. This shows that the <i>K</i> <sub>On</sub>—the ability to form the initial interaction with the receptor—is the major limiting factor in the fiber knobs overall affinity for CAR. </p> </div> <div class="insilicoevaluationofhadv-d26/48kinteractionwithcd46" title="sec"> <div class="title" tagx="title" title="title">
In silico evaluation of HAdV-D26/48K interaction with CD46</div> <p>A similar approach was taken to model HAdV-D26K and HAdV-D48K in complex with CD46. Alignments with the previously published HAdV-B11K-CD46 complex were generated and energy minimised to obtain the lowest energy state of the complex <sup> <a href="#CR34">34</a>, <a href="#CR41">41</a> </sup>. This interface utilises loops DG, GH, HI, and IJ to form a network of polar interactions with the CD46 Sc1 and Sc2 domains (Fig. <a href="#Fig7">7a</a>) <sup> <a href="#CR35">35</a> </sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 7</span> </a> </div>   <p>Modelling of the HAdV-D26K and HAdV-D48K with CD46. Red dashes show contacts between the energy minimised crystal structure of CD46 SC1 and SC2 domains (grey cartoon) and Ad11K in complex (PDB 3O8E) ( <b>a</b>). The known CD46 interacting fiber-knob, HAdV-B35K (purple), is aligned to the above crystal structure and energy minimised ( <b>b</b>). Amino acid sequence alignment of the tested fiber-knob proteins ( <b>c</b>) shows conservation of residues previously shown by Wang et al. (2007) <sup> <a href="#CR36">36</a> </sup> to be key to CD46 binding (underlined) between the known CD46 binding fiber-knobs, Ad11K and HAdV-B35K. Residues highlighted in blue are predicted to form direct contacts with CD46 in the energy minimised models. Similar alignments to that performed with HAdV-B35K are shown for HAdV-D26K (Green— <b>d</b>) and HAdV-D48K (cyan— <b>e</b>). In all models, red dashes indicate polar contacts between the residues shown as stick representations </p>   </div> <p /> <p>When HAdV-B35K was modelled in complex with CD46, a network of polar contacts between HAdV-B35K and CD46 was predicted (Fig. <a href="#Fig7">7b</a>) similar to that observed in the HAdV-B11K-CD46 complex crystal structure, PDB: 3O8E (Fig. <a href="#Fig7">7a</a>) <sup> <a href="#CR34">34</a> </sup>. Previously, HAdV-B35K residues Phe-243, Arg-244, Tyr-260, Arg-279, Ser-282, and Glu-302 (underlined in Fig. <a href="#Fig7">7c</a>) have been implicated as key contact residues for CD46 interaction, and are conserved in HAdV-B11K (highlighted in blue, Fig. <a href="#Fig7">7c</a>) <sup> <a href="#CR36">36</a> </sup>. Our modelling suggests that conversely, HAdV-D26K and HAdV-D48K are predicted to form very few polar contacts with CD46 with just two contacts predicted for HAdV-D26K (Fig. <a href="#Fig7">7d</a>) and three predicted for HAdV-D48K (Fig. <a href="#Fig7">7e</a>). Furthermore, they do not share any of the critical CD46 binding residues which have been reported previously (underlined, Fig. <a href="#Fig7">7c</a>) for HAdV-B11K and HAdV-B35K, or any of the predicted interacting residues (blue highlight, Fig. <a href="#Fig7">7c</a>). </p> <p>We again employed PISA to calculate the binding energy of the various modelled and energy minimised fiber-knob CD46 complexes (Fig. <a href="#Fig8">8a</a>). HAdV-B11K, the strongest known CD46 binding adenovirus <sup> <a href="#CR33">33</a> </sup>, was predicted to have the lowest binding energy reflecting its high stability interface, with HAdV-B35 demonstrating a similar but slightly reduced binding energy. Conversely, HAdV-D26K and HAdV-D48K are predicted to have lower binding energies, similar to that which may be expected for random proteins passing in solution, indicating that any interaction between either the HAdV-D26K, or HAdV-D48K with CD46 is unlikely <sup> <a href="#CR42">42</a> </sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 8</span> </a> </div>   <p>Binding energetics and affinities of HAdV-D26K and HAdV-D48K with CD46. Calculation of the predicted binding energies for the energy minimised fiber-knob: CD46 models are compared on the bar chart ( <b>a</b>), lower kcal mol <sup>−1</sup> values indicate a stronger interaction, the red line at 3.0  kcal mol <sup>−1</sup>, denotes an interface energy which can be considered negligible (random proteins passing in solution), <i>n</i> = 3, where each calculation is an independent fiber-knob: CD46 interface. The HI loop (red) of the HAdV-C5 fiber-knob (orange) is seen to extend between CD46 (grey) and the DG loop ( <b>b</b>). The antibody competition inhibition assay ( <b>c</b>) shows the relative inhibitory ability of the HAdV-C5, HAdV-B35, HAdV-D26, and HAdV-D48 fiber-knob domains in CD46 expressing CHO-BC1 cells, with the calculated IC <sub>50</sub> values shown in boxes. <i>n</i> = 3 biological replicates. Surface plasmon resonance (SPR) traces are shown by coloured lines, and the fitted curves by black lines ( <b>d</b>). The calculated binding coefficients on rate ( <i>K</i> <sub>On</sub>), off rate ( <i>K</i> <sub>Off</sub>), and dissociation coefficient ( <i>K</i> <sub>D</sub>) are given in the table ( <b>e</b>), nm (not measured) indicates that the kinetics were too fast to measure, nb denotes no binding. IC <sub>50</sub> curves are fitted by non-linear regression. Error bars represent standard deviation of 3 biological replicates. Error bars indicate mean ± SD </p>   </div> <p /> <p>While still low compared to the known CD46 interacting HAdV-B11K and HAdV-B35K binding energies, that for HAdV-C5K was higher than expected for a known non-CD46 interacting adenovirus (Fig. <a href="#Fig8">8a</a>). Inspection of the model shows that this is due to the close proximity of the large HAdV-C5 HI loop to CD46 (Fig. <a href="#Fig8">8b</a>). The residues involved in the predicted interaction are not conserved in any known CD46 interface and suggesting these are random interactions. Furthermore, interaction between the DG loop and CD46 is integral to known CD46 binding interfaces and is prevented by the HAdV-C5K HI loop laying between them (Fig. <a href="#Fig8">8b</a>). </p> </div> <div class="biologicalevaluationofhadv-d26/48kinteractionwithcd46" title="sec"> <div class="title" tagx="title" title="title">
Biological evaluation of HAdV-D26/48K interaction with CD46</div> <p>Antibody competition inhibition assays in CHO-BC1 cells (CHO cells transduced to express the BC1 isoform of CD46) were used to test the predictions made by modelling (Fig. <a href="#Fig8">8c</a>). These data confirm that recombinant HAdV-D26K and HAdV-D48K proteins are incapable of inhibiting antibody binding to CD46 at any concentration tested (up to 2 ng/cell), whilst the well-defined CD46 interacting HAdV-B35K demonstrates strong inhibition, with a calculated IC <sub>50</sub> of 0.003 μg/10 <sup>5</sup> cells. </p> <p>SPR analysis of the interaction between recombinant fiber-knob protein with CD46 confirms these findings. The known CD46 utilising HAdV-B35K is seen to bind CD46 even at low concentration, while HAdV-D48K shows no interaction (Fig. <a href="#Fig8">8d</a>). HAdV-D26K shows a very low affinity interaction with CD46, however the kinetics are extremely fast making it impossible to measure an accurate <i>K</i> <sub>On/Off</sub> at any of the concentrations measured, suggesting an unstable interface. The calculated <i>K</i> <sub>D</sub> for HAdV-D26K is seen to be more than 1.5 × 10 <sup>3</sup> times lower than that of HAdV-B35K (Fig. <a href="#Fig8">8e</a>). </p> </div> <div class="insilicoevaluationofhadv-d26/48kinteractionwithdsg-2" title="sec"> <div class="title" tagx="title" title="title">
In silico evaluation of HAdV-D26/48K interaction with DSG-2</div> <p>The third major protein receptor for human adenoviruses is Desmoglein 2 (DSG2), shown to enable infection by HAdV-B3, B7, B11, and B14 <sup> <a href="#CR43">43</a>, <a href="#CR44">44</a> </sup>. Whilst we have not been able to model the interaction of HAdV-D26 or D48 with DSG2, due to the lack of an available high-resolution complexed structure at the time of writing, we investigated the interaction by SPR analysis. HAdV-B3 is the best studied DSG2 binding adenovirus and showed binding in the μM range when tested by SPR (Fig. <a href="#Fig9">9a</a>), however, no binding was observed when the same experiment was run with HAdV-D26K or HAdV-D48K (Fig. <a href="#Fig9">9b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 9</span> </a> </div>   <p>Desmoglein 2 is unlikely to be a receptor for HAdV-D26K or HAdV-D48K. The dissociation constant was calculated for HAdV-B3K binding to DSG2, but kinetics were too fast to determine <i>K</i> <sub>On</sub> or <i>K</i> <sub>Off</sub> ( <b>a</b>), the <i>K</i> <sub>D</sub> curve is shown for HAdV-B3K while HAdV-D26K and HAdV-D48K are seen to form no interaction with DSG2 ( <b>b</b>). nm (not measured) indicates that the kinetics were too fast to measure, nb denotes no binding </p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>This study reveals the crystal structure of two adenovirus proteins critical to primary receptor engagement, HAdV-D26 and HAdV-D48 fiber-knob, which are important viral vectors currently in human clinical trials <sup> <a href="#CR3">3</a>, <a href="#CR11">11</a>, <a href="#CR12">12</a> </sup>. Despite their advanced development, the field lacks fundamental knowledge regarding the mechanisms of infection for these viral vector platforms. The work we described here provides a combined crystallographic, in silico, and in vitro approach to investigate adenovirus fiber-knob: receptor interactions with CAR and CD46, two receptors previously proposed to be utilised by these viruses <sup> <a href="#CR5">5</a>, <a href="#CR9">9</a>, <a href="#CR17">17</a> </sup>. </p> <p>Analysis of the phylogenetic relationship between 56 adenovirus serotypes, by both whole genome and fiber-knob domain alignment (Fig. <a href="#Fig1">1a</a>), confirms diversification into the widely accepted seven adenoviral species <sup> <a href="#CR7">7</a> </sup>. However, generating the phylogenetic tree with fiber-knob sequences, rather than whole genomes, shows additional diversity, not revealed by the whole virus taxonomy. Adenovirus species D breaks up into several additional sub-clades when focused on the fiber-knob, suggesting greater receptor diversity than might be expected based on the whole virus phylogeny. Similar observations have previously been made in species D hexon and penton <sup> <a href="#CR13">13</a> </sup>. </p> <p>In contrast to species D, the phylogeny of species B adenoviruses, which are known to utilise Desmoglein 2 and CD46 as primary receptors, is simplified when focused upon the fiber-knob, indicating less diverse receptor usage <sup> <a href="#CR39">39</a>, <a href="#CR43">43</a>– <a href="#CR45">45</a> </sup>. This simplification in comparison to the whole genomic alignment implies that much of the species diversity must lay in other proteins. The E3 protein, for example, is known to be highly diverse within species B adenoviruses, having previously been exploited in the selection of the oncolytic (cancer killing) virus enadenotucirev, which is currently in clinical trials <sup> <a href="#CR13">13</a>, <a href="#CR46">46</a>, <a href="#CR47">47</a> </sup>. </p> <p>That we see such opposing effects on the species B and D phylogenetic trees when focusing on the fiber-knob, highlights the limitations of simple taxonomic approaches. The current adenoviral taxonomy is based on antibody neutralisation assays, which are limited by antibody's reliance on surface accessible proteins in the capsid, and does not account for diversity in other viral proteins, as the above suggests for species B. This supports a taxonomic proposal based upon viral genetics rather than antibody neutralisation, as has previously been suggested <sup> <a href="#CR13">13</a>, <a href="#CR48">48</a> </sup>. </p> <p>Many studies on adenovirus neutralisation have focused upon NAbs which bind to the hexon <sup> <a href="#CR49">49</a>– <a href="#CR51">51</a> </sup>. Following intramuscular vaccination with non-replicating adenoviral vectors, most NAbs are targeted to the hexon; a reflection of its high abundance and surface availability in each viral capsid <sup> <a href="#CR49">49</a>, <a href="#CR52">52</a> </sup>. However, during natural infection, many NAbs target the fiber protein <sup> <a href="#CR52">52</a> </sup>, presumably due to the abundance of fiber produced in the early stages of hAdV replication to loosen cell–cell junctions and facilitate viral spread, prior to lysis and entry of large amounts of whole virus to the bloodstream <sup> <a href="#CR53">53</a> </sup>. For individuals with pre-existing anti-adenovirus immunity derived from natural infections, anti-fiber NAbs are likely to limit the use of vectors with common adenovirus fiber proteins by neutralisation of the viral vector prior to its therapeutic effect. Thus, for the development of vectors to circumvent pre-existing anti-adenovirus immunity for therapeutic use further exploration of this fiber protein diversity may be beneficial, as well as the on-going studies using hexon HVR pseudotypes to circumvent anti-hexon immunity <sup> <a href="#CR11">11</a>, <a href="#CR54">54</a> </sup>. </p> <p>Analysis of the adenovirus loops (Fig. <a href="#Fig4">4</a>, Supplementary Figure <a href="#MOESM1">3</a>) reveals an intricate network of polar interactions which stabilise their three-dimensional structures. These bonds appear to hold the loops in a conformation which, in the case of HAdV-B11K and HAdV-B35K, facilitates receptor binding. In the HAdV-D26 and HAdV-D48 fiber-knob structures presented in this study the loops are also held in a stable conformation, though not one which enables CD46 interaction. </p> <p>Modelling of HAdV-D26K/HAdV-D48K in complex with CD46 (Fig. <a href="#Fig7">7</a>) suggested few contacts, and interface energy calculations using these models predict a weak binding energy (Fig. <a href="#Fig8">8</a>). SPR indicated that HAdV-D26K has an affinity for CD46 that is approximately 1500× weaker than that of HAdV-B35K (Fig. <a href="#Fig8">8d, e</a>). Combined with the extremely fast kinetics, this is suggestive of a highly unstable interface. HAdV-D48K showed no affinity for CD46 at all. This was confirmed by in vitro competition inhibition assays, in which no tested quantity of recombinant fiber-knob was capable of inhibiting antibody binding to CD46 (Fig. <a href="#Fig8">8c</a>). These findings appear contradictory to previous studies which suggest CD46 as the primary receptor for these viruses <sup> <a href="#CR16">16</a>, <a href="#CR17">17</a> </sup>. Our findings improve knowledge of the cell entry mechanisms of these viruses and the vectors derived from them, and do not diminish the observed effectiveness of these vaccines. However, if CD46, a protein expressed on the surface of all nucleated cells, is not the receptor for these viruses then it is as yet unknown what the primary tissue tropism determinant is for these clinically significant viruses <sup> <a href="#CR55">55</a>, <a href="#CR56">56</a> </sup>. </p> <p>A similar methodology was applied to the interaction HAdV-D26K/HAdV-D48K with CAR. Inspection of the modelled complexes (Fig. <a href="#Fig5">5</a>) indicated a conserved α-interface enabling CAR binding in adenovirus 5, 26, and 48, fiber knobs. However, the structure of the β-interface interaction appears to indicate a mechanism modulating the fiber-knob’s CAR affinity (Fig. <a href="#Fig6">6</a>). When occupying the intermonomer cleft in the conformation shown in Fig. <a href="#Fig6">6b</a>, the DG loops of HAdV-D26K and HAdV-D48K are likely to inhibit CAR binding by steric hindrance, but if the loops were to shift into a conformation which relieves this clash CAR binding could occur. Therefore, the ability of these vectors to interact with CAR is likely a function of the steric hindrance provided by these loops, reducing the ability of the fiber-knob domain to engage CAR in a permissive conformation. SPR analysis supports this hypothesis (Fig. <a href="#Fig6">6d, e</a>), where the larger the DG loop of the investigated fiber-knob the slower the <i>K</i> <sub>On</sub>. </p> <p>The inter-loop contacts described in Fig. <a href="#Fig4">4a, b</a>, and Supplementary Figure <a href="#MOESM1">3</a>, and the normalised B-factors described in Fig. <a href="#Fig6">6b</a> will influence the molecular dynamics of the DG-loops. Loops which can occupy a CAR inhibitory conformation but have fewer stabilising contacts, such as that of HAdV-D26K (Supplementary Figure <a href="#MOESM1">3A, C</a>), should be more permissive to CAR binding. While loops which are less flexible and/or stabilised in a CAR inhibitory conformation, such as HAdV-D48K (Fig. <a href="#Fig6">6b</a>, Supplementary Figure <a href="#MOESM1">3B, D</a>) should result in a fiber-knob which is less able to bind CAR. This hypothesis fits the competition inhibition studies shown in Fig. <a href="#Fig6">6c</a>, which demonstrate that HAdV-D26K has an approximately ~15× lower affinity for CAR than HAdV-C5K, and HAdV-D48K has 500× lower affinity. </p> <p>Interestingly, the affinity of HAdV-D48K for CAR as measured by SPR is approximately 2× higher than that for HAdV-D26K, due to the slower <i>K</i> <sub>Off</sub> of HAdV-D48K (Fig. <a href="#Fig6">6e</a>) which is in contrast with the IC <sub>50</sub> curves (Fig. <a href="#Fig6">6c</a>) in which HAdV-D48K is observed to bind to CAR less strongly than HAdV-D26K. The incongruity may be explained by the methodology. It is possible that the large fluid volume in the wells during the inhibition assay (in comparison to the BIAcore microfluidics system), favoured greater binding by HAdV-D26K due to its faster <i>K</i> <sub>On</sub>, compared to HAdV-D48K. This discrepancy does not alter the proposed model of CAR interaction, and seems to confirm the importance of the <i>K</i> <sub>On</sub>, presumably mediated by the β-interface. </p> <p>Species D adenoviruses have a large range of different DG loops (Supplementary Figure <a href="#MOESM1">5</a>). Most sequences have lengths equal to, or greater than, that of HAdV-D26K, making it plausible that they too could modulate the fiber-knob interaction with CAR. However, the magnitude of this effect will be dependent on the individual molecular dynamics of the DG-loops and its interactions with adjacent residues. </p> <p>Assuming this mechanism of CAR binding regulation is broadly applicable, it may have important implications for adenoviral vector design. The presence of a high affinity receptor for the virus can mask the low affinity CAR interaction, creating a hidden tropism only observed if the virus is forced to rely upon it. Expression of CAR on human erythrocytes suggests the potential for sequestration of virotherapies in the blood <sup> <a href="#CR57">57</a> </sup>. CAR expression in lung epithelial tissues offers another site for potential off target activity <sup> <a href="#CR58">58</a>, <a href="#CR59">59</a> </sup>. Therefore, many virotherapies previously thought to be non-CAR binding adenoviruses may in fact demonstrate weak CAR tropism, driving off target infections or resulting in sequestration of the vector in tissues other than that target. This may not be of grave consequence for non-replicating vectors, such as viral vaccines, but in vectors which rely upon controlled replication in targeted tissues, such as oncolytic virotherapies, this could result in off-target infection, dysregulated expression of therapeutic protein, and reduced delivery to the point of need. </p> <p>DSG2 was also shown to be unable to bind HAdV-D26K or HAdV-D48K at any concentration by SPR. It is notable that the <i>K</i> <sub>D</sub> measured for the HAdV-B3K (66.9 μM) is much lower than that measured during the original investigation of DSG2 as an HAdV-B3K receptor (2.3 nM) <sup> <a href="#CR44">44</a> </sup>. This is likely due to our use of recombinant knob trimers, rather than the multivalent penton dodecahedrons. </p> <p>The final, known, adenovirus fiber-knob receptor, which has thus far not been addressed in this study is sialic acid, as part of glycosylation motifs. Several adenoviruses have been shown to bind to sialylated glycans, including HAdV-D37 <sup> <a href="#CR32">32</a>, <a href="#CR38">38</a> </sup>, HAdV-G52 <sup> <a href="#CR30">30</a>, <a href="#CR60">60</a> </sup>, and Canine adenovirus serotype 2 (CAdV-2) <sup> <a href="#CR57">57</a> </sup>. Each of these three viruses binds to sialic acid by different mechanisms (Supplementary Figure <a href="#MOESM1">6</a>). Supplementary Figure <a href="#MOESM1">6</a> shows that HAdV-D26/48K do not share the sialic acid binding residues found in HAdV-G52K or CAV-2 but do share the Tyr-142 and Lys-178 contact residues with HAdV-D37K. Further, the HAdV-D37K contact residue Pro-147 is between the sialic acid and the main chain oxygen which is functionally identical at the similar position in HAdV-D26/48K. Taken together, it remains plausible that HAdV-D26/48K may be capable of binding sialic acid in an HAdV-D37K-like manner. However, binding does not equate to functional infection, as seen with HAdV-D19pK <sup> <a href="#CR32">32</a> </sup> and further studies are required to ascertain whether HAdV-D26/48 are capable of utilising sialic acid to generate a productive infection. Further, HAdV-D37 was shown to require a specific glycosylation motif (GD1a) in order form a functional infection, so any assessment of sialic acid as an adenoviral receptor must be in the context of its glycan carrier <sup> <a href="#CR38">38</a> </sup>. </p> <p>The work undertaken in this study presents, for the first time, the crystal structures of the fiber-knob domain protein of HAdV-D26 (PDB: 6FJN), and HAdV-D48 (PDB: 6FJQ) fiber-knob protein. In addition, we report a new crystal structure for HAdV-C5 fiber-knob protein (PDB: 6HCN) with improved resolution compared to the existing structure (PDB: 1KNB) <sup> <a href="#CR27">27</a> </sup>. We utilised these structures to investigate the ability of these proteins to interact with the putative receptors, CAR and CD46, by an integrative structural, in silico, and in vitro workflow. We demonstrate that HAdV-D26 and HAdV-D48 fiber-knob domains have a weak ability to bind CAR, and negligible CD46 interaction, suggesting that these viruses are unlikely to utilise these proteins as a primary receptor in vivo. Finally, we showed that DSG2 is also unable to form a stable interaction in the context of SPR analysis. We suggest that CAR binding is moderated by a previously unreported mechanism of steric inhibition which may apply to other adenoviruses and demonstrate an in silico methodology capable of rapidly predicting these interactions. These findings enhance our understanding of the virology of adenovirus infection, and have direct implications for virotherapy vector design, which often rely upon carefully controlled receptor tropisms to achieve specificity and efficacy <sup> <a href="#CR9">9</a>, <a href="#CR18">18</a>, <a href="#CR28">28</a> </sup>. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="genomealignmentandanalysisofgeneticdiversity" title="sec"> <div class="title" tagx="title" title="title">
Genome alignment and analysis of genetic diversity</div> <p>Representative whole genomes (nucleotide) of adenoviral species 1–56 were selected from the National Center for Biotechnology Information (NCBI), and aligned using the EMBL-EBI Clustal Omega tool <sup> <a href="#CR61">61</a>, <a href="#CR62">62</a> </sup>. Fiber-knob domain amino acid sequences were derived from the same genome sequences, defined as the translated nucleotide sequence of the fiber protein (pIV) from the conserved TLW hinge motif to the protein C-terminus, and aligned in the same manner as the whole genomes. Phylogenetic relationships were inferred using the maximum likelihood method based upon the Jukes Cantor model for the whole genome nucleotide analysis <sup> <a href="#CR63">63</a> </sup>, and the Poisson correction model for the fiber-knob amino acid analysis <sup> <a href="#CR64">64</a> </sup>, using MEGA X software <sup> <a href="#CR65">65</a> </sup>. Confidence was determined by bootstrap analysis (500 replicates) <sup> <a href="#CR66">66</a> </sup> and trees displayed condensed at 70% confidence (percentage confidence values shown at each node) where stated. </p> </div> <div class="fiber-knobaminoacidvariability" title="sec"> <div class="title" tagx="title" title="title">
Fiber-knob amino acid variability</div> <p>Amino acid sequence variability scores were calculated from the Clustal Omega aligned amino acid sequences of the fiber-knob domains of either adenoviruses 1–56, or only the species D adenoviruses. Analysis was performed using the Protein Variability Server (PVS), using a consensus base sequence and the Wu–Kabat method <sup> <a href="#CR67">67</a> </sup>. </p> </div> <div class="generationofrecombinantfiber-knobprotein" title="sec"> <div class="title" tagx="title" title="title">
Generation of recombinant fiber-knob protein</div> <p>SG13009 <i>Escherichia coli</i> har bouring pREP-4 plasmid and pQE-30 expression vector containing the fiber-knob DNA sequence were cultured in 20 ml LB broth with 100 μg/ml ampicillin and 50 μg/ml kanamycin overnight from glycerol stocks made in previous studies <sup> <a href="#CR18">18</a>, <a href="#CR68">68</a>, <a href="#CR69">69</a> </sup>. 1 L of TB (Terrific Broth, modified, Sigma-Aldrich) containing 100 μg/ml ampicillin and 50 μg/ml were inoculated with the overnight <i>E. coli</i> culture and incubated at 37 °C until they reached OD0.6. IPTG was then added to a final concentration of 0.5 mM and the culture incubated at 37 °C for 4 h. Cells were then harvested by centrifugation at 3000 <i>g</i>, resuspended in lysis buffer (50 mM Tris, pH 8.0, 300 mM NaCl, 1% (v/v) NP40, 1 mg/ml Lysozyme, 1 mM β-mercaptoethanol), and incubated at room temperature for 30 min. Lysate was clarified by centrifugation at 30,000 <i>g</i> for 30 min and filtered through a 0.22 μm syringe filter (Millipore, Abingdon, UK). Clarified lysate was then loaded onto a 5 ml HisTrap FF nickel affinity chromatography column (GE) at 2.0 ml/min and washed with 5 column volumes into elution buffer A (50 mM Tris [pH 8.0], 300 mM NaCl, 1 mM β-mercaptoethanol). Protein was eluted by 30 min gradient elution from buffer A to B (buffer A + 400 mM Imidazole). Fractions were analysed by reducing SDS-PAGE, and fiber-knob containing fractions further purified using a superdex 200 10/300 size exclusion chromatography column (GE) in crystallisation buffer (10 mM Tris [pH 8.0] and 30 mM NaCl). Fractions were analysed by SDS-PAGE and pure fractions concentrated by centrifugation in Vivaspin 10,000 MWCO (Sartorius, Goettingen, Germany) proceeding crystallisation. </p> </div> <div class="competitioninhibitionassays" title="sec"> <div class="title" tagx="title" title="title">
Competition inhibition assays</div> <p>CHO cells expressing the appropriate receptor (CAR: CHO-CAR, or CD46: CHO-BC1) were seeded at a density of 30,000 cells per well in a flat bottomed 96-well tissue culture plate and incubated at 37 °C overnight. Serial dilutions were made up in serum-free RPMI-1640 to give a final concentration range of 0.0001–100 μg/10 <sup>5</sup> cells of recombinant soluble knob protein. Cells were incubated on ice for 30 min, then washed twice with cold PBS. Fiber-knob dilutions were then added to the cells and incubated on ice for 30 min. Cells were then washed twice in cold PBS and stained with the primary CAR or CD46 antibody, RmcB (Millipore; 05-644) or MEM-258 (Abcam; Ab789), respectively, to complex receptors unbound by fiber-knob protein, and incubated for 1 h on ice. Cells were washed twice further in PBS and incubated on ice for 1 h with Alexa-647 labelled goat anti-mouse F(ab’)2 (ThermoFisher; A-21237) <sup> <a href="#CR18">18</a>, <a href="#CR68">68</a>, <a href="#CR69">69</a> </sup>. All antibodies were used at a concentration of 2 μg/ml. </p> <p>Samples were run in triplicate and analysed by flow cytometry on Attune NxT (ThermoFisher), and analysed using FlowJo v10 (FlowJo, LLC) by gating sequentially on singlets, cell population, and Alexa-647 positive cells. Total fluorescence (TF) was used as the measure of inhibition, rather than percentage of fluorescent cells in the total population, to account for the presence of multiple receptor copies per cell surface which can enable partial inhibition of antibody binding on the cell surface. TF was defined as the percentage of Alexa-647 positive cells in the single cell population for each sample and multiplied by the median fluorescent intensity (MFI) of the Alexa-647 positive single cell population in each sample. Data points are the mean total fluorescence of three biological replicates with error given as the standard deviation from the mean. IC <sub>50</sub> curves were fitted by non-linear regression, and used to determine the IC <sub>50</sub> concentrations <sup> <a href="#CR18">18</a>, <a href="#CR68">68</a>, <a href="#CR69">69</a> </sup>. CHO-CAR and CHO-BC1 cells were originally derived by Bergelson et al. <sup> <a href="#CR70">70</a> </sup> and Manchester et al. <sup> <a href="#CR71">71</a> </sup>, respectively. </p> </div> <div class="crystallisationandstructuredetermination" title="sec"> <div class="title" tagx="title" title="title">
Crystallisation and structure determination</div> <p>Protein samples were purified into crystallisation buffer (10 mM Tris [pH 8.0] and 30 mM NaCl). The final protein concentration was approximately 7.5 mg/ml. Commercial crystallisation screen solutions were dispensed into 96-well plates using an Art-Robbins Instruments Griffon dispensing robot (Alpha Biotech, Ltd.), in sitting-drop vapour-diffusion format. Drops containing 200 nl of screen solution and 200 nl of protein solution were equilibrated against a reservoir of 60 μl crystallisation solution. The plates were sealed and incubated at 18 °C. </p> <p>Crystals of HAdV-C5K appeared in PACT <i>Premier</i> condition D04 (0.1 M MMT, pH 7.0, 20% PEG 1500), within 1–7 days. Crystals of HAdV-D26K appeared within 1–7 days, in PACT <i>Premier</i> (Molecular Dimensions, Suffolk, UK) condition A04; 0.1 M MMT [DL-malic acid, MES monohydrate, Tris], pH 6.0, 25% PEG 1500. Crystals of HAdV-D48K appeared in PACT <i>Premier</i> condition D02 (0.1 M Bis–Tris-propane, pH 6.5, 20% PEG 3350, 0.2 M NaNO <sub>3</sub>), within 2 weeks. Crystals were cryoprotected with reservoir solution to which ethylene glycol was added at a final concentration of 25%. Crystals were harvested in thin plastic loops and stored in liquid nitrogen for transfer to the synchrotron. Data were collected at Diamond Light Source beamline I04, running at a wavelength of 0.9795 Å. During data collection, crystals were maintained in a cold air stream at 100 K. Dectris Pilatus 6M detectors recorded the diffraction patterns, which were analysed and reduced with XDS, Xia2 <sup> <a href="#CR72">72</a> </sup>, DIALS, and Autoproc <sup> <a href="#CR73">73</a> </sup>. Scaling and merging data was completed with Pointless, Aimless and Truncate from the CCP4 package <sup> <a href="#CR74">74</a> </sup>. Structures were solved with PHASER <sup> <a href="#CR75">75</a> </sup>, COOT <sup> <a href="#CR76">76</a> </sup> was used to correct the sequences and adjust the models, REFMAC5 <sup> <a href="#CR77">77</a> </sup> was used to refine the structures and calculate maps. Graphical representations were prepared with PyMOL <sup> <a href="#CR78">78</a> </sup>. Reflection data and final models were deposited in the PDB database with accession codes: HAdV-C5K, 6HCN; HAdV-D26k, 6FJN; and HAdV-D48k, 6JFQ. Full crystallographic refinement statistics are given in Supplementary Table <a href="#MOESM1">2</a>; stereo images depicting representative areas of the model and map are provided in Supplementary Figure <a href="#MOESM1">7</a>. </p> </div> <div class="modellingoffiber-knobligandinteractions" title="sec"> <div class="title" tagx="title" title="title">
Modelling of fiber-knob ligand interactions</div> <p>Fiber-knob proteins were modelled in complex with CAR or CD46 using the existing HAdV-D37K—CAR liganded (PDB 2J12) or the HAdV-B11K—CD46 liganded (PDB 3O8E) structures, respectively, as a template. Non-protein components and hydrogens were removed from the template model and the fiber-knob protein of interest. The two fiber-knob proteins were then aligned with respect to their Cα chains, in such a way as to achieve the lowest possible RMSD. Models containing only the fiber-knob protein of interest and the ligand were saved and subjected to energy minimisation, using the YASARA self-parametrising energy minimisation algorithm as performed by the YASARA energy minimisation server, and results were visualised in PyMol <sup> <a href="#CR78">78</a>, <a href="#CR79">79</a> </sup>. </p> </div> <div class="calculationofinterfaceenergy" title="sec"> <div class="title" tagx="title" title="title">
Calculation of interface energy</div> <p>Interface energies were calculated using QT-PISA using biological protein assemblies and excluding crystallographic interfaces <sup> <a href="#CR80">80</a> </sup>. Values are the mean of the three symmetrical interfaces in each trimer and error is the standard deviation from the mean, any values above −3.0 kcal mol <sup>−1</sup> were considered to be background as shown as a red dashed line on graphs <sup> <a href="#CR42">42</a> </sup>. </p> </div> <div class="sequencealignments" title="sec"> <div class="title" tagx="title" title="title">
Sequence alignments</div> <p>Alignments were performed using the Clustal Omega multiple sequence alignment algorithm and visualised with BioEdit <sup> <a href="#CR61">61</a>, <a href="#CR62">62</a> </sup>. </p> </div> <div class="b-factornormalisation" title="sec"> <div class="title" tagx="title" title="title">
B-factor normalisation</div> <p>Comparing order between different structures by comparing individual B-factors can be misleading. Post-refinement B-factors relate to the Wilson B-factor, which can vary widely between data sets, even from the same crystal preparation. A valid comparison between different structures can be achieved by comparing normalised B-factors instead. Normalisation was performed by dividing individual atomic B-factors by the average B-factor of the whole data set, quantifying the range of internal flexibility in a structure. This normalised B-factor can then be compared between different data sets. </p> </div> <div class="surfaceplasmonresonance(spr)analysis" title="sec"> <div class="title" tagx="title" title="title">
Surface plasmon resonance (SPR) analysis</div> <p>Binding analysis was performed using a BIAcore 3000™ equipped with a CM5 sensor chip. Approximately 5000 RU of CD46, CAR, and DSG2 was attached to the CM5 sensor chip, using amine coupling, at a slow flow-rate of 10 μl/min to ensure uniform distribution on the chip surface. A blank flow cell was used as a negative control surface on flow cell 1. All measurements were performed at 25 °C in PBS buffer (Sigma, UK) at a flow rate of 30 µl/min. For equilibrium binding analysis, the HAdV-D26K and HAdV-B3K fiber knob proteins were purified and concentrated to 367 and 3 μM respectively. 5× 1:3 serial dilutions were prepared for each sample and injected over the relevant sensor chip. The equilibrium binding constant ( <i>K</i> <sub>D</sub>) values were calculated assuming a 1:1 interaction by plotting specific equilibrium-binding responses against protein concentrations followed by non-linear least squares fitting of the Langmuir binding equation. For single cycle kinetic analysis, HAdV-D26K, HAdV-D48K, HAdV-B35K, HAdV-C5K, and HAdV-B3K were injected at a top concentration of around 200 µM, followed by four injections using serial 1:3 dilutions. The <i>K</i> <sub>D</sub> values were calculated assuming Langmuir binding (AB = B×ABmax/(KD + B)) and the data were analysed using the kinetic titration algorithm (BIAevaluationTM 3.1). Receptor proteins were obtained commercially, as follows: Recombinant Human Desmoglein-2 Fc Chimera Protein, R&amp;amp;D Systems, Catalogue number 947-DM-100. Recombinant Human CXADR Fc Chimera Protein (CAR), R&amp;amp;D Systems, Catalogue number 3336-CX-050. Recombinant Human CD46 Protein (His Tag), Sino Biological, Catalogue number 12239-H08H. </p> </div> <div class="reportingsummary" title="sec"> <div class="title" tagx="title" title="title">
Reporting summary</div> <p>Further information on experimental design is available in the <a href="#MOESM3">Nature Research Reporting Summary</a> linked to this article. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2019_8599_MOESM1_ESM.pdf">LINK</a>  <p>Supplementary Information</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2019_8599_MOESM2_ESM.pdf">LINK</a>  <p>Peer Review File</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2019_8599_MOESM3_ESM.pdf">LINK</a>  <p>Reporting Summary</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Journal peer review information</b>: <i>Nature Communications</i> thanks Pascal Fender and the other anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p> <b>Supplementary Information</b> accompanies this paper at 10.1038/s41467-019-08599-y. </p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>A.T.B. is supported by a Tenovus Cancer Care PhD studentship to A.L.P. (reference PhD2015/L13). A.G.-W. was supported by a Life Sciences Research Network Wales (LSRNW) PhD studentship. J.A.D. is supported by a Cancer Research UK Biotherapeutics Drug Discovery Project Award to A.L.P. (project reference C52915/A23946). H.U.-K. was supported by a Cancer Research Wales PhD studentship to A.L.P. L.C. is funded by The HC Roscoe Grant 2016 from the British Medical Association Foundation for Medical Research and by the National Institutes for Health (NIH)/National Institute of Allergy and Infectious Diseases (NIAID) under CEIRS contract HHSN272201400008C. A.L.P. and P.J.R. are funded by Higher Education Funding Council for Wales. The authors acknowledge the Diamond Light Source for beamtime (proposal mx14843) and the staff of beamline I04 for assistance with diffraction data collection. The authors acknowledge Johanne Pentier for technical assistance with SPR data collection, as well as Aaron Wall and Anna Fuller for assistance with FPLC. The authors also acknowledge Elmar Kreiger and the team who maintain the YASARA energy minimisation server, and Pedro Reche and the Immunomedicine Group for maintenance of the Protein Variability Server. </p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>A.T.B. and A.L.P. conceived and designed the study. Modelling of protein–protein interfaces, phylogenetics, protein variability, and interface energy calculations were performed by A.T.B. A.T.B. and A.G.-W. performed crystallisation experiments. A.T.B., A.G.-W., and P.J.R. solved and refined crystallographic structures, and analysed the resultant models. Competition inhibition studies were performed by A.T.B. with advice from L.C. A.T.B. and D.K.C. performed SPR experiments. H.U.-K., J.A.D., and L.C. provided DNA constructs and preliminary data. The manuscript was prepared by A.T.B. and A.L.P.; all other authors reviewed, edited, and approved the manuscript. The study was supervised by A.L.P. </p> <div class="title" tagx="title" title="title">
Data availability</div> <p>Macromolecular structures generated during this study have been deposited in wwPDB (worldwide Protein Data Bank; <a href="https://www.wwpdb.org/">https://www.wwpdb.org/</a>), and have PDB ID’s 6FJN, 6HCN, and 6FJQ. PDB ID’s for macromolecular structures utilised, but not generated in the course of this study, are as follows: HAdV-B11K in complex with CD46, PDB 3O8E. HAdV-D37K in complex with CAR-D1, PDB 2J12. HAdV-B35K, PDB 2QLK. Genomic sequences from which fiber-knob domain sequences were determined, which have been used in phylogenetic analysis, have the following NCBI accession numbers: AC_000017|AF532578|X73487|AY803294|AB562586|AY601636|AF108105|GU191019|JQ326209|AC_000007|JN226749|KF528688|FJ404771|JN226750|JN226751|JN226752|EF153474|JN226753|FJ824826|JN226754|JN226755|AM749299|JN226756|JN226758|AY737797|AC_000019|GQ384080|JN226759|JN226760|KU162869|DQ315364|JN226761|JN226762|JN226763|JN226764|AY875648|JN226757|EF153473|DQ393829|AC_000008|AY737798|JN226765|DQ923122|AB605243|NC_012959|FJ643676|HM770721|HQ413315|AC_000018|DQ086466|JN226746|JN226747|AB448776|AB448767|AJ854486|KF006344|. All other data pertaining to this manuscript are available from the authors upon request. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Combination Adenovirus+Pembrolizumab to Trigger Immune Virus Effects—Full Text View—ClinicalTrials.gov. Available at: <a href="https://clinicaltrials.gov/ct2/show/NCT02798406?term=adenovirus&amp;recrs=a&amp;cond=Glioblastoma&amp;rank=2">https://clinicaltrials.gov/ct2/show/NCT02798406?term=adenovirus&amp;amp;recrs=a&amp;amp;cond=Glioblastoma&amp;amp;rank=2</a> (accessed 23rd September 2017). </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">A Safety and Efficacy Study of ChAdOx1 LS2 and MVA LS2—Full Text View—ClinicalTrials.gov. Available at: <a href="https://clinicaltrials.gov/ct2/show/NCT03203421?term=adenovirus&amp;recrs=a&amp;cond=malaria&amp;rank=2">https://clinicaltrials.gov/ct2/show/NCT03203421?term=adenovirus&amp;amp;recrs=a&amp;amp;cond=malaria&amp;amp;rank=2</a> (accessed 23rd September 2017). </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Long-term Safety Follow-up of Participants Exposed to the Candidate Ebola Vaccines Ad26.ZEBOV and/or MVA-BN-Filo—Full Text View—ClinicalTrials.gov. Available at: <a href="https://clinicaltrials.gov/ct2/show/NCT02661464?term=adenovirus&amp;recrs=a&amp;cond=ebola&amp;rank=1">https://clinicaltrials.gov/ct2/show/NCT02661464?term=adenovirus&amp;amp;recrs=a&amp;amp;cond=ebola&amp;amp;rank=1</a> (accessed 23rd September 2017). </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus-mediated gene delivery: potential applications for gene and cell-based therapies in the new era of personalized medicine </span> <span class="source" tagx="source" title="source">Genes Dis.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">43</span> <span class="lpage" tagx="lpage" title="lpage">63</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gendis.2017.04.001">10.1016/j.gendis.2017.04.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28944281">28944281</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abbink</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative seroprevalence and immunogenicity of six rare serotype recombinant adenovirus vaccine vectors from subgroups B and D </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="fpage" tagx="fpage" title="fpage">4654</span> <span class="lpage" tagx="lpage" title="lpage">4663</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.02696-06">10.1128/JVI.02696-06</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17329340">17329340</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">King, A. M. Q., Adams, M. J., Carstens, E. B. &amp;amp; Lefkowitz, E. J. (eds). Family—Adenoviridae. in <i>Virus Taxonomy</i> 125–141 (Elsevier, 2012). 10.1016/B978-0-12-384684-6.00009-4. </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">International Committee on Taxonomy of Viruses. <i>Virus Taxonomy: Classification and Nomenclature of Viruses: Sixth Report of the International Committee on Taxonomy of Viruses </i> (Springer-Verlag, 1995). </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mast</span> <span class="given-names" tagx="given-names" title="given-names">TC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">International epidemiology of human pre-existing adenovirus (Ad) type-5, type-6, type-26 and type-36 neutralizing antibodies: correlates of high Ad5 titers and implications for potential HIV vaccine trials </span> <span class="source" tagx="source" title="source">Vaccine</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">950</span> <span class="lpage" tagx="lpage" title="lpage">957</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.vaccine.2009.10.145">10.1016/j.vaccine.2009.10.145</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19925902">19925902</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uusi-Kerttula</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hulin-Curtis</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davies</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parker</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Oncolytic adenovirus: strategies and insights for vector design and immuno-oncolytic applications </span> <span class="source" tagx="source" title="source">Viruses</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">6009</span> <span class="lpage" tagx="lpage" title="lpage">6042</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/v7112923">10.3390/v7112923</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26610547">26610547</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Priddy</span> <span class="given-names" tagx="given-names" title="given-names">FH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Safety and immunogenicity of a replication‐incompetent adenovirus type 5 HIV‐1 Clade B <i>gag/pol/nef</i> vaccine in healthy adults </span> <span class="source" tagx="source" title="source">Clin. Infect. Dis.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">1769</span> <span class="lpage" tagx="lpage" title="lpage">1781</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1086/587993">10.1086/587993</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18433307">18433307</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baden</span> <span class="given-names" tagx="given-names" title="given-names">LR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">First-in-human evaluation of a hexon chimeric adenovirus vector expressing HIV-1 Env (IPCAVD 002) </span> <span class="source" tagx="source" title="source">J. Infect. Dis.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">210</span> <span class="fpage" tagx="fpage" title="fpage">1052</span> <span class="lpage" tagx="lpage" title="lpage">1061</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/infdis/jiu217">10.1093/infdis/jiu217</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24719474">24719474</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barouch</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evaluation of a mosaic HIV-1 vaccine in a multicentre, randomised, double-blind, placebo-controlled, phase 1/2a clinical trial (APPROACH) and in rhesus monkeys (NHP 13-19) </span> <span class="source" tagx="source" title="source">Lancet</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">392</span> <span class="fpage" tagx="fpage" title="fpage">232</span> <span class="lpage" tagx="lpage" title="lpage">243</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0140-6736(18)31364-3">10.1016/S0140-6736(18)31364-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30047376">30047376</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular evolution of human adenoviruses</span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">1812</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep01812">10.1038/srep01812</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23657240">23657240</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rux</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuser</span> <span class="given-names" tagx="given-names" title="given-names">PR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burnett</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structural and phylogenetic analysis of adenovirus hexons by use of high-resolution X-ray crystallographic, molecular modeling, and sequence-based methods </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">9553</span> <span class="lpage" tagx="lpage" title="lpage">9566</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.77.17.9553-9566.2003">10.1128/JVI.77.17.9553-9566.2003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12915569">12915569</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Overreliance on the hexon gene, leading to misclassification of human adenoviruses</span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">4693</span> <span class="lpage" tagx="lpage" title="lpage">4695</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.06969-11">10.1128/JVI.06969-11</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22301156">22301156</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teigler</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iampietro</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barouch</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Vaccination with adenovirus serotypes 35, 26, and 48 elicits higher levels of innate cytokine responses than adenovirus serotype 5 in Rhesus monkeys </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">9590</span> <span class="lpage" tagx="lpage" title="lpage">9598</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00740-12">10.1128/JVI.00740-12</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22787208">22787208</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus serotype 26 utilizes CD46 as a primary cellular receptor and only transiently activates T lymphocytes following vaccination of rhesus monkeys </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">10862</span> <span class="lpage" tagx="lpage" title="lpage">10865</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00928-12">10.1128/JVI.00928-12</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22811531">22811531</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uusi-Kerttula</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Pseudotyped αvβ6 integrin-targeted adenovirus vectors for ovarian cancer therapies</span> <span class="source" tagx="source" title="source">Oncotarget</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">27926</span> <span class="lpage" tagx="lpage" title="lpage">27937</span> <span class="pub-id"> <a href="https://dx.doi.org/10.18632/oncotarget.8545">10.18632/oncotarget.8545</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27056886">27056886</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Camacho</span> <span class="given-names" tagx="given-names" title="given-names">ZT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turner</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barry</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weaver</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">CD46-mediated transduction of a species D adenovirus vaccine improves mucosal vaccine efficacy </span> <span class="source" tagx="source" title="source">Hum. Gene Ther.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">364</span> <span class="lpage" tagx="lpage" title="lpage">374</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1089/hum.2013.215">10.1089/hum.2013.215</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24635714">24635714</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Weaver, E. A. &amp;amp; Barry, M. A. Low seroprevalent species D adenovirus vectors as influenza vaccines. <i>PLoS ONE</i> <b>8</b>, e73313 (2013). </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geisbert</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Recombinant adenovirus serotype 26 (Ad26) and Ad35 vaccine vectors bypass immunity to Ad5 and protect nonhuman primates against ebolavirus challenge </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">85</span> <span class="fpage" tagx="fpage" title="fpage">4222</span> <span class="lpage" tagx="lpage" title="lpage">4233</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.02407-10">10.1128/JVI.02407-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21325402">21325402</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic evidence for intratypic recombinant events in a novel human adenovirus C that causes severe acute respiratory infection in children </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">23014</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep23014">10.1038/srep23014</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26960434">26960434</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ebner</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinsker</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lion</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative sequence analysis of the hexon gene in the entire spectrum of human adenovirus serotypes: phylogenetic, taxonomic, and clinical implications </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">79</span> <span class="fpage" tagx="fpage" title="fpage">12635</span> <span class="lpage" tagx="lpage" title="lpage">12642</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.79.20.12635-12642.2005">10.1128/JVI.79.20.12635-12642.2005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16188965">16188965</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al Qurashi</span> <span class="given-names" tagx="given-names" title="given-names">YMA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alkhalaf</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guiver</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cooper</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Sequencing and phylogenetic analysis of the hexon, fiber, and penton regions of adenoviruses isolated from AIDS patients </span> <span class="source" tagx="source" title="source">J. Med. Virol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">1157</span> <span class="lpage" tagx="lpage" title="lpage">1165</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/jmv.23331">10.1002/jmv.23331</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22711343">22711343</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pache</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venkataraman</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nemerow</span> <span class="given-names" tagx="given-names" title="given-names">GR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reddy</span> <span class="given-names" tagx="given-names" title="given-names">VS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Conservation of fiber structure and CD46 usage by subgroup B2 adenoviruses</span> <span class="source" tagx="source" title="source">Virology</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">375</span> <span class="fpage" tagx="fpage" title="fpage">573</span> <span class="lpage" tagx="lpage" title="lpage">579</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.virol.2008.02.013">10.1016/j.virol.2008.02.013</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18336857">18336857</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Russell</span> <span class="given-names" tagx="given-names" title="given-names">WC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenoviruses: update on structure and function</span> <span class="source" tagx="source" title="source">J. Gen. Virol.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">90</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/vir.0.003087-0">10.1099/vir.0.003087-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19088268">19088268</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henry</span> <span class="given-names" tagx="given-names" title="given-names">LJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerard</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deisenhofer</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Crystal structure of the receptor-binding domain of adenovirus type 5 fiberprotein at 1.7 Å resolution </span> <span class="source" tagx="source" title="source">Structure</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">1259</span> <span class="lpage" tagx="lpage" title="lpage">1270</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0969-2126(94)00126-X">10.1016/S0969-2126(94)00126-X</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7704534">7704534</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baker</span> <span class="given-names" tagx="given-names" title="given-names">AT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aguirre-Hernández</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Halldén</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parker</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Designer oncolytic adenovirus: coming of age</span> <span class="source" tagx="source" title="source">Cancers</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">201</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/cancers10060201">10.3390/cancers10060201</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amoureux</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Polysialic Acid Neural Cell Adhesion Molecule (PSA-NCAM) is an adverse prognosis factor in glioblastoma, and regulates olig2 expression in glioma cell lines </span> <span class="source" tagx="source" title="source">BMC Cancer</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">91</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2407-10-91">10.1186/1471-2407-10-91</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20219118">20219118</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lenman</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Human adenovirus 52 uses sialic acid-containing glycoproteins and the coxsackie and adenovirus receptor for binding to target cells </span> <span class="source" tagx="source" title="source">PLoS Pathog.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">e1004657</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.ppat.1004657">10.1371/journal.ppat.1004657</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25674795">25674795</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnberg</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus type 37 binds to cell surface sialic acid through a charge-dependent interaction</span> <span class="source" tagx="source" title="source">Virology</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">302</span> <span class="fpage" tagx="fpage" title="fpage">33</span> <span class="lpage" tagx="lpage" title="lpage">43</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/viro.2002.1503">10.1006/viro.2002.1503</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12429514">12429514</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnberg</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edlund</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kidd</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wadell</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus type 37 uses sialic acid as a cellular receptor</span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">42</span> <span class="lpage" tagx="lpage" title="lpage">48</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.74.1.42-48.2000">10.1128/JVI.74.1.42-48.2000</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10590089">10590089</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cupelli</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structure of adenovirus type 21 knob in complex with CD46 reveals key differences in receptor contacts among species B adenoviruses </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">3189</span> <span class="lpage" tagx="lpage" title="lpage">3200</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.01964-09">10.1128/JVI.01964-09</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20071571">20071571</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Persson</span> <span class="given-names" tagx="given-names" title="given-names">BD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structure of the extracellular portion of CD46 provides insights into its interactions with complement proteins and pathogens </span> <span class="source" tagx="source" title="source">PLoS Pathog.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">e1001122</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.ppat.1001122">10.1371/journal.ppat.1001122</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20941397">20941397</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Persson</span> <span class="given-names" tagx="given-names" title="given-names">BD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus type 11 binding alters the conformation of its receptor CD46</span> <span class="source" tagx="source" title="source">Nat. Struct. Mol. Biol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">164</span> <span class="lpage" tagx="lpage" title="lpage">166</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nsmb1190">10.1038/nsmb1190</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17220899">17220899</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of CD46 binding sites within the adenovirus serotype 35 fiber knob</span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="fpage" tagx="fpage" title="fpage">12785</span> <span class="lpage" tagx="lpage" title="lpage">12792</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.01732-07">10.1128/JVI.01732-07</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17898059">17898059</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kirby</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of contact residues and definition of the CAR-binding site of adenovirus type 5 fiber protein </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">2804</span> <span class="lpage" tagx="lpage" title="lpage">2813</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.74.6.2804-2813.2000">10.1128/JVI.74.6.2804-2813.2000</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10684297">10684297</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nilsson</span> <span class="given-names" tagx="given-names" title="given-names">EC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The GD1a glycan is a cellular receptor for adenoviruses causing epidemic keratoconjunctivitis</span> <span class="source" tagx="source" title="source">Nat. Med.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">105</span> <span class="lpage" tagx="lpage" title="lpage">109</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nm.2267">10.1038/nm.2267</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21151139">21151139</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fleischli</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Species B adenovirus serotypes 3, 7, 11 and 35 share similar binding sites on the membrane cofactor protein CD46 receptor </span> <span class="source" tagx="source" title="source">J. Gen. Virol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">2925</span> <span class="lpage" tagx="lpage" title="lpage">2934</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1099/vir.0.83142-0">10.1099/vir.0.83142-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17947513">17947513</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seiradake</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lortat-Jacob</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Billet</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kremer</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cusack</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structural and mutational analysis of human Ad37 and canine adenovirus 2 fiber heads in complex with the D1 domain of coxsackie and adenovirus receptor </span> <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">281</span> <span class="fpage" tagx="fpage" title="fpage">33704</span> <span class="lpage" tagx="lpage" title="lpage">33716</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M605316200">10.1074/jbc.M605316200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16923808">16923808</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krieger</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koraimann</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vriend</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Increasing the precision of comparative models with YASARA NOVA—a self‐parameterizing force field </span> <span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinformatics</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">393</span> <span class="lpage" tagx="lpage" title="lpage">402</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/prot.10104">10.1002/prot.10104</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Predicting accurate absolute binding energies in aqueous solution: thermodynamic considerations for electronic structure methods </span> <span class="source" tagx="source" title="source">Phys. Chem. Chem. Phys.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">12441</span> <span class="lpage" tagx="lpage" title="lpage">12451</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C5CP00628G">10.1039/C5CP00628G</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25901455">25901455</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Vassal-Stermann, E. et al. Mapping of adenovirus of serotype 3 fibre interaction to desmoglein 2 revealed a novel ‘non-classical’ mechanism of viral receptor engagement. <i>Sci. Rep</i>. <b>8</b>, 8381 (2018). </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Desmoglein 2 is a receptor for adenovirus serotypes 3, 7, 11 and 14</span> <span class="source" tagx="source" title="source">Nat. Med.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">96</span> <span class="lpage" tagx="lpage" title="lpage">104</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nm.2270">10.1038/nm.2270</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21151137">21151137</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Segerman</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnberg</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Erikson</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindman</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wadell</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">There are two different species B adenovirus receptors: sBAR, common to species B1 and B2 adenoviruses, and sB2AR, exclusively used by species B2 adenoviruses </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">1157</span> <span class="lpage" tagx="lpage" title="lpage">1162</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.77.2.1157-1162.2003">10.1128/JVI.77.2.1157-1162.2003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12502832">12502832</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuhn</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Directed evolution generates a novel oncolytic virus for the treatment of colon cancer</span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">e2409</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0002409">10.1371/journal.pone.0002409</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18560559">18560559</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Illingworth</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Preclinical safety studies of enadenotucirev, a chimeric group B human-specific oncolytic adenovirus </span> <span class="source" tagx="source" title="source">Mol. Ther. Oncolytics</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">62</span> <span class="lpage" tagx="lpage" title="lpage">74</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.omto.2017.03.003">10.1016/j.omto.2017.03.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28480328">28480328</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">HAdV Working Group. <a href="http://hadvwg.gmu.edu/">http://hadvwg.gmu.edu/</a>. </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bradley</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iampietro</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borducchi</span> <span class="given-names" tagx="given-names" title="given-names">EN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barouch</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus serotype 5 neutralizing antibodies target both hexon and fiber following vaccination and natural infection </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">625</span> <span class="lpage" tagx="lpage" title="lpage">629</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.06254-11">10.1128/JVI.06254-11</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22072776">22072776</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bradley</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus serotype 5-specific neutralizing antibodies target multiple hexon hypervariable regions </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">1267</span> <span class="lpage" tagx="lpage" title="lpage">1272</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.06165-11">10.1128/JVI.06165-11</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22072746">22072746</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ma, J. et al. Manipulating adenovirus hexon hypervariable loops dictates immune neutralisation and coagulation factor X-dependent cell interaction in vitro and in vivo. <i>PLoS Pathog</i>. <b>11</b>, e1004673 (2015). </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parker</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of neutralizing sera on factor x-mediated adenovirus serotype 5 gene transfer</span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">83</span> <span class="fpage" tagx="fpage" title="fpage">479</span> <span class="lpage" tagx="lpage" title="lpage">483</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.01878-08">10.1128/JVI.01878-08</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18945780">18945780</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walters</span> <span class="given-names" tagx="given-names" title="given-names">RW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus fiber disrupts CAR-mediated intercellular adhesion allowing virus escape</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">110</span> <span class="fpage" tagx="fpage" title="fpage">789</span> <span class="lpage" tagx="lpage" title="lpage">799</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0092-8674(02)00912-1">10.1016/S0092-8674(02)00912-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12297051">12297051</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keele</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Adenovirus prime, Env protein boost vaccine protects against neutralization-resistant SIVsmE660 variants in rhesus monkeys </span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">15740</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms15740">10.1038/ncomms15740</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28580942">28580942</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andrews</span> <span class="given-names" tagx="given-names" title="given-names">PW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A human cell-surface antigen defined by a monoclonal antibody and controlled by a gene on human chromosome 1 </span> <span class="source" tagx="source" title="source">Ann. Hum. Genet.</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">31</span> <span class="lpage" tagx="lpage" title="lpage">39</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-1809.1985.tb01673.x">10.1111/j.1469-1809.1985.tb01673.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2416262">2416262</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uhlen</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A pathology atlas of the human cancer transcriptome</span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">357</span> <span class="fpage" tagx="fpage" title="fpage">eaan2507</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.aan2507">10.1126/science.aan2507</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28818916">28818916</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seiradake</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The cell adhesion molecule "CAR" and sialic acid on human erythrocytes influence adenovirus in vivo biodistribution </span> <span class="source" tagx="source" title="source">PLoS Pathog.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">e1000277</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.ppat.1000277">10.1371/journal.ppat.1000277</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19119424">19119424</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohen</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The coxsackievirus and adenovirus receptor is a transmembrane component of the tight junction </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">98</span> <span class="fpage" tagx="fpage" title="fpage">15191</span> <span class="lpage" tagx="lpage" title="lpage">15196</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.261452898">10.1073/pnas.261452898</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11734628">11734628</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Awasthi</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meinken</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Springer</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freimuth</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biodistribution of radioiodinated adenovirus fiber protein knob domain after intravenous injection in mice </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">78</span> <span class="fpage" tagx="fpage" title="fpage">6431</span> <span class="lpage" tagx="lpage" title="lpage">6438</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.78.12.6431-6438.2004">10.1128/JVI.78.12.6431-6438.2004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15163736">15163736</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lenman, A. et al. Polysialic acid is a cellular receptor for human adenovirus 52. <i>Proc. Natl. Acad. Sci. USA</i>10.1073/pnas.1716900115 (2018). </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sievers</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega </span> <span class="source" tagx="source" title="source">Mol. Syst. Biol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">539</span> <span class="lpage" tagx="lpage" title="lpage">539</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/msb.2011.75">10.1038/msb.2011.75</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21988835">21988835</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The EMBL-EBI bioinformatics web and programmatic tools framework</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">W580</span> <span class="lpage" tagx="lpage" title="lpage">W584</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv279">10.1093/nar/gkv279</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25845596">25845596</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jukes, T. H. &amp;amp; Cantor, C. R. Evolution of protein molecules. in <i>Mammalian Protein Metabolism</i> 21–132 (Elsevier, 1969). 10.1016/B978-1-4832-3211-9.50009-7. </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zuckerkandl, E. &amp;amp; Pauling, L. Evolutionary divergence and convergence in proteins. in <i>Evolving Genes and Proteins</i> (eds Bryson, V. &amp;amp; Vogel, H. J.) 97–166 (Academic Press, 1965). 10.1016/B978-1-4832-2734-4.50017-6. </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stecher</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knyaz</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across computing platforms</span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">1547</span> <span class="lpage" tagx="lpage" title="lpage">1549</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msy096">10.1093/molbev/msy096</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29722887">29722887</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Felsenstein</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Confidence limits on phylogenies: an approach using the Bootstrap</span> <span class="source" tagx="source" title="source">Evolution</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">783</span> <span class="lpage" tagx="lpage" title="lpage">791</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1558-5646.1985.tb00420.x">10.1111/j.1558-5646.1985.tb00420.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28561359">28561359</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Boronat</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diez-Rivero</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinherz</span> <span class="given-names" tagx="given-names" title="given-names">EL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reche</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">PVS: a web server for protein sequence variability analysis tuned to facilitate conserved epitope discovery </span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">W35</span> <span class="lpage" tagx="lpage" title="lpage">W41</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkn211">10.1093/nar/gkn211</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18442995">18442995</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coughlan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Retargeting adenovirus serotype 48 fiber knob domain by peptide incorporation</span> <span class="source" tagx="source" title="source">Hum. Gene Ther.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">385</span> <span class="lpage" tagx="lpage" title="lpage">394</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1089/hum.2014.016">10.1089/hum.2014.016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24617540">24617540</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coughlan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">In vivo retargeting of adenovirus type 5 to αvβ6 integrin results in reduced hepatotoxicity and improved tumor uptake following systemic delivery </span> <span class="source" tagx="source" title="source">J. Virol.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">83</span> <span class="fpage" tagx="fpage" title="fpage">6416</span> <span class="lpage" tagx="lpage" title="lpage">6428</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00445-09">10.1128/JVI.00445-09</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19369326">19369326</a> </span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bergelson</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation of a common receptor for coxsackie B viruses and adenoviruses 2 and 5</span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">275</span> <span class="fpage" tagx="fpage" title="fpage">1320</span> <span class="lpage" tagx="lpage" title="lpage">1323</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.275.5304.1320">10.1126/science.275.5304.1320</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9036860">9036860</a> </span> </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manchester</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liszewski</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Atkinson</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oldstone</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Multiple isoforms of CD46 (membrane cofactor protein) serve as receptors for measles virus </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="fpage" tagx="fpage" title="fpage">2161</span> <span class="lpage" tagx="lpage" title="lpage">2165</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.91.6.2161">10.1073/pnas.91.6.2161</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8134365">8134365</a> </span> </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Winter</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lobley</span> <span class="given-names" tagx="given-names" title="given-names">CMC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prince</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Decision making in xia2</span> <span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">1260</span> <span class="lpage" tagx="lpage" title="lpage">1273</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1107/S0907444913015308">10.1107/S0907444913015308</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23793152">23793152</a> </span> </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vonrhein</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Data processing and analysis with the autoPROC toolbox</span> <span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">67</span> <span class="fpage" tagx="fpage" title="fpage">293</span> <span class="lpage" tagx="lpage" title="lpage">302</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1107/S0907444911007773">10.1107/S0907444911007773</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21460447">21460447</a> </span> </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dodson, E. J., Winn, M. &amp;amp; Ralph, A. Collaborative computational project, number 4: providing programs for protein crystallography, Vol. 277. in <i>Methods in Enzymology</i> 620–633 (Academic Press, 1997). </span> </li> <li tag="ref"> <a name="CR75" /> <span class="label" tagx="label" title="label">75.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCoy</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phaser crystallographic software</span> <span class="source" tagx="source" title="source">J. Appl. Crystallogr.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">658</span> <span class="lpage" tagx="lpage" title="lpage">674</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1107/S0021889807021206">10.1107/S0021889807021206</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19461840">19461840</a> </span> </span> </li> <li tag="ref"> <a name="CR76" /> <span class="label" tagx="label" title="label">76.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emsley</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cowtan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Coot: model-building tools for molecular graphics</span> <span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">60</span> <span class="fpage" tagx="fpage" title="fpage">2126</span> <span class="lpage" tagx="lpage" title="lpage">2132</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1107/S0907444904019158">10.1107/S0907444904019158</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15572765">15572765</a> </span> </span> </li> <li tag="ref"> <a name="CR77" /> <span class="label" tagx="label" title="label">77.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murshudov</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">REFMAC5 for the refinement of macromolecular crystal structures</span> <span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">67</span> <span class="fpage" tagx="fpage" title="fpage">355</span> <span class="lpage" tagx="lpage" title="lpage">367</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1107/S0907444911001314">10.1107/S0907444911001314</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21460454">21460454</a> </span> </span> </li> <li tag="ref"> <a name="CR78" /> <span class="label" tagx="label" title="label">78.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Schrödinger, LLC. The PyMOL Molecular Graphics System, Version 2.0.</span> </li> <li tag="ref"> <a name="CR79" /> <span class="label" tagx="label" title="label">79.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krieger</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improving physical realism, stereochemistry, and side‐chain accuracy in homology modeling: four approaches that performed well in CASP8 </span> <span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinformatics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">114</span> <span class="lpage" tagx="lpage" title="lpage">122</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/prot.22570">10.1002/prot.22570</a> </span> </span> </li> <li tag="ref"> <a name="CR80" /> <span class="label" tagx="label" title="label">80.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krissinel</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henrick</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inference of macromolecular assemblies from crystalline state</span> <span class="source" tagx="source" title="source">J. Mol. Biol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">372</span> <span class="fpage" tagx="fpage" title="fpage">774</span> <span class="lpage" tagx="lpage" title="lpage">797</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jmb.2007.05.022">10.1016/j.jmb.2007.05.022</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17681537">17681537</a> </span> </span> </li> </ul> </div> </div>  </body></html>