<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Genome Biol Evol</span> <span class="iso-abbrev" title="iso-abbrev">Genome Biol Evol</span> <span class="publisher-id" title="publisher-id">gbe</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Genome Biology and Evolution</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1759-6653</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6601870">6601870</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31173074">31173074</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/gbe/evz118">10.1093/gbe/evz118</a> </span> <span class="publisher-id" title="publisher-id">evz118</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Population Genomic Signatures of Genetic Structure and Environmental Selection in the Catadromous Roughskin Sculpin <i>Trachidermus fasciatus</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Yu-Long</span> </span> <a href="#evz118-aff1">1</a> <a href="#evz118-aff2">2</a> <a href="#evz118-aff3">3</a> <a href="#evz118-FM1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">Dong-Xiu</span> </span> <a href="#evz118-aff1">1</a> <a href="#evz118-aff2">2</a> <a href="#evz118-aff3">3</a> <a href="#evz118-FM1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Bai-Dong</span> </span> <a href="#evz118-aff1">1</a> <a href="#evz118-aff2">2</a> <a href="#evz118-aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Jin-Xian</span> </span> <a href="#evz118-aff1">1</a> <a href="#evz118-aff2">2</a> <a href="#evz118-aff3">3</a> <a href="#evz118-cor1" /> </span> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Piontkivska</span> <span class="given-names" tagx="given-names" title="given-names">Helen</span> </span> <span class="role" tagx="role" title="role">Associate Editor</span> </span> </div> <span class="citation_author_institution" id="evz118-aff1">[1], </span> <span class="citation_author_institution" id="evz118-aff2">[2], </span> <span class="citation_author_institution" id="evz118-aff3">[3], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Corresponding author: E-mail: <span class="email" tagx="email" title="email">jinxianliu@gmail.com</span>. </div> <div class="fn-type-" title=""> <p>These authors contributed equally to this work.</p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-7</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-6-6</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-6-6</span> </span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">1751</span> <span class="lpage" tagx="lpage" title="lpage">1764</span> <span class="history" title="history"> <span class="accepted" title="accepted">accepted: 2019-5-31</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <a class="self-uri" href="evz118.pdf" title="self-uri">evz118.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Understanding the patterns of genetic diversity and adaptation across species’ range is crucial to assess its long-term persistence and determine appropriate conservation measures. The impacts of human activities on the genetic diversity and genetic adaptation to heterogeneous environments remain poorly understood in the marine realm. The roughskin sculpin ( <i>Trachidermus fasciatus</i>) is a small catadromous fish, and has been listed as a second-class state protected aquatic animal since 1988 in China. To elucidate the underlying mechanism of population genetic structuring and genetic adaptations to local environments, RAD tags were sequenced for 202 individuals in nine populations across the range of <i>T. fasciatus</i> in China. The pairwise <i>F</i> <sub>ST</sub> values over 9,271 filtered SNPs were significant except that between Dongying and Weifang. All the genetic clustering analysis revealed significant population structure with high support for eight distinct genetic clusters. Both the minor allele frequency spectra and <i>N</i>e estimations suggested extremely small <i>N</i>e in some populations (e.g., Qinhuangdao, Rongcheng, Wendeng, and Qingdao), which might result from recent population bottleneck. The strong genetic structure can be partly attributed to genetic drift and habitat fragmentation, likely due to the anthropogenic activities. Annotations of candidate adaptive loci suggested that genes involved in metabolism, development, and osmoregulation were critical for adaptation to spatially heterogenous environment of local populations. In the context of anthropogenic activities and environmental change, results of the present population genomic work provided important contributions to the understanding of genetic differentiation and adaptation to changing environments. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
population structure</div> <div class="kwd" title="kwd">
local adaptation</div> <div class="kwd" title="kwd"> <i>Trachidermus fasciatus</i> </div> <div class="kwd" title="kwd">
habitat fragmentation</div> <div class="kwd" title="kwd">
RAD-seq</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">National Natural Science Foundation of China</span> <span class="named-content" title="named-content">10.13039/501100001809</span> </div> <span class="award-id" tagx="award-id" title="award-id">31502169</span> <span class="award-id" tagx="award-id" title="award-id">41776171</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">NSFC-Shandong Joint Fund for Marine Ecology and Environmental Sciences</span> <span class="named-content" title="named-content">10.13039/501100001809</span> </div> <span class="award-id" tagx="award-id" title="award-id">U1606404</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">AoShan Talents Program</span> </div> <span class="award-id" tagx="award-id" title="award-id">2015ASTP-ES05</span> </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>When different populations experience heterogeneous environments, local selection regimes can drive phenotypic divergence and modulate the underlying genomic architecture, thereby promoting local adaptation and ultimately initiating evolutionary diversification and speciation ( <a href="#evz118-B87">Schluter 2000</a>; <a href="#evz118-B32">Ferchaud and Hansen 2016</a>). Detecting selection is an important step toward understanding the genetic basis of adaptive traits and the potential vulnerability or resilience of biodiversity to environment change ( <a href="#evz118-B17">Brauer et al. 2016</a>; <a href="#evz118-B7">Attard et al. 2018</a>; <a href="#evz118-B84">Sandoval-Castillo et al. 2018</a>). Reduction in genetic diversity can compromise the potential of a population to evolutionarily adapt to changing environments ( <a href="#evz118-B3">Allendorf and Luikart 2007</a>; <a href="#evz118-B76">Ouborg et al. 2010</a>; <a href="#evz118-B5">Angeloni et al. 2012</a>). Small and isolated populations are potentially faced with high extinction risks, due to the strong impact of genetic drift and inbreeding with low level of genetic variation ( <a href="#evz118-B37">Frankham et al. 2002</a>; <a href="#evz118-B76">Ouborg et al. 2010</a>). Climate change and habitat loss are among the most important factors resulting in biodiversity decline ( <a href="#evz118-B10">Barbosa et al. 2018</a>). Evolutionary adaptations from standing genetic variation over generations are crucial for species to respond to the new or altered environments, leading to the expression of environment-dependent phenotypes ( <a href="#evz118-B11">Barrett and Schluter 2008</a>; <a href="#evz118-B42">Harrisson et al. 2014</a>; <a href="#evz118-B10">Barbosa et al. 2018</a>). Population with low level of genetic diversity is potentially faced with low effective population size ( <i>N</i>e). <i>N</i>e is a key parameter to understand evolutionary processes and the viability of the overexploited and endangered populations as it determines the rate of genetic drift and inbreeding ( <a href="#evz118-B8">Baalsrud et al. 2014</a>). For population with low <i>N</i>e, the most important evolutionary forces are genetic drift and inbreeding, both leading to rapid loss of genetic diversity, which will result in a low-fitness risk of the population ( <a href="#evz118-B71">McMahon et al. 2014</a>). Estimates of <i>N</i>e can be theoretically useful to predict the impact of management practices on the loss of genetic variability due to the anthropogenic induced effects of random drift of the overexploited and endangered fishery species ( <a href="#evz118-B79">Pita et al. 2017</a>). For species long‐term persistence, there is a need to maximize genetic variability of species in order to maximize evolutionary potential. In addition, the long-term viability is also dependent on local adaptation, and genetic variability may be expected to decrease but would not limit long term evolutionary potential. Considering the importance of genetic diversity, it is essential and is the first step to appropriately define conservation units (CUs) to help guide management and conservation efforts ( <a href="#evz118-B3">Allendorf and Luikart 2007</a>; <a href="#evz118-B38">Funk et al. 2012</a>). With the rapid development of high-throughput next-generation sequencing (NGS), genome-wide data make it easier to integrate information from neutral and adaptive loci to characterize CUs within a population genetics framework ( <a href="#evz118-B38">Funk et al. 2012</a>). <a href="#evz118-B38">Funk et al. (2012)</a> proposed the delineation of three categories of CUs using genomic data: Evolutionarily significant units (ESUs) are defined using all the loci, maintenance of different ESUs will maximize evolutionary potential, management units (MUs) are defined using the neutral loci and are the basis for the short‐term management of populations, and adaptive units (AUs) are defined using the adaptive loci and are intended to help with protection of adaptive processes ( <a href="#evz118-B72">Moritz 2004</a>; Funk et al. <a href="#evz118-B38">2012</a>). Maintenance of different CUs will help maximize genetic variability of species in the face of environmental change. </p> <p>The roughskin sculpin ( <i>Trachidermus fasciatus</i> Heckel 1837), belonging to the family Cottidae, is a small carnivorous catadromous fish dwelling on the bottom of water ( <a href="#evz118-B93">Wang 1999</a>). <i>Trachidermus</i> <i>fasciatus</i> migrates from freshwater toward estuary and spawns in empty bivalve shells in winter, then the juveniles move upstream in spring to freshwater for growth ( <a href="#evz118-B89">Shao et al. 1980</a>; <a href="#evz118-B75">Onikura et al. 2002</a>). When <i>T. fasciatus</i> migrates from freshwater toward estuary, it stops exogenous nutrition ingestion and its digestive organs show an involution in various degrees ( <a href="#evz118-B60">Li et al. 1984</a>). The thyroid gland of <i>T. fasciatus</i> changes seasonally in close relation to the seaward migration ( <a href="#evz118-B88">Shao 1978</a>). According to historical records, <i>T. fasciatus</i> was widely distributed along the coastlines of the Bohai Sea, the Yellow Sea, and the East China Sea as well as in rivers connected to seas ( <a href="#evz118-B94">Wang and Cheng 2010</a>). However, it has experienced severe population declines, habitat fragmentation and local extinctions in the past decades, probably due to human-mediated environmental change such as water pollution, dam construction, and blockage of migration channel ( <a href="#evz118-B94">Wang and Cheng 2010</a>). Besides, under the background of global climate change and human activities, runoff volume of some rivers (e.g., Dagu River-Qingdao, Luan River-Qinhuangdao, Wei River-Weifang) has been sharply reduced since recent decades, and drying-up of the river course has occurred frequently ( <a href="#evz118-B48">Jiang and Wang 2013</a>; <a href="#evz118-B65">Liu et al. 2013</a>). This condition has seriously threatened the migration of <i>T. fasciatus</i>, and probably resulted in changes of life history of some <i>T. fasciatus</i> populations (e.g., Qingdao, Weifang, and Rongcheng population) from catadromous to noncatadromous. Considering its great biological, ecological, and economic importance in China, <i>T. fasciatus</i> had been included in the List of the Wildlife under Special State Protection as a second-class state protected aquatic animal by Chinese government in 1988. </p> <p>In previous studies, researchers had studied population structure and estimated demographic parameters of <i>T. fasciatus</i>, mainly based on presumably neutral DNA-based markers ( <a href="#evz118-B99">Xu et al. 2008</a>; <a href="#evz118-B102">Zeng et al. 2012</a>; <a href="#evz118-B39">Gao et al. 2013</a>; <a href="#evz118-B62">Li et al. 2016</a>). In our previous study, we found significant population structure and five genetic clusters among seven populations in China based on 16 microsatellite loci ( <a href="#evz118-B62">Li et al. 2016</a>). The five genetic clusters have presumably evolved from standing genetic variation of the same genetic lineage following deglaciation around 120,000 years ago ( <a href="#evz118-B39">Gao et al. 2013</a>). To date, few studies were conducted to estimate the genetic effective population size ( <i>N</i>e) of <i>T. fasciatus</i>, likely due to the lack of power of genome-wide genotyping methods. Moreover, previous studies restricted to neutral markers can provide only limited insight to the mechanisms of local adaptation. Across its widely distributed range for <i>T. fasciatus</i>, the temperature, salinity, the levels of pollution, and pathogen conditions are largely different ( <a href="#evz118-B9">Bao et al. 2002</a>). Heterogeneous environment can result in patterns of localized adaptive divergence of populations ( <a href="#evz118-B86">Savolainen et al. 2013</a>; <a href="#evz118-B84">Sandoval-Castillo et al. 2018</a>). In these circumstances, to improve the precision and accuracy of estimating a variety of important population genetic parameters and elucidating the mechanisms of local adaption of <i>T. fasciatus</i>, genomic tools as one promising approach may assist by increasing the number of variable genetic markers including both neutral and nonneutral loci. Restriction site-associated DNA tags sequencing (RAD-seq) is one of the rapid and cost-effective methods for genome-wide single nucleotide polymorphism (SNP) discovery and genotyping of nonmodel species without reference genomes ( <a href="#evz118-B20">Catchen et al. 2013</a>; <a href="#evz118-B56">Larson et al. 2014</a>; <a href="#evz118-B70">McKinney et al. 2017</a>). </p> <p>In the present study, nine <i>T. fasciatus</i> populations were collected from its entire natural habitats in China. A comprehensive and high-resolution genome-wide assessment of population genetic status of <i>T. fasciatus</i> was conducted at both neutral loci and putative adaptive loci generated from RAD sequencing. We specifically aimed to address two questions that should have broad implications to conservation and management of <i>T. fasciatus</i>. First, do <i>T. fasciatus</i> exhibit fine genetic structure and what is the status of <i>N</i>e for different populations? Second, how the populations genetically adapted to local heterogeneous environments? The results of the present population genomic study can contribute to the understanding of genetic differentiation of aquatic species under anthropogenic activities and their adaptation to changing environments. </p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="samplingandradsequencing" title="sec"> <div class="title" tagx="title" title="title">
Sampling and RAD Sequencing</div> <p>A total of 202 individuals of <i>T. fasciatus</i> were collected from nine sites across its range-distribution in China during 2013–2017 ( <a href="#evz118-F1">fig. 1</a> and <a href="#sup1">supplementary table S1</a>, <a href="#sup1">Supplementary Material</a> online). Tissues (including fin clips and muscles) were kept individually in 95% ethanol and stored at –80 °C. Genomic DNA samples were extracted following the standard phenol-chloroform extraction method. DNA extracts were visualized on 1% agarose gels to assess quality and were subsequently quantified using Qubit Fluorometer. Paired-end RAD (RPE) libraries were prepared followed the protocol as described by <a href="#evz118-B29">Etter et al. (2011)</a>. Briefly, genomic DNA was digested with the restriction enzyme <i>Eco</i>RI, and fragments were then ligated to Illumina P1 adapter with individual-specific index. Adapter-ligated fragments were subsequently pooled and randomly sheared, end repaired, dA added and ligated to Illumina P2 adapter. The fragments with both P1 and P2 adapter were enriched using high-fidelity PCR amplification. The RPE libraries were then sequenced on Illumina HiSeq machine at Novogene (Beijing, China). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 1. </span> </a> </div>   <p>—Map of the sampling location for nine populations of <i>Trachidermus fasciatus</i>. Figure was generated by Ocean Data View 5.0.0 (Schlitzer, R., Ocean Data View, odv.awi.de, 2018). </p>   </div> </div> <div class="raddataprocessing,genotyping,andsnpfiltering" title="sec"> <div class="title" tagx="title" title="title">
RAD Data Processing, Genotyping, and SNP Filtering</div> <p>Raw reads from Illumina runs were demultiplexed into separated files according to individuals’ indexes. Reads with adaptors were removed using Cutadapt ( <a href="#evz118-B69">Martin 2011</a>), and further cleaned using "process_radtags" of Stacks 1.48 ( <a href="#evz118-B21">Catchen et al. 2011</a>) with a score limit of 13 and windows size of 0.1. Only those reads with sufficiently high sequencing quality, and that had an unambiguous RAD site, were retained. Finally, reads were processed using "clone_filter" of Stacks to remove potential PCR duplications. </p> <p>RAD contigs were assembled using an optimized method as implemented in RADassembler ( <a href="#evz118-B63">Li et al. 2018</a>). This pipeline software uses Stacks to cluster reads from multiple individuals, and uses CAP3 ( <a href="#evz118-B46">Huang and Madan 1999</a>) to do local de novo assembly, which makes full use of the advantages of RPE reads. The optimal similarity thresholds for clustering within and across individuals were chosen using "chooseM" and "chooseN" functions of RADassembler. Reads were then aligned to the assembled contigs using BWA-MEM 0.7.15 ( <a href="#evz118-B58">Li 2013</a>) with default parameters, and were subsequently processed by SAMtools 1.7 ( <a href="#evz118-B59">Li et al. 2009</a>). SNP calling were performed using BCFtools in SAMtools. The generated SNPs were stored in a Variant Call Format (VCF) file. </p> <p>In order to remove SNPs of low qualities, which might be resulted from false calling or paralogs, the following cut-offs were applied using in house Perl script (accessed at <a href="https://github.com/lyl8086/VCF_filter">https://github.com/lyl8086/VCF_filter</a>; Last accessed June 5, 2019): 1) only biallelic SNPs were retained; 2) SNP quality ≥ 30; 3) minimum coverage depth for an individual ≥ 6; 4) minimum genotype quality (GQ) for an individual ≥ 15; 5) minimum individual coverage for a population ≥ 12, that is at least 12 individuals were required to be genotyped for a SNP in each population; 6) total missing rate for all individuals of each SNP ≤ 0.1; 7) maximum observed heterozygosity ( <i>H</i> <sub>O</sub>) for each population ≤ 0.5; 8) minor allele frequency (MAF) ≥ 0.05, but SNPs with MAF ≥ 0.2 in at least one population were retained. The final VCF file was converted into other formats necessary for subsequent analyses using PGDSpider 2.0.1.0 ( <a href="#evz118-B64">Lischer and Excoffier 2012</a>) and Plink 1.90b5.3 ( <a href="#evz118-B82">Purcell et al. 2007</a>). To minimize the effect of linkage disequilibrium (LD) for structure analysis, only one SNP with the lowest missing rate was retained for each RAD locus using in house Shell script and VCFtools 0.1.17 ( <a href="#evz118-B25">Danecek et al. 2011</a>). </p> </div> <div class="geneticdiversityanddifferentiation" title="sec"> <div class="title" tagx="title" title="title">
Genetic Diversity and Differentiation</div> <p>Genetic diversity statistics within populations, including nucleotide diversity ( <i>P</i>i), observed ( <i>H</i> <sub>O</sub>) and expected heterozygosity ( <i>H</i> <sub>E</sub>) and <i>F</i> <sub>IS</sub>, were estimated using "populations" in Stacks. Pairwise genetic differentiation ( <i>F</i> <sub>ST</sub>) between populations and their significance were evaluated using Arlequin 3.5.2.2 ( <a href="#evz118-B31">Excoffier and Lischer 2010</a>) with 10,000 permutations. To further evaluate population subdivision, we performed an analysis of molecular variance (AMOVA) as implemented in Arlequin. To estimate differentiation among groups ( <i>F</i> <sub>CT</sub>) and differentiation among populations within groups ( <i>F</i> <sub>SC</sub>), three groups were defined considering both the geographic origin of populations (the Bohai sea, the Yellow sea and the East China sea) and geographic distance among them (see <a href="#evz118-T3">table 3</a> for grouping details). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Analysis of Molecular Variance (AMOVA) Performed for Three Groups ("Dandong, Dalian, Qinhuangdao, Dongying and Weifang," "Rongcheng and Wendeng, Qingdao," "Fuyang"), and among All Populations </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Source of Variation</th> <th>Sum of Squares</th> <th>Variance Components</th> <th>Percentage Variation</th> <th>Fixation Index</th> </tr> </thead> <tbody> <tr> <td>Among three groups</td> <td>12,032.974</td> <td>38.272</td> <td>6.146</td> <td> <i>F</i> <sub>CT</sub> =  <b>0.0615</b> </td> </tr> <tr> <td>Among populations within three groups</td> <td>11,613.018</td> <td>32.707</td> <td>5.252</td> <td> <i>F</i> <sub>SC</sub> =  <b>0.0560</b> </td> </tr> <tr> <td>Among nine populations</td> <td>23,645.992</td> <td>56.976</td> <td>9.360</td> <td> <i>F</i> <sub>ST</sub> =  <b>0.0936</b> </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <div class="sc_UNKNOWN">
Note</div>.—Significant <i>F</i>-statistic values were in bold ( <i>P</i> &amp;lt; <i> </i>0.001). </p> </div> </div> </div> <p>Inference of population structure and individual assignment were carried out using three methods. Firstly, a model-based maximum likelihood method implemented in Admixture ( <a href="#evz118-B2">Alexander et al. 2009</a>) was used to estimate individual ancestries, which calculates estimates much more rapidly using a fast numerical optimization algorithm. The number of genetic clusters ( <i>K</i>) was tested for values ranging from 1 to 10 with ten replications for each tested <i>K</i> value. In order to accurately estimate the true number of subpopulations, four supervised estimators (MedMedK, MedMeaK, MaxMedK, MaxMeaK) of <a href="#evz118-B81">Puechmaille (2016)</a> were used to assess the most likely number of <i>K</i> implemented in StructureSelector ( <a href="#evz118-B61">Li and Liu 2018</a>). StructureSelector also used Clumpak ( <a href="#evz118-B54">Kopelman et al. 2015</a>) to generate plots for the selected <i>K</i>. Secondly, discriminant analysis of principal components (DAPC) method implemented in the R package Adegenet 2.1.0 ( <a href="#evz118-B49">Jombart 2008</a>) was used to visualize relationships among groups of samples and assign individuals to these groups. The alpha-score procedure was applied to select an appropriate number of principal components (PCs) as suggested by the manual. Thirdly, to visualize large and fine-scale patterns of population structure, NetView P 0.7.1 ( <a href="#evz118-B90">Steinig et al. 2016</a>) was used to visualize network relationships among all individuals with kNN step from 5 to 30. All the above structure analyses were performed on both all loci and neutral loci. </p> </div> <div class="neestimationandmafspectrum" title="sec"> <div class="title" tagx="title" title="title"> <i>N</i>e Estimation and MAF Spectrum </div> <p>Contemporary effective population size ( <i>N</i>e) for each population was calculated with NeEstimator v2 ( <a href="#evz118-B26">Do et al. 2014</a>). This program implements three single-sample methods that based on LD, heterozygote excess, and molecular coancestry. Data sets with one SNP per locus were used to estimate <i>N</i>e, and three MAF limits (0.05, 0.01, and 0) were applied. Furthermore, in order to provide an illustration of the extent to which populations exhibit an L-shaped distribution, MAF spectrum for each population were plotted in R 3.5.0 ( <a href="#evz118-B83">R Core Team 2018</a>). MAF spectrum is expected to be L-shaped distribution due to the mutation–drift equilibrium ( <a href="#evz118-B74">Nei et al. 1976</a>). Deviations from an L-shaped distribution would indicate a population bottleneck ( <a href="#evz118-B67">Luikart et al. 1998</a>). In order not to bias allele frequency spectra, we did not set a lower minor allele threshold for the SNPs used in MAF calculation, the generated results were visualized in R. </p> </div> <div class="outlierdetection,annotation,anddelimitationofcus" title="sec"> <div class="title" tagx="title" title="title">
Outlier Detection, Annotation, and Delimitation of CUs</div> <p>To identify putative loci under selection, two genome scan methods were applied. Firstly, the <i>F</i> <sub>ST</sub>-based method implemented in BayeScan 2.1 ( <a href="#evz118-B35">Foll and Gaggiotti 2008</a>) was used to identify high- <i>F</i> <sub>ST</sub> outliers. BayeScan identifies outlier loci with a Bayesian model that decomposing <i>F</i> <sub>ST</sub> coefficients into a population-specific component (beta) shared by all loci, and a locus-specific component (alpha) shared by all the populations using a logistic regression. BayeScan was performed using default parameters, and outlier loci were identified using a posterior odds of 10 with a 5% FDR. Secondly, a genotype–environment association method as implemented in Bayenv 2.0 ( <a href="#evz118-B24">Coop et al. 2010</a>) was used to identify putative loci correlated with environmental variations. Bayenv implements a Bayesian method that estimates the empirical pattern of covariance in allele frequencies between populations, and then uses this pattern as a null model for a test at individual SNPs. Four environmental variables were used for the tests of genotype–environment associations, including latitude, longitude, annual mean sea surface temperature (ASST) and mean sea surface temperature during catadromous period (CSST, from November to April, <a href="#evz118-B95">Wang et al. 2001</a>). The high-resolution SST data (long term mean SST from 1971 to 2000) provided by the NOAA/OAR/ESRL PSD, Boulder, Colorado, USA, from their web site at <a href="https://www.esrl.noaa.gov/psd/">https://www.esrl.noaa.gov/psd/</a> (last accessed June 5, 2019) was used. Because minor alleles may also be useful for identifying selection in naturally occurring organisms ( <a href="#evz118-B1">Ahrens 2018</a>), and alleles associated with environment can be significantly rarer than the common allele ( <a href="#evz118-B36">Fournier-Level et al. 2011</a>), we did not set a MAF cut-off on the data used in Bayenv tests (but a minimum of three copies of rare allele were required in at least one population in order to minimize genotyping errors). A Bayes Factor (BF) &amp;gt; 20 was required for the selection of outliers as evidence for "strong support," following <a href="#evz118-B50">Kass and Raftery (1995)</a>. Three replicates were applied for Bayenv tests, and only those SNPs consistently screened out as outliers were retained. </p> <p>Contig sequences containing outlier SNPs were then used for BLAST searches using local BLAST+ ( <a href="#evz118-B18">Camacho et al. 2009</a>) implemented in SequenceServer ( <a href="#evz118-B80">Priyam et al. 2015</a>) with an e-value of 1e–5. For the sequences with multiple matches, only the best hits were retained. Two reference sequences were used as the query database: Firstly, the complementary DNA (cDNA) sequences of <i>Gasterosteus aculeatus</i> from Ensembl ( <a href="#evz118-B103">Zerbino et al. 2018</a>), we chose it because the phylogenetic relationship between Cottales and Gasterosteales was close as described in <a href="#evz118-B15">Betancur-R et al. (2017)</a> and the genome of <i>G. aculeatus</i> was well annotated; secondly, transcriptome reads of <i>T. fasciatus</i> from <a href="#evz118-B68">Ma et al. (2018)</a> were de novo assembled into transcripts using Trinity 2.6.6 ( <a href="#evz118-B40">Grabherr et al. 2011</a>), and were then used as the database. Loci not matched to the first database were then searched against the second database (transcripts), and the aligned transcript sequences of <i>T. fasciatus</i> were then searched against the cDNA of <i>G. aculeatus</i>. The sequences with blasted hits were annotated using the UniProtKB database ( <a href="#evz118-B91">The UniProt Consortium 2018</a>), and were further categorized by a gene ontology (GO) term analysis implemented in Blast2Go ( <a href="#evz118-B23">Conesa et al. 2005</a>) with default parameters. </p> <p>Three categories of CUs were defined based on different sets of loci. Firstly, the Delta K method ( <a href="#evz118-B30">Evanno et al. 2005</a>) was used to determine the most likely uppermost level of hierarchical population structure using the results of Admixture based on all loci for defining ESUs. A principal components analysis (PCA) was also performed to confirm the high-level genetic distinctiveness using GCTA ( <a href="#evz118-B100">Yang et al. 2011</a>). Secondly, MUs were defined according to the structure analysis results based on the neutral loci. Thirdly, to identify patterns of adaptive differentiation among populations, and to further define AUs, structure analysis was performed based on "adaptive loci." We used only the high <i>F</i> <sub>ST</sub> outliers as the putative adaptive loci following the method used in <a href="#evz118-B10">Barbosa et al. (2018)</a>, in which the high <i>F</i> <sub>ST</sub> outliers were used to determine AUs. The high <i>F</i> <sub>ST</sub> outliers were identified by BayeScan as being under directional selection. Inference of genetic clusters was carried using Admixture and the best <i>K</i> was accessed and visualized in StructureSelector. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="radsequencingandgenotyping" title="sec"> <div class="title" tagx="title" title="title">
RAD Sequencing and Genotyping</div> <p>RAD sequencing of 202 <i>T. fasciatus</i> individuals resulted in 1,448,111,215 read pairs, and 812,744,207 read pairs were retained after quality filtering. A total of 635,367,008 read pairs were discarded due to low quality, PCR duplications or ambiguous RAD tags. Using RADassembler, the optimal numbers of mismatch for clustering within and across individuals were set to 6 and 6, respectively, that is, <i>M</i> = 6 and <i>n</i> = 6 in Stacks. A total of 172,441 RAD loci were assembled and retained, with a mean contig length of 529 bp and N50 of 561 bp. After mapping read pairs of each individual to the generated RAD contigs, 2,057,173 SNPs were called. Of these, 13,681 SNPs passed the quality cut-off and were retained. The SNPs were with a mean depth of 57.65 and total genotyping rate of 93.93%. After retaining one SNP from each locus with the lowest missing rate, 9,271 SNPs were kept in a total of 202 individuals across nine sampling locations. These 9,271 SNPs (total genotyping rate = 94.13%) were used for downstream structure analysis. </p> </div> <div class="populationgeneticdiversityandstructure" title="sec"> <div class="title" tagx="title" title="title">
Population Genetic Diversity and Structure</div> <p>Estimates of <i>H</i> <sub>O</sub> and <i>H</i> <sub>E</sub> average over the filtered 13,681 loci varied across the nine sampling locations ( <i>H</i> <sub>O</sub> = 0.1135–0.1543, <i>H</i> <sub>E</sub> = 0.1039–0.1382, <a href="#evz118-T1">table 1</a>). Nucleotide diversity ( <i>P</i>i = 0.1066–0.1418) were similar to <i>H</i> <sub>E</sub>, and <i>F</i> <sub>IS</sub> were not high across all the sites. Qingdao showed the lowest <i>H</i> <sub>O</sub>, <i>H</i> <sub>E</sub>, and <i>P</i>i, indicating the lowest genetic diversity among the nine populations. Rongcheng, Wendeng, Qingdao, and Fuyang displayed generally low level of percentage of polymorphic loci (46.69–58.48%), suggesting that lots of SNPs were monomorphic with one allele fixed, likely resulting from small populations after bottlenecks. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Summary of Genetic Diversity Statistics for Nine Populations of <i>Trachidermus fasciatus</i> </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Pop ID</th> <th>Variant Sites</th> <th>% Polymorphic Loci</th> <th>Num Indv</th> <th> <i>H</i> <sub>O</sub> </th> <th> <i>H</i> <sub>E</sub> </th> <th> <i>P</i>i </th> <th> <i>F</i> <sub>IS</sub> </th> <th>N <i>e</i> <sub>LD</sub> (95% CI) </th> </tr> </thead> <tbody> <tr> <td>Dandong</td> <td>13,681</td> <td>70.2361</td> <td>20.3856</td> <td>0.1231</td> <td>0.1175</td> <td>0.1205</td> <td>–0.0109</td> <td>Infinite (∞)</td> </tr> <tr> <td>Dalian</td> <td>13,681</td> <td>72.0050</td> <td>23.4955</td> <td>0.1304</td> <td>0.127</td> <td>0.1298</td> <td>–0.0019</td> <td>2591.1 (2062.7–3481.8)</td> </tr> <tr> <td>Qinhuangdao</td> <td>13,681</td> <td>67.9629</td> <td>20.1335</td> <td>0.1543</td> <td>0.1382</td> <td>0.1418</td> <td>–0.0439</td> <td>32.0 (31.8–32.2)</td> </tr> <tr> <td>Dongying</td> <td>13,681</td> <td>74.4609</td> <td>21.5243</td> <td>0.1350</td> <td>0.1298</td> <td>0.1329</td> <td>–0.0084</td> <td>512.8 (485.1–543.8)</td> </tr> <tr> <td>Weifang</td> <td>13,681</td> <td>72.2973</td> <td>20.8344</td> <td>0.1374</td> <td>0.1292</td> <td>0.1324</td> <td>–0.0186</td> <td>313.9 (301.7–327.1)</td> </tr> <tr> <td>Rongcheng</td> <td>13,681</td> <td>56.7356</td> <td>20.8467</td> <td>0.1192</td> <td>0.1120</td> <td>0.1148</td> <td>–0.0152</td> <td>23.8 (23.7–23.9)</td> </tr> <tr> <td>Wendeng</td> <td>13,681</td> <td>58.4753</td> <td>21.4057</td> <td>0.1310</td> <td>0.1167</td> <td>0.1196</td> <td>–0.0414</td> <td>6.8 (6.8–6.9)</td> </tr> <tr> <td>Qingdao</td> <td>13,681</td> <td>46.6852</td> <td>20.5603</td> <td>0.1135</td> <td>0.1039</td> <td>0.1066</td> <td>–0.0236</td> <td>100.3 (98.3–102.4)</td> </tr> <tr> <td>Fuyang</td> <td>13,681</td> <td>57.0865</td> <td>20.5565</td> <td>0.1364</td> <td>0.1313</td> <td>0.1346</td> <td>–0.0091</td> <td>1766.0 (1391.6–2414.6)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <div class="sc_UNKNOWN">
Note</div>.—Variant Sites, number of total SNPs; % Polymorphic Loci, proportion of SNPs in this populations; Num Indv, number of genotyped individuals averaged over all SNPs; <i>H</i> <sub>O</sub>, observed heterozygosity; <i>H</i> <sub>E</sub>, expected heterozygosity; <i>P</i>i, nucleotide diversity; <i>F</i> <sub>IS</sub>, inbreeding coefficient of individuals relative to the subpopulation; N <i>e</i> <sub>LD</sub>, effective populations size estimated based on LD method. </p> </div> </div> </div> <p>Most of the pairwise <i>F</i> <sub>ST</sub> values over 9,271 SNPs between populations were significant ( <i>P </i>&amp;lt; <i> </i>0.0014 after B-Y FDR correction), except that between Dongying and Weifang. <i>F</i> <sub>ST</sub> ranged from –0.0038 between the two closest populations (Dongying and Weifang) to 0.1916 between Qingdao and Fuyang ( <a href="#evz118-T2">table 2</a>). The <i>F</i> <sub>ST</sub> across all populations was 0.0936 and was statistically significant ( <i>P </i>&amp;lt; <i> </i>0.001). The hierarchical AMOVA ( <a href="#evz118-T3">table 3</a>) showed strong and significant differentiation among the three groups of populations ( <i>F</i> <sub>CT</sub> = 0.0615, <i>P </i>&amp;lt; <i> </i>0.001). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Pairwise <i>F</i> <sub>ST</sub> Values Over 9,271 SNPs across Nine Populations of <i>Trachidermus fasciatus</i> </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th /> <th>Dandong</th> <th>Dalian</th> <th>Qinhuangdao</th> <th>Dongying</th> <th>Weifang</th> <th>Rongcheng</th> <th>Wendeng</th> <th>Qingdao</th> <th>Fuyang</th> </tr> </thead> <tbody> <tr> <td>Dandong</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>Dalian</td> <td> <b>0.0405</b> </td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>Qinhuangdao</td> <td> <b>0.0592</b> </td> <td> <b>0.0401</b> </td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>Dongying</td> <td> <b>0.0393</b> </td> <td> <b>0.0335</b> </td> <td> <b>0.0502</b> </td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>Weifang</td> <td> <b>0.0437</b> </td> <td> <b>0.0259</b> </td> <td> <b>0.0401</b> </td> <td>–0.0038</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>Rongcheng</td> <td> <b>0.1130</b> </td> <td> <b>0.1108</b> </td> <td> <b>0.1270</b> </td> <td> <b>0.0980</b> </td> <td> <b>0.1056</b> </td> <td>—</td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>Wendeng</td> <td> <b>0.0988</b> </td> <td> <b>0.1033</b> </td> <td> <b>0.1199</b> </td> <td> <b>0.0832</b> </td> <td> <b>0.0895</b> </td> <td> <b>0.0702</b> </td> <td>—</td> <td>—</td> <td>—</td> </tr> <tr> <td>Qingdao</td> <td> <b>0.1358</b> </td> <td> <b>0.1396</b> </td> <td> <b>0.1574</b> </td> <td> <b>0.1336</b> </td> <td> <b>0.1424</b> </td> <td> <b>0.1429</b> </td> <td> <b>0.1077</b> </td> <td>—</td> <td>—</td> </tr> <tr> <td>Fuyang</td> <td> <b>0.1045</b> </td> <td> <b>0.1283</b> </td> <td> <b>0.1404</b> </td> <td> <b>0.1213</b> </td> <td> <b>0.1189</b> </td> <td> <b>0.1764</b> </td> <td> <b>0.1676</b> </td> <td> <b>0.1916</b> </td> <td>—</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <div class="sc_UNKNOWN">
Note</div>.—Values in bold were significant after FDR correction ( <i>P </i>&amp;lt; <i> </i>0.0014). </p> </div> </div> </div> <p>The analysis using Admixture and StructureSelector indicated that there were highest supports for eight genetic groups (MedMedK, MedMeaK, MaxMedK, MaxMeaK = 8, <a href="#sup1">supplementary fig. S1</a>, <a href="#sup1">Supplementary Material</a> online) among individuals. The major mode for <i>K </i>= <i> </i>8 generated by Clumpak followed the patterns as indicated by the pairwise <i>F</i> <sub>ST</sub> values: Dongying and Weifang formed into one group, and the other seven populations came out as distinct groups respectively ( <a href="#evz118-F2">fig. 2 <i>b</i> </a>). DAPC analysis retained 13 PCs based on the alpha-score, which explained 25.45% of the total variance. The patterns of DAPC scatter plots were similar to the results revealed by Admixture, individuals of each population formed into distinct clusters respectively, except those from Dongying and Weifang ( <a href="#evz118-F3">fig. 3</a>), and several individuals of Rongcheng and Wendeng were overlapped. NetView P was able to reveal fine-scale clustering with kNN = 20 ( <a href="#sup1">supplementary fig. S2</a>, <a href="#sup1">Supplementary Material</a> online), individuals were grouped into eight different clusters, with most individuals from Dongying and Weifang clustered together. Both Fuyang and Qingdao displayed little connections with other populations, suggesting that the two populations might be highly isolated, which also showed high <i>F</i> <sub>ST</sub> values (&amp;gt; 0.1) in comparison with other populations. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 2. </span> </a> </div>   <p>—Plots of the individual ancestry inference for ( <i>a</i>) <i>K</i> = 3 based on all loci; ( <i>b</i>) <i>K</i> = 8 based on all loci; and ( <i>c</i>) <i>K</i> = 6 based on high- <i>F</i> <sub>ST</sub> adaptive loci in nine populations of <i>Trachidermus fasciatus</i>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 3. </span> </a> </div>   <p>—DAPC scatter plots with prior population information. The plots were generated using 13 principle components with the first three coordinate axes based on all loci (explained 25.45% of the total variance). </p>   </div> </div> <div class="neestimationandmafspectrum" title="sec"> <div class="title" tagx="title" title="title"> <i>N</i>e Estimation and MAF Spectrum </div> <p>Estimations of <i>N</i>e were varying across the sampling sites, and the MAF constraint had little effect on the values of <i>N</i>e, thus the results with no MAF cut-off were used. The <i>N</i>e ranged from 6.8 (Wendeng, 95% CI 6.8–6.9) to infinite (Dandong, 95% CI infinite; <a href="#evz118-T1">table 1</a>). Rongcheng, Wendeng, and Qingdao displayed generally small <i>N</i>e, which might result from small population size and subsample effect ( <a href="#evz118-B53">Kjeldsen et al. 2016</a>). The estimations of <i>N</i>e were consistent with the pattern of genetic diversity, with Rongcheng, Wendeng, and Qingdao all showed low levels of <i>H</i>o, <i>H</i> <sub>E</sub>, and <i>P</i>i. MAF spectra generally exhibited L-shaped distributions, but varied markedly among populations ( <a href="#evz118-F4">fig. 4</a>). A certain number of rare alleles (&amp;gt; 15%) were observed in Qinhuangdao, Rongcheng, Wendeng, Qingdao, and Fuyang, suggesting small effective population size or newly founded populations. Specially, distributions of MAF spectra in Wendeng and Qingdao were much more distorted than the other populations, indicating potential bottlenecks with small <i>N</i>e. These results were consistent with the estimates of genome-wide <i>H</i> <sub>E</sub>, <i>P</i>i, and percentage of polymorphic loci within populations ( <a href="#evz118-T1">table 1</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 4. </span> </a> </div>   <p>—SNP minor allele frequency (MAF) spectrum for nine populations of <i>Trachidermus fasciatus.</i> The <i>x</i>-axis represents categories of MAF with the corresponding allele counts on the <i>y</i>-axis. Color in black represents MAF &amp;gt; 15%. </p>   </div> </div> <div class="candidatelociunderselectionanddelimitationofcus" title="sec"> <div class="title" tagx="title" title="title">
Candidate Loci under Selection and Delimitation of CUs</div> <p>BayeScan detected a total 56 outlier SNPs, of these 9 SNPs were potentially under balancing selection and 47 SNPs were potentially under divergent selection with posterior odds &amp;gt; 10 and FDR ≤ 0.05 ( <a href="#sup1">supplementary fig. S3</a>, <a href="#sup1">Supplementary Material</a> online). To minimize false positives, only loci under divergent selection were used. Bayenv consistently identified 282 SNPs correlated with environmental variables after three independent replicates, of these 56 SNPs were associated with latitude, 18 SNPs were longitude related, 164 SNPs were ASST related, and 236 SNPs were CSST related. There were little overlaps between SNPs related with longitude and with other environmental variables ( <a href="#sup1">supplementary fig. S4</a>, <a href="#sup1">Supplementary Material</a> online), whereas most of the ASST and CSST related SNPs were shared with each other (149 SNPs). Thereafter we used the combined results from BayeScan and Bayenv as the candidate adaptive loci, which corresponded to 313 SNPs from 300 RAD loci. Contig sequences of these loci were then used for further annotations. </p> <p>Of the 300 candidate RAD loci, 69 were directly mapped to 59 cDNA of stickleback. A total of 136 of the remaining 231 loci were aligned to 135 transcripts of <i>T. fasciatus</i>, and 48 of these loci were further aligned to 48 cDNA of stickleback. In total, 117 of the 300 candidate RAD loci were aligned to 105 cDNAs (genes) of stickleback genome. These genes were found to be involved in diverse functions, and were putatively involved in local adaptive processes of <i>T. fasciatus</i>. Using Blast2Go, 74 loci were annotated and assigned to 138 GO terms. Most of the candidate loci containing SNPs putatively under selection could be categorized as having functions associated with catalytic activity, binding and hydrolase activity functions; cellular metabolic processes, biological regulation, development and response to simulation; or transport, signal transduction, cell communication and biosynthetic process ( <a href="#sup1">supplementary table S2</a>, <a href="#sup1">Supplementary Material</a> online). These biological processes are potentially influenced by environmental variations like temperature and osmotic pressure. </p> <p>Structure results based on neutral loci were similar to that on all loci (data not shown). Similarly, there was also high support for eight genetic clusters in nine populations of <i>T. fasciatus</i>, which displayed significant genetic divergence. The eight genetic clusters could be defined as eight MUs, including MU1 (Dandong), MU2 (Dalian), MU3 (Qinhuangdao), MU4 (Dongying &amp;amp; Weifang), MU5 (Rongcheng), MU6 (Wendeng), MU7 (Qingdao), and MU8 (Fuyang). The Delta K method could identify high-level genetic structure, which indicated the highest support for three major groups ( <i>K</i> = 3, <a href="#evz118-F2">fig. 2 <i>a</i> </a>), the similar patterns were also indicated by PCA ( <a href="#evz118-F5">fig. 5</a>). Thereby, the three distinct genetic clusters could be defined as three ESUs: ESU1 (Dandong, Dalian, Qinhuangdao, Dongying, Weifang), ESU2 (Rongcheng, Wendeng, Qingdao), ESU3 (Fuyang), which should reflect the major divergence among populations of <i>T. fasciatus</i>. Using 47 high <i>F</i> <sub>ST</sub> outlier SNPs, six genetic clusters were identified among individuals ( <i>K</i> = 6, <a href="#evz118-F2">fig. 2 <i>c</i> </a>), corresponding to six AUs. Including AU1: Dandong, AU2: Dalian &amp;amp; Qinhuangdao, AU3: Dongying &amp;amp; Weifang, AU4: Rongcheng &amp;amp; Wendeng, AU5: Qingdao, AU6: Fuyang. The pattern of adaptive variation was similar to that using all loci (9,271 SNPs), except that Dalian &amp;amp; Qinhuangdao formed into one group and Rongcheng &amp;amp; Wendeng formed into another group. Levels of genetic divergence ( <i>F</i> <sub>ST</sub>) between six clusters using outlier loci were listed in the <a href="#sup1">supplementary table S3</a>, <a href="#sup1">Supplementary Material</a> online. Among the six clusters, AU6 was the most distinctive one in terms of adaptive variation. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label"> <div class="sc_UNKNOWN">
Fig</div>. 5. </span> </a> </div>   <p>—PCA scatter plot using the first two principle components based on all loci.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The present population genomic study delineated fine population genetic characteristics of the nine <i>T. fasciatus</i> populations by integrating data of putatively neutral and adaptive genetic variation at the genomic level. This is the first use of genome-wide SNP markers to assess a broad-scale population differentiation and signatures of selection in the <i>T. fasciatus</i>. Our study contributes to the growing body of literature identifying significant population structure and local adaptation in marine systems and has important implications for the conservation management of <i>T. fasciatus</i> and other endangered marine species. </p> <div class="causesofpopulationstructureandlowneint.fasciatus" title="sec"> <div class="title" tagx="title" title="title">
Causes of Population Structure and Low <i>N</i>e in <i>T. fasciatu</i> <i>s</i> </div> <p>The results provided a higher resolution of population structure compared that identified in our previous study based on microsatellite data ( <a href="#evz118-B62">Li et al. 2016</a>). In particular, in contrast to the microsatellite results, Dandong, Dalian, and Qinhuangdao populations were further genetically distinguished by RAD data sets. In addition, pairwise <i>F</i> <sub>ST</sub> values were higher than those based on microsatellites. A good understanding of how ecological variants of fishes can impact their population structure will provide more comprehensive implications for conservation management and decision-making. </p> <p>Habitat fragmentation is detrimental to species by reducing population size and genetic diversity and by restraining population connectivity ( <a href="#evz118-B57">Lemer and Planes 2014</a>; <a href="#evz118-B101">Young et al. 1996</a>). Therefore, habitat fragmentation may have significant effects on population genetic structure because fragmentation usually leads to limited dispersal and thus lower connectivity levels among populations ( <a href="#evz118-B73">Mossman and Waser 2001</a>; <a href="#evz118-B52">Keyghobadi 2007</a>; <a href="#evz118-B57">Lemer and Planes 2014</a>). Because of its complex life history, the breeding and growth of <i>T. fasciatus</i> depends on different habitats including oceans, estuaries and rivers ( <a href="#evz118-B47">Islam et al. 2007</a>; <a href="#evz118-B19">Cao et al. 2009</a>; <a href="#evz118-B94">Wang and Cheng 2010</a>). In the past decades, the habitats of <i>T. fasciatus</i> has been markedly degraded and fragmented due to excessive anthropogenic activities (e.g., large-scale reclamation, pollution, dam construction, river flow modification), and the census populations size were sharply reduced ( <a href="#evz118-B94">Wang and Cheng 2010</a>). On the basis of our results, most populations were with low <i>N</i>e, thus are potentially faced with strong effect of genetic drift. Therefore, the divergence among the isolated populations may be caused by low gene flow and the effect of genetic drift among these populations due to habitat fragmentation. However, Dongying and Weifang populations are in close geographical proximity and may share the same spawning ground, which could result in high gene flow between the two populations. </p> <p>In the presence of limited gene flow between fragmented populations, random genetic drift is expected to increase genetic differentiation among populations ( <a href="#evz118-B41">Harrison and Hastings 1996</a>; <a href="#evz118-B57">Lemer and Planes 2014</a>). Currently, the populations of <i>T. fasciatus</i> have been severely declining due to the combination of overfishing and other anthropogenic activities. The strength of genetic drift is highly related with <i>N</i>e, which could be much lower than census abundance in marine species ( <a href="#evz118-B43">Hauser et al. 2002</a>; <a href="#evz118-B78">Pinsky and Palumbi 2014</a>). Theoretical simulations suggest that short-term conservation of genetic diversity and inbreeding avoidance requires a <i>N</i>e &amp;gt; 500, and a safety balloon to maintain evolutionary flexibility in natural populations requires a <i>N</i>e &amp;gt; 1000 ( <a href="#evz118-B79">Pita et al. 2017</a>). The smaller <i>N</i>e estimates in Qinhuangdao, Rongcheng, Wendeng, and Qingdao indicated the exceptional status of these small populations. According to the MAF spectrum, the high proportion of rare alleles in Qinhuangdao, Rongcheng, Wendeng, and Qingdao also suggested of bottlenecks and strong genetic drift in these populations. Under the background of global climate change and human activities, runoff volume of some rivers as the migration channel of <i>T. fasciatus</i> (e.g., Dagu River for Qingdao population, Luan River for Qinhuangdao population, Wei River for Weifang population) has been sharply reduced since recent decades, and the drying-up of the river course has occurred frequently. Specially, the Rongcheng population is mainly distributed in Swan Lake. Swan Lake is a semienclosed coastal lagoon, the major environment perturbations might result in bottlenecks and small <i>N</i>e values, which will further strengthen genetic drift effects. </p> </div> <div class="genomicsignaturesofenvironmentalselectionint.fasciatus" title="sec"> <div class="title" tagx="title" title="title">
Genomic Signatures of Environmental Selection in <i>T. fasciatus</i> </div> <p>With the advantages of identification of genome-wide polymorphic markers, RAD sequencing can facilitate insight into studies of the ecological, evolutionary and conservation genomics ( <a href="#evz118-B4">Andrews et al. 2016</a>), especially in nonmodel organisms. However, RAD sequencing only covers a fraction of the total genome, and may have missed many loci under selection in local adaptation ( <a href="#evz118-B66">Lowry et al. 2017</a>). Nonetheless, despite the limited ability to detect adaptive loci, RAD sequencing remains a powerful tool for understanding the genetics of local adaptation in natural populations ( <a href="#evz118-B22">Catchen et al. 2017</a>). Genome scan methods, especially genotype–environment associations are still useful for uncovering candidate loci of relatively important ecological effect. In the present study, we did detect a number of important candidate loci that appear to be affected by spatially divergent selection, although some crucial adaptive loci might have been lost. </p> <p>Environmental conditions can be important selective forces that shape the genotypic and phenotypic composition of local populations ( <a href="#evz118-B85">Sanford and Kelly 2011</a>). Environmental heterogeneity across different geographical locations may lead to phenotypic plasticity or genotypic responses ( <a href="#evz118-B44">Hoffmann and Willi 2008</a>). Aquatic organisms mainly exchange heat with their environment via conduction and convection, thus water temperature is one of the most important abiotic factor that influencing the phenotype and habitat of aquatic organisms ( <a href="#evz118-B6">Angilletta 2009</a>). The temperature of organism’s surrounding environment controls a variety of biological and physiological processes, including heat exchange ( <a href="#evz118-B51">Kearney and Porter 2009</a>), development ( <a href="#evz118-B55">Kroeker et al. 2013</a>), metabolic rates ( <a href="#evz118-B28">Elliott 1976</a>), responses ( <a href="#evz118-B12">Batty et al. 1993</a>), and aerobic scope ( <a href="#evz118-B77">Pörtner and Knust 2007</a>), which will further affect the higher functions (such as muscular activity, behavior, growth, and reproduction) and might thereby shape the long-term fate of species. We found evidence that a number of metabolic or developmental processes related genes (34 annotated candidate loci) were involved in the genetic adaptation of <i>T. fasciatus</i> to the local environment. One locus of interest is located in the <i>Glycerol-3-phosphate dehydrogenase (GPDH)</i> gene, which encodes enzyme that catalyzes the reversible redox conversion of dihydroxyacetone phosphate to sn-glycerol 3-phosphate. <i>GPDH</i> is a major link between carbohydrate metabolism and lipid metabolism, and is crucial in electron transport chain of the mitochondria. Indeed, <i>T. fasciatus</i> expends a vast amount of energy during the catadromous and reproduction processes. Previous physiological studies indicated that the digestive system of <i>T. fasciatus</i> was degenerative since its migration from freshwater to saltwater ( <a href="#evz118-B60">Li et al. 1984</a>), and the main energy sources were from intracorporal glycogen and lipid storage ( <a href="#evz118-B93">Wang 1999</a>). Thus, lipid and carbohydrate metabolism processes might play a crucial role in adaptation to environmental heterogeneity across different geographical populations, especially for the water temperature. Another candidate locus of interest was found in <i>the GLI family zinc finger 3 gene</i>, which could be related to embryonic organ development and dorsal/ventral pattern formation. This gene is also critical for specifying the fate of cortical neurons, and acts as the major negative transducer of the Sonic hedgehog (Shh) signaling pathway ( <a href="#evz118-B92">Wang et al. 2011</a>). The Shh pathway is one of the key regulators for animal development, suggesting that changes in body development of <i>T. fasciatus</i> could be implicated in the environmental temperature. This result is consistent with previous studies of <i>T. fasciatus</i>, for example studies on <i>T. fasciatus</i> suggested that high temperatures would restrict the body development ( <a href="#evz118-B97">Wei et al. 1997</a>), affect the growth rate of larvae ( <a href="#evz118-B89">Shao et al. 1980</a>). Therefore, genes related to metabolism and development were likely crucial candidates for the adaptation of <i>T. fasciatus</i> to water temperature. Studies in other marine organisms have also identified adaptive loci associated with temperature that were involved in similar diverse functions, examples including the greenlip abalone ( <a href="#evz118-B84">Sandoval-Castillo et al. 2018</a>), the Asian seabass ( <a href="#evz118-B96">Wang et al. 2016</a>), and the lake trout ( <a href="#evz118-B14">Bernatchez et al. 2016</a>). </p> <p>Evidences of genetic adaptation in osmoregulatory related genes were also found. Among the annotated candidate loci, 19 loci were categorized as having functions involved in iron binding, including the metal iron binding, cation binding, etc., which might be involved in osmoregulation. As a catadromous fish, <i>T. fasciatus</i> have to uptake ions from the environment to compensate for salts lost in fresh water, whereas in saltwater they have to diffuse ions to compensate for the passive diffusion of salts into their bodies. Importantly, during spawning migration, <i>T. fasciatus</i> actively migrate from freshwater to saltwater to uptake sufficient iodine for the development of thyroid and sexual maturity ( <a href="#evz118-B88">Shao 1978</a>; <a href="#evz118-B89">Shao et al. 1980</a>). The osmoregulatory and ion-exchange related genes may play a key role in adaptation to freshwater and saltwater systems. Indeed, some populations of <i>T. fasciatus</i> might not be able to complete the migration between seawater and freshwater due the lack of connections with rivers or drying-up of the rivers. Thus these populations (e.g., Weifang, Rongcheng, and Qingdao) might have been lived completely in saltwater, which has been confirmed by otolith microchemistry analysis ( <a href="#evz118-B16">Bi 2013</a>). It is possible that there may be a difference of iron regulation between these populations and other populations with complete catadromous life history. Interestingly, we did find the candidate adaptive loci in the solute carrier family 39 (SLC39) like and Cation/H+ exchanger like gene, which were involved in osmoregulation. SLC family genes are multifunctional and are often involved in acid–base balancing via movement of monovalent and divalent anions ( <a href="#evz118-B98">Willoughby et al. 2018</a>). Previous studies in stickleback and trout also identified similar gene (SLC26) being involved in adaptation to the freshwater–saltwater systems ( <a href="#evz118-B45">Hohenlohe et al. 2010</a>; <a href="#evz118-B98">Willoughby et al. 2018</a>). The documentation of genes involved in both metabolic and osmoregulatory pathways provides the first glimpse of how <i>T. fasciatus</i> have been adapted to the spatially varying environment of local populations. </p> <p>Several candidate loci were annotated as having functions involved in biological regulation, response to stimulus, immune and signaling processes. For instance, one locus was found in the <i>Transporter 1, ATP-binding cassette, subfamily B gene (TAP1)</i>, which was involved in MHC protein binding, antigen processing and presentation of exogenous peptide antigen. TAP1 functions as the membrane transporter associated with antigen processing that are involved in transport of peptides that bind to MHC molecules ( <a href="#evz118-B33">Flajnik and Kasahara 2001</a>), and plays a major role in transportation of foreign peptides through the endoplasmic reticulum for processing before loading onto MHC molecules ( <a href="#evz118-B27">Elgert 2009</a>). Actually, immune response was also shown to be related to the local adaptation to different water temperature in pipefish ( <a href="#evz118-B34">Flanagan et al. 2016</a>) and abalones ( <a href="#evz118-B84">Sandoval-Castillo et al. 2018</a>). Overall, these findings suggested that the observed adaptive divergence of <i>T. fasciatus</i> might involve diverse functional changes related to spatially varying environments. This information about the functions of annotated candidate loci could help us to understand the genetic basis of adaptation to important environmental variables like temperature, and provided useful leads for future research in marine species. </p> </div> <div class="conservationandmanagementimplicationsfort.fasciatus" title="sec"> <div class="title" tagx="title" title="title">
Conservation and Management Implications for <i>T. fasciatus</i> </div> <p>Our study used population genomic techniques to offer high resolution genetic information to improve conservation and management policies and decisions, which previously were not clear with respect to patterns of adaptive genetic variation. The results presented here would provide complementary information for management, especially in defining CUs and setting conservation priorities and management plans for <i>T. fasciatus</i>. </p> <p>The results demonstrated that the nine populations of <i>T. fasciatus</i> should be defined as eight MUs (based on neutral loci), six AUs (based on adaptive loci), and three ESUs (based on all loci). CUs with lower genetic diversity and <i>N</i>e require high conservation priority, such as MU7 (Qingdao) &amp;amp; MU8 (Fuyang) or AU5 (Qingdao) &amp;amp; AU6 (Fuyang), which show high level of isolation with other populations. In particular, MU6 (Wendeng) deserves special attention given the lowest <i>N</i>e, which is related to low fitness risk. However, the census population size of MU6 might be relative large, which could be much larger than that of MU8 ( <a href="#evz118-B94">Wang and Cheng 2010</a>), but the <i>N</i>e of MU6 were much smaller than that of MU8. A large population census size with an extremely low genetic variation and <i>N</i>e could be attributed to overharvest or natural bottlenecks ( <a href="#evz118-B79">Pita et al. 2017</a>). Overharvest and habitat fragmentation due to anthropogenic activities can lead to the loss of genetic variation and a consequent reduction in evolutionary potential and adaptive ability ( <a href="#evz118-B43">Hauser et al. 2002</a>; <a href="#evz118-B78">Pinsky and Palumbi 2014</a>). Although there is no known targeted fishery for this species, <i>T. fasciatus</i> is often caught as bycatch from bottom-trawl fishery and trap net fishery in China ( <a href="#evz118-B94">Wang and Cheng 2010</a>). As a ferocious carnivorous fish with large food consumption, <i>T. fasciatus</i> needs to live in area with plenty of food resources. The various type of habitats (inshore sea area, estuaries, and rivers) for the breeding and growth of <i>T. fasciatus</i> also suggested its sensitive to environmental changes. Currently it is difficult for <i>T. fasciatus</i> to survive, migrate, breed, and completed the entire life cycle ( <a href="#evz118-B94">Wang and Cheng 2010</a>; <a href="#evz118-B62">Li et al. 2016</a>). Thus, carrying out prohibited fishing areas and prohibited fishing periods as well as habitat restoration and protection are of great concerns and are probably the most immediate action to take at this time to protect <i>T. fasciatus</i>. </p> <p>Another government-promoted conservation action on <i>T. fasciatus</i> was stock enhancement ( <a href="#evz118-B13">Bell et al. 2006</a>). A large number of evidences indicated that stock enhancement could improve the abundance of target species, but it might also impose some risks on the local populations (e.g., mixing genetic lineages, spreading pathogens). For the management and conservation of this species, the eight distinct clusters detected in the present study should be considered as different MUs. Furthermore, <i>T. fasciatus</i> that were recruited within their local adaptive cluster (AUs) might show higher fitness (with regard to variation in the thermal environment and migration routes) than recruited in clusters found elsewhere. </p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>By analysis of genome-wide data of nine <i>T. fasciatus</i> populations, fine population genetic structure and genomic signatures of adaptations across the species’ range were detected. Genetic clustering analysis revealed significant population structure with high support for eight distinct clusters among nine populations. The strong genetic structure could be attributed to habitat fragmentation and genetic drift due to anthropogenic activities. A substantial number of genetic variants appeared to be under differential selective pressure across the range of <i>T. fasciatus</i>. Annotations of candidate adaptive loci suggested that the genes involved in diverse functions like metabolism, development and osmoregulation are critical for the adaptation to environmental heterogeneity across the species’ range. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p> <a href="#sup1">Supplementary data</a> are available at <i>Genome Biology and Evolution</i> online. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary_Material_evz118</span> <div class="media" title="media"> <a href="supplementary_material_evz118.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We thank Zhenzhen Wan, Sudong Xia, Liang Cao, Honglei Wang, Jiqing Li, Huankun Gou, and Zhicai Lin for their help in collecting samples. The study was supported by National Natural Science Foundation of China (Nos. 31502169 and 41676137), NSFC-Shandong Joint Fund for Marine Ecology and Environmental Sciences (No. U1606404), and AoShan Talents Program supported by Qingdao National Laboratory of Marine Science and Technology (No. 2015ASTP-ES05). </p> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>The study was conceived by L.J.X. and X.D.X. The experiment was performed by L.Y.L. and X.D.X. The analysis was performed by L.Y.L. and Z.B.D. The manuscript was written by L.Y.L., X.D.X., and L.J.X. with contributions from Z.B.D. </p> </div> </div> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <p> <b>Data deposition:</b> This project has been deposited at the NCBI Nucleotide Short Read Archive (SRA) under the accession No. PRJNA360511. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
Literature Cited</div> <li tag="ref"> <a name="evz118-B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahrens</span> <span class="given-names" tagx="given-names" title="given-names">CW.</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">The search for loci under selection: trends, biases and progress</span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">27</span>( <span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">1342</span>– <span class="lpage" tagx="lpage" title="lpage">1356</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29524276">29524276</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alexander</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novembre</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lange</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Fast model-based estimation of ancestry in unrelated individuals</span>. <span class="source" tagx="source" title="source">Genome Res</span>. <span class="volume" tagx="volume" title="volume">19</span>( <span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">1655</span>– <span class="lpage" tagx="lpage" title="lpage">1664</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19648217">19648217</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allendorf</span> <span class="given-names" tagx="given-names" title="given-names">FW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luikart</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="source" tagx="source" title="source">Conservation and the genetics of populations</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Malden:Blackwell</span>. </span> </li> <li tag="ref"> <a name="evz118-B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andrews</span> <span class="given-names" tagx="given-names" title="given-names">KR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Good</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luikart</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohenlohe</span> <span class="given-names" tagx="given-names" title="given-names">PA.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Harnessing the power of RADseq for ecological and evolutionary genomics</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">81</span>– <span class="lpage" tagx="lpage" title="lpage">92</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26729255">26729255</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Angeloni</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagemaker</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vergeer</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ouborg</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Genomic toolboxes for conservation biologists</span>. <span class="source" tagx="source" title="source">Evol Appl</span>. <span class="volume" tagx="volume" title="volume">5</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">130</span>– <span class="lpage" tagx="lpage" title="lpage">143</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25568036">25568036</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Angilletta</span> <span class="given-names" tagx="given-names" title="given-names">MJ.</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="source" tagx="source" title="source">Thermal adaptation: a theoretical and empirical synthesis</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford:Oxford University Press</span>. </span> </li> <li tag="ref"> <a name="evz118-B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Attard</span> <span class="given-names" tagx="given-names" title="given-names">CRM</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Ecological disturbance influences adaptive divergence despite high gene flow in golden perch ( <i>Macquaria ambigua</i>): implications for management and resilience to climate change </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">27</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">196</span>– <span class="lpage" tagx="lpage" title="lpage">215</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29165848">29165848</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baalsrud</span> <span class="given-names" tagx="given-names" title="given-names">HT</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Effects of population characteristics and structure on estimates of effective population size in a house sparrow metapopulation </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">23</span>( <span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">2653</span>– <span class="lpage" tagx="lpage" title="lpage">2668</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24766660">24766660</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wan</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">The characteristics of the seasonal variability of the seasurface temperature field in the Bohai Sea, the Huanghai Sea and the East China Sea from AVHRR data </span>. <span class="source" tagx="source" title="source">Acta Oceanol Sin</span>. <span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">125</span>– <span class="lpage" tagx="lpage" title="lpage">133</span>. </span> </li> <li tag="ref"> <a name="evz118-B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbosa</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Integrative approaches to guide conservation decisions: using genomics to define conservation units and functional corridors </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">27</span>( <span class="issue" tagx="issue" title="issue">17</span>): <span class="fpage" tagx="fpage" title="fpage">3452</span>– <span class="lpage" tagx="lpage" title="lpage">3465</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30030869">30030869</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barrett</span> <span class="given-names" tagx="given-names" title="given-names">RDH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schluter</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Adaptation from standing genetic variation</span>. <span class="source" tagx="source" title="source">Trends Ecol Evol (Amst)</span>. <span class="volume" tagx="volume" title="volume">23</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">38</span>– <span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18006185">18006185</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Batty</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blaxter</span> <span class="given-names" tagx="given-names" title="given-names">JHS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fretwell</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> <span class="year" tagx="year" title="year">1993</span> <span class="mixed-article-title" title="mixed-article-title">Effect of temperature on the escape responses of larval herring, <i>Clupea harengus</i> </span>. <span class="source" tagx="source" title="source">Mar Biol</span>. <span class="volume" tagx="volume" title="volume">115</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">523</span>– <span class="lpage" tagx="lpage" title="lpage">528</span>. </span> </li> <li tag="ref"> <a name="evz118-B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartley</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lorenzen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loneragan</span> <span class="given-names" tagx="given-names" title="given-names">NR.</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Restocking and stock enhancement of coastal fisheries: potential, problems and progress</span>. <span class="source" tagx="source" title="source">Fish Res</span>. <span class="volume" tagx="volume" title="volume">80</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. </span> </li> <li tag="ref"> <a name="evz118-B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernatchez</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laporte</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perrier</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sirois</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernatchez</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Investigating genomic and phenotypic parallelism between piscivorous and planktivorous lake trout ( <i>Salvelinus namaycush</i>) ecotypes by means of RADseq and morphometrics analyses </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">25</span>( <span class="issue" tagx="issue" title="issue">19</span>): <span class="fpage" tagx="fpage" title="fpage">4773</span>– <span class="lpage" tagx="lpage" title="lpage">4792</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27497114">27497114</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Betancur-R</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic classification of bony fishes</span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">162</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28683774">28683774</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bi</span> <span class="given-names" tagx="given-names" title="given-names">XX.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span>. Studies on the morphology and genetics of Trachidermus fasciatus populations [doctoral thesis]. Qingdao:Ocean University of China. </span> </li> <li tag="ref"> <a name="evz118-B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brauer</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hammer</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beheregaray</span> <span class="given-names" tagx="given-names" title="given-names">LB.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Riverscape genomics of a threatened fish across a hydroclimatically heterogeneous river basin </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">25</span>( <span class="issue" tagx="issue" title="issue">20</span>): <span class="fpage" tagx="fpage" title="fpage">5093</span>– <span class="lpage" tagx="lpage" title="lpage">5113</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27575282">27575282</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Camacho</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">BLAST+: architecture and applications</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">421.</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20003500">20003500</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Threatened fishes of the world: <i>Trachidermus fasciatus</i> Heckel, 1837 (Cottidae) </span>. <span class="source" tagx="source" title="source">Environ Biol Fish</span>. <span class="volume" tagx="volume" title="volume">86</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">63.</span> </span> </li> <li tag="ref"> <a name="evz118-B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Catchen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">The population structure and recent colonization history of Oregon threespine stickleback determined using restriction-site associated DNA-sequencing </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">22</span>( <span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">2864</span>– <span class="lpage" tagx="lpage" title="lpage">2883</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23718143">23718143</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Catchen</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amores</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohenlohe</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cresko</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Postlethwait</span> <span class="given-names" tagx="given-names" title="given-names">JH.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Stacks: building and genotyping loci de novo from short-read sequences</span>. <span class="source" tagx="source" title="source">G3: Genes|Genomes|Genetics</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">171</span>– <span class="lpage" tagx="lpage" title="lpage">182</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22384329">22384329</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Catchen</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Unbroken: RADseq remains a powerful tool for understanding the genetics of adaptation in natural populations </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">362</span>– <span class="lpage" tagx="lpage" title="lpage">365</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28319339">28319339</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conesa</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">21</span>( <span class="issue" tagx="issue" title="issue">18</span>): <span class="fpage" tagx="fpage" title="fpage">3674</span>– <span class="lpage" tagx="lpage" title="lpage">3676</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474">16081474</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coop</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Witonsky</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Di Rienzo</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pritchard</span> <span class="given-names" tagx="given-names" title="given-names">JK.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Using environmental correlations to identify loci underlying local adaptation</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">185</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">1411</span>– <span class="lpage" tagx="lpage" title="lpage">1423</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20516501">20516501</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Danecek</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">The variant call format and VCFtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">27</span>( <span class="issue" tagx="issue" title="issue">15</span>): <span class="fpage" tagx="fpage" title="fpage">2156</span>– <span class="lpage" tagx="lpage" title="lpage">2158</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522">21653522</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Do</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">NeEstimator v2: re-implementation of software for the estimation of contemporary effective population size (Ne) from genetic data </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">14</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">209</span>– <span class="lpage" tagx="lpage" title="lpage">214</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23992227">23992227</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elgert</span> <span class="given-names" tagx="given-names" title="given-names">KD.</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="source" tagx="source" title="source">Immunology: understanding the immune system</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Hoboken:John Wiley &amp;amp; Sons</span>. </span> </li> <li tag="ref"> <a name="evz118-B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elliott</span> <span class="given-names" tagx="given-names" title="given-names">JM.</span> </span> </span> <span class="year" tagx="year" title="year">1976</span> <span class="mixed-article-title" title="mixed-article-title">The energetics of feeding, metabolism and growth of brown trout ( <i>Salmo trutta</i> L.) in relation to body weight, water temperature and ration size </span>. <span class="source" tagx="source" title="source">J Anim Ecol</span>. <span class="volume" tagx="volume" title="volume">45</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">923</span>– <span class="lpage" tagx="lpage" title="lpage">948</span>. </span> </li> <li tag="ref"> <a name="evz118-B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Etter</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preston</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bassham</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cresko</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">EA.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Local de novo assembly of RAD paired-end contigs using short sequencing reads</span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">6</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">e18561.</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21541009">21541009</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Evanno</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Regnaut</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goudet</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Detecting the number of clusters of individuals using the software structure: a simulation study </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">14</span>( <span class="issue" tagx="issue" title="issue">8</span>): <span class="fpage" tagx="fpage" title="fpage">2611</span>– <span class="lpage" tagx="lpage" title="lpage">2620</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739">15969739</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lischer</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">10</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">564</span>– <span class="lpage" tagx="lpage" title="lpage">567</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21565059">21565059</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferchaud</span> <span class="given-names" tagx="given-names" title="given-names">A-L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hansen</span> <span class="given-names" tagx="given-names" title="given-names">MM.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">The impact of selection, gene flow and demographic history on heterogeneous genomic divergence: three-spine sticklebacks in divergent environments </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">25</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">238</span>– <span class="lpage" tagx="lpage" title="lpage">259</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26412233">26412233</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flajnik</span> <span class="given-names" tagx="given-names" title="given-names">MF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasahara</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Comparative genomics of the MHC: glimpses into the evolution of the adaptive immune system </span>. <span class="source" tagx="source" title="source">Immunity</span> <span class="volume" tagx="volume" title="volume">15</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">351</span>– <span class="lpage" tagx="lpage" title="lpage">362</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11567626">11567626</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flanagan</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rose</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">AG.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Population genomics reveals multiple drivers of population differentiation in a sex-role-reversed pipefish </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">25</span>( <span class="issue" tagx="issue" title="issue">20</span>): <span class="fpage" tagx="fpage" title="fpage">5043</span>– <span class="lpage" tagx="lpage" title="lpage">5072</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27485274">27485274</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foll</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaggiotti</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">180</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">977</span>– <span class="lpage" tagx="lpage" title="lpage">993</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18780740">18780740</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fournier-Level</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">A map of local adaptation in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">334</span>( <span class="issue" tagx="issue" title="issue">6052</span>): <span class="fpage" tagx="fpage" title="fpage">86</span>– <span class="lpage" tagx="lpage" title="lpage">89</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21980109">21980109</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frankham</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Briscoe</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ballou</span> <span class="given-names" tagx="given-names" title="given-names">JD.</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="source" tagx="source" title="source">Introduction to conservation genetics</span>. Cambridge: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span>. </span> </li> <li tag="ref"> <a name="evz118-B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Funk</span> <span class="given-names" tagx="given-names" title="given-names">WC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKay</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohenlohe</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allendorf</span> <span class="given-names" tagx="given-names" title="given-names">FW.</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Harnessing genomics for delineating conservation units</span>. <span class="source" tagx="source" title="source">Trends Ecol Evol (Amst)</span>. <span class="volume" tagx="volume" title="volume">27</span>( <span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">489</span>– <span class="lpage" tagx="lpage" title="lpage">496</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22727017">22727017</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">TX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bi</span> <span class="given-names" tagx="given-names" title="given-names">XX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">LL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">CJ.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Population genetic structure of roughskin sculpin <i>Trachidermus fasciatus</i> based on the mitochondrial Cytb sequence </span>. <span class="source" tagx="source" title="source">Acta Hydrobiol Sin</span>. <span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">199</span>– <span class="lpage" tagx="lpage" title="lpage">207</span>. </span> </li> <li tag="ref"> <a name="evz118-B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grabherr</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="volume" tagx="volume" title="volume">29</span>( <span class="issue" tagx="issue" title="issue">7</span>): <span class="fpage" tagx="fpage" title="fpage">644</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harrison</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hastings</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Genetic and evolutionary consequences of metapopulation structure</span>. <span class="source" tagx="source" title="source">Trends Ecol Evol</span>. <span class="volume" tagx="volume" title="volume">11</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">180</span>– <span class="lpage" tagx="lpage" title="lpage">183</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21237801">21237801</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harrisson</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pavlova</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Telonis-Scott</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunnucks</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Using genomics to characterize evolutionary potential for conservation of wild populations</span>. <span class="source" tagx="source" title="source">Evol Appl</span>. <span class="volume" tagx="volume" title="volume">7</span>( <span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">1008</span>– <span class="lpage" tagx="lpage" title="lpage">1025</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25553064">25553064</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hauser</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adcock</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernal Ramirez</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carvalho</span> <span class="given-names" tagx="given-names" title="given-names">GR.</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Loss of microsatellite diversity and low effective population size in an overexploited population of New Zealand snapper ( <i>Pagrus auratus</i>) </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="volume" tagx="volume" title="volume">99</span>( <span class="issue" tagx="issue" title="issue">18</span>): <span class="fpage" tagx="fpage" title="fpage">11742</span>– <span class="lpage" tagx="lpage" title="lpage">11747</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12185245">12185245</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffmann</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willi</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Detecting genetic responses to environmental change</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span>. <span class="volume" tagx="volume" title="volume">9</span>( <span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">421</span>– <span class="lpage" tagx="lpage" title="lpage">432</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18463665">18463665</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohenlohe</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags </span>. <span class="source" tagx="source" title="source">PLoS Genet</span>. <span class="volume" tagx="volume" title="volume">6</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">e1000862.</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20195501">20195501</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madan</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">CAP3: a DNA sequence assembly program</span>. <span class="source" tagx="source" title="source">Genome Res</span>. <span class="volume" tagx="volume" title="volume">9</span>( <span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">868</span>– <span class="lpage" tagx="lpage" title="lpage">877</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846">10508846</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Islam</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hibino</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Tidal and diurnal variations in larval fish abundance in an estuarine inlet in Ariake Bay, Japan: implication for selective tidal stream transport </span>. <span class="source" tagx="source" title="source">Ecol Res</span>. <span class="volume" tagx="volume" title="volume">22</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">165</span>– <span class="lpage" tagx="lpage" title="lpage">171</span>. </span> </li> <li tag="ref"> <a name="evz118-B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">XL.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Variation of runoff volume in the Dagu river basin in the Jiaodong peninsula</span>. <span class="source" tagx="source" title="source">Arid Zone Res</span>. <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">965</span>– <span class="lpage" tagx="lpage" title="lpage">972</span>. </span> </li> <li tag="ref"> <a name="evz118-B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jombart</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">adegenet: a R package for the multivariate analysis of genetic markers</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">24</span>( <span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">1403</span>– <span class="lpage" tagx="lpage" title="lpage">1405</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18397895">18397895</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kass</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raftery</span> <span class="given-names" tagx="given-names" title="given-names">AE.</span> </span> </span> <span class="year" tagx="year" title="year">1995</span> <span class="mixed-article-title" title="mixed-article-title">Bayes factors</span>. <span class="source" tagx="source" title="source">J Am Stat Assoc</span>. <span class="volume" tagx="volume" title="volume">90</span>( <span class="issue" tagx="issue" title="issue">430</span>): <span class="fpage" tagx="fpage" title="fpage">773</span>– <span class="lpage" tagx="lpage" title="lpage">795</span>. </span> </li> <li tag="ref"> <a name="evz118-B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kearney</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Porter</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Mechanistic niche modelling: combining physiological and spatial data to predict species’ ranges </span>. <span class="source" tagx="source" title="source">Ecol Lett</span>. <span class="volume" tagx="volume" title="volume">12</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">334</span>– <span class="lpage" tagx="lpage" title="lpage">350</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19292794">19292794</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keyghobadi</span> <span class="given-names" tagx="given-names" title="given-names">NK.</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">The genetic implications of habitat fragmentation for animals</span>. <span class="source" tagx="source" title="source">Can J Zool</span>. <span class="volume" tagx="volume" title="volume">85</span>( <span class="issue" tagx="issue" title="issue">10</span>): <span class="fpage" tagx="fpage" title="fpage">1049</span>– <span class="lpage" tagx="lpage" title="lpage">1064</span>. </span> </li> <li tag="ref"> <a name="evz118-B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kjeldsen</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide SNP loci reveal novel insights into koala ( <i>Phascolarctos cinereus</i>) population variability across its range </span>. <span class="source" tagx="source" title="source">Conserv Genet</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">337</span>– <span class="lpage" tagx="lpage" title="lpage">353</span>. </span> </li> <li tag="ref"> <a name="evz118-B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kopelman</span> <span class="given-names" tagx="given-names" title="given-names">NM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mayzel</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jakobsson</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosenberg</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mayrose</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Clumpak: a program for identifying clustering modes and packaging population structure inferences across K </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">15</span>( <span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">1179</span>– <span class="lpage" tagx="lpage" title="lpage">1191</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25684545">25684545</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kroeker</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Impacts of ocean acidification on marine organisms: quantifying sensitivities and interaction with warming </span>. <span class="source" tagx="source" title="source">Glob Change Biol</span>. <span class="volume" tagx="volume" title="volume">19</span>( <span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">1884</span>– <span class="lpage" tagx="lpage" title="lpage">1896</span>. </span> </li> <li tag="ref"> <a name="evz118-B56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larson</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Genotyping by sequencing resolves shallow population structure to inform conservation of Chinook salmon ( <i>Oncorhynchus tshawytscha</i>) </span>. <span class="source" tagx="source" title="source">Evol Appl</span>. <span class="volume" tagx="volume" title="volume">7</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">355</span>– <span class="lpage" tagx="lpage" title="lpage">369</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24665338">24665338</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemer</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Planes</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Effects of habitat fragmentation on the genetic structure and connectivity of the black-lipped pearl oyster Pinctada <i>margaritifera populations</i> in French Polynesia </span>. <span class="source" tagx="source" title="source">Mar Biol</span>. <span class="volume" tagx="volume" title="volume">161</span>( <span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">2035</span>– <span class="lpage" tagx="lpage" title="lpage">2049</span>. </span> </li> <li tag="ref"> <a name="evz118-B58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span>. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv: 1303.3997 q-bio. doi: http://arxiv.org/abs/1303.3997. </span> </li> <li tag="ref"> <a name="evz118-B59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">The sequence alignment/map format and SAMtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span>( <span class="issue" tagx="issue" title="issue">16</span>): <span class="fpage" tagx="fpage" title="fpage">2078</span>– <span class="lpage" tagx="lpage" title="lpage">2079</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">KB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">BX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rui</span> <span class="given-names" tagx="given-names" title="given-names">JS.</span> </span> </span> <span class="year" tagx="year" title="year">1984</span> <span class="mixed-article-title" title="mixed-article-title">A histological study on the digestive organs of <i>Trachidermus fasciatus</i> during seaward migration </span>. <span class="source" tagx="source" title="source">J Fudan Univ</span>. <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">7</span>– <span class="lpage" tagx="lpage" title="lpage">16</span>. </span> </li> <li tag="ref"> <a name="evz118-B61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">JX.</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">StructureSelector: a web-based software to select and visualize the optimal number of clusters using multiple methods </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">18</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">176</span>– <span class="lpage" tagx="lpage" title="lpage">177</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28921901">28921901</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">DX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">TX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">JX.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Genetic diversity and population structure of the roughskin sculpin ( <i>Trachidermus fasciatus</i> Heckel) inferred from microsatellite analyses: implications for its conservation and management </span>. <span class="source" tagx="source" title="source">Conserv Genet</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">921</span>– <span class="lpage" tagx="lpage" title="lpage">930</span>. </span> </li> <li tag="ref"> <a name="evz118-B63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">DX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">BD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">JX.</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">An optimized approach for local de novo assembly of overlapping paired-end RAD reads from multiple individuals </span>. <span class="source" tagx="source" title="source">R Soc Open Sci</span>. <span class="volume" tagx="volume" title="volume">5</span>:171589. http://dx.doi.org/10.1098/rsos.171589. </span> </li> <li tag="ref"> <a name="evz118-B64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lischer</span> <span class="given-names" tagx="given-names" title="given-names">HEL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">PGDSpider: an automated data conversion tool for connecting population genetics and genomics programs </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">28</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">298</span>– <span class="lpage" tagx="lpage" title="lpage">299</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22110245">22110245</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">XF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhai</span> <span class="given-names" tagx="given-names" title="given-names">JQ.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Quantitative study of impacts for environmental change on runoff in Luanhe river basin</span>. <span class="source" tagx="source" title="source">J Nat Resour</span>. <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">244</span>– <span class="lpage" tagx="lpage" title="lpage">252</span>. </span> </li> <li tag="ref"> <a name="evz118-B66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lowry</span> <span class="given-names" tagx="given-names" title="given-names">DB</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Breaking RAD: an evaluation of the utility of restriction site-associated DNA sequencing for genome scans of adaptation </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">142</span>– <span class="lpage" tagx="lpage" title="lpage">152</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27860289">27860289</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luikart</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allendorf</span> <span class="given-names" tagx="given-names" title="given-names">FW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cornuet</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sherwin</span> <span class="given-names" tagx="given-names" title="given-names">WB.</span> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Distortion of allele frequency distributions provides a test for recent population bottlenecks </span>. <span class="source" tagx="source" title="source">J Hered</span>. <span class="volume" tagx="volume" title="volume">89</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">238</span>– <span class="lpage" tagx="lpage" title="lpage">247</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9656466">9656466</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">XF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhuang</span> <span class="given-names" tagx="given-names" title="given-names">ZM.</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Analyses of the molecular mechanisms associated with salinity adaption of <i>Trachidermus fasciatus</i> through combined iTRAQ-based proteomics and RNA sequencing-based transcriptomics </span>. <span class="source" tagx="source" title="source">Prog Biophys Mol Biol</span>. <span class="volume" tagx="volume" title="volume">136</span>: <span class="fpage" tagx="fpage" title="fpage">40</span>– <span class="lpage" tagx="lpage" title="lpage">53</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29447837">29447837</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <span class="source" tagx="source" title="source">Embnet J</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">10</span>– <span class="lpage" tagx="lpage" title="lpage">12</span>. </span> </li> <li tag="ref"> <a name="evz118-B70" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKinney</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larson</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seeb</span> <span class="given-names" tagx="given-names" title="given-names">LW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seeb</span> <span class="given-names" tagx="given-names" title="given-names">JE.</span> </span> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">RADseq provides unprecedented insights into molecular ecology and evolutionary genetics: comment on Breaking RAD by Lowry et al. (2016) </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">17</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">356</span>– <span class="lpage" tagx="lpage" title="lpage">361</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28028934">28028934</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B71" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McMahon</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teeling</span> <span class="given-names" tagx="given-names" title="given-names">EC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Höglund</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">How and why should we implement genomics into conservation?</span> <span class="source" tagx="source" title="source">Evol Appl</span>. <span class="volume" tagx="volume" title="volume">7</span>( <span class="issue" tagx="issue" title="issue">9</span>): <span class="fpage" tagx="fpage" title="fpage">999</span>– <span class="lpage" tagx="lpage" title="lpage">1007</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25553063">25553063</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B72" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moritz</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Conservation units and translocations: strategies for conserving evolutionary processes</span>. <span class="source" tagx="source" title="source">Hereditas</span> <span class="volume" tagx="volume" title="volume">130</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">217</span>– <span class="lpage" tagx="lpage" title="lpage">228</span>. </span> </li> <li tag="ref"> <a name="evz118-B73" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mossman</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waser</span> <span class="given-names" tagx="given-names" title="given-names">PM.</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Effects of habitat fragmentation on population genetic structure in the white-footed mouse ( <i>Peromyscus leucopus</i>) </span>. <span class="source" tagx="source" title="source">Can J Zool</span>. <span class="volume" tagx="volume" title="volume">79</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">285</span>– <span class="lpage" tagx="lpage" title="lpage">295</span>. </span> </li> <li tag="ref"> <a name="evz118-B74" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chakraborty</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fuerst</span> <span class="given-names" tagx="given-names" title="given-names">PA.</span> </span> </span> <span class="year" tagx="year" title="year">1976</span> <span class="mixed-article-title" title="mixed-article-title">Infinite allele model with varying mutation rate</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="volume" tagx="volume" title="volume">73</span>( <span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">4164</span>– <span class="lpage" tagx="lpage" title="lpage">4168</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1069306">1069306</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B75" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Onikura</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takeshita</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsui</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kimura</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Spawning grounds and nests of <i>Trachidermus fasciatus</i> (Cottidae) in the Kashima and Shiota estuaries system facing Ariake Bay, Japan </span>. <span class="source" tagx="source" title="source">Ichthyol Res</span>. <span class="volume" tagx="volume" title="volume">49</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">198</span>– <span class="lpage" tagx="lpage" title="lpage">201</span>. </span> </li> <li tag="ref"> <a name="evz118-B76" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ouborg</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertoldi</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loeschcke</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bijlsma</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hedrick</span> <span class="given-names" tagx="given-names" title="given-names">PW.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Conservation genetics in transition to conservation genomics</span>. <span class="source" tagx="source" title="source">Trends Genet</span>. <span class="volume" tagx="volume" title="volume">26</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">177</span>– <span class="lpage" tagx="lpage" title="lpage">187</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20227782">20227782</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B77" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pörtner</span> <span class="given-names" tagx="given-names" title="given-names">HO</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knust</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Climate change affects marine fishes through the oxygen limitation of thermal tolerance</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">315</span>( <span class="issue" tagx="issue" title="issue">5808</span>): <span class="fpage" tagx="fpage" title="fpage">95</span>– <span class="lpage" tagx="lpage" title="lpage">97</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17204649">17204649</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B78" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinsky</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palumbi</span> <span class="given-names" tagx="given-names" title="given-names">SR.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Meta-analysis reveals lower genetic diversity in overfished populations</span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">23</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">29</span>– <span class="lpage" tagx="lpage" title="lpage">39</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24372754">24372754</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B79" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pita</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pérez</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Velasco</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Presa</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Trends of the genetic effective population size in the Southern stock of the European hake </span>. <span class="source" tagx="source" title="source">Fish Res</span>. <span class="volume" tagx="volume" title="volume">191</span>: <span class="fpage" tagx="fpage" title="fpage">108</span>– <span class="lpage" tagx="lpage" title="lpage">119</span>. </span> </li> <li tag="ref"> <a name="evz118-B80" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Priyam</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2015</span>. Sequenceserver: a modern graphical user interface for custom BLAST databases. bioRxiv 033142, doi: https://doi.org/10.1101/033142 </span> </li> <li tag="ref"> <a name="evz118-B81" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puechmaille</span> <span class="given-names" tagx="given-names" title="given-names">SJ.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">The program structure does not reliably recover the correct population structure when sampling is uneven: subsampling and new estimators alleviate the problem </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">16</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">608</span>– <span class="lpage" tagx="lpage" title="lpage">627</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26856252">26856252</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B82" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purcell</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="source" tagx="source" title="source">Am J Hum Genet</span>. <span class="volume" tagx="volume" title="volume">81</span>( <span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">559</span>– <span class="lpage" tagx="lpage" title="lpage">575</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17701901">17701901</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B83" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="collab'">collab: R Core Team</span> <span class="year" tagx="year" title="year">2018</span> <span class="source" tagx="source" title="source">R: a Language and Environment for Statistical Computing</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span> <a href="https://www.R-project.org/">https://www.R-project.org/</a>; Last accessed June 5, 2019. </span> </li> <li tag="ref"> <a name="evz118-B84" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sandoval-Castillo</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hart</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strain</span> <span class="given-names" tagx="given-names" title="given-names">LWS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beheregaray</span> <span class="given-names" tagx="given-names" title="given-names">LB.</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Seascape genomics reveals adaptive divergence in a connected and commercially important mollusc, the greenlip abalone ( <i>Haliotis laevigata</i>), along a longitudinal environmental gradient </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">27</span>( <span class="issue" tagx="issue" title="issue">7</span>): <span class="fpage" tagx="fpage" title="fpage">1603</span>– <span class="lpage" tagx="lpage" title="lpage">1620</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29420852">29420852</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B85" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanford</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelly</span> <span class="given-names" tagx="given-names" title="given-names">MW.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Local adaptation in marine invertebrates</span>. <span class="source" tagx="source" title="source">Ann Rev Mar Sci</span>. <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">509</span>– <span class="lpage" tagx="lpage" title="lpage">535</span>. </span> </li> <li tag="ref"> <a name="evz118-B86" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Savolainen</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lascoux</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Merila</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Ecological genomics of local adaptation</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span>. <span class="volume" tagx="volume" title="volume">14</span>( <span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">807</span>– <span class="lpage" tagx="lpage" title="lpage">820</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24136507">24136507</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B87" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schluter</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Ecological character displacement in adaptive radiation</span>. <span class="source" tagx="source" title="source">Am Nat</span>. <span class="volume" tagx="volume" title="volume">156</span>( <span class="issue" tagx="issue" title="issue">S4</span>): <span class="fpage" tagx="fpage" title="fpage">S4</span>– <span class="lpage" tagx="lpage" title="lpage">S16</span>. </span> </li> <li tag="ref"> <a name="evz118-B88" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">BX.</span> </span> </span> <span class="year" tagx="year" title="year">1978</span> <span class="mixed-article-title" title="mixed-article-title">The seasonal change of the thyroid gland of <i>Trachidermus fasciatus</i> in relation to the seaward migration </span>. <span class="source" tagx="source" title="source">Oceanol Limnol Sin</span>. <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">230</span>– <span class="lpage" tagx="lpage" title="lpage">235</span>. </span> </li> <li tag="ref"> <a name="evz118-B89" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">BX</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">1980</span> <span class="mixed-article-title" title="mixed-article-title">On the breeding habit of <i>Trachidermus fasciatus</i> Heckel </span>. <span class="source" tagx="source" title="source">J Fish China</span>. <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">81</span>– <span class="lpage" tagx="lpage" title="lpage">86</span>. </span> </li> <li tag="ref"> <a name="evz118-B90" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinig</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neuditschko</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khatkar</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raadsma</span> <span class="given-names" tagx="given-names" title="given-names">HW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zenger</span> <span class="given-names" tagx="given-names" title="given-names">KR.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">netview p: a network visualization tool to unravel complex population structure using genome-wide SNPs </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span>. <span class="volume" tagx="volume" title="volume">16</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">216</span>– <span class="lpage" tagx="lpage" title="lpage">227</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26129944">26129944</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B91" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="collab'">collab: The UniProt Consortium</span>. <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">UniProt: a worldwide hub of protein knowledge</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">D506</span>– <span class="lpage" tagx="lpage" title="lpage">D515</span>. </span> </li> <li tag="ref"> <a name="evz118-B92" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uchida</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahn</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Gli3</i> is required for maintenance and fate specification of cortical progenitors </span>. <span class="source" tagx="source" title="source">J Neurosci</span>. <span class="volume" tagx="volume" title="volume">31</span>( <span class="issue" tagx="issue" title="issue">17</span>): <span class="fpage" tagx="fpage" title="fpage">6440</span>– <span class="lpage" tagx="lpage" title="lpage">6448</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21525285">21525285</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B93" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">JQ.</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Advances in studies on the ecology and reproductive biology of <i>Trachidermus fasciatus</i> Heckel </span>. <span class="source" tagx="source" title="source">Acta Hydrobiol Sin</span>. <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">729</span>– <span class="lpage" tagx="lpage" title="lpage">734</span>. </span> </li> <li tag="ref"> <a name="evz118-B94" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">JQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">The historical variance and causes of geographical distribution of a roughskin sculpin ( <i>Trachidermus fasciatus</i> Heckel) in Chinese territory </span>. <span class="source" tagx="source" title="source">Acta Ecol Sin</span>. <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">6845</span>– <span class="lpage" tagx="lpage" title="lpage">6853</span>. </span> </li> <li tag="ref"> <a name="evz118-B95" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">JQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">ZP.</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">The distribution of roughskin sculpin ( <i>Trachidermus fasciatus</i> Heckel) in Yalu river basin, China </span>. <span class="source" tagx="source" title="source">J Fudan Univ Nat Sci</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">471</span>– <span class="lpage" tagx="lpage" title="lpage">476</span>. </span> </li> <li tag="ref"> <a name="evz118-B96" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wan</span> <span class="given-names" tagx="given-names" title="given-names">ZY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yue</span> <span class="given-names" tagx="given-names" title="given-names">GH.</span> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Genetic variability, local selection and demographic history: genomic evidence of evolving towards allopatric speciation in Asian seabass </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">25</span>( <span class="issue" tagx="issue" title="issue">15</span>): <span class="fpage" tagx="fpage" title="fpage">3605</span>– <span class="lpage" tagx="lpage" title="lpage">3621</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27262162">27262162</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B97" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">1997</span> <span class="mixed-article-title" title="mixed-article-title">Environmental factors controlling the growth of <i>Trachidermus fasciatus</i> </span>. <span class="source" tagx="source" title="source">J Fudan Univ</span>. <span class="volume" tagx="volume" title="volume">36</span>: <span class="fpage" tagx="fpage" title="fpage">581</span>– <span class="lpage" tagx="lpage" title="lpage">585</span>. </span> </li> <li tag="ref"> <a name="evz118-B98" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willoughby</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harder</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tennessen</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scribner</span> <span class="given-names" tagx="given-names" title="given-names">KT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christie</span> <span class="given-names" tagx="given-names" title="given-names">MR.</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Rapid genetic adaptation to a novel environment despite a genome-wide reduction in genetic diversity </span>. <span class="source" tagx="source" title="source">Mol Ecol</span>. <span class="volume" tagx="volume" title="volume">27</span>( <span class="issue" tagx="issue" title="issue">20</span>): <span class="fpage" tagx="fpage" title="fpage">4041</span>– <span class="lpage" tagx="lpage" title="lpage">4051</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29802799">29802799</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B99" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Analysis of genetic diversity in roughskin sculpin <i>Trachidermus fasciatus</i> by AFLP markers </span>. <span class="source" tagx="source" title="source">J Dalian Fish Univ</span>. <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">437</span>– <span class="lpage" tagx="lpage" title="lpage">441</span>. </span> </li> <li tag="ref"> <a name="evz118-B100" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goddard</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Visscher</span> <span class="given-names" tagx="given-names" title="given-names">PM.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">GCTA: a tool for genome-wide complex trait analysis</span>. <span class="source" tagx="source" title="source">Am J Hum Genet</span>. <span class="volume" tagx="volume" title="volume">88</span>( <span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">76</span>– <span class="lpage" tagx="lpage" title="lpage">82</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21167468">21167468</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B101" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boyle</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">The population genetic consequences of habitat fragmentation for plants</span>. <span class="source" tagx="source" title="source">Trends Ecol Evol</span>. <span class="volume" tagx="volume" title="volume">11</span>( <span class="issue" tagx="issue" title="issue">10</span>): <span class="fpage" tagx="fpage" title="fpage">413</span>– <span class="lpage" tagx="lpage" title="lpage">418</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21237900">21237900</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B102" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Analysis of genetic diversity in wild populations of <i>Trachidermus fasciatus</i> by RAPD and the transformation of two SCAR markers </span>. <span class="source" tagx="source" title="source">Zool Res</span>. <span class="volume" tagx="volume" title="volume">33</span>( <span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">203</span>– <span class="lpage" tagx="lpage" title="lpage">210</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22467397">22467397</a> </span> </span> </li> <li tag="ref"> <a name="evz118-B103" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zerbino</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Ensembl 2018</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="volume" tagx="volume" title="volume">46</span>( <span class="issue" tagx="issue" title="issue">D1</span>): <span class="fpage" tagx="fpage" title="fpage">D754</span>– <span class="lpage" tagx="lpage" title="lpage">D761</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29155950">29155950</a> </span> </span> </li> </ul> </div> </div>  </body></html>