<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Proc Natl Acad Sci U S A</span> <span class="iso-abbrev" title="iso-abbrev">Proc. Natl. Acad. Sci. U.S.A</span> <span class="hwp" title="hwp">pnas</span> <span class="pmc" title="pmc">pnas</span> <span class="publisher-id" title="publisher-id">PNAS</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">0027-8424</span> <span class="issn-epub" tagx="issn" title="issn-epub">1091-6490</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">National Academy of Sciences</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6452661">6452661</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30894495">30894495</a> </span> <span class="publisher-id" title="publisher-id">201817580</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1073/pnas.1817580116">10.1073/pnas.1817580116</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">PNAS Plus</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Sciences</span>: <span class="subject" title="subject">Plant Biology</span> </div> <span class="series-title" tagx="series-title" title="series-title">PNAS Plus</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genome of <i>Crucihimalaya himalaica</i>, a close relative of <i>Arabidopsis</i>, shows ecological adaptation to high altitude </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Ticao</span> </span> <a href="#aff1"> <sup>a</sup> </a> <a href="#aff2"> <sup>b</sup> </a> <a href="#fn1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiao</span> <span class="given-names" tagx="given-names" title="given-names">Qin</span> </span> <a href="#aff3"> <sup>c</sup> </a> <a href="#fn1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novikova</span> <span class="given-names" tagx="given-names" title="given-names">Polina Yu.</span> </span> <a href="#aff4"> <sup>d</sup> </a> <a href="#aff5"> <sup>e</sup> </a> <a href="#fn1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Qia</span> </span> <a href="#aff1"> <sup>a</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yue</span> <span class="given-names" tagx="given-names" title="given-names">Jipei</span> </span> <a href="#aff1"> <sup>a</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guan</span> <span class="given-names" tagx="given-names" title="given-names">Yanlong</span> </span> <a href="#aff1"> <sup>a</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ming</span> <span class="given-names" tagx="given-names" title="given-names">Shengping</span> </span> <a href="#aff6"> <sup>f</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Tianmeng</span> </span> <a href="#aff6"> <sup>f</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De</span> <span class="given-names" tagx="given-names" title="given-names">Ji</span> </span> <a href="#aff6"> <sup>f</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Yixuan</span> </span> <a href="#aff6"> <sup>f</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Shehbaz</span> <span class="given-names" tagx="given-names" title="given-names">Ihsan A.</span> </span> <a href="#aff7"> <sup>g</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Hang</span> </span> <a href="#aff2"> <sup>b</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Montagu</span> <span class="given-names" tagx="given-names" title="given-names">Marc</span> </span> <a href="#aff4"> <sup>d</sup> </a> <a href="#aff5"> <sup>e</sup> </a> <a href="#cor1"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">Jinling</span> </span> <a href="#aff1"> <sup>a</sup> </a> <a href="#aff8"> <sup>h</sup> </a> <a href="#aff9"> <sup>i</sup> </a> <a href="#cor1"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-4327-3730</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van de Peer</span> <span class="given-names" tagx="given-names" title="given-names">Yves</span> </span> <a href="#aff4"> <sup>d</sup> </a> <a href="#aff5"> <sup>e</sup> </a> <a href="#aff10"> <sup>j</sup> </a> <a href="#cor1"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiong</span> <span class="given-names" tagx="given-names" title="given-names">La</span> </span> <a href="#aff6"> <sup>f</sup> </a> <a href="#cor1"> <sup>2</sup> </a> </span> <span class="citation_author_institution" id="aff1">[], <sup>a</sup> <span class="institution" title="institution">Kunming Institute of Botany, Chinese Academy of Sciences</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>b</sup> <span class="institution" title="institution">Yunnan University of Chinese Medicine</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff3">[], <sup>c</sup> <span class="institution" title="institution">Yunnan University</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff4">[], <sup>d</sup> <span class="institution" title="institution">Ghent University</span> <span class="country" tagx="country" title="country">Belgium</span> </span> <span class="citation_author_institution" id="aff5">[], <sup>e</sup> <span class="institution" title="institution">VIB</span> <span class="country" tagx="country" title="country">Belgium</span> </span> <span class="citation_author_institution" id="aff6">[], <sup>f</sup> <span class="institution" title="institution">Tibet University</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff7">[], <sup>g</sup> <span class="institution" title="institution">Missouri Botanical Garden</span> <span class="addr-line" title="addr-line">MO</span> </span> <span class="citation_author_institution" id="aff8">[], <sup>h</sup> <span class="institution" title="institution">East Carolina University</span> <span class="addr-line" title="addr-line">NC</span> </span> <span class="citation_author_institution" id="aff9">[], <sup>i</sup> <span class="institution" title="institution">Henan University</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff10">[], <sup>j</sup> <span class="institution" title="institution">University of Pretoria</span> <span class="country" tagx="country" title="country">South Africa</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <sup>2</sup>To whom correspondence may be addressed. Email: <span class="email" tagx="email" title="email">marc.vanmontagu@vib-ugent.be</span>, <span class="email" tagx="email" title="email">huangj@ecu.edu</span>, <span class="email" tagx="email" title="email">yves.vandepeer@psb.vib-ugent.be</span>, or <span class="email" tagx="email" title="email">lhagchong@163.com</span>. </div> <div class="fn-type-edited-by" title="edited-by"> <p>Contributed by Marc Van Montagu, February 21, 2019 (sent for review October 19, 2018; reviewed by Ya-Long Guo, Matthew B. Hufford, and Martin Lascoux) </p> </div> <div class="fn-type-con" title="con"> <p>Author contributions: T.Z., Q.Q., M.V.M., J.H., Y.V.d.P., and L.Q. designed research; T.Z., Q.Q., Q.W., J.Y., Y.G., S.M., T.L., J.D., Y.L., I.A.A.-S., H.S., and L.Q. performed research; T.Z., P.Y.N., Q.W., and Y.V.d.P. analyzed data; and T.Z., Q.Q., P.Y.N., J.H., and Y.V.d.P. wrote the paper. </p> </div> <div class="fn-type-con" title="con"> <p>Reviewers: Y.-L.G. Chinese Academy of Sciences; M.B.H., Iowa State University; and M.L., Uppsala University. </p> </div> <div class="fn-type-equal" title="equal"> <p> <sup>1</sup>T.Z., Q.Q., and P.Y.N. contributed equally to this work. </p> </div> </div> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-4-4</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-3-3</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-3-3</span> </span> <span class="volume" tagx="volume" title="volume">116</span> <span class="issue" tagx="issue" title="issue">14</span> <span class="fpage" tagx="fpage" title="fpage">7137</span> <span class="lpage" tagx="lpage" title="lpage">7146</span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license">[license_ref] <span class="license-p" title="license-p">This open access article is distributed under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution-NonCommercial-NoDerivatives License 4.0 (CC BY-NC-ND)</a>. </span> </span> </div> <a class="self-uri" href="pnas.201817580.pdf" title="self-uri">pnas.201817580.pdf</a> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Significance</div> <p> <i>Crucihimalaya himalaica</i> is a close relative of <i>Arabidopsis</i> with typical Qinghai–Tibet Plateau (QTP) distribution. Here, by combining short- and long-read sequencing technologies, we provide a de novo genome sequence of <i>C. himalaica</i>. Our results suggest that the quick uplifting of the QTP coincided with the expansion of repeat elements. Gene families showing dramatic contractions and expansions, as well as genes showing clear signs of natural selection, were likely responsible for <i>C. himalaica</i>’s specific adaptation to the harsh environment of the QTP. We also show that the transition to self-pollination of <i>C. himalaica</i> might have enabled its occupation of the QTP. This study provides insights into how plants might adapt to extreme environmental conditions. </p> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Crucihimalaya himalaica</i>, a close relative of <i>Arabidopsis</i> and <i>Capsella</i>, grows on the Qinghai–Tibet Plateau (QTP) about 4,000 m above sea level and represents an attractive model system for studying speciation and ecological adaptation in extreme environments. We assembled a draft genome sequence of 234.72 Mb encoding 27,019 genes and investigated its origin and adaptive evolutionary mechanisms. Phylogenomic analyses based on 4,586 single-copy genes revealed that <i>C. himalaica</i> is most closely related to <i>Capsella</i> (estimated divergence 8.8 to 12.2 Mya), whereas both species form a sister clade to <i>Arabidopsis thaliana</i> and <i>Arabidopsis lyrata</i>, from which they diverged between 12.7 and 17.2 Mya. LTR retrotransposons in <i>C. himalaica</i> proliferated shortly after the dramatic uplift and climatic change of the Himalayas from the Late Pliocene to Pleistocene. Compared with closely related species, <i>C. himalaica</i> showed significant contraction and pseudogenization in gene families associated with disease resistance and also significant expansion in gene families associated with ubiquitin-mediated proteolysis and DNA repair. We identified hundreds of genes involved in DNA repair, ubiquitin-mediated proteolysis, and reproductive processes with signs of positive selection. Gene families showing dramatic changes in size and genes showing signs of positive selection are likely candidates for <i>C. himalaica</i>’s adaptation to intense radiation, low temperature, and pathogen-depauperate environments in the QTP. Loss of function at the S-locus, the reason for the transition to self-fertilization of <i>C. himalaica</i>, might have enabled its QTP occupation. Overall, the genome sequence of <i>C. himalaica</i> provides insights into the mechanisms of plant adaptation to extreme environments. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
adaptive evolution</div> <div class="kwd" title="kwd">
natural selection</div> <div class="kwd" title="kwd">
extreme environment</div> <div class="kwd" title="kwd">
Qinghai–Tibet Plateau</div> <div class="kwd" title="kwd">
S-locus</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
National Natural Science Foundation of China (NSFC) <span class="named-content" title="named-content">501100001809</span> </div> <span class="award-id" tagx="award-id" title="award-id">31770408</span> <span class="award-id" tagx="award-id" title="award-id">31590823</span> <span class="award-id" tagx="award-id" title="award-id">31760082</span> <span class="award-id" tagx="award-id" title="award-id">37160127</span> <span class="award-id" tagx="award-id" title="award-id">91131901</span> <span class="principal-award-recipient" title="principal-award-recipient">Ticao Zhang</span> <span class="principal-award-recipient" title="principal-award-recipient">Qin Qiao</span> <span class="principal-award-recipient" title="principal-award-recipient">Hang Sun</span> <span class="principal-award-recipient" title="principal-award-recipient">La Qiong</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
National High Technology Research and Development Program of China</div> <span class="award-id" tagx="award-id" title="award-id">2014AA020528</span> <span class="principal-award-recipient" title="principal-award-recipient">Ticao Zhang</span> <span class="principal-award-recipient" title="principal-award-recipient">Polina Yu. Novikova</span> <span class="principal-award-recipient" title="principal-award-recipient">Jinling Huang</span> <span class="principal-award-recipient" title="principal-award-recipient">Yves Van de Peer</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Kunming Institute Botany</div> <span class="award-id" tagx="award-id" title="award-id">Light of West China</span> <span class="principal-award-recipient" title="principal-award-recipient">Ticao Zhang</span> <span class="principal-award-recipient" title="principal-award-recipient">Polina Yu. Novikova</span> <span class="principal-award-recipient" title="principal-award-recipient">Jinling Huang</span> <span class="principal-award-recipient" title="principal-award-recipient">Yves Van de Peer</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
European Union 7th Framework Programme</div> <span class="award-id" tagx="award-id" title="award-id">FP7/2007-2013</span> <span class="principal-award-recipient" title="principal-award-recipient">Ticao Zhang</span> <span class="principal-award-recipient" title="principal-award-recipient">Polina Yu. Novikova</span> <span class="principal-award-recipient" title="principal-award-recipient">Jinling Huang</span> <span class="principal-award-recipient" title="principal-award-recipient">Yves Van de Peer</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
European Research Council Advanced Grant</div> <span class="award-id" tagx="award-id" title="award-id">Agreement 322739 - DOUBLEUP</span> <span class="principal-award-recipient" title="principal-award-recipient">Ticao Zhang</span> <span class="principal-award-recipient" title="principal-award-recipient">Polina Yu. Novikova</span> <span class="principal-award-recipient" title="principal-award-recipient">Jinling Huang</span> <span class="principal-award-recipient" title="principal-award-recipient">Yves Van de Peer</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Research Foundation-Flanders</div> <span class="award-id" tagx="award-id" title="award-id">12S9618N</span> <span class="principal-award-recipient" title="principal-award-recipient">Ticao Zhang</span> <span class="principal-award-recipient" title="principal-award-recipient">Polina Yu. Novikova</span> <span class="principal-award-recipient" title="principal-award-recipient">Jinling Huang</span> <span class="principal-award-recipient" title="principal-award-recipient">Yves Van de Peer</span> </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <p>The Qinghai–Tibet Plateau (QTP), also known as Himalayan Plateau, is the highest (average elevation above 4,000 m) and largest ( <i>ca</i>. 2.5 million km <sup>2</sup>) plateau in the world. The QTP mountains reached their current elevations mainly between the late Miocene and late Pliocene, according to recent syntheses of different studies ( <a href="#r1">1</a>). The uplift of the QTP resulted in profound climatic and environmental changes in both the plateau region and Asia at large ( <a href="#r2">2</a>). Conditions on the QTP are now characterized by low temperature, low oxygen, reduced pathogen incidence, and strong UV radiation, which together provide a unique environment to study adaptive evolution ( <a href="#r3">3</a>, <a href="#r4">4</a>). Previous genome-wide studies on adaptive evolution on the QTP have focused mainly on humans and vertebrates (reviewed in ref. <a href="#r5">5</a>). These studies have revealed that genes involved in hypoxia responses, energy metabolism, and skeletal development are under positive selection and rapid evolution. In contrast, hitherto, no study of adaptive evolution based on whole-genome analysis has been conducted on wild plants (excluding the domesticated Tibetan highland barley) ( <a href="#r6">6</a>) in this region. </p> <p> <i>Arabidopsis thaliana</i> (Cruciferae/Brassicaceae) is the best-known model organism in plant biology. Relatives of <i>A. thaliana</i> are important for studies on evolutionary ecology and comparative genomics of plants ( <a href="#r7">7</a> <a href="#r8" />– <a href="#r9">9</a>), especially those found in specific and challenging environments. <i>Crucihimalaya himalaica</i> (Edgew.) Al-Shehbaz, O’Kane &amp;amp; R.A.Price, a self-fertilizing, diploid (2n = 16) relative of <i>A. thaliana</i>, was previously recognized as <i>A. himalaica</i> (Edgew.) O.E.Schulz, and the names <i>Arabis rupestris</i> Edgew. and <i>Arabis brevicaulis</i> Jafri are also accepted as synonyms ( <a href="#r10">10</a>). <i>C. himalaica</i> mainly grows on rocky hillsides, sandy slopes, alpine meadows, and screes in the Himalayas and Hengduan Mountains. Previous estimates suggested that <i>Crucihimalaya</i> diverged from the closely related <i>Pachycladon</i> ∼10.94 Mya ( <a href="#r11">11</a>), while the species <i>C. himalaica</i> originated about 3.56 Mya ( <a href="#r12">12</a>). Given the postulated timing of the most recent rapid uplift of the QTP during the period of late Miocene to late Pliocene ( <a href="#r1">1</a>, <a href="#r13">13</a>), many of the genetic changes in <i>C. himalaica</i> might reflect its adaptation to the extreme environment of QTP ( <a href="#r4">4</a>). </p> <p>In a previous study ( <a href="#r4">4</a>), we identified 487 positively selected genes (PSGs) in the transcriptome of <i>C. himalaica</i>. Predicted functions of these PSGs indicate that they potentially contribute to miscellaneous traits of adaptive importance, such as response to UV radiation, DNA repair, and membrane stabilization, which presumably are important for the adaptation of <i>C. himalaica</i> to the specialized environment on the QTP. Therefore, we believe that this species represents a model system for the study of speciation and ecological adaptation in extreme environments. In the present study, we generated de novo whole-genome sequences of <i>C. himalaica</i> and applied comparative and evolutionary genomics approaches to clarify the origin of this species and to investigate signals of adaptation. Our goal is to gain a deeper understanding into the mechanisms by which <i>C. himalaica</i> has adapted to the complex extreme conditions on the QTP at the whole-genome level. </p> <div class="results_discussion" title="results|discussion"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="genomeassemblyandannotation." title="sec"> <div class="title" tagx="title" title="title">
Genome Assembly and Annotation.</div> <p>The genome size of <i>C. himalaica</i> was estimated to be 265.23 Mb with a heterozygosity of 0.70% on the basis of <i>k</i>-mer statistics ( <a href="#r14">14</a>) ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S1 </a>). In total, we generated 50.68 Gb of paired-end (PE) and mate-pair (MP) reads with different insert sizes using an Illumina HiSeq platform combined with PacBio sequencing technology ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S2 </a>). A total assembly of 234.72Mb, consisting of 583 scaffolds (scaffold N50 length, 2.09 Mb; longest scaffold, 8.34 Mb), was achieved by combination of the ALLPATHS–LG ( <a href="#r15">15</a>) and PBJelly2 ( <a href="#r16">16</a>) assembly strategies ( <a href="#t01">Table 1</a>). To assess assembly accuracy, we remapped raw sequencing reads of a small fragment library to the assembled genome. With a 96.69% mapping rate and 224.57× average sequence depth, the reads covered 96.43% of the genome, which implies that the current assembly covered almost all unique genomic regions. About 94.05% of the assembly was covered by at least 20× reads, which guaranteed a highly accurate assembly at the single-nucleotide level ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S3 </a>). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1.</span>  <p>Genome assembly of <i>C. himalaica</i> </p>  <table> <thead> <tr> <td>Genome features</td> <td>Contigs</td> <td>Scaffolds</td> </tr> </thead> <tbody> <tr> <td>Total length, bp</td> <td>230,905,116</td> <td>234,722,603</td> </tr> <tr> <td>Total no.</td> <td>3,983</td> <td>583</td> </tr> <tr> <td>Longest length, bp</td> <td>1,756,581</td> <td>8,343,586</td> </tr> <tr> <td>No. of length ≥2,000</td> <td>3,586</td> <td>429</td> </tr> <tr> <td>Length of N50, bp</td> <td>136,392</td> <td>2,088,603</td> </tr> <tr> <td>No. of N50</td> <td>406</td> <td>34</td> </tr> <tr> <td>Length of N90, bp</td> <td>32,421</td> <td>470,087</td> </tr> <tr> <td>No. of N90</td> <td>1,711</td> <td>129</td> </tr> <tr> <td>GC content, %</td> <td>36.38</td> <td>—</td> </tr> <tr> <td>No. of genes</td> <td>—</td> <td>27,019</td> </tr> </tbody> </table> </div> <p>A total of 27,019 protein-coding genes were predicted ( <a href="#t01">Table 1</a> and <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S4 </a>). Among these, 26,806 genes (99.21%) were functionally annotated ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S5 </a>). In addition to protein-coding genes, various noncoding RNA sequences were identified and annotated ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S6 </a>), including 448 microRNAs, 577 transfer RNAs, 153 ribosomal RNAs, and 974 small nuclear RNAs. Gene region completeness was evaluated by RNA sequencing data ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S7 </a>). Of the 29,420 transcripts assembled by Trinity, 99.74% were mapped to the genome assembly, and 92.37% were complete. The completeness of gene regions was further assessed using BUSCO (Benchmarking Universal Single Copy Orthologs) ( <a href="#r17">17</a>), which showed that 96% of the plant single-copy orthologs were complete ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S8 </a>). Compared with other close relatives, the <i>C. himalaica</i> genome contains a similar number of transcription factors (1,711) and transcription regulators (413) ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S9 </a>). The <i>C. himalaica</i> genome showed strong homology with the genome of <i>A. thaliana</i>, <i>Arabidopsis lyrata</i>, and <i>Capsella rubella</i>, except in centromeres and certain high-repeat regions ( <a href="#fig01">Fig. 1</a> and <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Figs. S1–S3 </a>). These results implied that the assembly was of high quality, thus ensuring the reliability of subsequent comparative genomic analyses in this study. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1.</span> </a> </div>   <p>Comparative analyses of genomic features of <i>C. himalaica</i> vs. <i>A. thaliana</i> ( <i>A</i>) and <i>C. himalaica</i> vs. <i>C. rubella</i> ( <i>B</i>). Tracks from inside to outside are collinearity between both genomes, number of chromosomes/scaffolds, gene density, and TE density. </p>   </div> </div> <div class="repeatelements." title="sec"> <div class="title" tagx="title" title="title">
Repeat Elements.</div> <p>Compared with the genome size of closely related species, such as <i>C. rubella</i> (which has an assembled genome size of 134.8 Mb, whereas the genome size estimated from flow cytometry was 219 Mb) ( <a href="#r18">18</a>) and <i>A. thaliana</i> [125 Mb ( <a href="#r19">19</a>) and the estimated genome size of the reference accession Col-0 from flow cytometry is 166 Mb ( <a href="#r20">20</a>)], the <i>C. himalaica</i> genome is considerably larger. It has been well documented that polyploidization (whole-genome duplication, WGD) events and transposable element (TE) amplification are major causes of genome expansion ( <a href="#r21">21</a>, <a href="#r22">22</a>). Analyses of age distributions built from transversion substitutions at fourfold degenerate sites (4DTv) indicated that, except for the α (4DTv distance = ∼0.3) and β (4DTv distance = ∼0.6) polyploidy events shared among members of the Brassicaceae ( <a href="#r23">23</a>), <i>C. himalaica</i> has not undergone an additional species-specific WGD event ( <a href="#fig02">Fig. 2 <i>A</i> </a>). Next, we investigated the content and evolutionary history of TEs in <i>C. himalaica</i>. Using de novo prediction of TEs ( <a href="#s9"> <i>Materials and Methods</i> </a>), we identified and marked 46.91% of the assembly as repeat regions, among which TEs occupied 45.78% of the genome assembly length ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S10 </a>), which is higher than that reported for <i>A. lyrata</i> and <i>A. thaliana</i> (29.7% and 23.7%, respectively) ( <a href="#r24">24</a>). This was also apparent in a CIRCOS genomic comparison plot, which showed that the density of TEs in <i>C. himalaica</i> was higher than that in <i>A. thaliana</i> or <i>C. rubella</i>, whereas the density of genes in <i>C. himalaica</i> was lower than that in <i>A. thaliana</i> or <i>C. rubella</i> ( <a href="#fig01">Fig. 1</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2.</span> </a> </div>   <p>Evolutionary analyses of the <i>C. himalaica</i> genome. ( <i>A</i>) Age distribution of 4DTv distance values between orthologs of <i>C. himalaica</i> and <i>A. thaliana</i>, <i>C. himalaica</i> and <i>A. lyrata</i>, and <i>C. himalaica</i> and <i>C. rubella</i>. ( <i>B</i>) Insertion time distribution of LTR retrotransposons. ( <i>C</i>) Estimation of divergence times of nine species in the Brassicaceae. ( <i>D</i>) Venn diagram showing unique and shared gene families between genomes of <i>C. himalaica</i> and four close relatives. </p>   </div> <p>In particular, a high proportion of TEs in <i>C. himalaica</i> were LTR retrotransposons (30.37%), whereas other retrotransposons (short and long interspersed nuclear elements) only constituted 3.31% collectively ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S10 </a>). The retrotransposon proliferation might be responsible for the genome-size expansion in <i>C. himalaica</i>. To investigate the evolutionary dynamics of LTR retrotransposons, we estimated their insertion dates in four closely related species ( <a href="#s9"> <i>Materials and Methods</i> </a>). Compared with <i>A. thaliana</i>, which shows a large number of microdeletions in noncoding DNA and transposons ( <a href="#r24">24</a>), <i>A. lyrata</i> has a comparatively high proportion of recent insertions ( <a href="#r18">18</a>) ( <a href="#fig02">Fig. 2 <i>B</i> </a>), possibly contributing to its larger genome size (207 Mb). The proliferation of LTR retrotransposons in <i>C. himalaica</i> ( <a href="#fig02">Fig. 2 <i>B</i> </a> and <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Fig. S4 and Table S11 </a>) peaks at ∼2.0 Mya, shortly after the dramatic elevational and climatic changes of the QTP between the late Miocene and late Pliocene ( <a href="#r1">1</a>, <a href="#r13">13</a>). The activity of TEs (including LTR retrotransposons) can lead to diverse genetic changes (e.g., chromosomal rearrangements and gene duplication, creation, and disruption) that may drive lineage-specific diversification and adaptation ( <a href="#r25">25</a>, <a href="#r26">26</a>). We therefore hypothesize that the proliferation of LTR retrotransposons likely contributed to the diversification, speciation, and adaptation of <i>C. himalaica</i>, similar to the proliferation of <i>BARE</i>-1 retrotransposons in wild barley ( <a href="#r27">27</a>) and <i>DIRS1</i> retrotransposons in Antarctic teleost ( <a href="#r28">28</a>), which might have facilitated the adaptation of plants to drier slopes and subzero temperatures, respectively. Another example from apples involves a major burst of retrotransposons ∼21.0 Mya, which coincided with the uplift of the Tianshan mountains, the postulated center of origin of apples ( <a href="#r29">29</a>). </p> </div> <div class="phylogenetictreeconstructionandestimationofdivergencetimes." title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Tree Construction and Estimation of Divergence Times.</div> <p>Previous studies suggest that <i>Crucihimalaya</i> forms an endemic genus in the Himalayas and is most closely related to <i>Arabidopsis</i> ( <a href="#r30">30</a>, <a href="#r31">31</a>). However, our earlier phylogenetic analyses of transcriptome sequences found that <i>C. himalaica</i> formed a clade sister to <i>C. rubella</i> ( <a href="#r4">4</a>). Applying OrthoMCL ( <a href="#r32">32</a>) to nine published whole-genome sequences from Brassicaceae ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S12 </a>), we identified a total of 22,670 orthogroups. Among these orthogroups, 4,586 contained putative single-copy gene families. To verify the phylogenetic position of <i>C. himalaica</i>, we generated a maximum likelihood phylogenetic tree with a trimmed and concatenated protein sequence alignment from 4,586 single-copy genes in nine species. The resulting phylogeny indicated that <i>C. himalaica</i> was most closely related to <i>C. rubella</i>, and that these two species in turn formed a clade with <i>A. lyrata</i> and <i>A. thaliana</i> ( <a href="#fig02">Fig. 2 <i>C</i> </a>), confirming our previous results ( <a href="#r4">4</a>) <sub>.</sub> The above-mentioned four genera (which are classified in the tribe Camelineae), together with the allied <i>Cardamine hirsuta</i>, were often recognized as Lineage I or Clade A in previous phylogenetic studies ( <a href="#r33">33</a>, <a href="#r34">34</a>). </p> <p> <i>C. himalaica</i> and <i>C. rubella</i> were estimated to have diverged ∼10.6 (8.8 to 12.2) Mya in our analyses using MCMCtree ( <a href="#r35">35</a>) with three calibration points ( <a href="#s9"> <i>Materials and Methods</i> </a> and <a href="#fig02">Fig. 2 <i>C</i> </a>); the two species diverged from <i>Arabidopsis</i> ∼14.6 (12.7 to 17.2) Mya and from <i>Brassica rapa</i> ∼23.6 (20.9 to 27.8) Mya. These results are in agreement with previous estimates ( <a href="#r11">11</a>, <a href="#r36">36</a>). Given the absence of genomic information for the remaining species of <i>Crucihimalaya</i>, we are unable to date the exact origin of <i>C. himalaica</i>, which was estimated previously at ∼3.56 Mya ( <a href="#r11">11</a>, <a href="#r12">12</a>). However, <i>C. himalaica</i> must have evolved after the split with <i>Capsella</i> and almost certainly less than 10.6 Mya. Such estimation is in accordance with the timing of the most recent rapid uplift of the QTP from late Miocene to late Pliocene ( <a href="#r1">1</a>, <a href="#r13">13</a>). The genus <i>Crucihimalaya</i> is assigned to the tribe Crucihimalayeae, which includes two additional genera, <i>Ladakiella</i> and <i>Transberingia</i> ( <a href="#r37">37</a>, <a href="#r38">38</a>). The tribe Crucihimalayeae is mainly distributed in Central Asia, with one species ( <i>Transberingia bursifolia</i>) extending into North America. Therefore, we speculate that the ancestor of <i>C. himalaica</i> dispersed from Central Asia into the Himalayas, and speciation of <i>C. himalaica</i> was likely triggered by the rapid uplift of the Himalayas, which resulted in the evolution of specialized phenotypic and physiological characters as adaptations to the extreme environment (discussed further below). </p> </div> <div class="genefamilyexpansionandcontraction." title="sec"> <div class="title" tagx="title" title="title">
Gene Family Expansion and Contraction.</div> <p>Significant expansion or contraction in the size of particular gene families is often associated with the adaptive divergence of closely related species ( <a href="#r39">39</a>, <a href="#r40">40</a>). Comparisons of the genomes of <i>C. himalaica</i> and four close relatives ( <a href="#fig02">Fig. 2 <i>D</i> </a>) identified a total of 151 gene families that are significantly ( <i>P</i> &amp;lt; 0.01) expanded in <i>C. himalaica</i> and 89 gene families that are significantly contracted ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S13 </a>). Based on Kyoto Encyclopedia of Genes and Genomes (KEGG) and Gene Ontology (GO) annotations, expanded gene families were highly enriched in ubiquitin-conjugating enzyme (E2) activity ( <i>P</i> = 7.17E-40) and DNA repair pathways ( <i>P</i> = 1.45E-12) ( <a href="#t02">Table 2</a>). It is notable that the most significantly contracted gene families in the <i>C. himalaica</i> genome were found to be functionally related to disease and immune responses, such as the Toll-like receptor and NF-κB signaling pathway ( <a href="#t02">Table 2</a>). The majority of bacteria interact with Toll-like receptors on the surface of the host cell membrane, stimulating the NF-κB signaling pathway in the immune response ( <a href="#r41">41</a>). The Toll and interleukin-1 receptor (TIR) is an N-terminal component of the nucleotide-binding site (NBS) disease resistance protein family, which includes the TIR-NBS-LRR (TNL) and CC-NBS-LRR (CNL) subfamilies. In the <i>C. himalaica</i> genome, both TNL and CNL subfamilies underwent severe contractions compared with their close relatives ( <a href="#fig03">Fig. 3 <i>A</i> </a>). In particular, disease resistance RPP8-like proteins (belonging to CNL) have undergone the most significant contraction in the <i>C. himalaica</i> genome (6 vs. 16∼29 members, <i>P</i> = 1.26E-10). These proteins are major players in plant defense against pathogens by triggering hypersensitive responses ( <a href="#r42">42</a>). The rapid evolutionary expansion or contraction of the NBS gene family may be a fundamental strategy for plants to adapt to the rapidly changing species-specific pathogen spectrum ( <a href="#r43">43</a>, <a href="#r44">44</a>). As fewer microorganisms exist on the QTP owing to the harsh environments characterized by cold temperatures, aridity, and high UV radiation ( <a href="#r45">45</a>), it is reasonable to presume that the reduction in number of NBS genes in the <i>C. himalaica</i> genome is due to a lighter load of pathogens present in the environment and therefore the scarcity of pathogen infection on the QTP. We performed additional searches for NBS pseudogenes, which were assumed to have frameshift mutations and/or premature stop codons. Only one pseudogene could be found in <i>A. thaliana</i> and <i>A. lyrata</i>, but 20, 15, and 7 NBS pseudogenes could be identified in the <i>C. himalaica</i>, <i>C. rubella</i>, and <i>C. hirsuta</i> genomes, respectively ( <a href="#fig03">Fig. 3 <i>B</i> </a>). Therefore, pseudogenization of NBS disease resistance genes in <i>C. himalaica</i> is at least partly responsible for the contraction of this gene family. Similar observations were made for the genome of the ground tit ( <i>Parus humilis</i>), native to the QTP, where MHC genes involved in the cellular immune defense against pathogens also show significant contractions ( <a href="#r46">46</a>). Although the mechanism of MHC gene evolution in ground tit remains unclear, it is worth noting that both ground tit and <i>C. himalaica</i> show a convergent evolution with respect to the contraction of gene families involved in defense against pathogens. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2.</span>  <p>Functional annotation of the most significantly expansive and contract gene families in <i>C. himalaica</i> </p>  <table> <thead> <tr> <td>Gene families</td> <td>KEGG terms</td> <td>Input no.</td> <td>Background no.</td> <td> <i>P</i> value </td> </tr> </thead> <tbody> <tr> <td>Expanded gene</td> <td>Ubiquitin-mediated proteolysis</td> <td>19</td> <td>137</td> <td>5.59E-29</td> </tr> <tr> <td>families</td> <td>Mismatch repair</td> <td>10</td> <td>80</td> <td>9.44E-12</td> </tr> <tr> <td /> <td>Homologous recombination</td> <td>10</td> <td>105</td> <td>9.12E-11</td> </tr> <tr> <td /> <td>DNA replication</td> <td>10</td> <td>103</td> <td>9.12E-11</td> </tr> <tr> <td /> <td>Nucleotide excision repair</td> <td>10</td> <td>137</td> <td>9.31E-10</td> </tr> <tr> <td /> <td>RNA polymerase</td> <td>4</td> <td>32</td> <td>1.59E-06</td> </tr> <tr> <td /> <td>Pyrimidine metabolism</td> <td>4</td> <td>105</td> <td>9.53E-05</td> </tr> <tr> <td /> <td>Purine metabolism</td> <td>4</td> <td>176</td> <td>5.66E-04</td> </tr> <tr> <td>Contracted gene</td> <td>NF-κB signaling pathway</td> <td>64</td> <td>93</td> <td>1.96E-106</td> </tr> <tr> <td>families</td> <td>Toll-like receptor signaling pathway</td> <td>64</td> <td>106</td> <td>5.32E-104</td> </tr> <tr> <td /> <td>Retinol metabolism</td> <td>21</td> <td>65</td> <td>3.60E-29</td> </tr> <tr> <td /> <td>Metabolism of xenobiotics by P450</td> <td>18</td> <td>73</td> <td>2.89E-23</td> </tr> <tr> <td /> <td>Tryptophan metabolism</td> <td>13</td> <td>40</td> <td>2.89E-18</td> </tr> <tr> <td /> <td>Steroid hormone biosynthesis</td> <td>13</td> <td>58</td> <td>1.73E-16</td> </tr> <tr> <td /> <td>Metabolic pathways</td> <td>31</td> <td>1,243</td> <td>1.03E-12</td> </tr> <tr> <td /> <td>Arachidonic acid metabolism</td> <td>10</td> <td>62</td> <td>1.35E-11</td> </tr> <tr> <td /> <td>Linoleic acid metabolism</td> <td>8</td> <td>29</td> <td>4.96E-11</td> </tr> <tr> <td /> <td>Lysosome</td> <td>7</td> <td>123</td> <td>1.27E-05</td> </tr> <tr> <td /> <td>Apoptosis</td> <td>4</td> <td>33</td> <td>9.75E-05</td> </tr> </tbody> </table> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3.</span> </a> </div>   <p>Size of the NBS gene family ( <i>A</i>) and number of pseudo-NBS genes ( <i>B</i>) in the <i>C. himalaica</i> genome compared with those of related species. </p>   </div> </div> <div class="positiveselectiononsingle-copygenes." title="sec"> <div class="title" tagx="title" title="title">
Positive Selection on Single-Copy Genes.</div> <p>Orthologs that show signs of positive selection usually underwent adaptive divergence ( <a href="#r47">47</a>). Previously, we observed that the ratio of nonsynonymous to synonymous substitutions (dN/dS or ω) in <i>C. himalaica</i> is higher than those of closely related species, suggestive of accelerated evolution in <i>C. himalaica</i> after divergence from its ancestral lineage ( <a href="#r4">4</a>). In the present study, we conducted a positive selection analysis using the genomic sequences of <i>C. himalaica</i> and four close relatives. Among the 21,383 orthogroups, 11,085 contained single-copy orthologous genes. We used the branch-site model of the PAML 4 package ( <a href="#r35">35</a>) to identify genes with signs of positive selection. As a result, 844 genes possibly under positive selection were identified in the <i>C. himalaica</i> genome (ω &amp;gt; 1, <i>P</i> &amp;lt; 0.05). Of these genes, 610 showed highly significant ( <i>P</i> &amp;lt; 0.01) positive selection ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S14 </a>). A KEGG functional classification of the 610 significant PSGs in the <i>C. himalaica</i> genome ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S15 </a>) showed that several categories associated with DNA repair, the ubiquitin system, as well as plant hormone biosynthesis and signal transduction were enriched. Genes involved in DNA repair were also identified as being under positive selection pressure in a previous transcriptomic study of <i>C. himalaica</i> ( <a href="#r4">4</a>). It is notable that significantly expanded gene families and PSGs were both enriched in DNA repair and protein ubiquitination pathways. Signal transduction-related CheY-like genes were also found to have undergone both significant positive selection and expansion events in our previous study on the adaptive evolution of the cyanobacterium <i>Trichormus</i> sp. NMC-1 on the QTP ( <a href="#r3">3</a>), again providing evidence that expansion/duplication and (subsequent) positive selection of genes are an important mechanism for plant adaptive evolution. </p> <p>The extremely intense UV radiation on the QTP may influence plant growth and developmental processes such as photoperiodism and flowering, or cause DNA, RNA, and protein damage ( <a href="#r48">48</a>). Our results showed that 10 PSGs in the <i>C. himalaica</i> genome are involved in the DNA repair pathway ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S15 </a>). Notably, several genes in the nucleotide excision repair pathway showed signatures of positive selection. One of them encodes the DNA excision repair protein ERCC-1 (or UV hypersensitive 7), which was also identified previously ( <a href="#r4">4</a>) and is reported to function as a component of a structure-specific endonuclease that cleaves on the 5ʹ side of UV photoproducts in DNA ( <a href="#r49">49</a>). In addition to ERCC-1, we identified PSGs that encode the AP endonuclease 2 protein (APEX2), DNA polymerase delta subunit 2 (POLD2), DNA mismatch repair protein MLH1, and DNA repair and recombination protein RAD54 ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S14 </a>). These PSGs are involved in base excision repair, mismatch excision repair, and homologous recombination, suggesting that <i>C. himalaica</i> has evolved an integrated DNA-repair mechanism to adapt to the harsh habitats caused by the uplift of the QTP. Moreover, high UV-B radiation is a common stress that both animals and plants on the QTP must cope with. The DNA repair and radiation responses pathways similarly have played crucial roles in the highland adaptation of the Tibetan highland barley ( <a href="#r6">6</a>), Tibetan antelope ( <a href="#r50">50</a>), Tibetan chicken ( <a href="#r51">51</a>), and Tibetan hot-spring snake ( <a href="#r52">52</a>). </p> <p>Our analyses also indicated that ubiquitin system-related gene families underwent significant expansion and natural selection. Ubiquitin-mediated proteolysis impacts almost every aspect of plant growth and development, including plant hormone signal transduction, photomorphogenesis, reproduction, and abiotic stress responses ( <a href="#r53">53</a>). Such a multifunctional biological process is of even greater importance for plants in the complex harsh environment on the QTP. Among the seven PSGs in the ubiquitin system based on the KEGG annotation, for instance, one encodes an ortholog of the COP10 protein in <i>A. thaliana</i>, which can enhance the activity of ubiquitin-conjugating enzymes (E2s) ( <a href="#r54">54</a>) and participates in light signal transduction and photomorphogenesis ( <a href="#r55">55</a>). An additional PSG encodes the ortholog of the auxin transport protein BIG (E3 ubiquitin-protein ligase UBR4) in <i>A. thaliana</i>, which not only influences multiple auxin-mediated developmental processes (e.g., lateral root production and inflorescence architecture) but also plays a critical role in a multitude of light and phytohormone pathways ( <a href="#r56">56</a>). These PSGs enriched in the ubiquitin system might be important for <i>Crucihimalaya</i> to better survive the harsh environment in the QTP. Similarly, in three mangrove species, the ubiquitin-mediated proteolysis pathway includes 12 genes that have experienced convergent evolution at conservative sites, presumably important for adaptation of mangroves to the harsh coastal habitat ( <a href="#r57">57</a>). </p> <p>Moreover, many of the identified PSGs were associated with reproduction pathways (92 PSGs, <i>P</i> = 5.40E-06) based on GO annotation ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S16 </a>). One of the PSGs encodes the phytochrome and flowering time regulatory protein 1 (PFT1), which represses the PhyB-mediated light signaling and regulates the expression of FLOWERING LOCUS T (FT) and CONSTANS (CO) ( <a href="#r58">58</a>, <a href="#r59">59</a>). In a low-red to far-red ratio environment, plants require CO and FT to fully accelerate flowering in long days ( <a href="#r58">58</a>, <a href="#r59">59</a>). PFT1 acts as a hub to integrate a variety of interdependent environmental stimuli, including light quality and jasmonic acid-dependent defense responses. An additional gene encodes TERMINAL FLOWER 1 (TFL1), which controls inflorescence meristem identity and regulates flowering time in the long-day flowering pathway. As the QTP experiences a long sunshine duration and a short vegetation growing season, flowering time is particularly critical and affects both the life cycles and reproductive success of many alpine plants ( <a href="#r60">60</a>). In the wild, <i>C. himalaica</i> initiates flowering relatively early (April). However, this species rarely blossoms in low-altitude areas outside the QTP. These observations suggest that <i>C. himalaica</i> has evolved specific reproductive strategies on photoperiodism and flowering-related pathways as an adaptation to the long sunshine duration and short growing season on the QTP. </p> </div> <div class="s-locusstructureandself-fertilizationofc.himalaica." title="sec"> <div class="title" tagx="title" title="title">
S-Locus Structure and Self-Fertilization of <i>C. himalaica.</i> </div> <p>Self-incompatibility in Brassicaceae is controlled by the S-locus recognition system, which prevents self-fertilization ( <a href="#r61">61</a>, <a href="#r62">62</a>). Whereas self-fertilization often leads to decreased fitness of homozygous offspring, it also ensures reproduction in absence of pollinators or suitable mates, and therefore can be advantageous for plants to expand their distribution edges and occupy new niches ( <a href="#r63">63</a>, <a href="#r64">64</a>). The QTP is characterized by harsh conditions including a short growing season and low pollinator activities. Therefore, it is not surprising that <i>C. himalaica</i> is a selfing plant ( <a href="#r9">9</a>). The <i>Arabidopsis</i>/ <i>Capsella</i>-like S-locus contains the female specificity gene (S-receptor kinase, SRK) and the male specificity gene (S-locus cysteine-rich protein, SCR) ( <a href="#r65">65</a>). Self-compatibility is achieved when the SRK receptor fails to recognize the SCR ligand, usually through a loss of SCR function ( <a href="#r66">66</a> <a href="#r67" /> <a href="#r68" />– <a href="#r69">69</a>). We manually annotated the S-locus flanking genes ( <i>AKR3</i> and <i>U-box</i>) and the <i>SCR</i> gene on scaffold 29 of the <i>C. himalaica</i> genome assembly. Alignment of <i>Crucihimalaya SCR</i> coding sequences with homologs from miscellaneous <i>Arabidopsis halleri</i>, <i>A. lyrata</i>, <i>A. thaliana</i>, <i>Capsella grandiflora</i>, <i>C. rubella</i>, and <i>C. hirsuta</i> S-locus haplotypes showed that the closest haplotype to <i>C. himalaica</i> was the one from <i>A. halleri S15</i> ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Fig. S5 </a>) ( <a href="#r66">66</a>, <a href="#r68">68</a>, <a href="#r70">70</a>, <a href="#r71">71</a>). Note that, due to strong balancing selection on the S-locus, highly diverged S-alleles are shared across species and <i>SCR</i> gene phylogeny does not follow species phylogeny ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Fig. S5 </a>) ( <a href="#r72">72</a>, <a href="#r73">73</a>). SCR protein sequence alignment showed that <i>C. himalaica</i> appears to have lost two out of eight conserved cysteines ( <a href="#fig04">Fig. 4 <i>A</i> </a>) that are essential for structural and functional integrity of SCR ligand ( <a href="#r66">66</a>, <a href="#r74">74</a>). This alone may explain the transition to self-fertilization in <i>C. himalaica.</i> Interestingly, neither <i>SRK</i> from the S15 haplogroup nor any other <i>SRK</i> had a BLAST hit to the <i>C. himalaica</i> genome ( <a href="#fig04">Fig. 4 <i>B</i> </a>). To investigate whether SRK is indeed lost from the genome, we mapped short reads (450-bp insert size library) of <i>C. himalaica</i> to the combination of the <i>C. himalaica</i> genome assembly and <i>A. halleri</i> S15 haplogroup of S-locus. The <i>SRK</i> gene region did not have any reads mapped, and therefore we conclude that <i>SRK</i> is not present in the <i>C. himalaica</i> genome and likely has been lost. Taken together, we reason that the transition to self-compatibility of <i>C. himalaica</i> was accomplished by both ( <i>i</i>) loss of function in the male recognition gene <i>SCR</i> and ( <i>ii</i>) loss of the female recognition gene <i>SRK</i>. It is often suggested that self-fertilization rates of alpine plants increase at higher altitudes ( <a href="#r75">75</a>), due to the reduction in pollinator abundance and seed production in alpine plants ( <a href="#r76">76</a>, <a href="#r77">77</a>). This suggestion is consistent with the fact that self-fertilizing populations also exist in <i>Arabis alpina,</i> a close relative of <i>C. himalaica</i> that grows at high altitudes ( <a href="#r78">78</a>, <a href="#r79">79</a>). Moreover, previous studies also found that the rate of self-fertilizing hermaphroditic plants is increasing on the QTP and autonomous self-fertilization provides substantial reproductive assurance in the pollinator scarcity condition of the QTP ( <a href="#r80">80</a>, <a href="#r81">81</a>). Therefore, the transition to a self-compatibility mating system in <i>C. himalaica</i> likely facilitated its occupation of the QTP. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4.</span> </a> </div>   <p>Transitioning to self-fertilization in <i>C. himalaica</i>. ( <i>A</i>) Protein sequence alignment of S-locus <i>SCR</i> genes from <i>C. himalaica</i>, <i>A. halleri</i>, and <i>A. lyrata</i>. Cysteine residues are highlighted in red, showing eight conserved sites important for structural and functional integrity of the protein ( <a href="#r66">66</a>, <a href="#r74">74</a>). The <i>C. himalaica</i> SCR protein appears to have lost two out of eight conserved cysteines and therefore is probably nonfunctional. ( <i>B</i>) Structure of the S-locus part of scaffold 29 in <i>C. himalaica</i> genome assembly compared with <i>A. halleri</i> S15 haplogroup ( <a href="#r70">70</a>). Colors of the lines between S-loci correspond to BLAST scores from highest (blue) to lowest (gray). </p>   </div> </div> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Organisms that live on the QTP face a variety of abiotic stresses under the harsh environmental conditions and presumably have been subjected to a series of adaptive evolutionary changes. In the present study, we de novo-sequenced the genome of <i>C. himalaica</i>, a species exclusively found in the Himalayan region. Based on phylogenetic reconstruction and estimated divergence times, we propose that the speciation of <i>C. himalaica</i> was triggered by the uplift of the QTP, mediated by genomic evolution to adapt to the altered environment. The proliferation of LTR retrotransposons may at least partly be responsible for the increased genome size of <i>C. himalaica</i>. The significant contraction and pseudogenization of NBS gene families reflect the strong reduction in pathogen incidence on the QTP. Gene families that underwent significant expansion and genes that show signs of positive selection are enriched in DNA repair and protein ubiquitination pathways, which probably reflect to the adaptation of <i>C. himalaica</i> to high radiation, low temperature, and pathogen-depauperate environments on the QTP. Occupation of the QTP by <i>C. himalaica</i> was likely facilitated by self-compatibility, which, as we have shown here, involved both male and female components of the recognition system. Both similarities (e.g., DNA repair and disease-resistance pathways) and differences (e.g., reproduction and plant hormone-related pathways) in adaptive mechanisms have been identified among plants and animals that grow at high altitudes. Although further experimental verification is needed, our results provide insights into how plants adapt to harsh and extreme environments. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterial,genomesequencing,andassembly." title="sec"> <div class="title" tagx="title" title="title">
Plant Material, Genome Sequencing, and Assembly.</div> <p>Seedlings of <i>C. himalaica</i> were sampled from Batang County (altitude 4,010 m, N 30.313°, E 99.358°) of QTP. Seedlings from the same individual were cultivated in the greenhouse at Kunming Institute of Botany. High-quality genomic DNA was extracted using the Qiagen DNeasy Plant Mini Kit. The <i>C. himalaica</i> genome assembly was performed using sequence data obtained from a combination of sequencing technologies: Illumina PE reads, Illumina MP reads, and PacBio RS II reads ( <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental"> <i>SI Appendix</i>, Table S2 </a>). The whole step of library construction and sequencing was performed at Novogene Bioinformatics Technology Co., Ltd. First, six PE libraries were prepared to sequence the <i>C. himalaica</i> genome. These included two PE libraries with insert sizes of 250 and 450 bp and four MP libraries with insert sizes of 2, 5, 10 and 15 kb. Whole genomic sequence (44.49 Gb) data were generated solely using Illumina platforms (HiSEq. 2500) and assembled using ALLPATHS -LG ( <a href="#r15">15</a>). Next, the PacBio reads (6.19 Gb) were used to fill gaps using the PBJelly2 tool ( <a href="#r16">16</a>), followed by scaffold assembly with SSPACE ( <a href="#r82">82</a>) using long-insert-size PE reads. The accuracy and completeness of the assemblies were assessed by aligning the reads from short-insert-size libraries back to the scaffolds using BWA-mem (v. 0.7.17) ( <a href="#r83">83</a>). </p> </div> <div class="genepredictionandannotation." title="sec"> <div class="title" tagx="title" title="title">
Gene Prediction and Annotation.</div> <p>Gene prediction was performed using a combination of homology, de novo, and transcriptome-based approaches. Gene models were integrated by EvidenceModeler ( <a href="http://evidencemodeler.github.io/">evidencemodeler.github.io/</a>). Gene models were further updated by PASA ( <a href="#r84">84</a>) to generate UTRs and provide information on alternative splicing variants. The predicted genes were analyzed for functional domains and homologs using InterProScan and BLAST against the NCBI nonredundant protein sequence database, TrEMBL and SwissProt with an E-value cutoff 1E-15 and Blast2GO with default parameters. Completeness of the genome was also assessed by performing core gene annotation using the BUSCO ( <a href="#r17">17</a>) methods. Transcription factors were identified and classified into different families using the iTAK pipeline ( <a href="http://bioinfo.bti.cornell.edu/cgi-bin/itak/index.cgi">bioinfo.bti.cornell.edu/cgi-bin/itak/index.cgi</a>) ( <a href="#r85">85</a>). </p> </div> <div class="repetitiveelementsidentificationanddynamicanalysis." title="sec"> <div class="title" tagx="title" title="title">
Repetitive Elements Identification and Dynamic Analysis.</div> <p>A combined strategy based on homologous sequence alignment and de novo searches was used to identify repeat elements in the <i>C. himalaica</i> genome. We de novo predicted TEs using RepeatModeler ( <a href="http://www.repeatmasker.org/RepeatModeler">www.repeatmasker.org/RepeatModeler</a>), RepeatScout ( <a href="http://www.repeatmasker.org/">www.repeatmasker.org</a>), Piler ( <a href="#r86">86</a>) ( <a href="http://www.drive5.com/piler/">www.drive5.com/piler/</a>), and LTR-Finder ( <a href="#r87">87</a>) ( <a href="http://tlife.fudan.edu.cn/tlife/ltr_finder">tlife.fudan.edu.cn/tlife/ltr_finder</a>) with default parameters. For alignment of homologous sequences to identify repeats in the assembled genome, we used RepeatProteinMask and RepeatMasker ( <a href="http://www.repeatmasker.org/">www.repeatmasker.org</a>) with the repbase library ( <a href="#r88">88</a>). TEs overlapping with the same type of repeats were integrated, while those with low scores were removed if they overlapped over 80% of their lengths and belonged to different types. </p> <p>Intact LTR retrotransposons were identified by searching the genomes of <i>C. himalaica</i> with LTRharvest ( <a href="#r89">89</a>) (-motif tgca -motifmis 1) and LTR_Finder (LTR length 100 to 5,000 nt; length between two LTRs: 1,000 to 2,0000 nt). We combined results from both analyses after integrating the overlap sites. Candidate sequences were filtered by a two-step procedure to reduce false positives. First, LTRdigest ( <a href="#r90">90</a>) was used to identify the primer binding site (PBS) motif, and only elements containing PBS were retained; then, protein domains (pol, gag, and env) in candidate LTR retrotransposons were identified by searching against HMM profiles considered as intact. Second, clustering of candidate LTR retrotransposons was performed using the SiLiX software package ( <a href="http://lbbe.univ-lyon1.fr/SiLiX">lbbe.univ-lyon1.fr/SiLiX</a>). Using a substitution rate (r) of 7*10 <sup>−9</sup> substitutions per site per year ( <a href="#r91">91</a>, <a href="#r92">92</a>), the insertion date (T) can be calculated for each LTR retrotransposons (T = K/2r, K: genetic distance). Tandemly repeated gene arrays were identified using BLASTP with a threshold E value &amp;lt; 10 <sup>−8</sup>; fragmental alignments were conjoined for each gene pair. The tandem repeat gene clusters were identified using a cutoff of sequence identity &amp;gt;70% and distance &amp;lt;150 kb. Each cluster with more than two genes was retained. </p> </div> <div class="whole-genomealignmentandduplicationanalysis." title="sec"> <div class="title" tagx="title" title="title">
Whole-Genome Alignment and Duplication Analysis.</div> <p>We aligned the <i>C. himalaica</i> genome to those of <i>A. thaliana, A. lyrata</i>, and <i>C. rubella</i> using LASTZ ( <a href="#r93">93</a>) with the following parameter values: M = 254 K = 4500 L = 3000 Y = 15000 -seed = match 12 -step = 20 -identity = 85. To avoid the interference caused by repetitive sequences for sequence alignment, RepeatMasker and RepBase library were used to mask repetitive sequences of the above four genomes. The raw alignments were combined into larger blocks using the ChainNet algorithm. We identified orthologous genes among the <i>C. himalaica</i>, <i>A. thaliana</i>, <i>A. lyrata</i>, and <i>C. rubella</i> genomes and paralogous genes within <i>C. himalaica</i> and other relatives using BLASTP (E value &amp;lt; 1E-7). MCscanx ( <a href="#r94">94</a>) was used to identify syntenic blocks within the genome. For each gene pair in a syntenic block, the 4DTv distance was calculated; values of all gene pairs were plotted to identify putative whole-genome duplication events and divergence in two species. </p> </div> <div class="phylogenetictreeconstructionandestimationofspeciesdivergencetimes." title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Tree Construction and Estimation of Species Divergence Times.</div> <p>We selected genomes of <i>C. himalaica</i> and eight other species ( <i>A. thaliana</i>, <i>A. lyrata</i>, <i>C. rubella</i>, <i>C. hirsuta</i>, <i>Eutrema salsugineum</i>, <i>B. rapa</i>, <i>Schrenkiella parvula</i>, and <i>Aethionema arabicum</i>) to identify orthologs. To remove redundancy caused by alternative splicing variations, we retained only gene models at each gene locus that encoded the longest protein sequence. To exclude putative fragmented genes, genes encoding protein sequences shorter than 50 aa were filtered out. All filtered protein sequences of these plants were compared with each other using BLASTP (E value &amp;lt; 1E-7) and clustered into orthologous groups using OrthoMCL (inflation parameter, 1.5) ( <a href="#r33">33</a>). Protein sequences from 4,586 single-copy gene families were used for phylogenetic tree construction. MUSCLE ( <a href="#r95">95</a>) was used to generate multiple sequence alignment for protein sequences in each single-copy family with default parameters. The alignments of each family were concatenated to a super alignment matrix, which was then used for phylogenetic tree reconstruction through the PROTCATJTT model in RAxML software. </p> <p>Divergence time between nine species was estimated using MCMCtree in PAML ( <a href="#r35">35</a>) with the options "independent rates" and "GTR" model. A Markov chain Monte Carlo analysis was run for 10,000 generations, using a burn-in of 1,000 iterations. Three calibration points were applied based on a recent study using 53 plastomes of Brassicales: <i>A. arabicum</i> and other crucifers divergence time (29.0 to 41.8 Mya), core Brassicaceae origination time (21.3 to 29.8 Mya), and core <i>Arabidopsis</i> origination time (4.8 to 9.7 Mya) ( <a href="#r36">36</a>). </p> </div> <div class="genefamilyexpansionandcontraction." title="sec"> <div class="title" tagx="title" title="title">
Gene Family Expansion and Contraction.</div> <p>We selected four close relatives ( <i>A. thaliana</i>, <i>A</i>. <i>lyrata</i>, <i>C. rubella</i>, and <i>C. hirsuta</i>) of <i>C. himalaica</i> to identify orthogroups (gene families). Expansions and contractions of orthologous gene families were determined using CAFÉ v. 4.1 ( <a href="#r96">96</a>). The program uses a birth and death process to model gene gain and loss over a phylogeny. For each significantly expanded and contracted gene family in <i>C. himalaica</i>, functional information was inferred based on its ortholog in <i>A. thaliana</i>. GO enrichment analyses of genes were conducted using web-based agriGO ( <a href="systemsbiology.cau.edu.cn/agriGOv2">systemsbiology.cau.edu.cn/agriGOv2</a>) ( <a href="#r97">97</a>) with the singular enrichment analysis method and TAIR10 database. The KOBAS and BlastKOALA software ( <a href="#r98">98</a>) was also used to test the statistical enrichment of genes in KEGG pathways ( <a href="#r99">99</a>). </p> </div> <div class="identificationandclassificationofputativenbsresistancegenes." title="sec"> <div class="title" tagx="title" title="title">
Identification and Classification of Putative NBS Resistance Genes.</div> <p>A hidden Markov model search ( <a href="http://hmmer.janelia.org/">hmmer.janelia.org</a>) was used to identify NBS-encoding R genes in the <i>C. himalaica</i> genome and four close relatives ( <i>A. thaliana</i>, <i>A. lyrata</i>, <i>C. rubella</i>, and <i>C. hirsuta</i>). The sequences were screened using HMMs to search for the Pfam NBS (NB-ARC) family PF00931 domain ( <a href="http://pfam.xfam.org/">pfam.xfam.org/</a>). Pfam HMM searches were performed using a TIR model (PF01582) and several LRR models (PF00560, PF07723, PF07725, PF12799, PF13306, PF13516, PF13504, PF13855, and PF08263) to detect TIR domains and LRR motifs in the NBS-encoding amino acid sequences. CC motifs were detected using the COILS prediction program 2.2 ( <a href="http://embnet.vital-it.ch/software/COILS_form.html">embnet.vital-it.ch/software/COILS_form.html</a>) with a <i>p</i>-score cutoff of 0.9. </p> </div> <div class="genefamily-basedpositiveselectiontests." title="sec"> <div class="title" tagx="title" title="title">
Gene Family-Based Positive Selection Tests.</div> <p>For positive selection analyses we also selected the four most closely related species with assembled genomes ( <i>A. thaliana</i>, <i>A</i>. <i>lyrata</i>, <i>C. rubella</i>, and <i>C. hirsuta</i>) and <i>C. himalaica</i> to identify orthologs based on our phylogenetic tree. The analysis procedure is similar to the phylogenetic tree analysis. First, we used OrthoMCL ( <a href="#r32">32</a>) to identify homologous gene clusters (orthogroups) among the five genomes. OrthoMCL was run with an E-value cutoff of 1E-15 and an inflation parameter of 1.5 due to the close genetic relationship between five relatives. Orthogroups with single-copy genes shared by all five genomes were retained for further analyses. Multiple sequence alignment was performed for each orthogroup using MUSCLE v. 3.8.31 ( <a href="#r97">97</a>) with default parameters. Poorly aligned regions were further trimmed using the trimAl v. 1.4 software ( <a href="#r100">100</a>) with the parameter "-gt 0.8 -st 0.001". Maximum likelihood trees were generated using RAxML v7.0.4 ( <a href="#r101">101</a>) with the PROTCATJTT model. To calculate the nonsynonymous ( <i>K</i> <sub>a</sub>) and synonymous (Ks) substitution rates between pairs of orthogroups, we reverse-translated amino acid alignments to the corresponding codon-based nucleotide alignments using PAL2NAL ( <a href="#r102">102</a>). To increase the power of tests for positive selection, we applied the branch-site model ( <a href="#r103">103</a>) implemented in codeml program ( <a href="#r35">35</a>) to estimate the dN/dS substitution rates (ω value). We also deleted all gaps (clean data = 1) from the alignments to lower the effect of ambiguous bases on the inference of positive selection. A foreground branch was specified as the clade of <i>C. himalaica</i>. A likelihood ratio test was conducted to determine whether positive selection is operating in the foreground branch. In this study, PSGs were inferred if the <i>P</i> value was less than 0.01. The functional annotation of PSGs in <i>C. himalaica</i> was also conducted using the same approach and the same <i>P-</i>value cutoff with the gene family expansion and contraction analysis. </p> </div> <div class="s-locusstructureandself-fertilizationofc.himalaica." title="sec"> <div class="title" tagx="title" title="title">
S-Locus Structure and Self-Fertilization of <i>C. himalaica.</i> </div> <p>Known S-haplogroups from <i>Arabidopsis</i> and <i>Capsella</i> were used as query to search the genome assembly of <i>C. himalaica</i> ( <a href="#r70">70</a>, <a href="#r104">104</a>) using BLASTP. We manually annotated both flanking genes of the S-locus on scaffold 29: the <i>ARK3</i> gene and <i>U-box</i> gene. However, no homologs were detected in our search for the <i>SRK</i> gene. Using the standard BWA-mem (v. 0.7.17) ( <a href="#r83">83</a>) and Samtools (v. 1.6) ( <a href="#r105">105</a>) pipeline, we mapped short reads (450-bp insert size library) of <i>C. himalaica</i> to the combined reference of <i>C. himalaica</i> genome and <i>A. halleri</i> S15 haplogroup of S-locus ( <a href="#r70">70</a>) and then manually explored the coverage of the <i>SRK</i> region of the S15 haplogroup using IGV (v. 2.4.13) ( <a href="#r106">106</a>). The coding sequence of the predicted <i>SCR</i> gene (ID = AT4G22105.1_Ath-D2 on scaffold29:1034290-1034749), together with SCR sequences from <i>A. thaliana</i>, <i>C. rubella</i>, <i>C. grandiflora</i>, and <i>C. hirsuta,</i> was added to the curated alignment of the publicly available SCR genes kindly provided by Vincent Castric (Unité Evo-Eco-Paléo, CNRS/Université de Lille 1, Villeneuve d’Ascq, France) from <i>A. halleri</i> and <i>A. lyrata</i> using MUSCLE (v. 3.8.31) ( <a href="#r97">97</a>), and a maximum likelihood tree with 500 bootstrap replicates was constructed using MEGAX ( <a href="#r107">107</a>) and visualized using Jalview2 ( <a href="#r108">108</a>). A comparative structure plot of <i>C. himalaica</i> and <i>A. halleri</i> S15 S-loci ( <a href="#fig04">Fig. 4 <i>B</i> </a>) was constructed using the R library genoPlotR ( <a href="#r109">109</a>). </p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We dearly cherish the memory of our respected mentor and friend Prof. Yang Zhong at Fudan University for his full support to this project. We thank Takashi Tsuchimatsu and Vincent Castric for their feedback on the S-locus part. This work was supported by National Natural Science Foundation of China Grants 31770408, 31590823, 31760082, 31760127, U1802232, and 91131901 (to T.Z., H.S., Q.Q., and L.Q.), National Key R &amp;amp; D Program of China Grant 2017YF0505200 (to H.S.), the Strategic Priority Research Program of Chinese Academy of Sciences Grant XDA 20050203 (to H.S.), National High Technology Research and Development Program of China Grant 2014AA020528 (to T.Z.), the Chinese Academy of Sciences "Light of West China" Program (J.H.), and European Union Seventh Framework Programme Grant FP7/2007-2013 under European Research Council Advanced Grant Agreement 322739 – DOUBLEUP (to Y.V.d.P.). P.Y.N. is a postdoctoral fellow of the Research Foundation–Flanders (12S9618N). </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors declare no conflict of interest.</p> </div> <div class="fn-type-other" title="other"> <p>Data deposition: The raw genomic reads generated in this study have been deposited in the NCBI Sequence Read Archive (BioProject <a href="PRJNA521295">PRJNA521295</a>). </p> </div> <div class="fn-type-supplementary-material" title="supplementary-material"> <p>This article contains supporting information online at <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental">www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817580116/-/DCSupplemental</a>. </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="r1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xing</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ree</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Uplift-driven diversification in the Hengduan Mountains, a temperate biodiversity hotspot </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">114</span> <span class="fpage" tagx="fpage" title="fpage">E3444</span> <span class="lpage" tagx="lpage" title="lpage">E3451</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28373546">28373546</a> </span> </span> </li> <li tag="ref"> <a name="r2" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">XD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">BW</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Influence of the Tibetan Plateau uplift on the Asian monsoon-arid environment evolution</span> <span class="source" tagx="source" title="source">Chin Sci Bull</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">4277</span> <span class="lpage" tagx="lpage" title="lpage">4291</span> </span> </li> <li tag="ref"> <a name="r3" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">The genome and transcriptome of <i>Trichormus</i> sp. NMC-1: Insights into adaptation to extreme environments on the Qinghai-Tibet Plateau </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">29404</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27381465">27381465</a> </span> </span> </li> <li tag="ref"> <a name="r4" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome sequencing of <i>Crucihimalaya himalaica</i> (Brassicaceae) reveals how <i>Arabidopsis</i> close relative adapt to the Qinghai-Tibet Plateau </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">21729</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26906946">26906946</a> </span> </span> </li> <li tag="ref"> <a name="r5" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simonson</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Altitude adaptation: A glimpse through various lenses</span> <span class="source" tagx="source" title="source">High Alt Med Biol</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">125</span> <span class="lpage" tagx="lpage" title="lpage">137</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26070057">26070057</a> </span> </span> </li> <li tag="ref"> <a name="r6" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">The draft genome of Tibetan hulless barley reveals adaptive patterns to the high stressful Tibetan Plateau </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">112</span> <span class="fpage" tagx="fpage" title="fpage">1095</span> <span class="lpage" tagx="lpage" title="lpage">1100</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25583503">25583503</a> </span> </span> </li> <li tag="ref"> <a name="r7" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koenig</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Beyond the thale: Comparative genomics and genetics of <i>Arabidopsis</i> relatives </span> <span class="source" tagx="source" title="source">Nat Rev Genet</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">285</span> <span class="lpage" tagx="lpage" title="lpage">298</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25854181">25854181</a> </span> </span> </li> <li tag="ref"> <a name="r8" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitchell-Olds</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis thaliana</i> and its wild relatives: A model system for ecology and evolution </span> <span class="source" tagx="source" title="source">Trends Ecol Evol</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">693</span> <span class="lpage" tagx="lpage" title="lpage">700</span> </span> </li> <li tag="ref"> <a name="r9" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">AE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fiebig</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preuss</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Beyond the <i>Arabidopsis</i> genome: Opportunities for comparative genomics </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">129</span> <span class="fpage" tagx="fpage" title="fpage">1439</span> <span class="lpage" tagx="lpage" title="lpage">1447</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12177458">12177458</a> </span> </span> </li> <li tag="ref"> <a name="r10" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Shehbaz</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Kane</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Generic placement of species excluded from <i>Arabidopsis</i> (Brassicaceae) </span> <span class="source" tagx="source" title="source">Novon</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">296</span> <span class="lpage" tagx="lpage" title="lpage">307</span> </span> </li> <li tag="ref"> <a name="r11" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohmann</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lysak</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">A time-calibrated road map of Brassicaceae species radiation and evolutionary history</span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">2770</span> <span class="lpage" tagx="lpage" title="lpage">2784</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26410304">26410304</a> </span> </span> </li> <li tag="ref"> <a name="r12" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beilstein</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagalingum</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clements</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manchester</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mathews</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Dated molecular phylogenies indicate a Miocene origin for <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="fpage" tagx="fpage" title="fpage">18724</span> <span class="lpage" tagx="lpage" title="lpage">18728</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20921408">20921408</a> </span> </span> </li> <li tag="ref"> <a name="r13" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Uplift of the Tibetan Plateau and environmental changes</span> <span class="source" tagx="source" title="source">Chin Sci Bull</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">2117</span> <span class="lpage" tagx="lpage" title="lpage">2124</span> </span> </li> <li tag="ref"> <a name="r14" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Estimation of genomic characteristics by analyzing k-mer frequency in de novo genome projects </span> <span class="source" tagx="source" title="source">Quant Biol</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">62</span> <span class="lpage" tagx="lpage" title="lpage">67</span> </span> </li> <li tag="ref"> <a name="r15" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butler</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">ALLPATHS: De novo assembly of whole-genome shotgun microreads</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">810</span> <span class="lpage" tagx="lpage" title="lpage">820</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18340039">18340039</a> </span> </span> </li> <li tag="ref"> <a name="r16" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">English</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Mind the gap: Upgrading genomes with Pacific biosciences RS long-read sequencing technology</span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">e47768</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23185243">23185243</a> </span> </span> </li> <li tag="ref"> <a name="r17" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simão</span> <span class="given-names" tagx="given-names" title="given-names">FA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waterhouse</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ioannidis</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kriventseva</span> <span class="given-names" tagx="given-names" title="given-names">EV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zdobnov</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">3210</span> <span class="lpage" tagx="lpage" title="lpage">3212</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717">26059717</a> </span> </span> </li> <li tag="ref"> <a name="r18" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Slotte</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">The <i>Capsella rubella</i> genome and the genomic consequences of rapid mating system evolution </span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">831</span> <span class="lpage" tagx="lpage" title="lpage">835</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23749190">23749190</a> </span> </span> </li> <li tag="ref"> <a name="r19" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="collab'">collab: Arabidopsis Genome Initiative</span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Analysis of the genome sequence of the flowering plant <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">408</span> <span class="fpage" tagx="fpage" title="fpage">796</span> <span class="lpage" tagx="lpage" title="lpage">815</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711">11130711</a> </span> </span> </li> <li tag="ref"> <a name="r20" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Long</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Massive genomic variation and strong selection in <i>Arabidopsis thaliana</i> lines from Sweden </span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">884</span> <span class="lpage" tagx="lpage" title="lpage">890</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23793030">23793030</a> </span> </span> </li> <li tag="ref"> <a name="r21" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van de Peer</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maere</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">The evolutionary significance of ancient genome duplications</span> <span class="source" tagx="source" title="source">Nat Rev Genet</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">725</span> <span class="lpage" tagx="lpage" title="lpage">732</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19652647">19652647</a> </span> </span> </li> <li tag="ref"> <a name="r22" /> <span class="label" tagx="label" title="label">22</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bennetzen</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Mechanisms and rates of genome expansion and contraction in flowering plants</span> <span class="source" tagx="source" title="source">Genetica</span> <span class="volume" tagx="volume" title="volume">115</span> <span class="fpage" tagx="fpage" title="fpage">29</span> <span class="lpage" tagx="lpage" title="lpage">36</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12188046">12188046</a> </span> </span> </li> <li tag="ref"> <a name="r23" /> <span class="label" tagx="label" title="label">23</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iorizzo</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">A high-quality carrot genome assembly provides new insights into carotenoid accumulation and asterid genome evolution </span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">48</span> <span class="fpage" tagx="fpage" title="fpage">657</span> <span class="lpage" tagx="lpage" title="lpage">666</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27158781">27158781</a> </span> </span> </li> <li tag="ref"> <a name="r24" /> <span class="label" tagx="label" title="label">24</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">TT</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis lyrata</i> genome sequence and the basis of rapid genome size change </span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">476</span> <span class="lpage" tagx="lpage" title="lpage">481</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21478890">21478890</a> </span> </span> </li> <li tag="ref"> <a name="r25" /> <span class="label" tagx="label" title="label">25</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Warren</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Evolutionary impact of transposable elements on genomic diversity and lineage-specific innovation in vertebrates </span> <span class="source" tagx="source" title="source">Chromosome Res</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">505</span> <span class="lpage" tagx="lpage" title="lpage">531</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26395902">26395902</a> </span> </span> </li> <li tag="ref"> <a name="r26" /> <span class="label" tagx="label" title="label">26</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliver</span> <span class="given-names" tagx="given-names" title="given-names">KR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McComb</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greene</span> <span class="given-names" tagx="given-names" title="given-names">WK</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Transposable elements: Powerful contributors to angiosperm evolution and diversity</span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1886</span> <span class="lpage" tagx="lpage" title="lpage">1901</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24065734">24065734</a> </span> </span> </li> <li tag="ref"> <a name="r27" /> <span class="label" tagx="label" title="label">27</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalendar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanskanen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Immonen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nevo</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schulman</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Genome evolution of wild barley ( <i>Hordeum spontaneum</i>) by BARE-1 retrotransposon dynamics in response to sharp microclimatic divergence </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">97</span> <span class="fpage" tagx="fpage" title="fpage">6603</span> <span class="lpage" tagx="lpage" title="lpage">6607</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10823912">10823912</a> </span> </span> </li> <li tag="ref"> <a name="r28" /> <span class="label" tagx="label" title="label">28</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Auvinet</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Mobilization of retrotransposons as a cause of chromosomal diversification and rapid speciation: The case for the Antarctic teleost genus <i>Trematomus</i> </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">339</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29739320">29739320</a> </span> </span> </li> <li tag="ref"> <a name="r29" /> <span class="label" tagx="label" title="label">29</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daccord</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">High-quality <i>de novo</i> assembly of the apple genome and methylome dynamics of early fruit development </span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">1099</span> <span class="lpage" tagx="lpage" title="lpage">1106</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28581499">28581499</a> </span> </span> </li> <li tag="ref"> <a name="r30" /> <span class="label" tagx="label" title="label">30</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="source" tagx="source" title="source">The Families and Genera of Angiosperms in China</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Beijing</span> </span> </li> <li tag="ref"> <a name="r31" /> <span class="label" tagx="label" title="label">31</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="source" tagx="source" title="source">Floristics of Seed Plants from China</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Beijing</span> </span> </li> <li tag="ref"> <a name="r32" /> <span class="label" tagx="label" title="label">32</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stoeckert</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> <span class="suffix" tagx="suffix" title="suffix">Jr</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roos</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">OrthoMCL: Identification of ortholog groups for eukaryotic genomes</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">2178</span> <span class="lpage" tagx="lpage" title="lpage">2189</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885">12952885</a> </span> </span> </li> <li tag="ref"> <a name="r33" /> <span class="label" tagx="label" title="label">33</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beilstein</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Shehbaz</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kellogg</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Brassicaceae phylogeny and trichome evolution</span> <span class="source" tagx="source" title="source">Am J Bot</span> <span class="volume" tagx="volume" title="volume">93</span> <span class="fpage" tagx="fpage" title="fpage">607</span> <span class="lpage" tagx="lpage" title="lpage">619</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21646222">21646222</a> </span> </span> </li> <li tag="ref"> <a name="r34" /> <span class="label" tagx="label" title="label">34</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Resolution of Brassicaceae phylogeny using nuclear genes uncovers nested radiations and supports convergent morphological evolution </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">394</span> <span class="lpage" tagx="lpage" title="lpage">412</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26516094">26516094</a> </span> </span> </li> <li tag="ref"> <a name="r35" /> <span class="label" tagx="label" title="label">35</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">PAML 4: Phylogenetic analysis by maximum likelihood</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">1586</span> <span class="lpage" tagx="lpage" title="lpage">1591</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a> </span> </span> </li> <li tag="ref"> <a name="r36" /> <span class="label" tagx="label" title="label">36</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Plastome phylogeny and early diversification of Brassicaceae</span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">176</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28209119">28209119</a> </span> </span> </li> <li tag="ref"> <a name="r37" /> <span class="label" tagx="label" title="label">37</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Shehbaz</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Kane</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Transberingia</i>, a new generic name replacing the <i>Illegitimate Beringia</i> (Brassicaceae) </span> <span class="source" tagx="source" title="source">Novon</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">396</span> </span> </li> <li tag="ref"> <a name="r38" /> <span class="label" tagx="label" title="label">38</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">German</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Six new synonyms in the central Asian Cruciferae (Brassicaceae)</span> <span class="source" tagx="source" title="source">Nord J Bot</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">38</span> <span class="lpage" tagx="lpage" title="lpage">40</span> </span> </li> <li tag="ref"> <a name="r39" /> <span class="label" tagx="label" title="label">39</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dassanayake</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">The genome of the extremophile crucifer <i>Thellungiella parvula</i> </span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">913</span> <span class="lpage" tagx="lpage" title="lpage">918</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21822265">21822265</a> </span> </span> </li> <li tag="ref"> <a name="r40" /> <span class="label" tagx="label" title="label">40</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sudmant</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="person-group'"> <span class="collab'">collab: 1000 Genomes Project</span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Diversity of human copy number variation and multicopy genes</span> <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">330</span> <span class="fpage" tagx="fpage" title="fpage">641</span> <span class="lpage" tagx="lpage" title="lpage">646</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21030649">21030649</a> </span> </span> </li> <li tag="ref"> <a name="r41" /> <span class="label" tagx="label" title="label">41</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Vaccinia tricks Toll</span> <span class="source" tagx="source" title="source">Genome Biol</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">reports0079</span> </span> </li> <li tag="ref"> <a name="r42" /> <span class="label" tagx="label" title="label">42</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McDowell</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Intragenic recombination and diversifying selection contribute to the evolution of downy mildew resistance at the RPP8 locus of <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">1861</span> <span class="lpage" tagx="lpage" title="lpage">1874</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9811794">9811794</a> </span> </span> </li> <li tag="ref"> <a name="r43" /> <span class="label" tagx="label" title="label">43</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Unique evolutionary pattern of numbers of gramineous NBS-LRR genes</span> <span class="source" tagx="source" title="source">Mol Genet Genomics</span> <span class="volume" tagx="volume" title="volume">283</span> <span class="fpage" tagx="fpage" title="fpage">427</span> <span class="lpage" tagx="lpage" title="lpage">438</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20217430">20217430</a> </span> </span> </li> <li tag="ref"> <a name="r44" /> <span class="label" tagx="label" title="label">44</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Uncovering the dynamic evolution of nucleotide-binding site-leucine-rich repeat (NBS-LRR) genes in Brassicaceae </span> <span class="source" tagx="source" title="source">J Integr Plant Biol</span> <span class="volume" tagx="volume" title="volume">58</span> <span class="fpage" tagx="fpage" title="fpage">165</span> <span class="lpage" tagx="lpage" title="lpage">177</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25926337">25926337</a> </span> </span> </li> <li tag="ref"> <a name="r45" /> <span class="label" tagx="label" title="label">45</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">XJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">XJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">NL</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Microorganisms in a high altitude glacier ice in Tibet</span> <span class="source" tagx="source" title="source">Folia Microbiol (Praha)</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">241</span> <span class="lpage" tagx="lpage" title="lpage">245</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12094732">12094732</a> </span> </span> </li> <li tag="ref"> <a name="r46" /> <span class="label" tagx="label" title="label">46</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Ground tit genome reveals avian adaptation to living at high altitudes in the Tibetan plateau </span> <span class="source" tagx="source" title="source">Nat Commun</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">2071</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23817352">23817352</a> </span> </span> </li> <li tag="ref"> <a name="r47" /> <span class="label" tagx="label" title="label">47</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fitch</span> <span class="given-names" tagx="given-names" title="given-names">WM</span> </span> </span> <span class="year" tagx="year" title="year">1970</span> <span class="mixed-article-title" title="mixed-article-title">Distinguishing homologous from analogous proteins</span> <span class="source" tagx="source" title="source">Syst Zool</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">99</span> <span class="lpage" tagx="lpage" title="lpage">113</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/5449325">5449325</a> </span> </span> </li> <li tag="ref"> <a name="r48" /> <span class="label" tagx="label" title="label">48</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frohnmeyer</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Staiger</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Ultraviolet-B radiation-mediated responses in plants. Balancing damage and protection</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">133</span> <span class="fpage" tagx="fpage" title="fpage">1420</span> <span class="lpage" tagx="lpage" title="lpage">1428</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14681524">14681524</a> </span> </span> </li> <li tag="ref"> <a name="r49" /> <span class="label" tagx="label" title="label">49</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reed</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larkins</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chau</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Figg</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">DNA repair: ERCC1, nucleotide excision repair, and platinum resistance</span> <span class="source" tagx="source" title="source">Handbook of Anticancer Pharmacokinetics and Pharmacodynamics, Cancer Drug Discovery and Development </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rudek</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chau</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Figg</span> <span class="given-names" tagx="given-names" title="given-names">WD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McLeod</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span> </span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span> <span class="fpage" tagx="fpage" title="fpage">333</span> <span class="lpage" tagx="lpage" title="lpage">349</span> </span> </li> <li tag="ref"> <a name="r50" /> <span class="label" tagx="label" title="label">50</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Draft genome sequence of the Tibetan antelope</span> <span class="source" tagx="source" title="source">Nat Commun</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">1858</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23673643">23673643</a> </span> </span> </li> <li tag="ref"> <a name="r51" /> <span class="label" tagx="label" title="label">51</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Genome resequencing identifies unique adaptations of Tibetan chickens to hypoxia and high-dose ultraviolet radiation in high-altitude environments </span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">765</span> <span class="lpage" tagx="lpage" title="lpage">776</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26907498">26907498</a> </span> </span> </li> <li tag="ref"> <a name="r52" /> <span class="label" tagx="label" title="label">52</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J-T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Comparative genomic investigation of high-elevation adaptation in ectothermic snakes</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">115</span> <span class="fpage" tagx="fpage" title="fpage">8406</span> <span class="lpage" tagx="lpage" title="lpage">8411</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30065117">30065117</a> </span> </span> </li> <li tag="ref"> <a name="r53" /> <span class="label" tagx="label" title="label">53</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyzenga</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stone</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Abiotic stress tolerance mediated by protein ubiquitination</span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">599</span> <span class="lpage" tagx="lpage" title="lpage">616</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22016431">22016431</a> </span> </span> </li> <li tag="ref"> <a name="r54" /> <span class="label" tagx="label" title="label">54</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yanagawa</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> COP10 forms a complex with DDB1 and DET1 in vivo and enhances the activity of ubiquitin conjugating enzymes </span> <span class="source" tagx="source" title="source">Genes Dev</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">2172</span> <span class="lpage" tagx="lpage" title="lpage">2181</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15342494">15342494</a> </span> </span> </li> <li tag="ref"> <a name="r55" /> <span class="label" tagx="label" title="label">55</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yanagawa</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwok</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsui</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deng</span> <span class="given-names" tagx="given-names" title="given-names">X-W</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> COP10 is a ubiquitin-conjugating enzyme variant that acts together with COP1 and the COP9 signalosome in repressing photomorphogenesis </span> <span class="source" tagx="source" title="source">Genes Dev</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">554</span> <span class="lpage" tagx="lpage" title="lpage">559</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11877375">11877375</a> </span> </span> </li> <li tag="ref"> <a name="r56" /> <span class="label" tagx="label" title="label">56</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanyuka</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Mutations in the huge <i>Arabidopsis</i> gene BIG affect a range of hormone and light responses </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">57</span> <span class="lpage" tagx="lpage" title="lpage">70</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12834402">12834402</a> </span> </span> </li> <li tag="ref"> <a name="r57" /> <span class="label" tagx="label" title="label">57</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide convergence during evolution of mangroves from woody plants</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">1008</span> <span class="lpage" tagx="lpage" title="lpage">1015</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28087771">28087771</a> </span> </span> </li> <li tag="ref"> <a name="r58" /> <span class="label" tagx="label" title="label">58</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wollenberg</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strasser</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cerdán</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amasino</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Acceleration of flowering during shade avoidance in <i>Arabidopsis</i> alters the balance between FLOWERING LOCUS C-mediated repression and photoperiodic induction of flowering </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">148</span> <span class="fpage" tagx="fpage" title="fpage">1681</span> <span class="lpage" tagx="lpage" title="lpage">1694</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18790998">18790998</a> </span> </span> </li> <li tag="ref"> <a name="r59" /> <span class="label" tagx="label" title="label">59</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iñigo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alvarez</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strasser</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Califano</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cerdán</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">PFT1, the MED25 subunit of the plant mediator complex, promotes flowering through CONSTANS dependent and independent mechanisms in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">601</span> <span class="lpage" tagx="lpage" title="lpage">612</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21985558">21985558</a> </span> </span> </li> <li tag="ref"> <a name="r60" /> <span class="label" tagx="label" title="label">60</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turkington</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Flowering and fruiting phenology of 24 plant species on the north slope of Mt. Qomolangma (Mt. Everest) </span> <span class="source" tagx="source" title="source">J Mt Sci</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">45</span> <span class="lpage" tagx="lpage" title="lpage">54</span> </span> </li> <li tag="ref"> <a name="r61" /> <span class="label" tagx="label" title="label">61</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barrett</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">The evolution of plant sexual diversity</span> <span class="source" tagx="source" title="source">Nat Rev Genet</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">274</span> <span class="lpage" tagx="lpage" title="lpage">284</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11967552">11967552</a> </span> </span> </li> <li tag="ref"> <a name="r62" /> <span class="label" tagx="label" title="label">62</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xing</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Evolution of S-domain receptor-like kinases in land plants and origination of S-locus receptor kinases in Brassicaceae </span> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">69</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23510165">23510165</a> </span> </span> </li> <li tag="ref"> <a name="r63" /> <span class="label" tagx="label" title="label">63</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barringer</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Polyploidy and self-fertilization in flowering plants</span> <span class="source" tagx="source" title="source">Am J Bot</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">1527</span> <span class="lpage" tagx="lpage" title="lpage">1533</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21636519">21636519</a> </span> </span> </li> <li tag="ref"> <a name="r64" /> <span class="label" tagx="label" title="label">64</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodwillie</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalisz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eckert</span> <span class="given-names" tagx="given-names" title="given-names">CG</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">The evolutionary enigma of mixed mating systems in plants: Occurrence, theoretical explanations, and empirical evidence </span> <span class="source" tagx="source" title="source">Annu Rev Ecol Evol Syst</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">47</span> <span class="lpage" tagx="lpage" title="lpage">79</span> </span> </li> <li tag="ref"> <a name="r65" /> <span class="label" tagx="label" title="label">65</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takayama</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Isogai</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Self-incompatibility in plants</span> <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">467</span> <span class="lpage" tagx="lpage" title="lpage">489</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15862104">15862104</a> </span> </span> </li> <li tag="ref"> <a name="r66" /> <span class="label" tagx="label" title="label">66</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuchimatsu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Evolution of self-compatibility in <i>Arabidopsis</i> by a mutation in the male specificity gene </span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">464</span> <span class="fpage" tagx="fpage" title="fpage">1342</span> <span class="lpage" tagx="lpage" title="lpage">1346</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20400945">20400945</a> </span> </span> </li> <li tag="ref"> <a name="r67" /> <span class="label" tagx="label" title="label">67</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novikova</span> <span class="given-names" tagx="given-names" title="given-names">PY</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Genome sequencing reveals the origin of the allotetraploid <i>Arabidopsis suecica</i> </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">957</span> <span class="lpage" tagx="lpage" title="lpage">968</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28087777">28087777</a> </span> </span> </li> <li tag="ref"> <a name="r68" /> <span class="label" tagx="label" title="label">68</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boggs</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nasrallah</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nasrallah</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Independent S-locus mutations caused self-fertility in <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">PLoS Genet</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">e1000426</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19300485">19300485</a> </span> </span> </li> <li tag="ref"> <a name="r69" /> <span class="label" tagx="label" title="label">69</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">The evolution of selfing in <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">317</span> <span class="fpage" tagx="fpage" title="fpage">1070</span> <span class="lpage" tagx="lpage" title="lpage">1072</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17656687">17656687</a> </span> </span> </li> <li tag="ref"> <a name="r70" /> <span class="label" tagx="label" title="label">70</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goubet</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Contrasted patterns of molecular evolution in dominant and recessive self-incompatibility haplotypes in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">PLoS Genet</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">e1002495</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22457631">22457631</a> </span> </span> </li> <li tag="ref"> <a name="r71" /> <span class="label" tagx="label" title="label">71</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lanz</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Evolution of the S-locus region in <i>Arabidopsis</i> relatives </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">157</span> <span class="fpage" tagx="fpage" title="fpage">937</span> <span class="lpage" tagx="lpage" title="lpage">946</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21810962">21810962</a> </span> </span> </li> <li tag="ref"> <a name="r72" /> <span class="label" tagx="label" title="label">72</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castric</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vekemans</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Plant self-incompatibility in natural populations: A critical assessment of recent theoretical and empirical advances </span> <span class="source" tagx="source" title="source">Mol Ecol</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">2873</span> <span class="lpage" tagx="lpage" title="lpage">2889</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15367105">15367105</a> </span> </span> </li> <li tag="ref"> <a name="r73" /> <span class="label" tagx="label" title="label">73</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Llaurens</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Does frequency-dependent selection with complex dominance interactions accurately predict allelic frequencies at the self-incompatibility locus in <i>Arabidopsis halleri?</i> </span> <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">62</span> <span class="fpage" tagx="fpage" title="fpage">2545</span> <span class="lpage" tagx="lpage" title="lpage">2557</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18647339">18647339</a> </span> </span> </li> <li tag="ref"> <a name="r74" /> <span class="label" tagx="label" title="label">74</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusaba</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Self-incompatibility in the genus <i>Arabidopsis</i>: Characterization of the S locus in the outcrossing <i>A. lyrata</i> and its autogamous relative <i>A. thaliana</i> </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">627</span> <span class="lpage" tagx="lpage" title="lpage">643</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11251101">11251101</a> </span> </span> </li> <li tag="ref"> <a name="r75" /> <span class="label" tagx="label" title="label">75</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcíacamacho</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Totland</span> <span class="given-names" tagx="given-names" title="given-names">Ø</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Pollen limitation in the alpine: A meta-analysis</span> <span class="source" tagx="source" title="source">Arct Antarct Alp Res</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">103</span> <span class="lpage" tagx="lpage" title="lpage">111</span> </span> </li> <li tag="ref"> <a name="r76" /> <span class="label" tagx="label" title="label">76</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bingham</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orthner</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Efficient pollination of alpine plants</span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">391</span> <span class="fpage" tagx="fpage" title="fpage">238</span> <span class="lpage" tagx="lpage" title="lpage">239</span> </span> </li> <li tag="ref"> <a name="r77" /> <span class="label" tagx="label" title="label">77</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalisz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vogler</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Benefits of autonomous selfing under unpredictable pollinator environments</span> <span class="source" tagx="source" title="source">Ecology</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">2928</span> <span class="lpage" tagx="lpage" title="lpage">2942</span> </span> </li> <li tag="ref"> <a name="r78" /> <span class="label" tagx="label" title="label">78</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laenen</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Demography and mating system shape the genome-wide impact of purifying selection in <i>Arabis alpina</i> </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">115</span> <span class="fpage" tagx="fpage" title="fpage">816</span> <span class="lpage" tagx="lpage" title="lpage">821</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29301967">29301967</a> </span> </span> </li> <li tag="ref"> <a name="r79" /> <span class="label" tagx="label" title="label">79</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tedder</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ansell</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vogel</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mable</span> <span class="given-names" tagx="given-names" title="given-names">BK</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Sporophytic self-incompatibility genes and mating system variation in <i>Arabis alpina</i> </span> <span class="source" tagx="source" title="source">Ann Bot</span> <span class="volume" tagx="volume" title="volume">108</span> <span class="fpage" tagx="fpage" title="fpage">699</span> <span class="lpage" tagx="lpage" title="lpage">713</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21821832">21821832</a> </span> </span> </li> <li tag="ref"> <a name="r80" /> <span class="label" tagx="label" title="label">80</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">ZQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">ZM</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Patterns of flower morphology and sexual systems in the subnival belt of the Hengduan Mountains, SW China </span> <span class="source" tagx="source" title="source">Alp Bot</span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">65</span> <span class="lpage" tagx="lpage" title="lpage">73</span> </span> </li> <li tag="ref"> <a name="r81" /> <span class="label" tagx="label" title="label">81</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">ZQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">QJ</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Autonomous selfing provides reproductive assurance in an alpine ginger <i>Roscoea schneideriana</i> (Zingiberaceae) </span> <span class="source" tagx="source" title="source">Ann Bot</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">531</span> <span class="lpage" tagx="lpage" title="lpage">538</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18682439">18682439</a> </span> </span> </li> <li tag="ref"> <a name="r82" /> <span class="label" tagx="label" title="label">82</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boetzer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henkel</span> <span class="given-names" tagx="given-names" title="given-names">CV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butler</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pirovano</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Scaffolding pre-assembled contigs using SSPACE</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">578</span> <span class="lpage" tagx="lpage" title="lpage">579</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342">21149342</a> </span> </span> </li> <li tag="ref"> <a name="r83" /> <span class="label" tagx="label" title="label">83</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Durbin</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1754</span> <span class="lpage" tagx="lpage" title="lpage">1760</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">19451168</a> </span> </span> </li> <li tag="ref"> <a name="r84" /> <span class="label" tagx="label" title="label">84</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Analysis of alternative splicing in plants with bioinformatics tools</span> <span class="source" tagx="source" title="source">Curr Top Microbiol Immunol</span> <span class="volume" tagx="volume" title="volume">326</span> <span class="fpage" tagx="fpage" title="fpage">17</span> <span class="lpage" tagx="lpage" title="lpage">37</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18630745">18630745</a> </span> </span> </li> <li tag="ref"> <a name="r85" /> <span class="label" tagx="label" title="label">85</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">The draft genome of watermelon ( <i>Citrullus lanatus</i>) and resequencing of 20 diverse accessions </span> <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">51</span> <span class="lpage" tagx="lpage" title="lpage">58</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23179023">23179023</a> </span> </span> </li> <li tag="ref"> <a name="r86" /> <span class="label" tagx="label" title="label">86</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myers</span> <span class="given-names" tagx="given-names" title="given-names">EW</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">PILER: Identification and classification of genomic repeats</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">i152</span> <span class="lpage" tagx="lpage" title="lpage">i158</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15961452">15961452</a> </span> </span> </li> <li tag="ref"> <a name="r87" /> <span class="label" tagx="label" title="label">87</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">LTR_FINDER: An efficient tool for the prediction of full-length LTR retrotransposons</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">W265</span> <span class="lpage" tagx="lpage" title="lpage">W268</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17485477">17485477</a> </span> </span> </li> <li tag="ref"> <a name="r88" /> <span class="label" tagx="label" title="label">88</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jurka</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Repbase Update, a database of eukaryotic repetitive elements</span> <span class="source" tagx="source" title="source">Cytogenet Genome Res</span> <span class="volume" tagx="volume" title="volume">110</span> <span class="fpage" tagx="fpage" title="fpage">462</span> <span class="lpage" tagx="lpage" title="lpage">467</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699">16093699</a> </span> </span> </li> <li tag="ref"> <a name="r89" /> <span class="label" tagx="label" title="label">89</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellinghaus</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurtz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willhoeft</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">LTRharvest, an efficient and flexible software for <i>de novo</i> detection of LTR retrotransposons </span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">18</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18194517">18194517</a> </span> </span> </li> <li tag="ref"> <a name="r90" /> <span class="label" tagx="label" title="label">90</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinbiss</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willhoeft</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gremme</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurtz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Fine-grained annotation and classification of de novo predicted LTR retrotransposons</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">7002</span> <span class="lpage" tagx="lpage" title="lpage">7013</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19786494">19786494</a> </span> </span> </li> <li tag="ref"> <a name="r91" /> <span class="label" tagx="label" title="label">91</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ossowski</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">The rate and molecular spectrum of spontaneous mutations in <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">327</span> <span class="fpage" tagx="fpage" title="fpage">92</span> <span class="lpage" tagx="lpage" title="lpage">94</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20044577">20044577</a> </span> </span> </li> <li tag="ref"> <a name="r92" /> <span class="label" tagx="label" title="label">92</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Exposito-Alonso</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">The rate and potential relevance of new mutations in a colonizing plant lineage</span> <span class="source" tagx="source" title="source">PLoS Genet</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">e1007155</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29432421">29432421</a> </span> </span> </li> <li tag="ref"> <a name="r93" /> <span class="label" tagx="label" title="label">93</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harris</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="comment">Improved pairwise alignment of genomic DNA. PhD dissertation (Pennsylvania State Univ, State College, PA) </span> </span> </li> <li tag="ref"> <a name="r94" /> <span class="label" tagx="label" title="label">94</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">MCScanX: A toolkit for detection and evolutionary analysis of gene synteny and collinearity</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">e49</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22217600">22217600</a> </span> </span> </li> <li tag="ref"> <a name="r95" /> <span class="label" tagx="label" title="label">95</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">MUSCLE: Multiple sequence alignment with high accuracy and high throughput</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">1792</span> <span class="lpage" tagx="lpage" title="lpage">1797</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a> </span> </span> </li> <li tag="ref"> <a name="r96" /> <span class="label" tagx="label" title="label">96</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">GWC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lugo-Martinez</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3 </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">1987</span> <span class="lpage" tagx="lpage" title="lpage">1997</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23709260">23709260</a> </span> </span> </li> <li tag="ref"> <a name="r97" /> <span class="label" tagx="label" title="label">97</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ling</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">agriGO: A GO analysis toolkit for the agricultural community</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">W64</span> <span class="lpage" tagx="lpage" title="lpage">W70</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20435677">20435677</a> </span> </span> </li> <li tag="ref"> <a name="r98" /> <span class="label" tagx="label" title="label">98</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">KOBAS 2.0: A web server for annotation and identification of enriched pathways and diseases </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">W316</span> <span class="lpage" tagx="lpage" title="lpage">W322</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21715386">21715386</a> </span> </span> </li> <li tag="ref"> <a name="r99" /> <span class="label" tagx="label" title="label">99</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">KEGG: Kyoto encyclopedia of genes and genomes</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">27</span> <span class="lpage" tagx="lpage" title="lpage">30</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a> </span> </span> </li> <li tag="ref"> <a name="r100" /> <span class="label" tagx="label" title="label">100</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Capella-Gutiérrez</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silla-Martínez</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gabaldón</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">trimAl: A tool for automated alignment trimming in large-scale phylogenetic analyses</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1972</span> <span class="lpage" tagx="lpage" title="lpage">1973</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505945">19505945</a> </span> </span> </li> <li tag="ref"> <a name="r101" /> <span class="label" tagx="label" title="label">101</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ludwig</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meier</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">RAxML-III: A fast program for maximum likelihood-based inference of large phylogenetic trees </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">456</span> <span class="lpage" tagx="lpage" title="lpage">463</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15608047">15608047</a> </span> </span> </li> <li tag="ref"> <a name="r102" /> <span class="label" tagx="label" title="label">102</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suyama</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Torrents</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bork</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">PAL2NAL: Robust conversion of protein sequence alignments into the corresponding codon alignments </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">W609</span> <span class="lpage" tagx="lpage" title="lpage">W612</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16845082">16845082</a> </span> </span> </li> <li tag="ref"> <a name="r103" /> <span class="label" tagx="label" title="label">103</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">2472</span> <span class="lpage" tagx="lpage" title="lpage">2479</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16107592">16107592</a> </span> </span> </li> <li tag="ref"> <a name="r104" /> <span class="label" tagx="label" title="label">104</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bachmann</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tedder</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laenen</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steige</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Slotte</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">Targeted long-read sequencing of a locus under long-term balancing selection in <i>Capsella</i> </span> <span class="source" tagx="source" title="source">G3 (Bethesda)</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">1327</span> <span class="lpage" tagx="lpage" title="lpage">1333</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29476024">29476024</a> </span> </span> </li> <li tag="ref"> <a name="r105" /> <span class="label" tagx="label" title="label">105</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="person-group'"> <span class="collab'">collab: 1000 Genome Project Data Processing Subgroup</span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">The sequence alignment/map format and SAMtools</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">2078</span> <span class="lpage" tagx="lpage" title="lpage">2079</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a> </span> </span> </li> <li tag="ref"> <a name="r106" /> <span class="label" tagx="label" title="label">106</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thorvaldsdóttir</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mesirov</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Integrative genomics viewer (IGV): High-performance genomics data visualization and exploration </span> <span class="source" tagx="source" title="source">Brief Bioinform</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">178</span> <span class="lpage" tagx="lpage" title="lpage">192</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22517427">22517427</a> </span> </span> </li> <li tag="ref"> <a name="r107" /> <span class="label" tagx="label" title="label">107</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stecher</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knyaz</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">MEGA X: Molecular evolutionary genetics analysis across computing platforms</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">1547</span> <span class="lpage" tagx="lpage" title="lpage">1549</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29722887">29722887</a> </span> </span> </li> <li tag="ref"> <a name="r108" /> <span class="label" tagx="label" title="label">108</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waterhouse</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Procter</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clamp</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barton</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Jalview Version 2–A multiple sequence alignment editor and analysis workbench</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1189</span> <span class="lpage" tagx="lpage" title="lpage">1191</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151095">19151095</a> </span> </span> </li> <li tag="ref"> <a name="r109" /> <span class="label" tagx="label" title="label">109</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guy</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kultima</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andersson</span> <span class="given-names" tagx="given-names" title="given-names">SG</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">genoPlotR: Comparative gene and genome visualization in R</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">2334</span> <span class="lpage" tagx="lpage" title="lpage">2335</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20624783">20624783</a> </span> </span> </li> </ul> </div> </div>  </body></html>