<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">FEMS Microbiol Ecol</span> <span class="iso-abbrev" title="iso-abbrev">FEMS Microbiol. Ecol</span> <span class="publisher-id" title="publisher-id">femsec</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">FEMS Microbiology Ecology</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">0168-6496</span> <span class="issn-epub" tagx="issn" title="issn-epub">1574-6941</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6736398">6736398</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31429869">31429869</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/femsec/fiz128">10.1093/femsec/fiz128</a> </span> <span class="publisher-id" title="publisher-id">fiz128</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Biogeographical patterns in soil bacterial communities across the Arctic region</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malard</span> <span class="given-names" tagx="given-names" title="given-names">Lucie A</span> </span> <a href="#aff1">1</a> <a href="#cor1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anwar</span> <span class="given-names" tagx="given-names" title="given-names">Muhammad Z</span> </span> <a href="#aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">Carsten S</span> </span> <a href="#aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pearce</span> <span class="given-names" tagx="given-names" title="given-names">David A</span> </span> <a href="#aff1">1</a> <a href="#aff3">3</a> </span> </div> <span class="citation_author_institution" id="aff1">[1], </span> <span class="citation_author_institution" id="aff2">[2], </span> <span class="citation_author_institution" id="aff3">[3], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <b>Corresponding author</b>: Faculty of Health and Life Sciences, Northumbria University, Newcastle-upon-Tyne NE1 8ST, UK. E-mail: <span class="email" tagx="email" title="email">lucie.malard@northumbria.ac.uk</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-8-8</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-9</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-8-8</span> </span> <span class="volume" tagx="volume" title="volume">95</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">fiz128</span> <span class="history" title="history"> <span class="received" title="received">received: 2019-4-05</span> <span class="accepted" title="accepted">accepted: 2019-8-19</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <a class="self-uri" href="fiz128.pdf" title="self-uri">fiz128.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
ABSTRACT</div> <p>The considerable microbial diversity of soils and key role in biogeochemical cycling have led to growing interest in their global distribution and the impact that environmental change might have at the regional level. In the broadest study of Arctic soil bacterial communities to date, we used high-throughput DNA sequencing to investigate the bacterial diversity from 200 independent Arctic soil samples from 43 sites. We quantified the impact of spatial and environmental factors on bacterial community structure using variation partitioning analysis, illustrating a nonrandom distribution across the region. pH was confirmed as the key environmental driver structuring Arctic soil bacterial communities, while total organic carbon (TOC), moisture and conductivity were shown to have little effect. Specialist taxa were more abundant in acidic and alkaline soils while generalist taxa were more abundant in acidoneutral soils. Of the 48 147 bacterial taxa, a core microbiome composed of only 13 taxa that were ubiquitously distributed and present within 95% of samples was identified, illustrating the high potential for endemism in the region. Overall, our results demonstrate the importance of spatial and edaphic factors on the structure of Arctic soil bacterial communities. </p> </div> <div class="abstract" title="abstract"> <p>Arctic soil bacterial communities are influenced by spatial and edaphic factors and exhibit biogeographical distribution. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
16S rRNA</div> <div class="kwd" title="kwd">
Arctic soil</div> <div class="kwd" title="kwd">
microbial diversity</div> <div class="kwd" title="kwd">
indicator species</div> <div class="kwd" title="kwd">
core microbiome</div> <div class="kwd" title="kwd">
biogeography</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">European Commission's Marie Sklowdowska Curie</span> </div> <span class="award-id" tagx="award-id" title="award-id">675546</span> </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Frozen soils in the Arctic region store over 1500 Pg of carbon (Koven <i>et al</i>. <a href="#bib31">2011</a>; Mackelprang <i>et al</i>. <a href="#bib37">2011</a>) and as Arctic warming is exacerbated and permafrost thaw accelerates, the depth of the active layer is increasing. As previously frozen carbon becomes available, it is expected that microbial activity will increase, which may lead to increased atmospheric release rates of climate active gases such as carbon dioxide (CO <sub>2</sub>), methane (CH <sub>4</sub>) and nitrous oxide (N <sub>2</sub>O) (Ma <i>et al</i>. <a href="#bib36">2007</a>; Mackelprang <i>et al</i>. <a href="#bib37">2011</a>). Carbon-climate feedback studies of permafrost affected regions use temperature, soil moisture and precipitation as the main drivers of decomposition rates (Koven <i>et al</i>. <a href="#bib31">2011</a>; Schuur <i>et al</i>. <a href="#bib56">2015</a>). While models are useful to gain a global understanding of the impact of climate change on permafrost thaw and greenhouse gas release, the accuracy of results obtained is highly variable when compared with data collected in the field or laboratory (Schuur <i>et al</i>. <a href="#bib56">2015</a>) due to empirical and modelling uncertainties which still need to be addressed (Bradford <i>et al</i>. <a href="#bib5">2016</a>). </p> <p>As microorganisms drive biogeochemical cycling and participate in the uptake and release of CO <sub>2</sub>, CH <sub>4</sub> and N <sub>2</sub>O, microbial data should be incorporated in climate models. Current models use soil properties to model changes in fluxes, without considering microbial communities and the changes in community composition induced by climate change (Bardgett, Freeman and Ostle <a href="#bib3">2008</a>; Nazaries <i>et al</i>. <a href="#bib45">2013</a>). Adding microbial information into models will improve their predictions; however, detailed microbial data is still required, with a focus on the microbial community, diversity, function, stability and long-term changes in these communities (Graham <i>et al</i>. <a href="#bib24">2012</a>; Nazaries <i>et al</i>. <a href="#bib45">2013</a>). While global surveys of microbial diversity have already been conducted (Tedersoo <i>et al</i>. <a href="#bib63">2014</a>; Delgado-Baquerizo <i>et al</i>. <a href="#bib14">2018</a>), the number of Arctic samples is restricted (Malard and Pearce <a href="#bib39">2018</a>) and therefore, microbial data is still lacking for most permafrost-affected regions. </p> <p>To accurately model spatial distribution patterns of microbes and incorporate microbial data from permafrost-affected regions in climate models in the near future, the first step is to produce a baseline database of microbial diversity with respect to biogeographical distribution. Biogeography is the study of biodiversity across space and time; it gives insights into ecological mechanisms such as speciation, extinction, dispersal and species interactions (Martiny <i>et al</i>. <a href="#bib42">2006</a>; Fierer <a href="#bib19">2008</a>), improving our understanding of community assembly. Theoretically, distant and isolated habitats are expected to present high endemicity as a consequence of intrinsic dispersal limitations and environmental filtering (Mittelbach and Schemske <a href="#bib44">2015</a>; Kleinteich <i>et al</i>. <a href="#bib30">2017</a>; Bahram <i>et al</i>. <a href="#bib1">2018</a>). Thus, isolated, pristine ecosystems with limited human presence, such as the Arctic region, should harbor endemic communities. However, microbial communities may be less constrained by geographical barriers and thus, have long been considered ubiquitous (Finlay <a href="#bib21">2002</a>; O'Malley <a href="#bib47">2007</a>). Yet, recent studies have uncovered patterns of microbial biogeography on global scales (Fierer and Jackson <a href="#bib20">2006</a>; Lauber <i>et al</i>. <a href="#bib34">2009</a>; Tedersoo <i>et al</i>. <a href="#bib63">2014</a>; Henschel, Anwar and Manohar <a href="#bib27">2015</a>; Bahram <i>et al</i>. <a href="#bib1">2018</a>; Delgado-Baquerizo <i>et al</i>. <a href="#bib14">2018</a>). They demonstrated that microbial distribution is nonrandom, generally influenced by biotic and abiotic factors such as plant cover, soil pH, C/N ratio or precipitation, and that communities vary from one region to another. However, whether these patterns apply to Arctic microbial communities is still subject to debate as these studies tend to have a low number of Arctic samples and Arctic studies have generally focused on small-scale patterns in restricted areas, as shown by Metcalfe <i>et al</i>. ( <a href="#bib1_217_1567079012999">2018</a>) highlighting the focuses on Abisko, Sweden and Toolik lake, Alaska. Large-scale studies are therefore required to improve our understanding not only of the spatial distribution of microorganisms but also on the processes of community assembly. Producing a baseline database of diversity in the region will enable the accurate modelling of spatial distribution, linking with patterns of functional processes to evaluate the potential consequences of environmental change on ecosystem properties. </p> <p>In this study, we conducted a Pan-Arctic survey of bacterial communities in Arctic soils to provide a baseline of Arctic bacterial diversity. We addressed the following questions: (i) Are bacterial taxa ubiquitously distributed or are there biogeographical patterns? (ii) What are the key factors influencing bacterial community structure? (iii) Are there taxa closely associated with the key edaphic properties and what can we learn from them? (iv) If bacterial taxa are not ubiquitous, is there a core microbiome? </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
METHODS</div> <div class="samplecollection" title="sec"> <div class="title" tagx="title" title="title">
Sample collection</div> <p>Soil samples were collected across the Arctic region between April 2017 and September 2017, and the global positioning system (GPS) coordinates of each site were recorded with a portable GPS (Fig. <a href="#fig1">1</a>). All sites were generally remote with limited human presence; however, in Iceland and Norway where tourist activities are frequent, we aimed to sample at least 500 m away from human-related activities. At each location, three to five soil samples were collected within 100 m <sup>2</sup> under the most common vegetation, for a total of 200 independent Arctic samples across 43 sites, with the aim to cover the broadest geographical area possible. Despite the extent, it should be noted that large areas of Canada and Russia remain uncharacterized and under-represented, highlighting areas where sampling should be prioritized in the future. Approximately 150 g of soil per sample was collected from the top 15 cm, avoiding plant roots and rocks and using an ethanol-cleaned shovel and Whirl-Pak bags (Nasco, WI, USA). Remaining plant roots and rocks were removed manually in a class II microbiological safety cabinet; samples were homogenized by manual mixing and frozen at −20°C before transportation to the United Kingdom. Samples were conserved at –20°C until analyzed. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1.</span> </a> </div>   <p>Map of sampling sites within the Arctic Region, as characterized by the Arctic circle (black line), the 10°C isotherm (red) and the tree line boundary (green). 200 individual soil samples at 43 different sites were collected for this study, all within at least one of the definitions of the Arctic region. </p>   </div> </div> <div class="soilproperties" title="sec"> <div class="title" tagx="title" title="title">
Soil properties</div> <p>We focused our investigation on environmental variables previously identified as influencing bacterial communities and included pH, conductivity, moisture and organic carbon. Moisture content was measured gravimetrically for each soil sample after drying at 150°C for 24 h and total organic carbon (TOC) was measured gravimetrically by heating previously dried soils to 550°C for 4 h. pH and conductivity were measured in the laboratory in a 1:5 freshly thawed soil to water ratio, using a Mettler-Toledo FE20 pH meter (Mettler-Toledo Instruments Co., Shanghai, China) and a CMD500 conductivity meter (WPA, Cambridge, UK). </p> </div> <div class="dnaextraction" title="sec"> <div class="title" tagx="title" title="title">
DNA extraction</div> <p>Soil DNA was extracted for each sample using the PowerSoil kit (Qiagen, Hilden, Germany), for a total of 400 DNA extracts. Each sample was amplified by polymerase chain reaction (PCR) using the universal primers 515F-806R, as per the Schloss lab standard operating procedure (Kozich <i>et al</i>. <a href="#bib32">2013</a>) and the Earth Microbiome Project (Thompson <i>et al</i>. <a href="#bib64">2017</a>), under the following conditions: initial denaturation at 95°C for 2 min then 30 cycles of 20 s denaturation at 95°C; primer annealing at 55°C for 15 s; elongation at 72°C for 5 min and then a final elongation at 72°C for 10 min. Negative controls, DNA extraction kit controls and ZymoBIOMICS mock communities (Zymo Research, Irvine, CA, USA) were included alongside the soil DNA and sequenced. PCR amplicons were cleaned and normalized using SequalPrep Plate Normalization Kit (Invitrogen, Carlsbad, CA, USA) and combined into four pools. The fragment size in each pool was assessed using the BioAnalyzer hsDNA assay (Agilent technologies, Santa Clara, CA, USA) and the concentration was measured using the Qubit hsDNA kit (Invitrogen). The library was supplemented with 5% PhiX and loaded on an Illumina MiSeq V2 500 cycles cartridge (Illumina, San Diego, Ca, USA). </p> </div> <div class="illuminasequencinganddataprocessing" title="sec"> <div class="title" tagx="title" title="title">
Illumina sequencing and data processing</div> <p>Raw amplicon sequences were demultiplexed with the associated barcodes. Cutadapt (Martin <a href="#bib41">2011</a>) was used for adaptor and primer clipping. Forward and reverse reads longer than 240 bp were merged (98% ± 0.8%/sample) using FLASH (fast length adjustment of short reads) (Magoč and Salzberg <a href="#bib38">2011</a>) for a total of 20 million reads (∼50 000 ± 30 000 reads/sample) initially. Vsearch (Rognes <i>et al</i>. <a href="#bib52">2016</a>) was used for downstream analyses. Quality filtering was carried out with an expected error &amp;lt;1.5. Dereplication was performed to identify unique sequences. A two-step chimera detection method was used, first by aligning against ChimeraSlayer Gold database provided with SILVA (Pruesse <i>et al</i>. <a href="#bib51">2007</a>), second by using the <i>de novo</i> detection module in vsearch. <i>De novo</i> operational taxonomic unit (OTU) calling was performed on high-quality trimmed sequences at 97% similarity level using the USEARCH (Edgar <a href="#bib18">2010</a>) algorithm for clustering implemented in vsearch to generate OTUs. Unique chimera filtered sequences were aligned using the Python Nearest Alignment Space Termination (PyNAST) (Caporaso <i>et al</i>. <a href="#bib7">2009</a>) tool with a relaxed neighbor-joining tree built using FastTree (Price, Dehal and Arkin <a href="#bib50">2010</a>). The taxonomy was determined using the Classification Resources for Environmental Sequence Tags (CREST) (Lanzén <i>et al</i>. <a href="#bib33">2012</a>) classifier with a confidence threshold of 0.80 against SILVA release 128 as a reference database. </p> <p>Samples less than at least 2000 reads/sample were filtered from the OTU table in order to have sufficient reads as described in Caporaso <i>et al</i>. ( <a href="#bib8">2011</a>). After filtering, 386 samples were used for the statistical analyses, corresponding to 386 DNA extracts from 200 independent samples and ∼19.5 million reads (50 609 ± 26 700 reads/sample) assigned against 48 147 OTUs. </p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>The dataset is deposited at European Nucleotide Archive/SRA under the accession number PRJEB29109. </p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>All statistical analyses were performed with a combination of QIIME1 V 1.90 (Caporaso <i>et al</i>. <a href="#bib9">2010</a>) and the R environment (R Core Team <a href="#bib62">2013</a>). Multiple rarefactions were performed in QIIME with the smallest sample size as maximum depth. Alpha diversity (richness, Shannon and Simpson indices) were calculated in QIIME on the matrices resulting from the multiple rarefactions. The differences in alpha diversity indices were tested in QIIME using nonparametric (Monte-Carlo) tests across different pH categories with a Bonferroni correction. </p> <p>The nonrarefied OTU table was normalized using cumulative-sum scaling (CSS) in QIIME (Paulson <i>et al</i>. <a href="#bib49">2013</a>). The resulting OTU table was input into R for subsequent analyses. Using the phyloseq package (McMurdie and Holmes <a href="#bib43">2013</a>), we calculated the Bray–Curtis dissimilarity distance and visualized the ordination using principal coordinate analysis (PCoA). Using the cmdscale, envfit and ordisurf functions in the vegan package (Dixon <a href="#bib15">2003</a>), we calculated and visualized the significant correlations between edaphic properties and bacterial community dissimilarity in relation to the pH gradient. Permutational multivariate analyses of variance (PerMANOVA) was conducted using the adonis function with 999 permutations. Pearson's correlation with 999 permutations was used to evaluate the collinearity of environmental variables. </p> <p>To evaluate the spatial component, the geographic locations of the sampling sites were transformed into cartesian coordinates using the SoDA package (Chambers <a href="#bib11">2008</a>) and the euclidean distance was calculated using the vegan package. The presence of a linear trend was tested by redundancy discriminant analysis and ANOVA as prescribed in Borcard, Gillet and Legendre ( <a href="#bib4">2018</a>). A significant linear trend was identified, violating the second-order stationarity assumption where the mean of the variable and its spatial covariance are the same over the study area and its variance is finite. In other words, spatial correlation coefficients cannot be tested for significance if an overall trend is present in the data (Franklin and Mills <a href="#bib22">2007</a>; Borcard, Gillet and Legendre <a href="#bib4">2018</a>) and therefore, the data was detrended by linear regression of the <i>x</i>, <i>y</i> coordinates. To carry meaningful spatial analysis, constructing spatial variables representing spatial structures at all relevant scales is necessary (Borcard, Gillet and Legendre <a href="#bib4">2018</a>). To do so, we used distance-based Moran's Eigenvector Maps (dbMEM) on detrended data and <i>x</i>, <i>y</i> coordinates using the adespatial R package (Dray <i>et al</i>. <a href="#bib16">2017</a>). Significance of the spatial vectors was assessed using ANOVA. Forward selection was carried out to identify significant dbMEM vectors. </p> <p>Variation partitioning analysis (VPA) was used to assess the impact of environmental and spatial factors on Bray–Curtis dissimilarity. The linear trend was considered to represent a source of variation like any other and therefore, we performed variation partitioning analysis using the environmental variables, <i>x</i>, <i>y</i> coordinates (trend) and significant dbMEM vectors. </p> <p>Generalist and specialist taxa were identified by selecting taxa present in over 95% of samples within each unique and overlapped pH category. Indicator taxa were determined by the Dufrene-Legendre indicator species analysis using the multipatt function (Cáceres and Legendre <a href="#bib6">2009</a>) to identify abundant OTUs that were specifically associated with the different pH ranges. The phylogenetic tree of indicator species was built using the aligned sequences from the identified indicator OTUs and input into FastTree. The tree was visualized using the Interactive Tree of Life (iTOL) online tool (Letunic and Bork <a href="#bib35">2016</a>). </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
RESULTS</div> <div class="bacterialcommunitycomposition" title="sec"> <div class="title" tagx="title" title="title">
Bacterial community composition</div> <p>We identified 48 147 bacterial taxa, mainly classified as Proteobacteria (20%) and Planctomycetes (15%). Acidobacteria represented 13% while Actinobacteria, Bacteroidetes, Chloroflexi and Verrucomicrobia each accounted for 4–6% of the community. Of these 48 147 OTUs, only 135 had overall abundances over 0.1% across all 386 samples and were defined as abundant taxa. They accounted for 32% of all the reads, illustrating the dominance of a few taxa over the rest of the community. Within these abundant taxa, Acidobacteria dominated the subcommunity at ∼32% with Blastocatellia (12.8%) and Acidobacteria subgroup 6 (7.2%) as abundant classes. Alphaproteobacteria (10.6%) and Betaproteobacteria (6.6%) were the most commonly identified Proteobacteria (21% overall) and Verrucomicrobia was largely represented by Spartobacteria (17.2%). Actinobacteria (10.7%), Chloroflexi (7%) and Bacteroidetes (3.5%) were also among the abundant phyla classified. </p> </div> <div class="spatialandenvironmentalfactorsinfluencingbacterialcommunities" title="sec"> <div class="title" tagx="title" title="title">
Spatial and environmental factors influencing bacterial communities</div> <p>Distance–decay curves were used to evaluate the influence of spatial factors on bacterial community structure. Figure <a href="#fig2">2A</a> illustrates the increase in community dissimilarity with increasing geographic distance (linear regression: <i>R</i> <sup>2</sup> = 0.108, <i>P &amp;lt;</i> 2.2 × 10 <i> <sup>−</sup> </i> <sup>16</sup>). The best-fitted power model ( <i>R</i> <sup>2</sup> = 0.221, <i>P</i> = 0.005) suggested the presence of spatial autocorrelation, also qualified as dispersal limitation, within the first 500 m. For a more accurate estimate of dispersal limitation, we plotted the same graph within 0 and 500 m (Fig. <a href="#fig2">2B</a>) and observed spatial autocorrelation approximately up to 20 m. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2.</span> </a> </div>   <p>Distance–decay curves with the linear regression (red), the best-fitted power model (green). <b>(A)</b> Community dissimilarity of the total bacterial community by geographic distance in km. <b>(B)</b> Zoom into Fig. <a href="#fig2">2A</a> from 0 to 500 m to assess the spatial autocorrelation distance. </p>   </div> <p>To determine the influence of environmental factors on the bacterial community structure, we used the adonis function with 999 permutations. Results indicated that pH explained 18.7% of the variance in bacterial community composition overall (Table <a href="#tbl1">1</a>). In comparison, conductivity, moisture and TOC explained almost 10% of the variance altogether. The measured environmental parameters explained 29% of the variance in the bacterial subcommunity, leaving 71% unexplained. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1.</span>  <p>The relative importance of environmental factors on bacterial communities, calculated by adonis. </p>  <table> <thead> <tr> <th /> <th>Total</th> </tr> <tr> <th>Variable</th> <th> <sub> <i>R</i> </sub>2 </th> <th>Pr( <i>&amp;gt;</i>F) </th> </tr> </thead> <tbody> <tr> <td>pH</td> <td>0.187</td> <td>0.0001***</td> </tr> <tr> <td>Conductivity</td> <td>0.048</td> <td>0.0001***</td> </tr> <tr> <td>Moisture</td> <td>0.027</td> <td>0.0001***</td> </tr> <tr> <td>TOC</td> <td>0.021</td> <td>0.0001***</td> </tr> <tr> <td>Residuals</td> <td>0.717</td> <td>N/A</td> </tr> </tbody> </table> </div> <p>To quantify the influence of edaphic properties and spatial factors, we conducted variation partitioning on the bacterial community. We built spatial vectors using distance-based Moran's Eigenvector Maps (dbMEM) on <i>x</i>, <i>y</i> geographic coordinates. Three vectors (MEMs) were produced; after forward selection, two significant MEMs were identified for the total community and used in further analyses. The variation partitioning analysis differentiated the effect of environmental factors, linear trend and spatial vectors on Bray–Curtis dissimilarity (Fig. <a href="#fig3">3</a>). Results indicated that the environmental factors (X1 = [a]+[d]+[g]+[f]) explained 30.7% of the variance in the community. These results are equivalent to the finding of the adonis test (Table <a href="#tbl1">1</a>), confirming the success of the variation partitioning analysis. Nonspatially structured environmental variables (fraction [a]) explained 15% of the variation while spatial factors only (fractions [b], [e] and [c]) explained ∼10% of the variation. Of the fraction X1 (environmental factors), 15% represented spatially structured environmental variables, also called induced spatial dependence, where the spatial structure of these environmental variables induced a similar spatial structure in the response data (detailed results in <a href="#sup1">Table S1</a>, Supporting Information). In total, 63.3% of the total bacterial community dissimilarity could not be explained by environmental and spatial factors after <i>R</i> <sup>2</sup> adjustment. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3.</span> </a> </div>   <p>Venn diagram illustrating the results of the variation partitioning analysis and indicating the amount of variation in bacterial community explained by environmental variables and spatial factors including the linear trend. Detailed results are available in <a href="#sup1">Table S1</a> (Supporting Information). </p>   </div> </div> <div class="phasakeyenvironmentalfactor" title="sec"> <div class="title" tagx="title" title="title">
pH as a key environmental factor</div> <p>Of the environmental variables we measured, we identified pH as the primary factor structuring Arctic soil bacterial communities. The Pearson correlations indicated that pH did not have strong collinearity (coefficient over |0.8|) with any other variable measured ( <a href="#sup1">Table S2</a>, Supporting Information). The clustering of samples by pH range was clearly observed on the PCoA of bacterial communities (Fig. <a href="#fig4">4A</a>). The envfit analysis confirmed that pH was the key measured factor driving the ordination, although TOC, conductivity and moisture all had some influence. Using ordisurf, we visualized the PCoA ordination of the bacterial community in relation to the pH gradient ( <a href="#sup1">Figure S1</a>, Supporting Information). The Bray–Curtis dissimilarity heatmap and dendrogram (Fig. <a href="#fig4">4B</a>) identified three main clusters illustrating community differences. The first cluster was composed of acidic samples in a pH gradient from Norwegian soils at pH = 4.07 (±0.35) to samples from Alaska (pH = 4.64 ± 0.41) and West Greenland (pH = 4.90 ± 0.85). The second cluster included the lower acidoneutral range of samples from East Greenland (pH = 5.96 ± 0.69), Svalbard (pH = 5.65 ± 0.53) and Iceland (pH = 5.84 ± 0.46). Finally, the last cluster included the higher range of acidoneutral and alkaline soils from Russia (pH = 6.19 ± 0.22) and Canada (pH = 7.94 ± 0.67). The full spectrum of soil pH was covered from pH = 3.5 to pH = 9.0 and geographical locations often had samples from more than one pH category ( <a href="#sup1">Figure S2</a>, Supporting Information). As pH was the primary factor we identified influencing bacterial community structure, we defined three pH categories to focus on: acidic, acidoneutral and alkaline. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4.</span> </a> </div>   <p> <b>(A)</b> PCoA of the Bray–Curtis dissimilarity of bacterial communities across the Arctic region by pH range. <b>(B)</b> Bray–Curtis dissimilarity heatmap and dendrogram illustrating the clustering of samples into three pH categories. </p>   </div> </div> <div class="bacterialdiversitybyphcategory" title="sec"> <div class="title" tagx="title" title="title">
Bacterial diversity by pH category</div> <p>We compared the observed richness and Shannon diversity index by pH range (Fig. <a href="#fig5">5A</a> and <a href="#fig5">B</a>) using a Monte-Carlo test to assess differences. Alpha diversity was significantly lower in acidic samples than in acidoneutral and alkaline soils ( <a href="#sup1">Table S3</a>, Supporting Information). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5.</span> </a> </div>   <p>Alpha diversity differences in the total communities by pH category. <b>(A)</b> Richness by observed OTUs. <b>(B)</b> Shannon index. </p>   </div> <p>In terms of community composition, we differentiated the abundant community, composed of only 135 OTUs, from the total community composed of 48 147 OTUs and observed the changes in relative abundance by pH range. In the total community, Acidobacteria, Armatimonadetes, Planctomycetes and Verrucomicrobia decreased with increasing pH. On the other hand, Bacteroidetes, Chloroflexi and Gemmatimonadetes increased in alkaline soils; Proteobacteria remained stable throughout. In the abundant community, Acidobacteria, Bacteroidetes and Gemmatimonadetes increased with increasing pH while Actinobacteria, Planctomycetes and Verrucomicrobia remained stable (Fig. <a href="#fig6">6</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6.</span> </a> </div>   <p>Relative abundance of the abundant and total communities by pH range classified at the phylum level. </p>   </div> </div> <div class="generalistvsspecialisttaxa" title="sec"> <div class="title" tagx="title" title="title">
Generalist vs specialist taxa</div> <p>The differentiation of specialist from generalist taxa associated with pH range was conducted by considering all taxa identified in this study and present in a minimum 95% of all samples from each pH category (Fig. <a href="#fig7">7</a>). 125 acidic specialist OTUs were identified, unique to acidic soil samples. Of these 125 unique OTUs, most belonged to the Acidobacteria (27%), Verrucomicrobia (20%), Actinobacteria (14%), Planctomycetes (14%) and Proteobacteria (14%). At the class level, Acidobacteria group 1 dominated at 14%, while the rest of the identified classes had balanced relative abundances oscillating between 5 and 8%. Shared OTUs, or generalists were present in low numbers in acidic soils. 12 OTUs were shared with acidoneutral soils only, which were dominated by Alphaproteobacteria (65%), mainly Rhizobiales, while 3 OTUs were shared with alkaline only, belonging to the Spartobacteria (Verrucomicrobia), Phycisphaerae (Planctomycetes) and Chitinophagia (Bacteroidetes) classes. In comparison, acidoneutral soils had 76 shared taxa with alkaline soils, against 12 shared OTUs with acidic. Taxa shared with alkaline soils were mainly classified as Blastocatellia (Acidobacteria), Spartobacteria and Acidobacteria subgroup 6. Taxa exclusively found in acidoneutral soils belonged mainly to Actinobacteria (20%), Verrucomicrobia (20%) and Acidobacteria (19%). At the class level, unique taxa living in acidoneutral soils were dominated by Spartobacteria, Alphaproteobacteria and Holophagae. Alkaline soils presented a combination of both, a high number of shared (79 OTUs in total) and 125 exclusive taxa. Alkaline unique taxa were mostly composed of Acidobacteria (36%) and Proteobacteria (22%). From these unique taxa, the Acidobacteria subgroup 6 (20%), Blastocatellia (12%) and Alphaproteobacteria (12%) dominated the community. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7.</span> </a> </div>   <p>Venn diagram illustrating generalist (shared) and specialist (unique) OTUs by pH category.</p>   </div> </div> <div class="indicatorspecies" title="sec"> <div class="title" tagx="title" title="title">
Indicator species</div> <p>We conducted the indicator species analysis on the abundant taxa only as they may be more useful for monitoring purposes. We identified 17 unique taxa-habitat associations. In acidic soils, 10 indicator species, mainly Acidobacteria group 1 and 2 and Ca. Methylacidiphilum were identified. The six OTUs associated with Acidoneutral soils were mainly Acidobacteria group 4 (Blastocatellia) and Spartobacteria (Verrucomicrobia). Finally, only one OTU was identified as an indicator species for alkaline soils and belonged to the Holophagae (Acidobacteria). We also conducted the indicator species analysis of abundant taxa to combine pH ranges and identified 84 OTUs. 21 were identified as indicator species of acidic and acidoneutral soils combined, and mainly belonged to the Verrucomicrobia. Only two OTUs were associated with acidic and alkaline soils, Acidothermus (Actinobacteria) and Candidatus Xiphinematobacter (Verrucomicrobia), further illustrating the low overlap of taxa between these ecosystems and inferring the large ecosystem differences. Finally, 61 OTUs were associated with acidoneutral and alkaline soils, mainly belonging to the Acidobacteria and Proteobacteria phyla (see <a href="#sup1">Table S3</a>, Supporting Information for the detailed list). The phylogenetic tree (Figure S3, Supporting Information) illustrates the classification of the identified abundant indicator taxa with the pH range associated. Although displaying diversity, they were mainly classified as Acidobacteria and Verrucomicrobia, with the vast majority of taxa associated with acidoneutral and alkaline soils. </p> </div> <div class="thecoremicrobiome" title="sec"> <div class="title" tagx="title" title="title">
The core microbiome</div> <p>While some taxa displayed biogeographical patterns with nonrandom distribution across space and environmental gradients, others were cosmopolitan and identified in over 95% of all samples. The core microbiome represented 0.026% of bacterial communities and accounted for 2.7% of all reads. It was composed of 13 OTUs, mainly Proteobacteria and Acidobacteria, notably belonging to the Rhizobiales and Acidobacteria subgroup 6. The Alphaproteobacteria, especially the Bradyrhizobiaceae family, was most abundant in acidic soils and decreased with increasing pH. The Betaproteobacteria, especially the Comamonadaceae family, decreased with decreasing pH. The Acidobacteria were highest in the acidoneutral soils. Although relative abundances differed by pH range, these 13 OTUs were present in all categories. <a href="#sup1">Figure S4</a> (Supporting Information) illustrates the taxonomic classes of the core microbiome and distribution between each pH category. </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
DISCUSSION</div> <p>The coverage of this study allowed the identification of a core microbiome, present in over 95% of the samples analyzed and composed of only 13 OTUs. The most abundant of these taxa belonged to the Bradyrhizobiaceae family. This is one of the most common families worldwide, as identified by Delgado-Baquerizo <i>et al</i>. ( <a href="#bib14">2018</a>). However, whether this is the same taxa remains unclear and highlights the need for global studies incorporating extreme environments. The identification of a core Arctic soil microbiome is novel and this low number of cosmopolitan OTUs illustrates the potential for endemism. For the rest of the bacterial community, spatial and edaphic factors influenced distribution across the region. </p> <div class="arcticbacterialcommunityassembly" title="sec"> <div class="title" tagx="title" title="title">
Arctic bacterial community assembly</div> <p>The distance–decay curve (Fig. <a href="#fig2">2A</a> and <a href="#fig2">B</a>) illustrated the influence of geography on bacterial communities. The spatial autocorrelation is a proxy for dispersal limitation, which was restricted to ∼20 m (Fig. <a href="#fig2">2B</a>). However, the lack of small-scale data is problematic in estimating the real autocorrelation range. The variation partitioning analysis quantified the importance of both, selection (deterministic) and dispersal (stochastic), on the bacterial community structure. Environmental variables explained 30% of the total variation (selection), of which 15% was spatially structured, corresponding to the induced spatial dependence, as described in Borcard, Gillet and Legendre ( <a href="#bib4">2018</a>). Spatial components (trend + MEMs) alone explained 9% of the variation, illustrating the spatial autocorrelation (Borcard, Gillet and Legendre <a href="#bib4">2018</a>), or dispersal. </p> <p>Here, we focused on environmental variables previously identified as influencing bacterial communities elsewhere and included pH, conductivity, moisture and organic carbon. However, 63% of the variation remained unexplained by either environmental or spatial factors in the variation partitioning analysis. This unexplained variance could be the result of unmeasured biotic or abiotic factors, directly or indirectly influencing bacterial communities. Examples of unmeasured edaphic properties could include total nitrogen, zinc, calcium oxide or nickel, all shown to have an influence on bacterial community structure in the Arctic (Wojcik <i>et al</i>. <a href="#bib67">2018</a>). Other ecosystem properties such as ice presence, active layer depth or soil texture are likely to have some influence. Climatic and topographic variables such as temperature, precipitation (Yergeau and Kowalchuk <a href="#bib69">2008</a>; Castro <i>et al</i>. <a href="#bib10">2010</a>; Nielsen and Ball <a href="#bib46">2015</a>) and altitude (Shen <i>et al</i>. <a href="#bib57">2013</a>; Xu <i>et al</i>. <a href="#bib68">2014</a>) are also known to influence bacterial assemblage and were not included in this study. Vegetation cover is a known driver of bacterial community assemblage in the Arctic region (as reviewed by Malard and Pearce ( <a href="#bib39">2018</a>)) which was not included in this analysis. Biotic interactions may also be a major part of the unexplained variance observed and could include competition and predation within bacterial communities or with other members of the soil biota (Wardle <a href="#bib65">2006</a>; Singh <i>et al</i>. <a href="#bib59">2009</a>). Other processes influencing bacterial communities include diversification and drift, both likely to account for some of the unexplained variance (Martiny <i>et al</i>. <a href="#bib42">2006</a>). </p> </div> <div class="theimportanceofph" title="sec"> <div class="title" tagx="title" title="title">
The importance of pH</div> <p>We evaluated the impact of environmental variables known to influence global soil bacterial communities. Overall, pH explained the most variation (18.7%) in bacterial community structure and diversity while TOC, moisture and conductivity explained between 2 and 5% of the variance, respectively. The identification of pH as a key factor influencing Arctic soil bacterial community composition is in line with previous Arctic studies over both, small and large scales (Männistö, Tiirola and Häggblom <a href="#bib40">2006</a>; Chu <i>et al</i>. <a href="#bib12">2010</a>; Ganzert, Bajerski and Wagner <a href="#bib23">2014</a>; Siciliano <i>et al</i>. <a href="#bib58">2014</a>; Schostag <i>et al</i>. <a href="#bib55">2015</a>). However, this study is the first to quantify the magnitude of influence, highlighting the large variation in bacterial community remaining unexplained. While pH has been identified globally as a major factor influencing microbial diversity and community structure (Fierer and Jackson <a href="#bib20">2006</a>; Lauber <i>et al</i>. <a href="#bib34">2009</a>; Tedersoo <i>et al</i>. <a href="#bib63">2014</a>; Delgado-Baquerizo <i>et al</i>. <a href="#bib14">2018</a>), the underlying processes and mechanisms by which it does remain unclear. Studies have demonstrated that the soil pH is correlated with other elements of the geochemistry and has a strong impact on nutrient and water availability as well as solubility and adsorption (Gray <i>et al</i>. <a href="#bib26">2014</a>). For instance, acidic pH increases aluminum, hydrogen and manganese solubility, retarding plant root growth due to high toxicity (Clark and Baligar <a href="#bib13">2000</a>; Singh <i>et al</i>. <a href="#bib60">2017</a>). Acidic soils have not only nutrient deficiencies such as calcium magnesium and potassium but also decreased phosphorus and molybdenum solubilities (Clark and Baligar <a href="#bib13">2000</a>; Gray <i>et al</i>. <a href="#bib26">2014</a>). Alkaline soils are generally the result of low precipitation and high evapotranspiration, leading to low water availability and in common with acidic soils, nutrient deficiencies are found with decreased phosphorus, iron, copper or zinc for instance (Clark and Baligar <a href="#bib13">2000</a>). In similar ways, acidic and alkaline soils are generally considered harsh environments requiring a wide range of adaptations from microorganisms while acidoneutral soils are considered the optimum environment for microbial life (Fierer and Jackson <a href="#bib20">2006</a>; Rousk <i>et al</i>. <a href="#bib53">2010</a>); such differences in soil composition are likely driving the observed differences in microbial community composition by pH range. </p> </div> <div class="distributionofgeneralistandspecialisttaxa" title="sec"> <div class="title" tagx="title" title="title">
Distribution of generalist and specialist taxa</div> <p>Microbial communities are assembled by deterministic (selection) and stochastic (dispersal) processes. It has been hypothesized that communities primarily structured by deterministic processes will host more specialist taxa, highly adapted to the ecosystem, while communities influenced by dispersal will harbor primarily generalist taxa, more resilient to change (Pandit, Kolasa and Cottenie <a href="#bib48">2009</a>; Graham and Stegen <a href="#bib25">2017</a>; Sriswasdi, Yang and Iwasaki <a href="#bib61">2017</a>). While specialist taxa are restricted to certain niches, they can be locally abundant; shared taxa, or generalists, are distributed across many niches (Barberán <i>et al</i>. <a href="#bib2">2012</a>). In most cases, specialist will be more abundant because generalists rapidly become specialists to adapt to their ecosystems, despite generalists having evolutionary advantages (Sriswasdi, Yang and Iwasaki <a href="#bib61">2017</a>). </p> <p>In this study, bacterial communities were specialist-dominated in acidic soils, generalist-dominated in acidoneutral soils and a mixed community in alkaline samples. The higher abundance of specialists in acidic soils (considered the harshest systems) could illustrate the need for environmental adaptations to survive in these ecosystems and suggests that deterministic processes likely structure microbial communities. Geographically, the first cluster identified by the Bray–Curtis dissimilarity matrix (Fig. <a href="#fig4">4B</a>) grouped all acidic samples from northern Norway, western Greenland and Alaska, illustrating the similarities of their bacterial communities despite the large distances separating these locations, further suggesting the strong influence of selection over dispersal. The dominance of generalists in acidoneutral soils illustrates the lower environmental pressure to have specific survival adaptations and infers the dominance of stochastic processes in community structuring. Eastern Greenland, Svalbard and Iceland were grouped together by the Bray–Curtis dissimilarity matrix. Geographically, these locations connected through the Greenland sea, a possible route of dispersal through aerosolization and long-distance aeolian transport (Šantl-Temkiv <i>et al</i>. <a href="#bib54">2018</a>). Finally, alkaline soils hosted a mixed community of specialists and generalists, suggesting the combination of both, selection and dispersal, in community structuring. The abundance of generalist taxa in alkaline soils, generally considered a harsh system in many ways similar to acidic soils, highlights the adaptability of generalists to a wide range of environmental conditions. Interestingly, the Bray–Curtis dissimilarity matrix clustered the Canadian (alkaline) and Russian (acidoneutral) samples together. The Russian samples were on the higher end of the acidoneutral pH scale and the grouping of these samples illustrates the blurred boundary between acidoneutral and alkaline pH categories. While environmental selection may be driven by many variables, dispersal likely occurs via long-distance aeolian dispersal (Šantl-Temkiv <i>et al</i>. <a href="#bib54">2018</a>). </p> </div> <div class="theimportanceofindicatortaxa" title="sec"> <div class="title" tagx="title" title="title">
The importance of indicator taxa</div> <p>The indicator species analysis determined OTU-pH range associations identifying different classes of Acidobacteria and Verrucomicrobia, primarily, characteristic of each pH range. The identification of abundant indicator species with strong habitat associations opens the possibility of predicting the presence and abundance of these taxa across the Arctic region. This is especially important for taxa such as Ca. Methylacidiphilum, a known methanotroph unlike others as it belongs to the Verrucomicrobia phylum instead of the Proteobacteria (Dunfield <i>et al</i>. <a href="#bib17">2007</a>; Khadem <i>et al</i>. <a href="#bib29">2010</a>). While natural CH <sub>4</sub> emissions come primarily from wetlands, the identification of these phylotypes in acidic soils suggests wetland may not be the only major source of northern methane emissions. Understanding the distribution and abundance of such indicator taxa, combined with field gas measurements, may allow large-scale estimates of climate-active gases release rates in the region, from soils and wetlands (Wartiainen <i>et al</i>. <a href="#bib66">2006</a>; Jørgensen <i>et al</i>. <a href="#bib28">2015</a>). </p> </div> </div> <div class="concludingremarks" title="sec"> <div class="title" tagx="title" title="title">
CONCLUDING REMARKS</div> <p>This study investigated patterns in soil bacterial diversity across the Arctic and demonstrated that bacterial communities differed across the region. However, the dispersal range was limited to ∼20 m and the variation in communities across the Arctic was primarily driven by environmental rather than spatial factors. The key edaphic property influencing communities we identified was pH, in accordance with that seen elsewhere. However, 63% of the variation remained unexplained, calling for more in-depth investigations of the drivers of bacterial community structure in the region. We identified unique and indicator taxa closely associated to the soil pH. Finally, a core microbiome composed of only 13 ubiquitously distributed taxa was identified, highlighting the potential for endemism in the region. While this study brings a deeper understanding of Arctic bacterial community assemblages, this is also a baseline for future functional studies in the region, which will be critical to forecast the ecological consequences of environmental change. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">fiz128_Supplemental_Files</span> <div class="media" title="media"> <a href="fiz128_supplemental_files.zip">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGEMENTS</div> <p>This work was supported by a grant from the European Commission's Marie Sklowdowska Curie Actions program under project number 675546. The authors also thank Dr Paul Mann, Dr Charles Greer, Dr Svetlana Evgrafova, Dr Bill Holben, Sam Pannoni, Edwin Sia and UNIS for their participation and support in fieldwork. MiSeq sequencing of the 16S rRNA gene was performed by the NU-OMICS sequencing service (Northumbria University). </p> </div> <div class="conflict" title="conflict"> <div class="title" tagx="title" title="title" /> <p> <b> <i>Conflicts of interest</i> </b>. None declared. </p> </div> <div class="author-bio" title="author-bio"> <div class="title" tagx="title" title="title">
Author Biographical</div> <p>LAM and DAP conceived and designed the study and sampling strategy. LAM carried the fieldwork and laboratory work. MZA conducted bioinformatics processing, LAM and MZA conducted the statistical analysis. LAM drafted the manuscript and MZA, DAP and CSJ revised and approved the final version. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"> <a name="bib1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bahram</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hildebrand</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forslund</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Structure and function of the global topsoil microbiome</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">560</span>: <span class="fpage" tagx="fpage" title="fpage">233</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30069051">30069051</a> </span> </span> </li> <li tag="ref"> <a name="bib2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barberán</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bates</span> <span class="given-names" tagx="given-names" title="given-names">ST</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casamayor</span> <span class="given-names" tagx="given-names" title="given-names">EO</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Using network analysis to explore co-occurrence patterns in soil microbial communities</span>. <span class="source" tagx="source" title="source">ISME J</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">343</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21900968">21900968</a> </span> </span> </li> <li tag="ref"> <a name="bib3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bardgett</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freeman</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ostle</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial contributions to climate change through carbon cycle feedbacks</span>. <span class="source" tagx="source" title="source">ISME J</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">805</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18615117">18615117</a> </span> </span> </li> <li tag="ref"> <a name="bib4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borcard</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gillet</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Legendre</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="source" tagx="source" title="source">Numerical Ecology with R</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="year" tagx="year" title="year">2018</span>. </span> </li> <li tag="ref"> <a name="bib5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bradford</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wieder</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bonan</span> <span class="given-names" tagx="given-names" title="given-names">GB</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Managing uncertainty in soil carbon feedbacks to climate change</span>. <span class="source" tagx="source" title="source">Nat Clim Change</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">751</span>. </span> </li> <li tag="ref"> <a name="bib7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caporaso</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bittinger</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bushman</span> <span class="given-names" tagx="given-names" title="given-names">FD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">PyNAST: a flexible tool for aligning sequences to a template alignment</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">266</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19914921">19914921</a> </span> </span> </li> <li tag="ref"> <a name="bib9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caporaso</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuczynski</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stombaugh</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">QIIME allows analysis of high-throughput community sequencing data</span>. <span class="source" tagx="source" title="source">Nat Methods</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">335</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20383131">20383131</a> </span> </span> </li> <li tag="ref"> <a name="bib8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caporaso</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lauber</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walters</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">4516</span>– <span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20534432">20534432</a> </span> </span> </li> <li tag="ref"> <a name="bib10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castro</span> <span class="given-names" tagx="given-names" title="given-names">HF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Classen</span> <span class="given-names" tagx="given-names" title="given-names">AT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Austin</span> <span class="given-names" tagx="given-names" title="given-names">EE</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Soil microbial community responses to multiple experimental climate change drivers</span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">76</span>: <span class="fpage" tagx="fpage" title="fpage">999</span>– <span class="lpage" tagx="lpage" title="lpage">1007</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20023089">20023089</a> </span> </span> </li> <li tag="ref"> <a name="bib11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chambers</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="source" tagx="source" title="source">Software for Data Analysis: Programming with R</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Science &amp;amp; Business Media</span>, <span class="year" tagx="year" title="year">2008</span>. </span> </li> <li tag="ref"> <a name="bib12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fierer</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lauber</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Soil bacterial diversity in the Arctic is not fundamentally different from that found in other biomes </span>. <span class="source" tagx="source" title="source">Environ Microbiol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">2998</span>– <span class="lpage" tagx="lpage" title="lpage">3006</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20561020">20561020</a> </span> </span> </li> <li tag="ref"> <a name="bib13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clark</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baligar</span> <span class="given-names" tagx="given-names" title="given-names">VC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Acidic and alkaline soil constraints on plant mineral nutrition</span>. In: <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilkinson</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span> </span>(ed). <span class="source" tagx="source" title="source">Plant–Environment Interactions</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Boca Raton</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">CRC press</span>, <span class="year" tagx="year" title="year">2000</span>, <span class="fpage" tagx="fpage" title="fpage">133</span>– <span class="lpage" tagx="lpage" title="lpage">77</span>. </span> </li> <li tag="ref"> <a name="bib6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cáceres</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Legendre</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Associations between species and groups of sites: indices and statistical inference</span>. <span class="source" tagx="source" title="source">Ecology</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">90</span>: <span class="fpage" tagx="fpage" title="fpage">3566</span>– <span class="lpage" tagx="lpage" title="lpage">74</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20120823">20120823</a> </span> </span> </li> <li tag="ref"> <a name="bib14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delgado-Baquerizo</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliverio</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brewer</span> <span class="given-names" tagx="given-names" title="given-names">TE</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">A global atlas of the dominant bacteria found in soil</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">359</span>: <span class="fpage" tagx="fpage" title="fpage">320</span>– <span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29348236">29348236</a> </span> </span> </li> <li tag="ref"> <a name="bib15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">VEGAN, a package of R functions for community ecology</span>. <span class="source" tagx="source" title="source">J Veg Sci</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">927</span>– <span class="lpage" tagx="lpage" title="lpage">30</span>. </span> </li> <li tag="ref"> <a name="bib16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dray</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanchet</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borcard</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Adespatial: multivariate multiscale spatial analysis</span>. <span class="comment">R package version 0.0-9.</span> <span class="year" tagx="year" title="year">2017</span>. </span> </li> <li tag="ref"> <a name="bib17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunfield</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuryev</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Senin</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Methane oxidation by an extremely acidophilic bacterium of the phylum Verrucomicrobia</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">450</span>: <span class="fpage" tagx="fpage" title="fpage">879</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18004300">18004300</a> </span> </span> </li> <li tag="ref"> <a name="bib18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Search and clustering orders of magnitude faster than BLAST</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">2460</span>– <span class="lpage" tagx="lpage" title="lpage">1</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691">20709691</a> </span> </span> </li> <li tag="ref"> <a name="bib20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fierer</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jackson</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The diversity and biogeography of soil bacterial communities</span>. <span class="source" tagx="source" title="source">PNAS</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">103</span>: <span class="fpage" tagx="fpage" title="fpage">626</span>– <span class="lpage" tagx="lpage" title="lpage">31</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16407148">16407148</a> </span> </span> </li> <li tag="ref"> <a name="bib19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fierer</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial biogeography: patterns in microbial diversity across space and time</span>. <span class="source" tagx="source" title="source">Accessing Uncultivated Microorganisms</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">American Society of Microbiology</span>, <span class="year" tagx="year" title="year">2008</span>, <span class="fpage" tagx="fpage" title="fpage">95</span>– <span class="lpage" tagx="lpage" title="lpage">115</span>. </span> </li> <li tag="ref"> <a name="bib21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finlay</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Global dispersal of free-living microbial eukaryote species</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">296</span>: <span class="fpage" tagx="fpage" title="fpage">1061</span>– <span class="lpage" tagx="lpage" title="lpage">3</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12004115">12004115</a> </span> </span> </li> <li tag="ref"> <a name="bib22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franklin</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mills</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Statistical analysis of spatial structure in microbial communities</span>. <span class="source" tagx="source" title="source">The Spatial Distribution of Microbes in the Environment</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">The Netherlands</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="year" tagx="year" title="year">2007</span>, <span class="fpage" tagx="fpage" title="fpage">31</span>– <span class="lpage" tagx="lpage" title="lpage">60</span>. </span> </li> <li tag="ref"> <a name="bib23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ganzert</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bajerski</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bacterial community composition and diversity of five different permafrost-affected soils of Northeast Greenland </span>. <span class="source" tagx="source" title="source">FEMS Microbiol Ecol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">89</span>: <span class="fpage" tagx="fpage" title="fpage">426</span>– <span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24819653">24819653</a> </span> </span> </li> <li tag="ref"> <a name="bib24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graham</span> <span class="given-names" tagx="given-names" title="given-names">DE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wallenstein</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vishnivetskaya</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Microbes in thawing permafrost: the unknown variable in the climate change equation</span>. <span class="source" tagx="source" title="source">ISME J</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">709</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22094350">22094350</a> </span> </span> </li> <li tag="ref"> <a name="bib25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graham</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stegen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dispersal-based microbial community assembly decreases biogeochemical function</span>. <span class="source" tagx="source" title="source">Processes</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">65</span>. </span> </li> <li tag="ref"> <a name="bib26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCann</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christgen</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Soil geochemistry confines microbial abundances across an arctic landscape: implications for net carbon exchange with the atmosphere </span>. <span class="source" tagx="source" title="source">Biogeochemistry</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">120</span>: <span class="fpage" tagx="fpage" title="fpage">307</span>– <span class="lpage" tagx="lpage" title="lpage">17</span>. </span> </li> <li tag="ref"> <a name="bib27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henschel</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anwar</span> <span class="given-names" tagx="given-names" title="given-names">MZ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manohar</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comprehensive meta-analysis of ontology annotated 16S rRNA profiles identifies beta diversity clusters of environmental bacterial communities </span>. <span class="source" tagx="source" title="source">PLoS Comput Biol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">e1004468</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26458130">26458130</a> </span> </span> </li> <li tag="ref"> <a name="bib28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jørgensen</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johansen</span> <span class="given-names" tagx="given-names" title="given-names">KML</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westergaard-Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Net regional methane sink in High Arctic soils of northeast Greenland</span>. <span class="source" tagx="source" title="source">Nat Geosci</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">20</span>. </span> </li> <li tag="ref"> <a name="bib29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khadem</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pol</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jetten</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Nitrogen fixation by the verrucomicrobial methanotroph ‘Methylacidiphilum fumariolicum'SolV</span>. <span class="source" tagx="source" title="source">Microbiology</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">156</span>: <span class="fpage" tagx="fpage" title="fpage">1052</span>– <span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20056702">20056702</a> </span> </span> </li> <li tag="ref"> <a name="bib30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kleinteich</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hildebrand</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bahram</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Pole-to-pole connections: Similarities between Arctic and Antarctic microbiomes and their vulnerability to environmental change </span>. <span class="source" tagx="source" title="source">Front Ecol Evol</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">137</span>. </span> </li> <li tag="ref"> <a name="bib31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koven</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ringeval</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Friedlingstein</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Permafrost carbon-climate feedbacks accelerate global warming</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">14769</span>– <span class="lpage" tagx="lpage" title="lpage">74</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21852573">21852573</a> </span> </span> </li> <li tag="ref"> <a name="bib32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kozich</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westcott</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baxter</span> <span class="given-names" tagx="given-names" title="given-names">NT</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Development of a dual-index sequencing strategy and curation pipeline for analyzing amplicon sequence data on the MiSeq Illumina sequencing platform </span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">79</span>: <span class="fpage" tagx="fpage" title="fpage">5112</span>– <span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23793624">23793624</a> </span> </span> </li> <li tag="ref"> <a name="bib33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lanzén</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jørgensen</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huson</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">CREST–classification resources for environmental sequence tags</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e49334</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23145153">23145153</a> </span> </span> </li> <li tag="ref"> <a name="bib34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lauber</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamady</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Pyrosequencing-based assessment of soil pH as a predictor of soil bacterial community structure at the continental scale </span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">75</span>: <span class="fpage" tagx="fpage" title="fpage">5111</span>– <span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19502440">19502440</a> </span> </span> </li> <li tag="ref"> <a name="bib35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Letunic</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bork</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">44</span>: <span class="fpage" tagx="fpage" title="fpage">W242</span>– <span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27095192">27095192</a> </span> </span> </li> <li tag="ref"> <a name="bib37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mackelprang</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waldrop</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeAngelis</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Metagenomic analysis of a permafrost microbial community reveals a rapid response to thaw </span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">480</span>: <span class="fpage" tagx="fpage" title="fpage">368</span>– <span class="lpage" tagx="lpage" title="lpage">71</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22056985">22056985</a> </span> </span> </li> <li tag="ref"> <a name="bib38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Magoč</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">2957</span>– <span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21903629">21903629</a> </span> </span> </li> <li tag="ref"> <a name="bib39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malard</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pearce</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial diversity and biogeography in Arctic soils</span>. <span class="source" tagx="source" title="source">Env Microbiol Rep</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">611</span>– <span class="lpage" tagx="lpage" title="lpage">25</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30028082">30028082</a> </span> </span> </li> <li tag="ref"> <a name="bib41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <span class="source" tagx="source" title="source">EMBnet.journal</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">10</span>– <span class="lpage" tagx="lpage" title="lpage">12</span>. </span> </li> <li tag="ref"> <a name="bib42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martiny</span> <span class="given-names" tagx="given-names" title="given-names">JBH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohannan</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Microbial biogeography: putting microorganisms on the map</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">102</span>– <span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16415926">16415926</a> </span> </span> </li> <li tag="ref"> <a name="bib36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">WK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schautz</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fishback</span> <span class="given-names" tagx="given-names" title="given-names">L-AE</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Assessing the potential of ammonia oxidizing bacteria to produce nitrous oxide in soils of a high arctic lowland ecosystem on Devon Island, Canada </span>. <span class="source" tagx="source" title="source">Soil Biol Biochem</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">2001</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. </span> </li> <li tag="ref"> <a name="bib43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McMurdie</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holmes</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data </span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e61217</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23630581">23630581</a> </span> </span> </li> <li tag="ref"> <a name="bib1_217_1567079012999" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Metcalfe</span> <span class="given-names" tagx="given-names" title="given-names">DB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hermans</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahlstrand</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Patchy field sampling biases understanding of climate change impacts across the Arctic</span>. <span class="source" tagx="source" title="source">Nature ecology and evolution</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">1443</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30013133">30013133</a> </span> </span> </li> <li tag="ref"> <a name="bib44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mittelbach</span> <span class="given-names" tagx="given-names" title="given-names">GG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schemske</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ecological and evolutionary perspectives on community assembly</span>. <span class="source" tagx="source" title="source">Trends Ecol Evol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">241</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25804867">25804867</a> </span> </span> </li> <li tag="ref"> <a name="bib40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Männistö</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiirola</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Häggblom</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bacterial communities in Arctic fjelds of Finnish Lapland are stable but highly pH-dependent</span>. <span class="source" tagx="source" title="source">FEMS Microbiol Ecol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">452</span>– <span class="lpage" tagx="lpage" title="lpage">65</span>. </span> </li> <li tag="ref"> <a name="bib45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nazaries</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murrell</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millard</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Methane, microbes and models: fundamental understanding of the soil methane cycle for future predictions </span>. <span class="source" tagx="source" title="source">Environ Microbiol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">2395</span>– <span class="lpage" tagx="lpage" title="lpage">417</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23718889">23718889</a> </span> </span> </li> <li tag="ref"> <a name="bib46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">UN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ball</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Impacts of altered precipitation regimes on soil communities and biogeochemistry in arid and semi‐arid ecosystems </span>. <span class="source" tagx="source" title="source">Glob Change Biol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">1407</span>– <span class="lpage" tagx="lpage" title="lpage">21</span>. </span> </li> <li tag="ref"> <a name="bib47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Malley</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The nineteenth century roots of 'everything is everywhere'</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">647</span>– <span class="lpage" tagx="lpage" title="lpage">51</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17603517">17603517</a> </span> </span> </li> <li tag="ref"> <a name="bib48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandit</span> <span class="given-names" tagx="given-names" title="given-names">SN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kolasa</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cottenie</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Contrasts between habitat generalists and specialists: an empirical extension to the basic metacommunity framework </span>. <span class="source" tagx="source" title="source">Ecology</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">90</span>: <span class="fpage" tagx="fpage" title="fpage">2253</span>– <span class="lpage" tagx="lpage" title="lpage">62</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19739387">19739387</a> </span> </span> </li> <li tag="ref"> <a name="bib49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paulson</span> <span class="given-names" tagx="given-names" title="given-names">JN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stine</span> <span class="given-names" tagx="given-names" title="given-names">OC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bravo</span> <span class="given-names" tagx="given-names" title="given-names">HC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Differential abundance analysis for microbial marker-gene surveys</span>. <span class="source" tagx="source" title="source">Nat Methods</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">1200</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24076764">24076764</a> </span> </span> </li> <li tag="ref"> <a name="bib50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">MN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dehal</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arkin</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FastTree 2–approximately maximum-likelihood trees for large alignments</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e9490</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a> </span> </span> </li> <li tag="ref"> <a name="bib51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pruesse</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quast</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knittel</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">7188</span>– <span class="lpage" tagx="lpage" title="lpage">96</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17947321">17947321</a> </span> </span> </li> <li tag="ref"> <a name="bib62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="collab'">collab: R Core Team</span>. <span class="comment">R: a language and environment for statistical computing</span> <span class="year" tagx="year" title="year">2013</span>. </span> </li> <li tag="ref"> <a name="bib52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rognes</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flouri</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nichols</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">VSEARCH: a versatile open source tool for metagenomics</span>. <span class="source" tagx="source" title="source">PeerJ</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">e2584</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27781170">27781170</a> </span> </span> </li> <li tag="ref"> <a name="bib53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rousk</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bååth</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brookes</span> <span class="given-names" tagx="given-names" title="given-names">PC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Soil bacterial and fungal communities across a pH gradient in an arable soil</span>. <span class="source" tagx="source" title="source">ISME J</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">1340</span>– <span class="lpage" tagx="lpage" title="lpage">51</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20445636">20445636</a> </span> </span> </li> <li tag="ref"> <a name="bib54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Šantl-Temkiv</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gosewinkel</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Starnawski</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Aeolian dispersal of bacteria in southwest Greenland: their sources, abundance, diversity and physiological states </span>. <span class="source" tagx="source" title="source">FEMS Microbiol Ecol</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">94</span>: <span class="fpage" tagx="fpage" title="fpage">fiy031</span>. </span> </li> <li tag="ref"> <a name="bib55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schostag</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stibal</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Distinct summer and winter bacterial communities in the active layer of Svalbard permafrost revealed by DNA-and RNA-based analyses </span>. <span class="source" tagx="source" title="source">Front Microbiol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">399</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25983731">25983731</a> </span> </span> </li> <li tag="ref"> <a name="bib56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schuur</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGuire</span> <span class="given-names" tagx="given-names" title="given-names">AD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schädel</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Climate change and the permafrost carbon feedback</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">520</span>: <span class="fpage" tagx="fpage" title="fpage">171</span>– <span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25855454">25855454</a> </span> </span> </li> <li tag="ref"> <a name="bib57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Soil pH drives the spatial distribution of bacterial communities along elevation on Changbai Mountain </span>. <span class="source" tagx="source" title="source">Soil Biol Biochem</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">204</span>– <span class="lpage" tagx="lpage" title="lpage">11</span>. </span> </li> <li tag="ref"> <a name="bib58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siciliano</span> <span class="given-names" tagx="given-names" title="given-names">SD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Winsley</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Soil fertility is associated with fungal and bacterial richness, whereas pH is associated with community composition in polar soil microbial communities </span>. <span class="source" tagx="source" title="source">Soil Biol Biochem</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">78</span>: <span class="fpage" tagx="fpage" title="fpage">10</span>– <span class="lpage" tagx="lpage" title="lpage">20</span>. </span> </li> <li tag="ref"> <a name="bib59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">BK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dawson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macdonald</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Impact of biotic and abiotic interaction on soil microbial communities and functions: a field study </span>. <span class="source" tagx="source" title="source">Appl Soil Ecol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">239</span>– <span class="lpage" tagx="lpage" title="lpage">48</span>. </span> </li> <li tag="ref"> <a name="bib60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Toxicity of aluminium on various levels of plant cells and organism: a review</span>. <span class="source" tagx="source" title="source">Environ Exp Bot</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">137</span>: <span class="fpage" tagx="fpage" title="fpage">177</span>– <span class="lpage" tagx="lpage" title="lpage">93</span>. </span> </li> <li tag="ref"> <a name="bib61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sriswasdi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C-c</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iwasaki</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Generalist species drive microbial dispersion and evolution</span>. <span class="source" tagx="source" title="source">Nature Communications</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">1162</span>. </span> </li> <li tag="ref"> <a name="bib63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tedersoo</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bahram</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Põlme</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Global diversity and geography of soil fungi</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">346</span>: <span class="fpage" tagx="fpage" title="fpage">1256688</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25430773">25430773</a> </span> </span> </li> <li tag="ref"> <a name="bib64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">LR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanders</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McDonald</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">A communal catalogue reveals Earth's multiscale microbial diversity</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">551</span>: <span class="fpage" tagx="fpage" title="fpage">457</span>– <span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29088705">29088705</a> </span> </span> </li> <li tag="ref"> <a name="bib65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wardle</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The influence of biotic interactions on soil biodiversity</span>. <span class="source" tagx="source" title="source">Ecol Lett</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">870</span>– <span class="lpage" tagx="lpage" title="lpage">86</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16796577">16796577</a> </span> </span> </li> <li tag="ref"> <a name="bib66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wartiainen</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hestnes</span> <span class="given-names" tagx="given-names" title="given-names">AG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McDonald</span> <span class="given-names" tagx="given-names" title="given-names">IR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Methylobacter tundripaludum sp. nov., a methane-oxidizing bacterium from Arctic wetland soil on the Svalbard islands, Norway (78 N) </span>. <span class="source" tagx="source" title="source">Int J Syst Evol Microbiol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">56</span>: <span class="fpage" tagx="fpage" title="fpage">109</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16403874">16403874</a> </span> </span> </li> <li tag="ref"> <a name="bib67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wojcik</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donhauser</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frey</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Linkages between geochemistry and microbiology in a proglacial terrain in the High Arctic </span>. <span class="source" tagx="source" title="source">Ann Glaciol</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">95</span>– <span class="lpage" tagx="lpage" title="lpage">110</span>. </span> </li> <li tag="ref"> <a name="bib68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. </span> <span class="mixed-article-title" title="mixed-article-title">Soil microbial community structure and activity along a montane elevational gradient on the Tibetan Plateau </span>. <span class="source" tagx="source" title="source">Eur J Soil Biol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">6</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. </span> </li> <li tag="ref"> <a name="bib69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yergeau</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kowalchuk</span> <span class="given-names" tagx="given-names" title="given-names">GA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Responses of Antarctic soil microbial communities and associated functions to temperature and freeze–thaw cycle frequency </span>. <span class="source" tagx="source" title="source">Environ Microbiol</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">2223</span>– <span class="lpage" tagx="lpage" title="lpage">35</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18479442">18479442</a> </span> </span> </li> </ul> </div> </div>  </body></html>